<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Bruno C. Vellutini1,✉, Marina B. Cuenca1, Abhijeet Krishna1,2,3, Alicja Szałapak1,2,3, Carl D. Modes1,2,3, and Pavel Tomancak1,2,3,✉" />
  <title>Patterned embryonic invagination evolved in response to mechanical instability</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  
</head>
<body>
<header id="title-block-header">
<h1 class="title">Patterned embryonic invagination evolved in response to mechanical instability</h1>
<p class="author">Bruno C. Vellutini<sup>1,✉</sup>, Marina B. Cuenca<sup>1</sup>, Abhijeet Krishna<sup>1,2,3</sup>, Alicja Szałapak<sup>1,2,3</sup>, Carl D. Modes<sup>1,2,3</sup>, and Pavel Tomancak<sup>1,2,3,✉</sup></p>
</header>
<p><sup>1</sup> Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, Germany<br />
<sup>2</sup> Center for Systems Biology Dresden, Dresden, Germany<br />
<sup>3</sup> Cluster of Excellence Physics of Life, Technische Universität Dresden, Dresden, Germany</p>
<p><sup>✉</sup> Correspondence: <a href="mailto:vellutini@mpi-cbg.de">Bruno C. Vellutini &lt;vellutini@mpi-cbg.de&gt;</a>, <a href="mailto:tomancak@mpi-cbg.de">Pavel Tomancak &lt;tomancak@mpi-cbg.de&gt;</a></p>
<h1 id="abstract">Abstract</h1>
<p>Mechanical forces are crucial for driving and shaping the morphogenesis of tissues and organs during embryonic development. However, their relevance for the evolution of morphogenetic processes remains poorly understood. Here we show that a morphogenetic innovation present in fly embryos—a deep epithelial fold known as the cephalic furrow—plays a mechanical role during <em>Drosophila</em> gastrulation. By integrating <em>in vivo</em> experiments and <em>in silico</em> simulations, we find that the formation of the cephalic furrow effectively prevents mechanical instabilities at the head–trunk epithelium by absorbing the compressive stresses generated by concurrent morphogenetic movements. Furthermore, by comparing the expression of known and novel genes involved in cephalic furrow formation between fly species, we find that the presence of the cephalic furrow is linked to the appearance of a novel <em>buttonhead</em> expression domain at the head–trunk boundary. These data suggest that the genetic control of cephalic furrow formation was established through the integration of a new player into the ancestral head–trunk patterning system, and that mechanical instability may have been the selective pressure associated with the evolution of the cephalic furrow. Our findings uncover empirical evidence for how mechanical forces can influence the evolution of morphogenetic innovations in early development.</p>
<h1 id="main">Main</h1>
<!--## Evolution of morphogenesis-->
<!--Interplay of patterning and mechanics in the evolution of morphogenesis-->
<p>Morphogenesis is a physical process.<span class="citation" data-cites="Keller2012-ca Heisenberg2013-pl"><sup><a href="#ref-Keller2012-ca" role="doc-biblioref">1</a>,<a href="#ref-Heisenberg2013-pl" role="doc-biblioref">2</a></sup></span> When embryonic tissues extend or invaginate, they push and pull on neighboring tissues. These mechanical interactions provide essential information to embryonic cells throughout development and ultimately shape the final morphology of tissues and organs.<span class="citation" data-cites="Collinet2021-qa"><sup><a href="#ref-Collinet2021-qa" role="doc-biblioref">3</a></sup></span> However, how mechanical forces influence the evolution of morphogenesis in early embryonic development remains unclear. To investigate the interplay between genetics and mechanics during the evolution of morphogenesis, we studied a prominent but enigmatic epithelial fold that forms at the head–trunk boundary of flies during gastrulation—the cephalic furrow.<span class="citation" data-cites="Hartenstein1985-zs Spencer2015-td"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a>,<a href="#ref-Spencer2015-td" role="doc-biblioref">5</a></sup></span></p>
<!--- Morphogenesis is a complex sequence of patterning and physical events.-->
<!--- Under genetic control, tissues can fold via intrinsic cell-generated forces.-->
<!--- Moving tissues generate extrinsic tissue-scale forces.-->
<!--- Self organized morphogenesis is important for regulating development.-->
<!--- Mechanical forces are crucial for the morphogenesis of embryonic tissues.-->
<!--- But it is unclear whether tissue mechanics can influence the origin and evolution of patterned morphogenetic processes.-->
<!--- What are the selective pressures and evolutionary events leading to an evolutionary innovation in early embryos where the tissues are confined in their shells?-->
<!--- How does the mechanics of embryonic tissues impact the evolution of morphogenetic processes?-->
<!--- How do novel morphogenetic processes in the embryo originate?-->
<!--Embryos develop through orchestrated cell and tissue movements known as morphogenesis.-->
<!--Morphogenetic movements are usually under tight genetic control The control of morphogenesis in early embryonic development can be patterned or self-organized, but it is usually an intricate mixture of both [@Collinet2021-qa].-->
<!--Genes determine the position and timing of cell shape changes which will drive coordinated tissue movements like in gastrulation.-->
<!--Morphogenetic processes are usually under tight genetic control.-->
<!--The expression of maternal or zygotic genes in early development determines cell fates and activate behaviors.-->
<!--The position and timing of morphogenetic processes is usually under tight genetic control and regulated by genes which instruct active cell shape changes driving morphogenesis [@Collinet2021-qa].-->
<!--Epithelial folding, tissue extension, and cell delamination are examples of morphogenetic processes occurring during embryonic development.-->
<!--These events are usually under tight genetic control.-->
<!--The position and timing is determined by the expression of maternal or zygotic genes which control cell shape changes that drive morphogenesis [@Collinet2021-qa].-->
<!--When tissues fold, they push and pull on their neighboring tissues, and these mechanical cues have been shown to be crucial as a source of information defining the final morphology.-->
<!--Epithelial folding is a fundamental process of animal embryogenesis [@Wang2021-iy].-->
<!--Tissues fold through intrinsic cellular mechanisms or extrinsic tissue-scale forces [@Denk-Lobnig2020-pr].-->
<!--Some theories suggest that mechanical forces may be crucial to generate novelties in animal development [@Newman2000-oz].-->
<!--However, these ideas have not been tested empirically.-->
<!--However, the selective pressures and evolutionary events leading to the evolution of novel morphogenetic processes in early embryos are poorly understood.-->
<!--## Cephalic furrow as a model-->
<!--Patterned epithelial folding and the case of the cephalic furrow-->
<p>Cephalic furrow formation in <em>Drosophila</em> is under strict genetic control. It is one of the earliest morphogenetic events of gastrulation, beginning as paired lateral indentations which quickly invaginate to form a deep epithelial fold at the boundary between the head (procephalon) and the trunk (germ band).<span class="citation" data-cites="Hartenstein1985-zs Foe1989-cw Spencer2015-td"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a>–<a href="#ref-Foe1989-cw" role="doc-biblioref">6</a></sup></span> The site of invagination is determined by the zygotic expression of two transcription factors, <em>buttonhead</em> (<em>btd</em>) and <em>even skipped</em> (<em>eve</em>), whose domains overlap at the head–trunk boundary by a narrow row of blastoderm cells.<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">7</a></sup></span> The position of the cephalic furrow is thus specified with remarkable accuracy.<span class="citation" data-cites="Liu2013-lc"><sup><a href="#ref-Liu2013-lc" role="doc-biblioref">8</a></sup></span> These so-called initiator cells shorten along the apical–basal axis by activating lateral myosin contractility which drives the infolding of the tissue.<span class="citation" data-cites="Eritano2020-mi"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a></sup></span> In addition, the mechanical coupling between cells ensures a precise and stereotypic morphogenetic process of tissue folding.<span class="citation" data-cites="Eritano2020-mi"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a></sup></span> The resulting fold spans the entire lateral surface, from dorsal to ventral, making the cephalic furrow a landmark of <em>Drosophila</em> gastrulation.<span class="citation" data-cites="Hartenstein1985-zs Foe1989-cw"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a>,<a href="#ref-Foe1989-cw" role="doc-biblioref">6</a></sup></span></p>
<p>Unlike other embryonic invaginations, however, the cephalic furrow has no obvious function. While the ventral furrow gives rise to mesodermal precursors and the midgut invaginations give rise to endodermal tissues, the cephalic furrow does not give rise to any specific structure and, after a couple of hours, unfolds leaving no trace.<span class="citation" data-cites="Hartenstein1985-zs"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a></sup></span> Therefore, despite forming under tight genetic control, the role of the cephalic furrow during development has remained unclear. While it has been thought to serve as a temporary tissue storage<span class="citation" data-cites="Turner1977-ol"><sup><a href="#ref-Turner1977-ol" role="doc-biblioref">10</a></sup></span> or as a tissue anchor during gastrulation,<span class="citation" data-cites="Costa1993-qj Dicko2017-wz"><sup><a href="#ref-Costa1993-qj" role="doc-biblioref">11</a>,<a href="#ref-Dicko2017-wz" role="doc-biblioref">12</a></sup></span> these hypotheses have not been investigated <em>in vivo</em> or considered in a phylogenetic context. Remarkably, recent evidence from an independent study by Dey <em>et al.</em> indicates that the cephalic furrow is an evolutionary novelty that originated within Diptera,<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> making it an ideal model for investigating how patterned morphogenetic processes in early embryonic development evolve.</p>
<!--^(footnote^[^note]^)^ -->
<!--[^note]: This study has been submitted back-to-back with the present manuscript. Both pre-prints are available on bioRxiv.-->
<!--The invagination is only present in a derived group of cyclorraphan flies, but is absent in the remaining basally-branching dipteran flies [@Dey2023-mo].-->
<!--The clear origin in the phylogenetic tree makes the cephalic furrow an ideal model for investigating how patterned morphogenetic processes evolve.-->
<p>Our work integrates genetics and mechanics to uncover the developmental role and evolutionary origins of the cephalic furrow. First, we analyzed how perturbing cephalic furrow formation impacts gastrulation in <em>Drosophila</em> by live imaging mutant embryos, and find that the absence of the cephalic furrow increases the mechanical instability of the blastoderm epithelium. Using a combination of <em>in vivo</em> experiments and <em>in silico</em> simulations, we show that the cephalic furrow effectively prevents this instability by absorbing compressive stresses, and thus, plays a mechanical role during gastrulation. Next, to uncover the changes in genetic patterning associated with the evolution of the cephalic furrow, we compared the expression of head–trunk patterning genes between <em>Drosophila</em> and <em>Clogmia</em>, a fly belonging to a basally-branching group of dipterans which do not form a cephalic furrow. We find that while the ancestral head–trunk patterning system is conserved between the two flies, <em>Clogmia</em> lacks the head–trunk domain of <em>btd</em> that is present in <em>Drosophila</em>, suggesting that the appearance of a novel <em>btd</em> domain at the head–trunk boundary was a key event associated with cephalic furrow evolution. Taken together, these data suggest that the evolution of the cephalic furrow patterning system occurred through the cooption of a novel genetic player, and that the underlying selective pressure may have been the mechanical instability during gastrulation. Our findings reveal an interplay between genetic patterning and mechanical forces during the evolution of morphogenesis in early development.</p>
<!--TODO: Improve this ending.-->
<!--The cephalic furrow retains a significant number of blastoderm cells beneath the surface of the embryo during gastrulation (about 10% according to recent estimates [@Stern2021-zv]),-->
<!--The mechanisms controlling the formation of the cephalic furrow remain poorly understood [@Gilmour2017-um]-->
<!--But the interactions of the cephalic furrow with other morphogenetic movements have not been investigated *in vivo* [@Kong2017-ac].-->
<!--Current models for the cephalic furrow account for the mechanics of the invagination [@Eritano2020-mi; @Jeffery2023; @OlderModel], but not the global context of how it interacts with neighbouring tissues during gastrulation.-->
<h1 id="analyses-of-cephalic-furrow-mutants">Analyses of cephalic furrow mutants</h1>
<!--Mechanical role of cephalic furrow formation-->
<!--TODO: Revise for clarity-->
<p>To understand the physical consequences of perturbing cephalic furrow formation in <em>Drosophila</em>, we investigated the tissue dynamics at the head–trunk boundary in known cephalic furrow mutants—<em>btd</em> and <em>eve</em>, where the invagination is absent,<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">7</a></sup></span> and <em>prd</em> where the invagination is delayed.<span class="citation" data-cites="Blankenship2001-tm"><sup><a href="#ref-Blankenship2001-tm" role="doc-biblioref">14</a></sup></span> We generated fluorescent lines carrying a loss of function allele of these genes, and imaged the embryos under high temporal resolution using <em>in toto</em> lightsheet microscopy, characterizing differences in timing and dynamics of developmental events during gastrulation (Figure <a href="#fig:ectopic-folding">1</a>a).</p>
<!--In *btd* and *eve* mutants, the cephalic furrow is absent [@Vincent1997-fa],  while in *prd* mutants, it is delayed [@Blankenship2001-tm].-->
<!--However, the dynamics of tissues at their head--trunk boundary have not been analyzed.-->
<!--The cephalic furrow is absent in *btd* and *eve* mutants [@Vincent1997-fa] and delayed in *prd* mutants [@Blankenship2001-tm], but the dynamics of their head--trunk boundary tissues have not been analyzed.-->
<h2 id="disruption-of-initiator-cell-behavior">Disruption of initiator cell behavior</h2>
<!--Initiator cell behavior is disrupted in cephalic furrow mutants-->
<!--TODO: Consider moving this paragraph to supplementary notes, if needed-->
<p>We first analyzed how the typical behavior of initiator cells is perturbed in the three mutant backgrounds. In wildtype embryos, the initiator cells shorten and undergo anisotropic apical constriction a few minutes before the end of cellularization<span class="citation" data-cites="Spencer2015-td"><sup><a href="#ref-Spencer2015-td" role="doc-biblioref">5</a></sup></span> (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b). In <em>prd</em> mutants, both behaviors are delayed and the epithelium only invaginates after gastrulation has started (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b). Moreover, the cells adjacent to the initiator cells lack the typical arched profile of wildtype cells and the shape of the resulting fold is slightly abnormal compared to a wildtype invagination (<a href="#fig:initiator">Extended Data Fig. 1</a>a). In <em>btd</em> and <em>eve</em> mutants, the phenotypes are stronger. Initiator cells in <em>btd</em> embryos exhibit a reduced degree of apical constriction but do not shorten (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b, <a href="#fig:vid-initiator">Video S1</a>). In contrast, <em>eve</em> mutants show neither apical constriction nor cell shortening; the epithelium remains flat for several minutes after gastrulation (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b). These observations reveal that cephalic furrow formation in <em>prd</em> mutants is not only delayed but also perturbed, and that the cellular mechanism for the invagination is severely disrupted in <em>btd</em> and <em>eve</em> mutants.</p>
<!--TODO: Create video for btd initiator cells {!@fig:vid-initiator-btd}-->
<!--TODO: Create video for prd initiator cells {!@fig:vid-initiator-prd}-->
<h2 id="formation-of-ectopic-folds">Formation of ectopic folds</h2>
<!--## Ectopic folds form at the head--trunk boundary-->
<!--TODO: Re-write paragraphs to incorporate wildtype and mutant EFs?-->
<!--TODO: Move observation of ectopic folding to the next paragraph?-->
<p>Our recordings show that mutant embryos whose initiator cells fail to shorten (<em>i.e.</em>, <em>btd</em> and <em>eve</em>) exhibit a peculiar phenotype—they form late epithelial folds near the canonical site of cephalic furrow invagination (Figure <a href="#fig:ectopic-folding">1</a>b,c, <a href="#fig:initiator">Extended Data Fig. 1</a>a). Such folds, herein referred to ectopic folds, were previously observed in <em>eve</em> mutants,<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">7</a></sup></span> and more recently in <em>btd</em> mutants.<span class="citation" data-cites="Eritano2020-mi"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a></sup></span> However, what drives their formation or how their appearance connects with the absence of the cephalic furrow in these mutants have remained unclear.</p>
<!--But what drives their formation, developmental significance, or if they are relate to the absence of the cephalic furrow has remained unclear.-->
<p>To understand the mechanisms underlying ectopic fold formation and developmental significance, we analyzed their frequency, position, and dynamics in different genetic backgrounds. Ectopic folds are delayed and more ephemeral when compared to the cephalic furrow. In <em>btd</em> and <em>eve</em> mutants, they appear around the head–trunk boundary about 20 min after gastrulation (Figure <a href="#fig:ectopic-folding">1</a>a,b,c, <a href="#fig:vid-lateral-btd">Video 1</a>, <a href="#fig:vid-lateral-eve">Video 2</a>), when the germ band is already extended 35% of the egg length (Figure <a href="#fig:ectopic-folding">1</a>a,c,d, Table <a href="#tbl:gbtime">1</a>). Ectopic folds only last about 20 min and then disappear while the cephalic furrow persists for a few hours. Ectopic folds are also less organized. Upon folding, they show a wider, often asymmetric cleft, and lack the structured cell morphologies of the cephalic furrow, such as the wedge-shaped initiator cells or the elongated arch-like cells at the edges of the invagination<span class="citation" data-cites="Spencer2015-td"><sup><a href="#ref-Spencer2015-td" role="doc-biblioref">5</a></sup></span> (Figure <a href="#fig:ectopic-folding">1</a>b, <a href="#fig:vid-profile-btd">Video 3</a>, <a href="#fig:vid-profile-eve">Video 4</a>). Moreover, ectopic fold formation shows greater variability. For example, the position of ectopic folds varies between individual mutant embryos. They can form anterior, posterior, or at the head–trunk boundary (Figure <a href="#fig:ectopic-folding">1</a>f, Figure <a href="#fig:features">Extended Data Fig. 2</a>b, <a href="#fig:vid-dynamics-btd">Video S2</a>, <a href="#fig:vid-variability-btd">Video S3</a> Figure ??). And they exhibit more variable kinetics (Figure <a href="#fig:features">Extended Data Fig. 2</a>d,e). Finally, ectopic folds are 25% shallower (Figure <a href="#fig:features">Extended Data Fig. 2</a>f) and occupy an area 50% smaller compared to the cephalic furrow (Figure <a href="#fig:ectopic-folding">1</a>g,h, Figure <a href="#fig:features">Extended Data Fig. 2</a>c). These differences in morphology and dynamics suggest that the cephalic furrow and the ectopic folds form via distinct mechanisms.</p>
<!--TODO: Add ectopic variability panel somewhere-->
<!--TODO: Merge with previous section?-->
<!--TODO: Improve description of ectopic folding frequency-->
<!--TODO: Select panels to include about frequency of ectopic folding-->
<p>To better understand the mechanisms of ectopic folding, we analyzed the phenomenon more broadly in our datasets and found that ectopic folds may also occur in embryos with a cephalic furrow (<em>i.e.</em>, <em>btd</em> or <em>eve</em> heterozygotes, <em>prd</em> hetero- or homozygotes, and wildtype). These ectopic folds, however, differ from the ectopic folds of cephalic mutants in two key aspects. While the frequency of ectopic folding is high in cephalic furrow mutants (&gt;93%)—nearly every <em>btd</em> and <em>eve</em> homozygote shows one or more ectopic folds per embryo side (2.0±1.0 and 1.8±0.6, respectively) (Figure <a href="#fig:ectopic-folding">1</a>b, Figure <a href="#fig:features">Extended Data Fig. 2</a>a)—the frequency of ectopic folding in the other embryos is lower, ranging between 12% (<em>eve</em> heterozygotes) and 80% (wildtype embryos) (Table <a href="#tbl:stats">2</a>). In addition, the area of ectopic folds in wildtype embryos is 3–4x smaller than the area of ectopic folds in cephalic furrow mutants (Figure <a href="#fig:features">Extended Data Fig. 2</a>g–k, Table <a href="#tbl:area">3</a>). Thus, cephalic furrow mutants have more frequent and more prominent ectopic folds compared to wildtype embryos. These data provide evidence that the absence of the cephalic furrow is causally linked to an increase in the frequency and in the magnitude of ectopic folding during <em>Drosophila</em> gastrulation.</p>
<p>Altogether, the variable positions and frequency of ectopic folding suggests that they are not under genetic control comparable to that of the cephalic furrow, but that they may form as a result of physical interactions in the tissue.</p>
<!--TODO:AB: Quantify the variability of ectopic folds in some way (plot it)-->
<h2 id="evidence-of-tissue-compression">Evidence of tissue compression</h2>
<!--## Ectopic folding coincides with mitotic domain expansion and germ band extension-->
<p>The formation of ectopic folds coincides spatiotemporally with two other concurrent processes of gastrulation: the expansion of mitotic domains and the extension of the germ band (Figure <a href="#fig:ectopic-folding">1</a>a,c,d,e). Mitotic domains are groups of blastoderm cells that divide in synchrony during cleavage cycle 14 in <em>Drosophila</em>, first appearing in the head around 20 min after gastrulation.<span class="citation" data-cites="Foe1989-cw"><sup><a href="#ref-Foe1989-cw" role="doc-biblioref">6</a></sup></span> In cephalic furrow mutants, ectopic folds appear in between mitotic domains, or between the posterior Mitotic Domain 6 (MD6) and the extending germ band (Figure <a href="#fig:mitotic-strain">2</a>a,c). The dividing cells lose their basal attachment, round up at the apical side, and more than double (2.3x) their apical area during anaphase (Figure <a href="#fig:apical-strain">Extended Data Fig. 3</a>a), compressing any adjacent cells. Compressed, non-dividing are the first to buckle (Figure <a href="#fig:mitotic-strain">2</a>b). The buckling of ectopic folds is about 3x faster than the progressive invagination of the cephalic furrow (about 14 min) (Figure <a href="#fig:initiator">Extended Data Fig. 1</a>a, <a href="#fig:vid-mitotic-fold">Video 5</a>). Mitotic expansions always precede ectopic folding (Figure <a href="#fig:mitotic-strain">2</a>c,d, Figure <a href="#fig:initiator">Extended Data Fig. 1</a>a, Table <a href="#tbl:gbtime">1</a>), suggesting that compressive stresses from the mitotic expansions and germ band extension may be contributing to their formation.</p>
<!--To uncover the physical interactions driving the ectopic fold formation, we investigated two concurrent gastrulation processes: cell divisions in mitotic domains and the extension of the germ band.-->
<!--TODO: Add pvalues somewhere.-->
<p>To estimate the forces acting on the buckling tissue, we measured the rate of tissue deformation (strain rate) at the head–trunk and trunk–germ boundaries using particle image velocimetry (Figure <a href="#fig:mitotic-strain">2</a>e). We find that the strain rate at the head–trunk boundary of <em>btd</em> mutants is 1.3x higher than in sibling controls, with a peak that coincides with the maximum expansion of mitotic domains and appearance of ectopic folds (Figure <a href="#fig:mitotic-strain">2</a>f, Figure <a href="#fig:apical-strain">Extended Data Fig. 3</a>b, <a href="#fig:vid-strain-rate">Video 6</a>). This indicates that ectopic folds form between mitotic domains at least in part due to the compressive stress generated by mitotic expansions. At the trunk–germ boundary, we observe a steady increase in the strain rate (Figure <a href="#fig:mitotic-strain">2</a>f) that correlates with the compressed profile of cells between MD6 and germ band (Figure <a href="#fig:mitotic-strain">2</a>e). To evaluate if the tissue at the trunk–germ boundary is under compression, we performed larger laser ablations orthogonal to the expected force vectors in wildtype embryos and tracked pairs of vertices from non-ablated cells on each side of the cut. While in control embryos the distance between vertices remains more or less constant, in ablated embryos this distance decreases over time (Figure <a href="#fig:mitotic-strain">2</a>g). This suggests that the tissue might be “collapsing on itself” due to adjacent compressive stresses, and is congruent with the hypothesis that the trunk–germ interface is under compression from the extending germ band. Taken together, these analyses suggest the expansion of mitotic domains, and the extension of the germ band are potential sources of mechanical stress to the epithelial monolayer, and that they may be involved in the formation of ectopic folds by tissue buckling.</p>
<!--TODO: Laser cut experiments. Consider adding them here starting with: “MD and GB generate mechanical forces and tissue flows that converge to the head--trunk boundary. If that’s the case, we expect that these tissues are under compression, and thus, would not show any recoil if cut using laser. To test that...”. The reasoning would almost the same as the one from the next paragraph: we observe that when MD and GB generate opposing flows at the same time the epithelial undergoes a buckling transition forming ectopic folds, we observe that the first epithelial cells to buckle are usually non-dividing between MDs and between MD/GB that become compressed immediately before the formation of the EF, these observations in combination with our models suggested that compressive stresses are driving a buckling transition in CF mutants, which in addition in a mutant situation there’s an excess of tissue in the surface due to the absence of the CF and the threshold for a buckling transition is even lower and these compressive forces are even more effective, if that’s the case, if these epithelial cells are being compressed and are under pressure we expect that a cut orthogonal to the predicted forces would show no recoil and possibly the tissue would collapse in itself. To test this hypothesis in vivo we performed painstaking laser cuts in cephalic furrow mutants and saw just that, if non-dividing cells between MDs or between MD/GB are cut we see no recoil, this supports our view that compressive forces are the essential driving forces underlying EF formation in cephalic furrow mutants and additionally it reveals that the HT is under compressive stresses from at least these two opposing morphogenetic events at gastrulation the MD and GB-->
<!--TODO:AB: Are there force measurements?-->
<!--TODO: Include cell shape analyses with density et al. around here-->
<!--TODO: Quantify compression of trunk-germ cells in btd mutants-->
<h2 id="contribution-of-mitotic-domains-and-germ-band">Contribution of mitotic domains and germ band</h2>
<!--## Role of mitotic domains and germ band-->
<!--## Involvement of mitotic domains and germ band-->
<!--## Mitotic domains and germ band extension drive tissue buckling-->
<!--Mitotic domains are required for ectopic folding in vivo-->
<!--TODO: Re-word this to test if MD are required, GB is secondary observation-->
<p>We first asked whether the extension of the germ band alone can generate ectopic folds. To that end, we generated double-mutant flies lacking both the cephalic furrow and the mitotic domains by combining <em>btd</em> mutants with a loss-of-function allele of <em>cdc25</em> or <em>string</em> (<em>stg</em>), a phosphatase that regulates the cell divisions in cleavage cycle 14 of <em>Drosophila</em>.<span class="citation" data-cites="Edgar1989-hv"><sup><a href="#ref-Edgar1989-hv" role="doc-biblioref">15</a></sup></span> In <em>stg</em> mutants, the formation of the cephalic furrow and of other morphogenetic movements during early gastrulation occur normally<span class="citation" data-cites="Edgar1989-hv"><sup><a href="#ref-Edgar1989-hv" role="doc-biblioref">15</a></sup></span> (Figure <a href="#fig:live-details">Extended Data Fig. 5</a>a,b, <a href="#fig:vid-stg-lateral">Video S4</a>, <a href="#fig:vid-stg-dorsal">Video S5</a>). Indeed, in <em>btd</em>–<em>stg</em> double-mutant embryos, we find no ectopic folds forming at the head–trunk interface (Figure <a href="#fig:live-experiments">4</a>a,b,j (left), <a href="#fig:vid-double-lateral">Video 10</a>, <a href="#fig:vid-double-dorsal">Video 11</a>). The experiment shows that mitotic domains are required for ectopic folding, and the push from the germ band extension alone is insufficient to induce ectopic buckling in cephalic furrow mutants.</p>
<!--TODO: Add here the wildtype control ablation and cauterization experiments-->
<!--Cephalic furrow invaginates autonomously pulling adjacent tissues-->
<!--TODO: Reduce and move data somewhere else?-->
<p>First, we determined how the cephalic furrow influences the mechanics of the surrounding tissue and how far such influence spreads away from the furrow. By laser ablating cell membranes at different distances from initiator cells in early wildtype embryos, we find that the recoil velocity increases with time and decreases with distance, with an approximate range of at least 40µm anterior and posterior to the initiator cells (Figure <a href="#fig:live-experiments">4</a>i). This suggests that the cell shortening of initiator cells, which is actively driven by lateral myosin contractility,<span class="citation" data-cites="Eritano2020-mi"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a></sup></span> pulls on the adjacent cells and generates tension in the surrounding tissues.</p>
<!--Cephalic furrow forms normally but is shallower in cauterized wildtype-->
<p>Since our experiments indicate that the germ band is compressing the head–trunk tissues, we asked whether the extension of the germ band can influence the depth of the cephalic furrow. To test <em>in vivo</em> whether the presence of the germ band impacts the formation of the cephalic furrow, we blocked the germ band extension by cauterization in wildtype embryos. The cephalic furrow forms normally in cauterized embryos indicating that the germ band extension is not required for its formation (Figure <a href="#fig:live-experiments">4</a>e,f,j (right)). We measured the maximum depth of the cephalic furrow between cauterized and non-cauterized embryos, and found it is only 15% shallower after cauterization (Figure <a href="#fig:live-experiments">4</a>h). These experiments corroborate the view that cephalic furrow initiation is autonomous and independent of other morphogenetic movements of gastrulation, such as the germ band extension which only has a mild influence on the final depth of the invagination.</p>
<!--This result needs to be corroborated with a larger sample size, but it provides initial evidence that the germ band extension has a mild influence on the final depth of the cephalic furrow.-->
<!--These experiments rule out the germ band extension as a driver of cephalic furrow invagination, and give support to the notion that the cephalic furrow forms autonomously by independent active mechanisms which are .-->
<!--The simulations show that the average depth of the cephalic furrow correlates with the percentage of germ band extension ({@fig:fold-model}i).-->
<!--Mitotic activity and germ band elongation contributes to the depth of the cephalic furrow [@Campos-Ortega1997-rd].-->
<!--TODO:AB: Is the increase in recoil velocity because CF or epithelial maturation?-->
<!--Germ band extension is required for ectopic folding in vivo-->
<!--TODO: Re-word this to test if GB is required, MD is secondary observation-->
<p>To test whether mitotic domains alone are sufficient to induce ectopic buckling <em>in vivo</em>, we mechanically blocked the extension of the germ band in cephalic furrow mutants. For that, we cauterized a patch of dorsal tissue, attaching it to the vitelline envelope at the onset of gastrulation (Figure <a href="#fig:live-experiments">4</a>c). When the germ band extension is blocked in <em>btd</em> and <em>eve</em> mutants, no ectopic folds appear at the head–trunk interface (Figure <a href="#fig:live-experiments">4</a>c,d,j (middle), <a href="#fig:vid-cautlat-eve">Video 7</a>, <a href="#fig:vid-cautdor-eve">Video 8</a>, <a href="#fig:vid-cautdor-btd">Video 9</a>). Mitotic expansions compress the neighboring non-dividing cells, but no buckling occurs (Figure <a href="#fig:live-experiments">4</a>c). Cauterized mutant embryos also show less epithelial deformations compared to non-cauterized mutant embryos (Figure <a href="#fig:live-experiments">4</a>d,g). These experiments reveal that <em>in vivo</em>, the local expansions of mitotic domains are insufficient to cause epithelial buckling at the head–trunk interface, and that the germ band extension is required. Altogether, the experiments show that, in cephalic furrow mutants, neither mitotic domain nor germ band can promote ectopic folding by themselves.</p>
<h1 id="physical-model-of-folding-dynamics">Physical model of folding dynamics</h1>
<!--## Physical model predicts increased mechanical instability-->
<p>To determine the relative contribution of mitotic domains and germ band as sources of mechanical stress on the head–trunk boundary, we created a physical model to simulate these tissue mechanics <em>in silico</em> and performed experimental perturbations to test our theoretical predictions <em>in vivo</em> (see below).</p>
<h2 id="simulation-design-and-properties">Simulation design and properties</h2>
<p>Our model represents an epithelial monolayer confined inside a rigid shell. It embodies one side of a frontal slice between the midline and the dorsal apex of a <em>Drosophila</em> embryo with its typical morphological proportions (Figure <a href="#fig:fold-model">3</a>a, Figure <a href="#fig:model-features">Extended Data Fig. 4</a>a). The blastoderm is modeled by an elliptical arc of equidistant particles connected by springs and enclosed on one side by a rigid barrier representing the vitelline envelope (Figure <a href="#fig:fold-model">3</a>b). The total energy per unit length of this tissue (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>T</mi></msub><annotation encoding="application/x-tex">W_T</annotation></semantics></math>) is a sum of a stretching energy component (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>s</mi></msub><annotation encoding="application/x-tex">W_s</annotation></semantics></math>) and a bending energy component (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>b</mi></msub><annotation encoding="application/x-tex">W_b</annotation></semantics></math>) (Figure <a href="#fig:fold-model">3</a>c). Each of these components have a rigidity associated with them. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>s</mi></msub><annotation encoding="application/x-tex">K_s</annotation></semantics></math> is the stretching rigidity and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math> is the bending rigidity. These two parameters can be combined into a single dimensionless bending rigidity, <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> (Figure <a href="#fig:fold-model">3</a>c). To simulate the physical interactions between mitotic domains, germ band, and cephalic furrow, we defined the mitotic domains as compressed regions which tend to expand (they contain more particles compared to the surrounding regions), and the cephalic furrow as a narrow region having an intrinsic negative curvature predisposing the tissue to invaginate (Figure <a href="#fig:fold-model">3</a>b). The germ band in our model is defined by the position of the posterior end of the tissue (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math>) which is fixed at different fractions of egg length for each simulation (Figure <a href="#fig:fold-model">3</a>b). The effect of germ band extension appears as a global compression in the blastoderm. Finally, to run the simulations we added a ground level of random noise, and iterated the model towards an equilibrium in the total energy of the system (see Methods).</p>
<p>We first characterized the general properties of the model. In simulations where folds begin to form, the bending energy increases while releasing a larger amount of stretching energy which in turn decreases the total energy of the system over the iterations (Figure <a href="#fig:fold-model">3</a>d). The increase in bending energy coincides with a rapid deepening of the fold. Once the bending energy reaches a peak, the fold continues to deepen, though more gradually, but the number of folds stabilizes and there are rarely changes in the overall folding patterns (Figure <a href="#fig:fold-model">3</a>d, Figure <a href="#fig:model-features">Extended Data Fig. 4</a>b). Folding events are stochastic and can happen at different iterations across different simulations. Also, we are mainly interested in analyzing the initial position and the number of folding events in each simulation. Hence, we used the state of each simulation at the peak of its bending energy as the reference point to compare with other simulations.</p>
<!--## Parameter sweep of mutant conditions-->
<!--## Parameter sweep of bending rigidity-->
<!--## Bending rigidity parameter sweep-->
<h2 id="bending-rigidity-sweep-in-mutant-conditions">Bending rigidity sweep in mutant conditions</h2>
<!--TODO:AB: Describe overfolding events. Mark them on phase diagram. Discuss as limitation-->
<p>To obtain realistic values of the dimensionless bending rigidity <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> where the model matches experimental observations, we performed a sweep across the parameter space in conditions mimicking the mutants without the cephalic furrow. As expected, the probability of buckling increases with the amount of germ band extension, mainly in lower bending rigidities (Figure <a href="#fig:fold-model">3</a>e, Figure <a href="#fig:model-features">Extended Data Fig. 4</a>c). At 10% germ band extension, we observe a buckling transition in the phase space of softer conditions (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>≈</mo><mn>0.7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} \approx 0.7 \times 10^{−4}</annotation></semantics></math>), but in stiffer conditions (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>≈</mo><mn>1.2</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} \approx 1.2 \times 10^{−4}</annotation></semantics></math>) the germ band alone, even at its maximum extension, cannot drive the formation of ectopic folds (Figure <a href="#fig:fold-model">3</a>e, Figure <a href="#fig:model-features">Extended Data Fig. 4</a>c). Adding mitotic domains to the simulations changed the phase diagram and dynamics of ectopic folding. We observe that mitotic domains alone can induce ectopic folds in softer conditions (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>&lt;</mo><mn>1.1</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} &lt; 1.1 \times 10^{−4}</annotation></semantics></math>) and that the number of folds increased for low values of germ band extension (Figure <a href="#fig:fold-model">3</a>f). In addition, the number of iterations until a first folding event occurs is lower in conditions with mitotic domains, especially in simulations with 0–20% of germ band extension (Figure <a href="#fig:model-features">Extended Data Fig. 4</a>d). These simulations suggest that, depending on the bending rigidity, the germ band or mitotic domains alone can drive ectopic folding, and that their combined action may increase the mechanical instabilities in the blastoderm epithelium. We thus set to test the individual contribution of mitotic domains and germ band to the epithelial instability by performing perturbation experiments <em>in vivo</em>.</p>
<!--TODO: Move definition of a fold to methods-->
<!--TODO: Add panel with the threshold for counting folds-->
<!--TODO:AB: Give a physics-based estimation of order of magnitude of dimensionless parameter from literature. Good sanity check to see if values are not off the charts!-->
<!--TODO: Move this part somewhere else after the model-->
<p>Altogether, the experiments show that, in cephalic furrow mutants, neither mitotic domain nor germ band can promote ectopic folding by themselves. With this crucial insight, we could then look back to our model and identify the bending rigidity value where these biological behaviors are recapitulated in the simulations. For that, we determined at which bending rigidity the average number of folds falls below 1 in mitotic-domains-only and germ-band-only conditions (Figure <a href="#fig:model-features">Extended Data Fig. 4</a>c,d). The criteria is fulfilled when the bending rigidity is <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>≈</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} \approx 1.0 \times 10^{−4}</annotation></semantics></math>. This specific value establishes an important bridge between our <em>in vivo</em> and <em>in silico</em> data by revealing where the embryo resides within the parameter space of the model, and by providing a biologically relevant reference for subsequent simulations.</p>
<p>To compare our reference bending rigidity with direct measurements, we calculated <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> for the bending rigidity estimates in MDCK monolayers (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>b</mi></msub><mo>≈</mo><mn>5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>13</mn></mrow></msup><mi>N</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">K_{b} \approx 5 \times 10^{-13} Nm</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>s</mi></msub><mo>≈</mo><mn>0.2</mn><mi>N</mi><msup><mi>m</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{s} \approx 0.2 Nm^{-1}</annotation></semantics></math>)<span class="citation" data-cites="Trushko2020-gf Fouchard2020-km"><sup><a href="#ref-Trushko2020-gf" role="doc-biblioref">16</a>,<a href="#ref-Fouchard2020-km" role="doc-biblioref">17</a></sup></span> (Supplementary Note 1). Given that in elastic sheets the bending and stretching rigidity ratio is proportional to the square of the height (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>b</mi></msub><mi>/</mi><msub><mi>K</mi><mi>s</mi></msub><mo>∝</mo><msup><mi>h</mi><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_{b}/K_{s} \propto h^2</annotation></semantics></math>),<span class="citation" data-cites="Efrati2009-ve"><sup><a href="#ref-Efrati2009-ve" role="doc-biblioref">18</a></sup></span> we adjusted the calculation to the estimated thickness of the MDCK monolayer (~18µm).<span class="citation" data-cites="Fouchard2020-km"><sup><a href="#ref-Fouchard2020-km" role="doc-biblioref">17</a></sup></span> The value we obtained in this way for the effective <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> of a height-adjusted MDCK monolayer, <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mo>≈</mo><mn>1.8</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">\approx 1.8 \times 10^{-4}</annotation></semantics></math>, is within a factor of 2 of our own reference bending rigidity, indicating that the relevant region of parameter space we identified in our model is consistent with existing direct measurements of bending rigidity in other tissues.</p>
<!--and is therefore likely to be physiologically relevant-->
<p>Simulations at our reference bending rigidity regime reveal a correlation between increased compressive stresses and buckling instability. When both mitotic domains and germ band are present <em>in silico</em>, there is an increase in tissue compression that maximizes the number and speed of ectopic fold formation (Figure <a href="#fig:model-features">Extended Data Fig. 4</a>c,d). Similarly, we propose the mechanical instability in cephalic furrow mutants may be due to an increase in the compressive stresses at the head–trunk boundary, generated by the combined formation of mitotic domains and germ band extension during early gastrulation.</p>
<h2 id="impact-of-cephalic-furrow-on-ectopic-folding">Impact of cephalic furrow on ectopic folding</h2>
<!--## Cephalic furrow formation prevents mechanical instability-->
<p>The increased mechanical instability in cephalic furrow mutants led us to hypothesize that the temporary storage of invaginated cells may contribute to absorbing the compressive stresses generated by mitotic domains and germ band in wildtype conditions. To explore the role of the cephalic furrow as a mechanical buffer, we analyzed <em>in vivo</em> and computationally how it interacts with mitotic domains and with the germ band, and how it influences the formation of ectopic folds.</p>
<p>Next, we tested the conditions that impact the effectiveness of the cephalic furrow in preventing the formation of ectopic folds in the model.</p>
<!--Timing rather than force more effective at preventing ectopic folding-->
<p>In our model, we programmed the cephalic furrow by establishing an intrinsic negative curvature (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math>) to a narrow region of the particle–spring blastoderm that matches the span of the initiator cells <em>in vivo</em> (Figure <a href="#fig:fold-model">3</a>a, Figure <a href="#fig:model-features">Extended Data Fig. 4</a>i). Using the bending rigidity fine-tuned with the previous simulations and experiments, we ran a parameter sweep for different values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> and established a baseline (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><mo>&gt;</mo><mn>0.3</mn></mrow><annotation encoding="application/x-tex">\kappa_o^{CF}&gt;0.3</annotation></semantics></math>) where the invagination forms in a robust manner with minimal variability, phenocopying the cephalic furrow <em>in vivo</em> (Figure <a href="#fig:model-features">Extended Data Fig. 4</a>i).</p>
<!--TODO: Rewrite this paragraph to be clearer-->
<p>We first evaluated how the strength of the cephalic furrow impacts the frequency and placement of ectopic folding. In conditions without the germ band (<em>g</em>=0), we find that an active invagination reduces the spread and frequency of ectopic folding at the head–trunk boundary (Figure <a href="#fig:fold-model">3</a>g) and that this reduction in variability correlates with the strength of the pull (Figure <a href="#fig:model-features">Extended Data Fig. 4</a>j). In conditions with an extended germ band, however, this buffering effect was diminished. Embryos with <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><mo>&lt;</mo><mo>=</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">\kappa_o^{CF}&lt;=1</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>&gt;</mo><mo>=</mo><mn>2</mn></mrow><annotation encoding="application/x-tex">g&gt;=2</annotation></semantics></math> fail to form the cephalic furrow and variable ectopic folding dominates the landscape (Figure <a href="#fig:model-features">Extended Data Fig. 4</a>j). These simulations show that higher values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are more effective at preventing ectopic folds. However, the increased spread of ectopic folding and shallow cephalic furrow with the extended germ band is not match what happens <em>in vivo</em>, suggesting that the simulations were missing something.</p>
<p>One important difference was the relative timing of developmental events. In the simulations above, the cephalic furrow and mitotic domains form at the same time, but in wildtype embryos, the cephalic furrow forms around 15 min before mitotic domains. Therefore, we asked whether their relative timing of formation may influence the effectiveness of the cephalic furrow. Indeed, when we added a delay to the formation of mitotic domains, the cephalic furrow is more effective in preventing ectopic folding even for lower <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> values and for greater percentages of germ band extension (Figure <a href="#fig:fold-model">3</a>g, Figure <a href="#fig:model-features">Extended Data Fig. 4</a>e, Figure <a href="#fig:model-features">Extended Data Fig. 4</a>b). Ectopic folds only form in the posterior region near the germ band tip, a position notably similar to where the dorsal folds form in the <em>Drosophila</em> embryo <em>in vivo</em>.</p>
<p>Finally, we tested how the position of the cephalic furrow impacts its ability to prevent ectopic folding. Similar to the simulations of the cephalic furrow at ~65% embryo length Figure <a href="#fig:fold-model">3</a>g (the wildtype position), formation of the cephalic furrow at &gt;70% embryo length, when the germ band is not extended (<em>g</em>=0), also effectively prevents ectopic folding in the head (Figure <a href="#fig:fold-model">3</a>h). However, when the cephalic furrow is moved further back &lt;50% embryo length, ectopic folding around mitotic domains becomes more frequent. Conversely, when the germ band is extended (<em>g</em>=0.4), a cephalic furrow located more posteriorly along the anteroposterior axis (&lt;30% embryo length) will prevent ectopic folding, while a cephalic furrow located more anteriorly (&gt;50% embryo length) will fail to do so and ectopic folding appears at the posterior region (Figure <a href="#fig:fold-model">3</a>h). These simulations reveal that when the cephalic furrow is positioned between 40-60% embryo length, it is the most effective at preventing ectopic folding throughout the different phases of germ band extension.</p>
<!--TODO: Add full parameter sweeps as supplementary notes-->
<p>Taken together, our physical model provides a theoretical basis that a fold like the cephalic furrow, when forming before other morphogenetic events and around the central portion of the anteroposterior axis, can effectively absorb compressive stresses and prevent mechanical instabilities in embryonic tissues during gastrulation.</p>
<!--TODO: Timing effectiveness is not exemplified in the last panel-->
<!--TODO: Add evolutionary question here or after genes?-->
<h1 id="evolution-of-gene-expression">Evolution of gene expression</h1>
<!--## Genetic innovation at the head--trunk boundary-->
<!--
- Function depends on timing and position that depends on patterning
- Is the patterning conserved in species with or without cephalic furrow?
- We identified slp as a cephalic furrow gene (supp, screen, mutants, etc)
- Comparative expression across species (include prd?)
-->
<p>As described above, the effectiveness of the cephalic furrow in preventing epithelial instabilities depends on the position and time of the invagination. In <em>Drosophila</em>, this spatiotemporal control is determined genetically by the combinatorial expression of <em>btd</em>, <em>eve</em>, and <em>prd</em> at the head–trunk boundary<span class="citation" data-cites="Vincent1997-fa Blankenship2001-tm"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">7</a>,<a href="#ref-Blankenship2001-tm" role="doc-biblioref">14</a></sup></span> (also this study). However, it remains unclear which elements of this genetic patterning cascade are associated with the evolution of the cephalic furrow. To address this question, we first sought to identify additional genes involved in cephalic furrow formation, and then asked how the expression of these cephalic furrow genes compare between dipteran species with and without the cephalic furrow.</p>
<h2 id="role-of-sloppy-paired-in-cephalic-furrow-patterning">Role of <em>sloppy paired</em> in cephalic furrow patterning</h2>
<!--## Ancestral *slp*--*eve* interaction defines head--trunk position-->
<p>To identify other cephalic furrow genes, we performed a live-imaging screen using loss-of-function alleles of several candidates expressed at the head–trunk interface (Supplementary Note 2). We found that null mutants for the <em>sloppy paired</em> (<em>slp</em>) transcription factors, <em>slp1</em> and <em>slp2</em>, show a strong phenotype where the cephalic furrow is delayed and shifted towards the anterior end by ~6% of the egg length (control = 67.6±1.4%, n=26; <em>slp</em> = 73.2±0.7%, n=7) (Figure <a href="#fig:genetic-patterning">5</a>a,b, Figure <a href="#fig:vid-slp-lateral">Video 12</a>). With this anterior shift, <em>slp</em> mutants exhibit a more prominent posterior dorsal fold and an early ectopic fold within MD6 appearing before cell divisions (Figure <a href="#fig:genetic-patterning">5</a>a, Figure <a href="#fig:vid-slp-lateral">Video 12</a>), which are congruent with the increase in posterior mechanical instability present in our simulations where the cephalic furrow is shifted forward (Figure <a href="#fig:fold-model">3</a>h).</p>
<p>Since <em>slp1</em> is a known anterior repressor that positions anterior pair rule stripes,<span class="citation" data-cites="Andrioli2004-qi Andrioli2012-fv"><sup><a href="#ref-Andrioli2004-qi" role="doc-biblioref">19</a>,<a href="#ref-Andrioli2012-fv" role="doc-biblioref">20</a></sup></span> we wondered if the shift in the position of the cephalic furrow in <em>slp</em> mutants coincided with a shift in typical overlap between <em>btd</em> and <em>eve</em> stripe 1 at the head–trunk boundary. Indeed, the expression patterns and typical overlap between the two genes remains almost unaltered, except for the small ectopic expression <em>btd</em> in the head and for the wider gap between <em>eve</em> stripe 1 and 2 (Figure <a href="#fig:genetic-patterning">5</a>c–d, Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>c). The anterior shift in cells expressing <em>btd</em> and <em>eve</em> corresponds to a few rows of blastoderm cells (control = 4.7±0.5 rows, n=4; <em>slp</em> = 7.3±0.7 rows, n=5) (Figure <a href="#fig:genetic-patterning">5</a>e). Despite the displacement, most <em>slp</em> embryos exhibit initiator cell behaviors, suggesting that the patterning is not entirely perturbed (Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>c). However, the symmetry of the resulting fold is altered (Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>d), indicating that <em>slp1</em> may contribute not only to the positioning but also to the patterning of individual cells that give rise to the cephalic furrow.</p>
<p>By analyzing the expression of <em>slp1</em> relative to <em>btd</em>, <em>eve</em>, and <em>prd</em> in wildtype <em>Drosophila</em> embryos, we find that the early <em>slp1</em> and <em>eve</em> expression domains demarcate the head–trunk boundary from the onset of zygotic activation until gastrulation (Figure <a href="#fig:genetic-patterning">5</a>f, Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>b). Early <em>slp1</em> transcripts are limited to the anterior end while <em>eve</em> transcripts, which are initially ubiquitous,<span class="citation" data-cites="Andrioli2012-fv"><sup><a href="#ref-Andrioli2012-fv" role="doc-biblioref">20</a></sup></span> begin to clear from the anterior end at cycle 11, and become limited to the posterior region of the body from cycle 12 (Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>b). At this stage, the two genes form broad, complementary territories that correspond to the head and trunk regions of the embryo, with the domains juxtaposed at ~70% of the embryo length (Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>b). We first detect <em>btd</em> and <em>prd</em> transcripts at this interface (Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>a,b). During subsequent stages, the <em>slp1</em>–<em>eve</em> boundary progressively resolves into narrow abutting stripes giving rise to the row of initiator cells (Figure <a href="#fig:genetic-patterning">5</a>f, Figure <a href="#fig:genetic-features">Extended Data Fig. 6</a>a). Altogether, the data suggests that <em>slp1</em> activity contributes to restricting the anterior boundary of <em>eve</em> expression during early stages of zygotic activation, an interaction that determines the site of invagination of the cephalic furrow along the anteroposterior axis.</p>
<p>At the onset of gastrulation, the expression of <em>btd</em>, <em>eve</em>, <em>slp</em>, and <em>prd</em> at the head–trunk boundary of <em>Drosophila</em>, forms a unique combinatorial code that coincides with the different portions of the cephalic furrow. The central row of <em>eve</em>-expressing initiator cells are surrounded by <em>slp1</em>-expressing adjacent cells, with <em>prd</em> expression offset by a single row of cells relative to the inner <em>slp1</em> stripes (Figure <a href="#fig:genetic-patterning">5</a>g,i). Moreover, <em>slp1</em>-expressing cells also demarcate the outer edges of the invagination (Figure <a href="#fig:genetic-patterning">5</a>i). This combinatorial expression suggests that each row has a unique transcriptional identity. This arrangement is disrupted in <em>btd</em>, <em>eve</em>, and <em>prd</em> mutants (Supplementary Note 3), embryos that exhibit cephalic furrow defects (Figure <a href="#fig:initiator">Extended Data Fig. 1</a>). This indicating that this specific molecular profile is important for the patterning and morphogenesis of the invagination in <em>Drosophila</em>, and provide a template for the comparison to other species.</p>
<!--Since *slp* is crucial to maintain parasegment polarity by activating *wingless* and repressing *engrailed* [@Cadigan1994-hz], the expression of *slp* may also be important for establishing the individual cell identities around the cephalic furrow.-->
<!--it must occur at the head--trunk boundary and the initiation of the process must begin before other morphogenetic movements take place.-->
<!--the dipteran fly *Clogmia albipunctata*, a species from the basally-branching Psychodidae family which has no cephalic furrow [@Jimenez-Guri2014-zs; @Dey2023-mo].-->
<!--Early slp/eve domains demarcate the head--trunk boundary-->
<!--how btd also regulates slp stripe 2-->
<!--- slp positions btd/eve, then btd positions slp stripe 2-->
<!--Late *slp* expression demarcates cephalic furrow edges-->
<!--TODO: Describe narrowing of btd--eve overlap to a single row-->
<!--- slp as well shows some adhesive differences.-->
<h2 id="association-of-buttonhead-with-cephalic-furrow-evolution">Association of <em>buttonhead</em> with cephalic furrow evolution</h2>
<!--## Novel *btd* head--trunk domain associated with cephalic furrow-->
<!--TODO: Update text to reflect the four species studied-->
<!--TODO: First mention early expression fairly similar-->
<!--TODO: Put early expression patterns Ccap, Aste, Calb on new extended data-->
<!--TODO: Put representative images of early patterns on main figure-->
<!--TODO: Add lateral head view for the four species-->
<!--TODO: Add profile view for the four species-->
<!--TODO: Complete the schematics with combinatorial pattern-->
<p>To uncover the differences in the head–trunk genetic patterning associated with the evolution of the cephalic furrow, we compared the expression patterns in <em>Drosophila</em> to three other dipteran species. One species, the mediterranean fruit fly <em>Ceratitis capitata</em> (Tephritidae),<span class="citation" data-cites="Strobl2022-og"><sup><a href="#ref-Strobl2022-og" role="doc-biblioref">21</a></sup></span> belongs to a family known to form a cephalic furrow. The other two species, the malaria mosquito <em>Anopheles stephensi</em> (Culicidae)<span class="citation" data-cites="Cheatle-Jarvela2020-go Cheatle-Jarvela2023-gd"><sup><a href="#ref-Cheatle-Jarvela2020-go" role="doc-biblioref">22</a>,<a href="#ref-Cheatle-Jarvela2023-gd" role="doc-biblioref">23</a></sup></span> and the drain fly <em>Clogmia albipunctata</em> (Psychodidae),<span class="citation" data-cites="Jimenez-Guri2014-zs Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a>,<a href="#ref-Jimenez-Guri2014-zs" role="doc-biblioref">24</a></sup></span> belong to families where the cephalic furrow has not been observed.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span></p>
<p>The three species show early, juxtaposing domains of <em>slp</em> and <em>eve</em> demarcating the head and trunk regions in a pattern highly similar to that of <em>Drosophila</em> (Figure <a href="#fig:genetic-patterning">5</a>f,j, Figure <a href="#fig:genetic-comparative">Extended Data Fig. 7</a>a,b,c). Moreover, the late pattern of abutting <em>slp</em> and <em>eve</em> stripes is nearly identical between the four species (Figure <a href="#fig:genetic-patterning">5</a>f,j,k). These observations suggest that the genetic interactions that establish the head–trunk boundary position in <em>Drosophila</em>, and much of the late patterning, might be conserved in other dipteran species.</p>
<p>However, we identified a key difference between species with and without the cephalic furrow in the expression of <em>btd</em>. In <em>Drosophila</em> and <em>Ceratitis</em>, the head–trunk domain of <em>btd</em> overlaps with <em>eve</em> stripe 1 (Figure <a href="#fig:genetic-patterning">5</a>k). <em>Anopheles</em> and <em>Clogmia</em> do not exhibit this pattern. Their head–trunk domain of <em>btd</em> either does not overlap with <em>eve</em> or is entirely absent (Figure <a href="#fig:genetic-patterning">5</a>k). Unlike <em>Drosophila</em> (Figure <a href="#fig:genetic-patterning">5</a>i), <em>eve</em> stripe 1 cells do not express <em>btd</em> in species without a cephalic furrow (Figure <a href="#fig:genetic-patterning">5</a>m). The absence of a <em>btd</em>–<em>eve</em> overlap in species without cephalic furrow has also been corroborated in other dipteran species.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> Since <em>btd</em> is a crucial gene for cephalic furrow formation, this suggests that the its evolution may have been associated with the establishment and positioning of <em>btd</em> at the head–trunk boundary.</p>
<!--*prd* expression relative to *slp* and *eve* in *Clogmia* is different to *Drosophila* ({@fig})-->
<!--The data indicates that the presence of a head--trunk domain of *btd* in *Drosophila*.-->
<!--shows that both flies share a conserved head--trunk patterning system, and that the only significant difference is the-->
<!--Therefore, one of the key differences between species with and without a cephalic furrow is that cells expressing eve -->
<!--in cells that give rise to the initiator cells of the cephalic furrow -->
<!--andfrom early stages and a more anterior domain localized in the acron is activated at the onset of gastrulation ({@fig:genetic-patterning}f).-->
<!--In *Clogmia*, the *btd* acron domain is expressed much earlier and, strikingly, the head--trunk domain is not present; *btd* is only expressed at a ventral patch in the foregut area ({@fig:genetic-patterning}f,h).-->
<!--Additionally, the combinatorial expression of *slp*, *eve*, and *prd* at the onset of gastrulation is conserved between *Clogmia* and *Drosophila* ({@fig:genetic-patterning}j).-->
<!--These data shows that both flies share a conserved head--trunk patterning system, and that the only significant difference is the presence of a head--trunk domain of *btd* in *Drosophila*.-->
<!--Since *btd* is a crucial gene for cephalic furrow formation, this suggests that the origin of a novel expression domain of *btd* at the head--trunk boundary was a key event associated with the evolution of the cephalic furrow.-->
<!--Head--trunk boundary slp/eve expression is conserved-->
<!--Clogmia lacks the head--trunk domain of btd-->
<!--- As an aside, curiously, the eye domain of btd is expressed very early in Clogmia while the same domain is delayed tremendously in Drosophila. Clogmia has larval eyes while Drosophila doesn’t. It could be that some portions of head development in Drosophila were just turned off.-->
<!--- It is probable that the expansion of btd along the head--trunk was or still is or originated with some other role in the patterning of the head, and that only later on it got coopted into the cephalic furrow program.-->
<!--- Could it be that this is a re-use of the anterior midgut program? There btd and eve overlap and there is an invagination. I wonder if it is also driven by lateral myosin.-->
<!--Clogmia is the sister group to Megaselia+Episyrphus+Drosophila [@Jimenez-Guri2013]-->
<!--Compare the dorsoventral patterning between Clogmia and Drosophila [@Wotton2017]. Apparently, Clogmia is ventralized while Drosophila dorsalized. The origin of this head--trunk btd domain could be that it expanded dorsally with the dorsalization.-->
<h1 id="discussion">Discussion</h1>
<!--Head--trunk boundary under compressive stress-->
<p>Our work investigates the function and evolution of the cephalic furrow—a patterned epithelial invagination that forms at the head–trunk boundary of dipteran flies. We find that perturbing cephalic furrow formation in <em>Drosophila</em> makes the head–trunk epithelium unstable due to the compressive stresses generated by the concomitant formation of mitotic domains and germ band extension. Dividing cells can impact the stability of epithelial monolayers because they generate in-plane outward forces during the elongation phase<span class="citation" data-cites="Gupta2021-oo"><sup><a href="#ref-Gupta2021-oo" role="doc-biblioref">25</a></sup></span> and undergo basal detachment, creating an imbalance in the tissue.<span class="citation" data-cites="Ko2020-wk"><sup><a href="#ref-Ko2020-wk" role="doc-biblioref">26</a></sup></span> Mitotic rounding indeed induces epithelial folds in the tracheal placode of flies<span class="citation" data-cites="Kondo2013-gy"><sup><a href="#ref-Kondo2013-gy" role="doc-biblioref">27</a></sup></span> and in intestinal villi of mice,<span class="citation" data-cites="Freddo2016-et"><sup><a href="#ref-Freddo2016-et" role="doc-biblioref">28</a></sup></span> but it only occurs in the presence of an additional intraepithelial compressive force. Our findings that only the combined action of mitotic domains and germ band can induce ectopic folds (Figure <a href="#fig:live-experiments">4</a>j) indicates that a similar mechanism could be at play in cephalic furrow mutants. The role of mitotic domains and germ band as stressors is corroborated by complementary experiments performed in the independent study by Dey <em>et al.</em>,<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> consolidating the evidence that the head–trunk interface of flies is under increased mechanical stress during gastrulation. Since our experiments (Figure <a href="#fig:features">Extended Data Fig. 2</a>g,i–k) and simulations (Figure <a href="#fig:fold-model">3</a>g,h) show that the early formation of the cephalic furrow effectively inhibits the occurrence of ectopic folds, it gives support to the idea that the cephalic furrow is preventing the build up of these compressive stresses at the head–trunk boundary. We propose, therefore, that the cephalic furrow safeguards the mechanical stability of the head–trunk boundary, and thus, accomplishes a physical role during <em>Drosophila</em> gastrulation.</p>
<!--has a stress-absorbing role.-->
<!--Compressive forces are a key mechanism forming epithelial folds in development [@Nelson2016-gz; @Collinet2021-qa].-->
<!--Generating mechanical instability in particular when epithelial monolayers are under confinement [@Trushko2020-gf].-->
<!--The fact that *eve* mutants have no cell intercalation [@Collinet2015-af] suggests that the formation of ectopic folds depends on the rapid phase of germ band extension [@Campos-Ortega1985-ma], driven mainly by the posterior midgut invagination.-->
<!--Cephalic furrow absorbs stresses and prevents mechanical instability-->
<!--Cephalic furrow evolution link to tissue mechanics-->
<p>This raises an intriguing evolutionary question—has the cephalic furrow evolved in response to mechanical instability? For this to be the case, we expect that increased mechanical instability would be detrimental to the fitness of individuals. While mechanical compression and tissue buckling have the potential to disrupt cell-to-cell interactions and short-range signaling at the head–trunk boundary, investigating these effects <em>in vivo</em> is challenging. However, Dey <em>et al.</em> show that inhibiting cephalic furrow formation in <em>Drosophila</em> increases the frequency of a distorted ventral midline where the body axis becomes partially rotated during gastrulation.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> This suggests that increased compressive stresses and variable ectopic buckling cause a mechanical imbalance between embryonic tissues which can impact the robustness of morphogenetic processes and potentially slow down embryogenesis. In this sense, a patterned head invagination may have improved the robustness of gastrulation or allowed for faster development, providing an evolutionary advantage which was then positively selected.</p>
<!--Such adverse effects may have been significant in certain situations, selection for developmental speed, for example which likely occurred during fly evolution.-->
<!--Out-of-plane divisions as alternative mechanical solution-->
<p>Furthermore, the mapping of cephalic furrow traits on a dipteran phylogeny supports the hypothesis of mechanical instability as a selective pressure (Figure <a href="#fig:summary">6</a>a). Mitotic domains and germ band extension (long germ) are ancestral traits common to Diptera while the cephalic furrow is a derived trait—an evolutionary novelty of cyclorraphan flies<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> (Figure <a href="#fig:summary">6</a>a). The sources of stress, mitotic domains and germ band, were probably present at the dawn of dipterans. This suggests that the head–trunk interface of flies may have endured mechanical instabilities long before the evolution of the cephalic furrow. Remarkably, flies with no cephalic furrow, <em>Clogmia</em> and <em>Chironomus</em>, exhibit out-of-plane cell divisions exactly at the head–trunk boundary, suggesting that they evolved an alternative solution to mitigate the effect of tissue compression load during gastrulation<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> (Figure <a href="#fig:summary">6</a>a). One key difference that might explain these divergent solutions for the same selective pressure is the expression of <em>btd</em> at the head–trunk boundary. Neither <em>Clogmia</em> (Figure <a href="#fig:genetic-patterning">5</a>) nor <em>Chironomus</em><span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> exhibit the <em>btd</em>–<em>eve</em> expression overlap at the head–trunk boundary (Figure <a href="#fig:summary">6</a>a), a trait essential to specify the initiator cells in species with cephalic furrow like <em>Drosophila</em>. Therefore, the establishment of a <em>btd</em>–<em>eve</em> overlap was probably a key event associated with the origin of the cephalic furrow. In this evolutionary scenario, the mechanical instability was the underlying selective pressure leading to the cooption of novel expression domains and the genetic stabilization of a patterned cephalic furrow invagination (Figure <a href="#fig:summary">6</a>b).</p>
<!--Facilitated by this novel transcriptional background-->
<!--the signaling pathway regulating lateral myosin activation-->
<!--coopted to control the behavior of initiator cells-->
<!--and genetically stabilized by natural selection.-->
<!--Scenario for cephalic furrow evolution-->
<!--Altogether, the genetic patterning and tissue mechanics data obtained in our study and in the study by Dey et al. raises the following evolutionary scenario for the cephalic furrow ({@fig:summary}b).-->
<!--The evolution of the long germ in Diptera had as a consequence the increase in global compressive stresses happening during fly gastrulation.-->
<!--These stresses led to more frequent and potentially detrimental mechanical instability in embryonic tissues.-->
<!--The establishment of out-of-plane divisions was one solution to mitigate and it probably occurred by the fixation of genetic mechanisms determining the orientation.-->
<!--Cyclorraphan took another route.-->
<!--The genetic stabilization occurred through the btd--eve overlap via cooption of yet unknown genetic signaling controlling lateral myosin activation.-->
<!--The invention of the cephalic furrow may have allowed for shorter developmental times or may be the responsible factor leading to increased mechanical instability.-->
<!--One idea is that shorter developmental times concatenated the formation of mitotic domains and germ band extension, increasing the mechanical instabilities at the head--trunk boundary, or alternatively, that flies that evolved the cephalic furrow allowed for shorter developmental times.-->
<!--TODO: Model to test if invagination is more effective than out-of-plane divisions.-->
<!--Broader implications for the interplay between mechanics and patterned morphogenesis-->
<!--Genetic stabilization of self-organized morphogenesis by natural selection-->
<!--Empirical support for this mechanism of generating morphogenetic innovations-->
<p>Our cephalic furrow data reveals how the interplay between genetic patterning and tissue mechanics may have shaped the evolution of morphogenesis in the early development of flies. However, we believe that these findings potentially uncover only one realization of a more general mechanism integrating tissue mechanics with developmental evolution. Classical theoretical work by Newman and Müller raised the hypothesis that self-organized morphogenesis were critical to generate morphogenetic innovations at the dawn of animal evolution.<span class="citation" data-cites="Newman2000-oz"><sup><a href="#ref-Newman2000-oz" role="doc-biblioref">29</a></sup></span> Or, in other words, that before genome-guided embryonic development, tissues would fold due to mechanical interactions between them and their environment, and that the developmental “programs” we know today were fixed secondarily by natural selection. Our work provides initial empirical evidence in support of this hypothesis, showing that mechanical forces might have had a critical role in generating morphogenetic innovations not only at the transition to multicellularity, but also after the establishment of developmental patterning systems across evolutionary time. We propose that the genetic resolution of mechanical conflicts between distinct embryonic processes may be a broadly occurring mechanism that contributes to generating the remarkable morphogenetic diversity of early animal embryogenesis.</p>
<!--While the interplay between tissue mechanics and developmental evolution is poorly understood, classical theoretical work by Newman and Müller have raised the hypothesis that self-organized morphogenesis were critical to generate morphogenetic innovations at the dawn of animal evolution [@Newman2000-oz].-->
<!--Or, in other words, that before genome guided embryonic development evolved, tissues would fold due to mechanical interactions between them and the environment, and the patterned morphogenetic programs we know today were fixed secondarily by natural selection [@Newman2000-oz].-->
<!--Our work support this hypothesis by bringing empirical evidence from developmental and comparative data that mechanical forces might have a critical role in generating morphogenetic innovations in early development.-->
<!--In the case of the cephalic furrow, compressive stresses originating from concurrent gastrulation processes and resulting in ectopic epithelial buckling may have had a detrimental effect to the fitness of individuals.-->
<!--This mechanical instability could have acted as a selective pressure underlying the cooption of novel head--trunk gene expression domains and the establishment of a patterned cephalic furrow invagination.-->
<!--These speculations, however, require further testing with experiments that prevent cephalic furrow formation without other genetic side effects, as well as a systematic comparative analysis of mitotic domains, germ band extension, and tissue mechanics across the fly phylogeny.-->
<!--Overall, our work proposes that the cephalic furrow plays a mechanical role in absorbing compressive stresses in fly gastrulation, and introduces the cephalic furrow formation as powerful context to test the hypothesis that mechanical interactions within embryonic tissues may play an important role for the evolution of morphogenesis in early development.-->
<!--Alternatively, the evolution of the cephalic furrow was in fact the facilitator to allowing for faster development.-->
<!--Physical interaction of embryonic tissues with the egg shell are important for morphogenesis [@Bailles2019-qs; @Munster2019-ww]-->
<!--Our simulations also revealed the buckling events are influenced by the roundness of the vitelline envelope. Buckling occurs more easily on straighter shells. This might explain why buckling at the tips is less frequent. Friction plays a role in the shape of the fold [Trushko]. Recent works show how interactions of the embryonic tissues with the shell [@Munster;@Bailles] and between gastrulation movements [@Guo2021] are important to embryonic development. Insect eggs have an outstanding diversity [@Church]. This observation might have implications to how egg shape might have influenced the evolution of development. And reveal a glimpse of the complex mechanical underpinnings of gastrulation dynamics around the head--trunk boundary of bilaterians.-->
<!--Mechanical instability perturbs patterning and causes the loss of ATP, which may lead to  developmental delays-->
<!--Our findings also explain why so few cephalic furrow mutants have been identified. Because even though the cephalic furrow is disrupted, the tissue compression from other gastrulation movements will still make the epithelium fold, making the cephalic furrow phenotypes less obvious-->
<!--Dividing cells can’t do morphogenesis. That’s why mitotic domains occur in patches. Otherwise, gastrulation movements would fail.-->
<h1 id="methods">Methods</h1>
<h2 id="drosophila-stocks-and-genetic-crossings"><em>Drosophila</em> stocks and genetic crossings</h2>
<p>To generate fluorescent cephalic furrow mutants, we performed genetic crosses using the loss-of-function alleles <em>btd<sup>XA</sup></em> (FBal0030657), <em>eve<sup>3</sup></em> (FBal0003885), <em>prd<sup>4</sup></em> (FBal0013967), <em>slp<sup>Δ34B</sup></em> (FBal0035631), and <em>stg<sup>2</sup></em> (FBal0247234); the membrane fluorescent marker <em>Gap43-mCherry</em> (FBal0258719, gift from Kassiani Skouloudaki); and the green fluorescent balancers <em>FM7c, Kr-GFP</em> (FBst0005193), <em>CyO, twi-GFP</em> (gift from Akanksha Jain), and <em>TM3, Kr-GFP</em> (FBst0005195). We established stable lines balancing the loss-of-function alleles with fluorescent balancers, and used the lack of GFP signal to identify homozygous embryos in our live-imaging recordings. For genes on chromosomes 1 and 2 (<em>btd</em>, <em>eve</em>, and <em>prd</em>), we added the membrane marker on chromosome 3 ([<em>btd<sup>XA</sup>/FM7c, Kr-GFP;; Gap43-mCherry/MKRS</em>], [<em>eve<sup>3</sup>/CyO, twi-GFP; Gap43-mCherry/MKRS</em>], and [<em>slp<sup>Δ34B</sup>/CyO, twi-GFP; Gap43-mCherry/TM6B</em>]). For <em>stg</em>, which is located on chromosome 3, we recombined the allele with Gap (<em>Gap43-mCherry, stg<sup>2</sup>/TM3, Kr-GFP</em>). Since the <em>btd</em>–<em>stg</em> double mutant stable line is weak, we imaged the progeny of <em>btd<sup>XA</sup>/FM7c, Kr-GFP;; Gap43-mCherry, stg<sup>2</sup>/Gap43-mCherry</em> flies, identifying <em>btd</em> homozygozity by the GFP, and <em>stg</em> homozygozity by the lack of cell divisions after gastrulation. For laser ablations, we used a <em>moe-GFP</em> line (gift from Elizabeth Knust). The wildtype stocks contain the Gap43-mCherry marker in the Oregon-R genetic background. We obtained the founder fly stocks from the Bloomington Drosophila Stock Center and the Kyoto Stock Center and deposited the lines in the MPI-CBG stock collection. The complete list of FlyBase<span class="citation" data-cites="Gramates2022-xe"><sup><a href="#ref-Gramates2022-xe" role="doc-biblioref">30</a></sup></span> accession numbers and genotypes is available in the project’s data repository.<span class="citation" data-cites="Vellutini2023-ng"><sup><a href="#ref-Vellutini2023-ng" role="doc-biblioref">31</a></sup></span></p>
<h2 id="animal-husbandry-and-embryo-collection">Animal husbandry and embryo collection</h2>
<p>We maintained the <em>Drosophila</em> stocks in 50mL hard plastic vials containing standard fly food and enclosed with a foam lid to allow air exchange. They were kept in an incubator with a constant 25°C temperature and 65% humidity and a 12:12h light cycle. For imaging, we first amplified the stocks in larger 200mL vials for a few weeks. We then narcotized the flies with CO<sub>2</sub>, and transferred them to a cage with a plate attached to one end containing a layer of apple juice agar and a slab of yeast paste on top. The flies were left to acclimatize in the cage for two days before the experiments. To guarantee the embryos are at a similar developmental stage, we exchanged the agar plate once per hour at least twice (pre-lays), and let the flies lay the eggs on the agar for one hour before collecting the plate. After filling the plate with water, we used a brush to release the eggs from the agar and transferred them to a cell strainer with 100µm nylon mesh (VWR). To remove the chorion, we immersed the embryos in 20% bleach (sodium hypochlorite solution, Merck 1.05614.2500) for 90s, washed abundantly with water, and proceeded to mounting for live imaging.</p>
<p>We maintained <em>Clogmia</em> flies in plastic petri dishes with a layer of wet cotton at 25°C and fed weekly with powder parsley. To obtain embryos for fixation, we collected the adult flies in a 200mL hard plastic vial with wet cotton, and let them mate for 2–3 days. Then, we anesthetized the flies with CO<sub>2</sub>, dissected the ovaries from ripe females, and released the eggs using tweezers in deionized water, which activates embryonic development.<span class="citation" data-cites="Sander1985-sg Garcia-Solache2010-lz"><sup><a href="#ref-Sander1985-sg" role="doc-biblioref">32</a>,<a href="#ref-Garcia-Solache2010-lz" role="doc-biblioref">33</a></sup></span> We let embryos develop in deionized water at room temperature until the desired stage. To remove the chorion, we transferred the embryos to a glass vial with 0.5x PBS using a fine brush, exchanged the medium for 5% bleach in 0.5x PBS for 2min, and washed abundantly with 0.5x PBS. Using the diluted PBS solution instead of water prevents the embryos from bursting after dechorionation.</p>
<!--TODO: Add width of petri dish-->
<h2 id="embryo-fixation-and-in-situ-hybridization">Embryo fixation and <em>in situ</em> hybridization</h2>
<p>For <em>Drosophila</em>, we transferred dechorionated embryos to a glass vial containing equal volumes of 4% paraformaldehyde in PBS and n-Heptane, and let the vial shaking at 215rpm for 45min. After removing the fixative (lower phase) using a glass pipet, we added an equal volume of 100% methanol, and shook the vial vigorously by hand for 1min. We then removed the n-Heptane (upper phase) and collected the embryos on the bottom to an eppendorf tube and washed several times with 100% methanol. Samples were stored at -20°C. For Clogmia, we used the same protocol, but diluted the fixative in 0.5x PBS.</p>
<p>We performed the <em>in situ</em> hybridization of <em>btd</em>, <em>eve</em>, <em>prd</em>, and <em>slp</em> genes in <em>Drosophila</em> and <em>Clogmia</em> using the Hybridization Chain Reaction v3.0 (HCR<sup>TM</sup>)<span class="citation" data-cites="Choi2018-fg"><sup><a href="#ref-Choi2018-fg" role="doc-biblioref">34</a></sup></span> reagents, except for the probe sets, which we designed using a custom script. The oligos were obtained from Sigma-Aldrich. We selected the HCR<sup>TM</sup> amplifiers to allow for triple (multiplexed) <em>in situ</em> combinations of <em>btd</em>+<em>eve</em>+<em>slp</em> or <em>prd</em>+<em>eve</em>+<em>slp</em>. Before starting, we rehydrated the embryos in 100% methanol with a series of washes to 100% PBT. We permeabilized <em>Clogmia</em> embryos with 1:5000 dilution of ProteinaseK (20 mg/mL) for 5min, but not <em>Drosophila</em> embryos. All samples were re-fixed in 4% paraformaldehyde for 40min and washed thoroughly with PBT. We then followed the <em>In situ HCR v3.0 protocol for whole-mount fruit fly embryos Revision 4 (2019-02-21)</em> from Molecular Instruments <a href="https://www.molecularinstruments.com/hcr-rnafish-protocols">molecularinstruments.com/hcr-rnafish-protocols</a>. After the protocol, we stained the embryos with 1:1000 DAPI in 5x SSCT solution for 2h and mounted the embryos in 80% glycerol in 5x SSCT for imaging.</p>
<h2 id="sample-mounting-for-microscopy">Sample mounting for microscopy</h2>
<p>For most of our live imaging, we used a Zeiss Lightsheet Z.1 microscope. To increase the throughput of samples imaged in one session, we optimized a mounting strategy developed previously in our laboratory.<span class="citation" data-cites="Schmied2016-oq"><sup><a href="#ref-Schmied2016-oq" role="doc-biblioref">35</a></sup></span> First, we cut a 22x22mm glass coverslip (0.17mm thickness) into 6x15mm strips using a diamond knife, and attached a single strip to a custom sample holder using silicon glue, letting it harden for 15min. We then coated the coverslip strip with a thin layer of heptane glue and let it dry while preparing the embryos. Using a fine brush, we transferred the embryos collected in the cell strainer onto an agar pad, and oriented them manually with a blunt cactus spine under a stereomicroscope. We aligned about 20 embryos in a single line (head to tail) along the main axis of the strip with the left or ventral sides up, depending on the experiment. To attach the embryos to the coverslip, we carefully lowered the sample holder over the agar pad until the glass coated with heptane glue touched the embryos. We placed the sample holder into the microscope chamber filled with water, and rotated it so that the samples are facing the detection objective directly, and the coverslip is orthogonal to the detection objective; this is important to prevent the lightsheet from hitting the glass edges. With the embryos oriented vertically along the coverslip, the lightsheet generated from the illumination objectives coming from the sides only needs to pass through the width of the embryo (about 200µm). This approach gives the best results for recording lateral and dorsal views and is ideal for live imaging homozygote embryos since they are only about one fourth of the total number of imaged embryos. For fixed imaging of <em>in situ</em> samples we used an inverted Zeiss LSM 700 Confocal microscope. We mounted the samples immersed in 80% glycerol between a slide and a glass coverslip supported by tape.</p>
<h2 id="microscopy-acquisition-parameters">Microscopy acquisition parameters</h2>
<p>For the lightsheet lateral datasets, we used a Zeiss 20x/1NA Plan-Apochromat water immersion objective to acquire stacks with 0.28µm XY-resolution and 3µm Z-resolution covering half of the embryo’s volume in a single view. This Z-resolution was restored to 1µm during image processing (see below). For the dorsal datasets, we used a Zeiss 40x/1NA Plan-Apochromat water immersion objective to acquire stacks with 0.14µm XY-resolution and 3µm Z-resolution covering a volume around in the middle section of the anterior end of the embryo. We adjusted the time resolution between 45–60s per frame to maximize the number of embryos acquired in one session. To visualize both the membrane signal (mCherry) and the green balancer signal (GFP), we acquired two channels simultaneously using the 488 and 561nm lasers at 3% power with an image splitter cube containing a LP560 dichromatic mirror with SP550 and LP585 emission filters. All live imaging recordings were performed at 25°C. For the confocal datasets, we used a 20x/0.8 Plan-Apochromat Zeiss air objective to acquire 4-channels using 3 tracks (405nm, 488 and 639nm, and 555nm) with a BP575-640 emission filter and about 0.4µm XY-resolution and 2µm Z-resolution covering about half the embryo’s volume.</p>
<!--using a pixel dwell time of x with 2 line averaging and 1 airy units.-->
<h2 id="image-processing-and-visualization">Image processing and visualization</h2>
<p>We converted the raw imaging datasets into individual TIFF stacks for downstream processing using a custom ImageJ macro in Fiji.<span class="citation" data-cites="Schindelin2012-di Rueden2017-ky"><sup><a href="#ref-Schindelin2012-di" role="doc-biblioref">36</a>,<a href="#ref-Rueden2017-ky" role="doc-biblioref">37</a></sup></span> To visualize the presence and dynamics of ectopic folds, we generated 3D renderings of the surface of embryos in lateral recordings using the plugin 3Dscript in Fiji.<span class="citation" data-cites="Schmid2019-bm"><sup><a href="#ref-Schmid2019-bm" role="doc-biblioref">38</a></sup></span> For analyzing the entire epithelial surface, we first improved the signal-to-noise ratio and z-resolution of lateral datasets from 3µm to 1µm by training a deep learning upsampling model using CARE.<span class="citation" data-cites="Weigert2018-ti"><sup><a href="#ref-Weigert2018-ti" role="doc-biblioref">39</a></sup></span> Then, we created cartographic projections of the lateral recordings using the ImSAnE toolbox<span class="citation" data-cites="Heemskerk2015-kv"><sup><a href="#ref-Heemskerk2015-kv" role="doc-biblioref">40</a></sup></span> by loading the restored data in MATLAB,<span class="citation" data-cites="Matlab2015-nd"><sup><a href="#ref-Matlab2015-nd" role="doc-biblioref">41</a></sup></span> segmenting the epithelial surface using ilastik,<span class="citation" data-cites="Berg2019-ab"><sup><a href="#ref-Berg2019-ab" role="doc-biblioref">42</a></sup></span> and generating 3D cartographic projections of lateral views following a workflow established for fly embryos.<span class="citation" data-cites="Vellutini2022-ya"><sup><a href="#ref-Vellutini2022-ya" role="doc-biblioref">43</a></sup></span> To visualize <em>in situ</em> hybridization data, we performed maximum intensity projections or extracted single slices from the raw volumes. For all microscopy images, we only performed minimal linear intensity adjustments to improve their contrast and brightness.<span class="citation" data-cites="Schmied2020-ip"><sup><a href="#ref-Schmied2020-ip" role="doc-biblioref">44</a></sup></span></p>
<h2 id="ectopic-fold-analyses">Ectopic fold analyses</h2>
<p>To characterize the relative timing of ectopic folding, we annotated the germ band position and the number of frames after the onset of gastrulation at the initial buckling, when the first cells disappear from the surface in the lateral 3D renderings. We defined the onset of gastrulation (T=0) as the moment immediately after the end of cellularization, and immediately before the beginning of the ventral furrow invagination. To visualize the variability of ectopic folding, we manually traced the fold outlines in lateral recordings using Fiji. Because embryos have different sizes, we first used the plugin <em>bUnwarpJ</em><span class="citation" data-cites="Arganda-Carreras2006-im"><sup><a href="#ref-Arganda-Carreras2006-im" role="doc-biblioref">45</a></sup></span> (<a href="https://imagej.net/plugins/bunwarpj/">imagej.net/plugins/bunwarpj</a>) to register individual frames and then applied the same transformation to the fold traces for a standardized comparison. We analyzed the dynamics of ectopic folds by measuring the relative angle and tortuosity of the segmented line traces over time, and to visualize the kinetics we generated color-coded temporal projections using the script <em>Temporal Color Code</em> (<a href="https://imagej.net/plugins/temporal-color-code">imagej.net/plugins/temporal-color-code</a>) with the perceptually uniform <em>mpl-viridis</em> color map (<a href="https://bids.github.io/colormap/">bids.github.io/colormap</a>) bundled in Fiji.</p>
<p>To estimate the folded area in the cephalic furrow and ectopic folds, we annotated the region of the blastoderm before gastrulation that infolded in the cartographic projections using Fiji, and calculated the area correcting the pixel dimensions according to the coordinates in the projection. For the fold depth, we measured the distance between the vitelline envelope to the tip of the fold at the moment of maximum depth in the dorsal recordings. For the analysis of the epithelial surface, we used the plugin <em>MorphoLibJ</em><span class="citation" data-cites="Legland2016-cp"><sup><a href="#ref-Legland2016-cp" role="doc-biblioref">46</a></sup></span> (<a href="https://imagej.net/plugins/morpholibj">imagej.net/plugins/morpholibj</a>) to segment, measure, and color-code the cell apical areas, and the plugin <em>Linear Stack Alignment with SIFT</em> (<a href="https://imagej.net/plugins/linear-stack-alignment-with-sift">imagej.net/plugins/linear-stack-alignment-with-sift</a>) to register cells between timepoints.</p>
<h2 id="laser-cauterization-experiments">Laser cauterization experiments</h2>
<p>We performed laser cauterization experiments in two microscope setups, a Luxendo MuVi SPIM with a photomanipulation module and a Zeiss LSM 780 NLO with multiphoton excitation. For the MuVi SPIM, we embedded dechorionated embryos in 2% low-melting agarose and mounted the samples in glass capillaries to obtain <em>in toto</em> recordings. We used a pulsed infrared laser 1030–1040nm with 200fs pulse duration and 1.5W power to cauterize the posterior region of the dorsal embryonic surface, attaching the blastoderm to the vitelline envelope. Using a Olympus 20x/1.0NA water immersion objective, we acquired stacks with 0.29µm XY-resolution and 1µm Z-resolution of four different angles every one minute. For the Zeiss microscope, we attached the embryos with the dorsal side down onto coverslips using heptane glue and immersing in halocarbon oil. We cauterized the embryos sequentially using a near infrared 800nm laser (Chameleon Vision II) through a single pixel line (210nm/px and 100µs/px) around the same dorsal region to block the germ band extension. We used a Zeiss 25x/0.8NA LD LCI Plan-Apochromat glycerol immersion objective to acquire every 2:38min two different planes of the blastoderm, (i) the surface to monitor the germ band extension, and (ii) 40µm deep in the equatorial region to monitor the occurrence of ectopic folding. The stacks had 0.21µm XY-resolution and one minute time resolution. To obtain a quantitative measure of ectopic folding, we analyzed the degree by which the tissues deform between non-cauterized and cauterized mutants using as a proxy the tortuosity of the epithelium outline. For that, we took the profile slices from dorsal recordings and transformed the curved vitelline envelope into a straight line using the <em>Straighten</em> tool of ImageJ (Figure <a href="#fig:tortuosity">Supplementary Method 1</a>a). We then cropped a 200×25µm region along the head–trunk interface and applied gaussian blur, thresholding, and edge detection to obtain the epithelium outline for individual timepoints covering about 50min after gastrulation (Figure <a href="#fig:tortuosity">Supplementary Method 1</a>a,b). We extracted measurements from the epithelium outlines using the ImageJ plugin <em>Analyze Skeleton</em><span class="citation" data-cites="Arganda-Carreras2010-la"><sup><a href="#ref-Arganda-Carreras2010-la" role="doc-biblioref">47</a></sup></span> (<a href="https://imagej.net/plugins/analyze-skeleton/">imagej.net/plugins/analyze-skeleton</a>), and generated the color-coded temporal projections as described above.</p>
<!--TODO: Temperature in MuVi and Zeiss?-->
<h2 id="laser-ablation-experiments">Laser ablation experiments</h2>
<p>We performed laser ablations in a Yokogawa CSU-X1 confocal spinning disk, an EMCCD camera (Andor iXon DU-888), and the software AndorIQ for image acquisition. We attached dechorionated embryos laterally to a MatTek glass-bottom petri dish and covered the samples with water, and performed the ablations using a Titanium Sapphire Chameleon Ultra II (Coherent) laser at 800nm tuned down from 80MHz to 20kHz with a pulse-picker. The laser power measured before the microscope port was 6 mW and the pixel dwell time for scanning was 2µs. To ensure the cut, we repeated the scan ten consecutive times along a single cell acquiring a single slice with a 60x/1.2NA water immersion objective with 0.18µm XY-resolution and 200ms time-steps. We ablated each embryo just once. The temperature was maintained at 28°C. To analyze the ablation data, we created a line crossing the edges of the ablated cell perpendicular to the cut and generated a kymograph using the <em>Multi Kymograph</em> Fiji plugin (Figure <a href="#fig:ablation">Supplementary Note 2</a>). We then binarized the kymographs, measured the distance between cell edges over the first 30s after the cut, and performed a linear fit of the data to obtain the recoil velocity (Figure <a href="#fig:ablation">Supplementary Note 2</a>).</p>
<!--TODO: What is the brand of the objective?-->
<!--TODO: Does a higher temperature affects the quantifications?-->
<h2 id="strain-rate-analysis">Strain rate analysis</h2>
<p>To estimate the strain rates, we first performed particle image velocimetry on cartographic projections using the ImageJ plugin <em>iterativePIV</em><span class="citation" data-cites="Tseng2012-ni"><sup><a href="#ref-Tseng2012-ni" role="doc-biblioref">48</a></sup></span> (<a href="https://sites.google.com/site/qingzongtseng/piv">sites.google.com/site/qingzongtseng/piv</a>). Then, we used the equation <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>E</mi><mo>=</mo><mrow><mo stretchy="true" form="prefix">|</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><mover><mi>∇</mi><mo accent="true">⃗</mo></mover><mi>.</mi><mover><mi>v</mi><mo accent="true">⃗</mo></mover><mo stretchy="true" form="postfix">)</mo></mrow><mo>+</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><msub><mi>∂</mi><mi>x</mi></msub><msub><mi>v</mi><mi>y</mi></msub><mo>+</mo><msub><mi>∂</mi><mi>y</mi></msub><msub><mi>v</mi><mi>x</mi></msub><mo stretchy="true" form="postfix">)</mo></mrow><mo stretchy="true" form="postfix">|</mo></mrow></mrow><annotation encoding="application/x-tex">E = \left| \frac{1}{2} \left( \vec \nabla . \vec v \right) + \frac{1}{2} \left( \partial_x v_y + \partial_y v_x \right) \right|</annotation></semantics></math> to define and calculate the magnitude of the strain rate, where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>v</mi><annotation encoding="application/x-tex">v</annotation></semantics></math> is the displacement obtained in the PIV divided by the time in minutes. The measurements combine isotropic and anisotropic strain rate. We used these values to create a color-coded overlay for the strain rate (Figure <a href="#fig:apical-strain">Extended Data Fig. 3</a>b). To generate the line plots, we averaged the strain rate along the dorsoventral axis in two pre-defined regions, the head–trunk (canonical cephalic furrow position) and the trunk–germ (posterior to the Mitotic Domain 6) (Figure <a href="#fig:apical-strain">Extended Data Fig. 3</a>b).</p>
<h2 id="model-and-simulations">Model and simulations</h2>
<p>Our model follows an approach similar to a previously published model of epithelial buckling under confinement.<span class="citation" data-cites="Trushko2020-gf"><sup><a href="#ref-Trushko2020-gf" role="doc-biblioref">16</a></sup></span> It represents the monolayer epithelium of the early <em>Drosophila</em> embryo in a cross section as a single line through the apical–basal midline of the epithelial cells. The tissue is modeled as an elastic rod with a stretching energy per unit length <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>s</mi></msub><annotation encoding="application/x-tex">W_s</annotation></semantics></math> and bending energy per unit length <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>b</mi></msub><annotation encoding="application/x-tex">W_b</annotation></semantics></math>, so that the total energy per unit length is <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>W</mi><mi>T</mi></msub><mo>=</mo><msub><mi>W</mi><mi>s</mi></msub><mo>+</mo><msub><mi>W</mi><mi>b</mi></msub></mrow><annotation encoding="application/x-tex">W_T = W_s + W_b</annotation></semantics></math>. In full, <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>W</mi><mi>T</mi></msub><mo>=</mo><msub><mo>∫</mo><msub><mi>L</mi><mi>o</mi></msub></msub><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>s</mi></msub><msup><mrow><mo stretchy="true" form="prefix">(</mo><mfrac><mrow><mi>d</mi><mi>s</mi></mrow><mrow><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow></mfrac><mo>−</mo><mn>1</mn><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub><mo>+</mo><msub><mo>∫</mo><msub><mi>L</mi><mi>o</mi></msub></msub><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>b</mi></msub><msup><mrow><mo stretchy="true" form="prefix">(</mo><mi>κ</mi><mo>−</mo><msub><mi>κ</mi><mi>o</mi></msub><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">W_T = \int_{L_o} \frac{1}{2} K_s \left( \frac{ds}{ds_o} - 1 \right)^2 ds_o + \int_{L_o} \frac{1}{2} K_b \left( \kappa - \kappa_o \right)^2 ds_o</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>s</mi></msub><annotation encoding="application/x-tex">K_s</annotation></semantics></math> is the stretching rigidity and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math> is the bending rigidity of the tissue; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">ds_o</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><mi>s</mi></mrow><annotation encoding="application/x-tex">ds</annotation></semantics></math> are the preferred and current lengths of the curve, respectively; and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>κ</mi><annotation encoding="application/x-tex">\kappa</annotation></semantics></math> is the curvature of the rod. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>L</mi><mi>o</mi></msub><annotation encoding="application/x-tex">L_o</annotation></semantics></math> is the total length of the tissue in a stress-free condition. To perform numerics, we discretize the curve into <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>N</mi><annotation encoding="application/x-tex">N</annotation></semantics></math> particles indexed by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math>. The total energy per unit length for this discretized model is given by <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>W</mi><mi>T</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>s</mi></msub><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>2</mn></mrow><mrow><mi>N</mi><mo>−</mo><mn>3</mn></mrow></munderover><msup><mrow><mo stretchy="true" form="prefix">(</mo><mfrac><mrow><mi>Δ</mi><msub><mi>r</mi><mi>i</mi></msub></mrow><mrow><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow></mfrac><mo>−</mo><mn>1</mn><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub><mo>+</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>b</mi></msub><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>2</mn></mrow><mrow><mi>N</mi><mo>−</mo><mn>3</mn></mrow></munderover><msup><mrow><mo stretchy="true" form="prefix">(</mo><msub><mi>κ</mi><mi>i</mi></msub><mo>−</mo><msub><mi>κ</mi><mrow><mi>o</mi><mo>,</mo><mi>i</mi></mrow></msub><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">W_T^{\text{*}} = \frac{1}{2} K_s \sum_{i = 2}^{N-3} \left( \frac{\Delta r_i}{\Delta r_o} - 1 \right)^2 \Delta r_o + \frac{1}{2} K_b \sum_{i = 2}^{N-3} \left( \kappa_i - \kappa_{o,i} \right)^2 \Delta r_o</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">\Delta r_o</annotation></semantics></math> is the preferred length of springs connecting consecutive points (equal for all springs); <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mi>i</mi></msub></mrow><annotation encoding="application/x-tex">\Delta r_i</annotation></semantics></math> is the current length between <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">i+1</annotation></semantics></math>; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>κ</mi><mi>i</mi></msub><annotation encoding="application/x-tex">\kappa_i</annotation></semantics></math> is the discretized curvature at point <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math>; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>κ</mi><mrow><mi>o</mi><mo>,</mo><mi>i</mi></mrow></msub><annotation encoding="application/x-tex">\kappa_{o,i}</annotation></semantics></math> is the preferred curvature at point <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math> (equal to 0, except when specified). The first and last two points of the curve are fixed in space. To obtain a physically meaningful dimensionless bending rigidity, we divide the bending rigidity by the factor <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>s</mi></msub><msup><mi>L</mi><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_sL^2</annotation></semantics></math> as <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mfrac><msub><mi>K</mi><mi>b</mi></msub><mrow><msub><mi>K</mi><mi>s</mi></msub><msup><mi>L</mi><mn>2</mn></msup></mrow></mfrac></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} = \frac{K_b}{K_sL^2}</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>L</mi><annotation encoding="application/x-tex">L</annotation></semantics></math> is the semi-major axis of the embryo. To minimize the total energy, we add a ground level of noise to the particles and let the particles move in the direction of the forces. The motion of the particles are governed by <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mfrac><mrow><mi>Δ</mi><msub><munder><mi>r</mi><mo accent="true">_</mo></munder><mi>i</mi></msub></mrow><mrow><mi>Δ</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mo>−</mo><mfrac><mi>L</mi><mrow><msub><mi>K</mi><mi>s</mi></msub><mi>τ</mi></mrow></mfrac><mfrac><mrow><mi>∂</mi><msup><mi>W</mi><mtext mathvariant="normal">*</mtext></msup></mrow><mrow><mi>∂</mi><msub><munder><mi>r</mi><mo accent="true">_</mo></munder><mi>i</mi></msub></mrow></mfrac><mo>+</mo><msub><munder><mi>ζ</mi><mo accent="true">_</mo></munder><mi>i</mi></msub></mrow><annotation encoding="application/x-tex">\frac{\Delta\underline{r}_i}{\Delta t} = - \frac{L}{K_s\tau}\frac{\partial W^{\text{*}}}{\partial \underline{r}_i} + \underline{\zeta}_i</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><munder><mi>r</mi><mo accent="true">_</mo></munder><mi>i</mi></msub><annotation encoding="application/x-tex">\underline{r}_i</annotation></semantics></math> is the current position of the <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math>th particle; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>τ</mi><annotation encoding="application/x-tex">\tau</annotation></semantics></math> represents an arbitrary timescale introduced here to balance dimensions (set to 1); <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><mi>t</mi></mrow><annotation encoding="application/x-tex">\Delta t</annotation></semantics></math> are the timesteps (set to <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>5</mn></mrow></msup><mo>×</mo><mi>τ</mi><msub><mi>K</mi><mi>s</mi></msub><mi>/</mi><mi>L</mi></mrow><annotation encoding="application/x-tex">10^{-5} \times \tau K_s/L</annotation></semantics></math>); and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><munder><mi>ζ</mi><mo accent="true">_</mo></munder><mi>i</mi></msub><annotation encoding="application/x-tex">\underline{\zeta}_i</annotation></semantics></math> is the noise, chosen from a gaussian distribution with mean 0 and standard distribution <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>5</mn></mrow></msup><mo>×</mo><mi>L</mi></mrow><annotation encoding="application/x-tex">10^{-5}\times L</annotation></semantics></math>. In our model the position of the germ band corresponds to the position of the last particle in the curve on the semi-ellipse that represents the embryonic blastoderm. The extent of the germ band is given by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math> which is the projection of the germ band arclength onto the mid-axis of the embryo normalized by the embryo length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>2</mn><mi>L</mi></mrow><annotation encoding="application/x-tex">2L</annotation></semantics></math>). When <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">g=0</annotation></semantics></math> the tissue is free of stretching stress, but at any other <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>0</mn><mo>&lt;</mo><mi>g</mi><mo>&lt;</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">0&lt;g&lt;1</annotation></semantics></math>, the blastoderm will be compressed. The preferred lengths of the individual springs is obtained by dividing the elliptical arclength into <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>N</mi><annotation encoding="application/x-tex">N</annotation></semantics></math> equal segments. The length of each segment is given by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mtext mathvariant="normal">o</mtext></msub><mo>=</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><mi>L</mi><msubsup><mo>∫</mo><mn>0</mn><mi>π</mi></msubsup><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>e</mi><mn>2</mn></msup><msup><mo>cos</mo><mn>2</mn></msup><mo stretchy="false" form="prefix">(</mo><mi>u</mi><mo stretchy="false" form="postfix">)</mo></mrow></msqrt><mi>d</mi><mi>u</mi><mo stretchy="true" form="postfix">)</mo></mrow></mrow><annotation encoding="application/x-tex">\Delta r_{\text{o}} = \frac{1}{N} \left( L\int_{0}^{\pi} \sqrt{1 - e^2 \cos^2(u)} du \right)</annotation></semantics></math>. To find the initial lengths of the springs, we use <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><mi>r</mi><mo stretchy="false" form="prefix">(</mo><mi>t</mi><mo>=</mo><mn>0</mn><mo stretchy="false" form="postfix">)</mo><mo>=</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><mi>L</mi><msubsup><mo>∫</mo><mrow><mi>u</mi><mi>′</mi></mrow><mi>π</mi></msubsup><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>e</mi><mn>2</mn></msup><msup><mo>cos</mo><mn>2</mn></msup><mo stretchy="false" form="prefix">(</mo><mi>u</mi><mo stretchy="false" form="postfix">)</mo></mrow></msqrt><mi>d</mi><mi>u</mi><mo stretchy="true" form="postfix">)</mo></mrow></mrow><annotation encoding="application/x-tex">\Delta r(t = 0) = \frac{1}{N} \left( L\int_{u&#39;}^{\pi} \sqrt{1 - e^2 \cos^2(u)} du \right)</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>e</mi><mo>=</mo><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mrow><mo stretchy="false" form="prefix">(</mo><mn>0.4</mn><mo stretchy="false" form="postfix">)</mo></mrow><mn>2</mn></msup></mrow></msqrt></mrow><annotation encoding="application/x-tex">e = \sqrt{1 - {(0.4)}^{2}}</annotation></semantics></math>; and the angle <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>u</mi><mi>′</mi></mrow><annotation encoding="application/x-tex">u&#39;</annotation></semantics></math> corresponds to the position of the blastoderm end. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>u</mi><mi>′</mi></mrow><annotation encoding="application/x-tex">u&#39;</annotation></semantics></math> is obtained for a given value of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math> by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>u</mi><mi>′</mi><mo>=</mo><msup><mo>cos</mo><mrow><mo>−</mo><mn>1</mn></mrow></msup><mo stretchy="false" form="prefix">(</mo><mn>1</mn><mo>−</mo><mn>2</mn><mi>g</mi><mo stretchy="false" form="postfix">)</mo></mrow><annotation encoding="application/x-tex">u&#39; = \cos^{-1}(1 - 2g)</annotation></semantics></math>. Here, we obtained the initial lengths by dividing the compressed blastoderm into <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>N</mi><annotation encoding="application/x-tex">N</annotation></semantics></math> equal segments. For any simulation, the value of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math> is constant (blastoderm end is static in position). To model mitotic domains, we introduced new particles and springs on the mid-points between two particles in specific regions of length <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>0.5</mn><mi>L</mi></mrow><annotation encoding="application/x-tex">0.5L</annotation></semantics></math>. The new springs are given the same <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">\Delta r_{o}</annotation></semantics></math> as the rest of the springs in the tissue. The blastoderm is confined by a rigid boundary in the shape of a semi-ellipse. Any particle that lands outside this boundary at any timestep is re-positioned onto the rigid boundary. This new position is prescribed by taking the intersection point of the rigid boundary curve and the line segment that connects the position before this iteration (which was inside or on the vitelline envelope) and the position outside the vitelline envelope. Finally, we define and count a <em>fold</em> when we find that a particle’s distance from the rigid boundary is greater than a threshold value. To calculate this threshold, we measure the maximum distance that particles can achieve when the tissue is in a stress-free state. This threshold was calculated to be <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>0.035</mn><mi>L</mi></mrow><annotation encoding="application/x-tex">0.035L</annotation></semantics></math>. The code for the model and simulations is available in a Zenodo repository.<span class="citation" data-cites="Krishna2023-hu"><sup><a href="#ref-Krishna2023-hu" role="doc-biblioref">49</a></sup></span></p>
<h2 id="data-visualization-and-figure-assembly">Data visualization and figure assembly</h2>
<p>We created illustrations and assembled the final figure plates using Inkscape v1.2.2.<span class="citation" data-cites="The_Inkscape_Project2003-ht"><sup><a href="#ref-The_Inkscape_Project2003-ht" role="doc-biblioref">50</a></sup></span> For microscopy videos, we exported the original stacks as AVI without compression with 10–15fps using Fiji and post-processed them to MPEG-4 format 1080p resolution using the H.264 encoding at a constant bitrate quality factor of 18 for visualization using HandBrake v1.6.1.<span class="citation" data-cites="HandBrake_Team2003-fq"><sup><a href="#ref-HandBrake_Team2003-fq" role="doc-biblioref">51</a></sup></span> The high-resolution figures and videos are available in a Zenodo repository.<span class="citation" data-cites="Vellutini2023-ix"><sup><a href="#ref-Vellutini2023-ix" role="doc-biblioref">52</a></sup></span> We performed the data wrangling, statistical analyses, and plotting in R v4.2.1<span class="citation" data-cites="R-Core-Team1993-jw"><sup><a href="#ref-R-Core-Team1993-jw" role="doc-biblioref">53</a></sup></span> using RStudio v2022.7.2.576,<span class="citation" data-cites="RStudio-Team2011-vj"><sup><a href="#ref-RStudio-Team2011-vj" role="doc-biblioref">54</a></sup></span> and Python 3.10.7 using Jupyter notebooks.<span class="citation" data-cites="Granger2021-zg"><sup><a href="#ref-Granger2021-zg" role="doc-biblioref">55</a></sup></span> All the data and analyses pipelines were deposited in a Zenodo repository.<span class="citation" data-cites="Vellutini2023-ng"><sup><a href="#ref-Vellutini2023-ng" role="doc-biblioref">31</a></sup></span></p>
<h2 id="data-availability">Data availability</h2>
<ul>
<li>Data, scripts and analyses:<span class="citation" data-cites="Vellutini2023-ng"><sup><a href="#ref-Vellutini2023-ng" role="doc-biblioref">31</a></sup></span>
<ul>
<li><a href="https://github.com/bruvellu/cephalic-furrow">github.com/bruvellu/cephalic-furrow</a> (Git repository)</li>
<li><a href="https://zenodo.org/record/7781947">zenodo.org/record/7781947</a> (Zenodo record)</li>
</ul></li>
<li>Model and simulations:<span class="citation" data-cites="Krishna2023-hu"><sup><a href="#ref-Krishna2023-hu" role="doc-biblioref">49</a></sup></span>
<ul>
<li><a href="https://github.com/abhijeetkrishna/cephalic-furrow-theory">github.com/abhijeetkrishna/cephalic-furrow-theory</a> (Git repository)</li>
<li><a href="https://zenodo.org/record/7784906">zenodo.org/record/7784906</a> (Zenodo record)</li>
</ul></li>
<li>Figures and videos:<span class="citation" data-cites="Vellutini2023-ix"><sup><a href="#ref-Vellutini2023-ix" role="doc-biblioref">52</a></sup></span>
<ul>
<li><a href="https://zenodo.org/record/7781916">zenodo.org/record/7781916</a> (Zenodo record)</li>
</ul></li>
</ul>
<h1 id="acknowledgments">Acknowledgments</h1>
<p>We thank all current and former LoPaTs for the discussions and support during this project. Akanksha Jain and Vladimir Ulman for initial help with cartographic projections, Giulia Serafini for help with fly crosses, Anaïs Bailles for constructive feedback, and Michaela Burkon and Pavel Mejstrik for technical support. Jan Brugués and Keisuke Ishihara for the laser ablation setup, the MPI-CBG Light Microscopy Facility and Computer Services Facility for the ample support, and Sven Ssykor and Cornelia Maas for help with fly stocks. Steffen Lemke and Yu-Chiun Wang for the cephalic furrow discussions and sharing unpublished data. The Lemke Lab for the generous help in setting up the <em>Clogmia</em> cultures. Carles Blanch-Mercader for feedback on simulations. Juliana Roscito for text revisions. Michael Akam for drawing BCV’s attention to the cephalic furrow in the first place. AS was supported by funding from the European Union’s Horizon 2020 Research and Innovation Programme under grant agreement no. 829010 (PRIME). BCV was supported by an EMBO Long Term Fellowship (ALTF 74-2018) This work was supported by the MPI-CBG core funding to CDM and PT, and by an European Research Council Advanced Grant (ERC-AdG 885504 GHOSTINTHESHELL) awarded to PT.</p>
<h1 id="author-contributions">Author contributions</h1>
<p>BCV and PT conceived the study. BCV designed experiments, generated fly stocks, acquired microscopy data, performed <em>in situ</em> hybridizations, and processed and analyzed the <em>in vivo</em> data. MBC conceived and conducted the laser ablation and cauterization experiments, and analyzed the laser ablation and strain rate data. CDM, AK, and AS designed the model. AK and AS programmed the model, performed the simulations, and analyzed the <em>in silico</em> data. BCV wrote the manuscript. All authors revised and contributed to the text.</p>
<h1 id="references">References</h1>
<div id="refs" class="references" role="doc-bibliography">
<div id="ref-Keller2012-ca">
<p>1. Keller, R. Physical biology returns to morphogenesis. <em>Science</em> <strong>338,</strong> 201–203 (2012).</p>
</div>
<div id="ref-Heisenberg2013-pl">
<p>2. Heisenberg, C.-P. &amp; Bellaïche, Y. Forces in tissue morphogenesis and patterning. <em>Cell</em> <strong>153,</strong> 948–962 (2013).</p>
</div>
<div id="ref-Collinet2021-qa">
<p>3. Collinet, C. &amp; Lecuit, T. Programmed and self-organized flow of information during morphogenesis. <em>Nat. Rev. Mol. Cell Biol.</em> <strong>22,</strong> 245–265 (2021).</p>
</div>
<div id="ref-Hartenstein1985-zs">
<p>4. Hartenstein, V. &amp; Campos-Ortega, J. A. Fate-mapping in wild-type <em>Drosophila melanogaster</em>. <em>Wilhelm Roux Arch. Entwickl. Mech. Org.</em> <strong>194,</strong> 181–195 (1985).</p>
</div>
<div id="ref-Spencer2015-td">
<p>5. Spencer, A. K., Siddiqui, B. A. &amp; Thomas, J. H. Cell shape change and invagination of the cephalic furrow involves reorganization of F-actin. <em>Dev. Biol.</em> <strong>402,</strong> 192–207 (2015).</p>
</div>
<div id="ref-Foe1989-cw">
<p>6. Foe, V. E. Mitotic domains reveal early commitment of cells in <em>Drosophila</em> embryos. <em>Development</em> <strong>107,</strong> 1–22 (1989).</p>
</div>
<div id="ref-Vincent1997-fa">
<p>7. Vincent, A., Blankenship, J. T. &amp; Wieschaus, E. Integration of the head and trunk segmentation systems controls cephalic furrow formation in <em>Drosophila</em>. <em>Development</em> <strong>124,</strong> 3747–3754 (1997).</p>
</div>
<div id="ref-Liu2013-lc">
<p>8. Liu, F., Morrison, A. H. &amp; Gregor, T. Dynamic interpretation of maternal inputs by the <em>Drosophila</em> segmentation gene network. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>110,</strong> 6724–6729 (2013).</p>
</div>
<div id="ref-Eritano2020-mi">
<p>9. Eritano, A. S., Bromley, C. L., Bolea Albero, A., Schütz, L., Wen, F.-L., Takeda, M., Fukaya, T., Sami, M. M., Shibata, T., Lemke, S. &amp; Wang, Y.-C. Tissue-scale mechanical coupling reduces morphogenetic noise to ensure precision during epithelial folding. <em>Dev. Cell</em> <strong>53,</strong> 212–228.e12 (2020).</p>
</div>
<div id="ref-Turner1977-ol">
<p>10. Turner, F. R. &amp; Mahowald, A. P. Scanning electron microscopy of <em>Drosophila melanogaster</em> embryogenesis. II. Gastrulation and segmentation. <em>Dev. Biol.</em> <strong>57,</strong> 403–416 (1977).</p>
</div>
<div id="ref-Costa1993-qj">
<p>11. Costa, M., Sweeton, D. &amp; Wieschaus, E. in <em>The development of <span style="font-style:normal;">Drosophila melanogaster</span></em> (eds. Bate, M. &amp; Arias, A. M.) <strong>1,</strong> 425–465 (Cold Spring Harbor Laboratory Press, 1993).</p>
</div>
<div id="ref-Dicko2017-wz">
<p>12. Dicko, M., Saramito, P., Blanchard, G. B., Lye, C. M., Sanson, B. &amp; Étienne, J. Geometry can provide long-range mechanical guidance for embryogenesis. <em>PLoS Comput. Biol.</em> <strong>13,</strong> e1005443 (2017).</p>
</div>
<div id="ref-Dey2023-mo">
<p>13. Dey, B., Kaul, V., Kale, G., Scorcelletti, M., Takeda, M., Wang, Y.-C. &amp; Lemke, S. Divergent evolutionary strategies preempt tissue collision in fly gastrulation. <em>bioRxiv</em> (2023). doi:<a href="https://doi.org/10.1101/2023.10.09.561568">10.1101/2023.10.09.561568</a></p>
</div>
<div id="ref-Blankenship2001-tm">
<p>14. Blankenship, J. T. &amp; Wieschaus, E. Two new roles for the <em>Drosophila</em> AP patterning system in early morphogenesis. <em>Development</em> <strong>128,</strong> 5129–5138 (2001).</p>
</div>
<div id="ref-Edgar1989-hv">
<p>15. Edgar, B. A. &amp; O’Farrell, P. H. Genetic control of cell division patterns in the <em>Drosophila</em> embryo. <em>Cell</em> <strong>57,</strong> 177–187 (1989).</p>
</div>
<div id="ref-Trushko2020-gf">
<p>16. Trushko, A., Di Meglio, I., Merzouki, A., Blanch-Mercader, C., Abuhattum, S., Guck, J., Alessandri, K., Nassoy, P., Kruse, K., Chopard, B. &amp; Roux, A. Buckling of an epithelium growing under spherical confinement. <em>Dev. Cell</em> <strong>54,</strong> 655–668.e6 (2020).</p>
</div>
<div id="ref-Fouchard2020-km">
<p>17. Fouchard, J., Wyatt, T. P. J., Proag, A., Lisica, A., Khalilgharibi, N., Recho, P., Suzanne, M., Kabla, A. &amp; Charras, G. Curling of epithelial monolayers reveals coupling between active bending and tissue tension. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>117,</strong> 9377–9383 (2020).</p>
</div>
<div id="ref-Efrati2009-ve">
<p>18. Efrati, E., Sharon, E. &amp; Kupferman, R. Elastic theory of unconstrained non-euclidean plates. <em>J. Mech. Phys. Solids</em> <strong>57,</strong> 762–775 (2009).</p>
</div>
<div id="ref-Andrioli2004-qi">
<p>19. Andrioli, L. P., Oberstein, A. L., Corado, M. S. G., Yu, D. &amp; Small, S. Groucho-dependent repression by Sloppy-paired 1 differentially positions anterior pair-rule stripes in the <em>D</em><em>rosophila</em> embryo. <em>Dev. Biol.</em> <strong>276,</strong> 541–551 (2004).</p>
</div>
<div id="ref-Andrioli2012-fv">
<p>20. Andrioli, L. P., Digiampietri, L. A., Barros, L. P. de &amp; Machado-Lima, A. Huckebein is part of a combinatorial repression code in the anterior blastoderm. <em>Dev. Biol.</em> <strong>361,</strong> 177–185 (2012).</p>
</div>
<div id="ref-Strobl2022-og">
<p>21. Strobl, F., Schetelig, M. F. &amp; Stelzer, E. H. K. In toto light sheet fluorescence microscopy live imaging datasets of <em>ceratitis capitata</em> embryonic development. <em>Sci. Data</em> <strong>9,</strong> 340 (2022).</p>
</div>
<div id="ref-Cheatle-Jarvela2020-go">
<p>22. Cheatle Jarvela, A. M., Trelstad, C. S. &amp; Pick, L. Regulatory gene function handoff allows essential gene loss in mosquitoes. <em>Commun. Biol.</em> <strong>3,</strong> 540 (2020).</p>
</div>
<div id="ref-Cheatle-Jarvela2023-gd">
<p>23. Cheatle Jarvela, A. M., Trelstad, C. S. &amp; Pick, L. Anterior-posterior patterning of segments in <em>Anopheles stephensi</em> offers insights into the transition from sequential to simultaneous segmentation in holometabolous insects. <em>J. Exp. Zool. B Mol. Dev. Evol.</em> <strong>340,</strong> 116–130 (2023).</p>
</div>
<div id="ref-Jimenez-Guri2014-zs">
<p>24. Jiménez-Guri, E., Wotton, K. R., Gavilán, B. &amp; Jaeger, J. A staging scheme for the development of the moth midge <em>Clogmia albipunctata</em>. <em>PLoS One</em> <strong>9,</strong> e84422 (2014).</p>
</div>
<div id="ref-Gupta2021-oo">
<p>25. Gupta, V. K., Nam, S., Yim, D., Camuglia, J., Martin, J. L., Sanders, E. N., O’Brien, L. E., Martin, A. C., Kim, T. &amp; Chaudhuri, O. The nature of cell division forces in epithelial monolayers. <em>J. Cell Biol.</em> <strong>220,</strong> (2021).</p>
</div>
<div id="ref-Ko2020-wk">
<p>26. Ko, C. S., Kalakuntla, P. &amp; Martin, A. C. Apical constriction reversal upon mitotic entry underlies different morphogenetic outcomes of cell division. <em>Mol. Biol. Cell</em> <strong>31,</strong> 1663–1674 (2020).</p>
</div>
<div id="ref-Kondo2013-gy">
<p>27. Kondo, T. &amp; Hayashi, S. Mitotic cell rounding accelerates epithelial invagination. <em>Nature</em> <strong>494,</strong> 125–129 (2013).</p>
</div>
<div id="ref-Freddo2016-et">
<p>28. Freddo, A. M., Shoffner, S. K., Shao, Y., Taniguchi, K., Grosse, A. S., Guysinger, M. N., Wang, S., Rudraraju, S., Margolis, B., Garikipati, K., Schnell, S. &amp; Gumucio, D. L. Coordination of signaling and tissue mechanics during morphogenesis of murine intestinal villi: A role for mitotic cell rounding. <em>Integr. Biol.</em> <strong>8,</strong> 918–928 (2016).</p>
</div>
<div id="ref-Newman2000-oz">
<p>29. Newman, S. A. &amp; Müller, G. B. Epigenetic mechanisms of character origination. <em>J. Exp. Zool.</em> <strong>288,</strong> 304–317 (2000).</p>
</div>
<div id="ref-Gramates2022-xe">
<p>30. Gramates, L. S., Agapite, J., Attrill, H., Calvi, B. R., Crosby, M. A., Dos Santos, G., Goodman, J. L., Goutte-Gattat, D., Jenkins, V. K., Kaufman, T., Larkin, A., Matthews, B. B., Millburn, G., Strelets, V. B. &amp; the FlyBase Consortium. FlyBase: A guided tour of highlighted features. <em>Genetics</em> <strong>220,</strong> iyac035 (2022).</p>
</div>
<div id="ref-Vellutini2023-ng">
<p>31. Vellutini, B. C., Cuenca, M. B., Krishna, A., Szałapak, A., Modes, C. D. &amp; Tomančák, P. Repository for: Patterned embryonic invagination evolved in response to mechanical instability. (2023). doi:<a href="https://doi.org/10.5281/zenodo.7781947">10.5281/zenodo.7781947</a></p>
</div>
<div id="ref-Sander1985-sg">
<p>32. Sander, K. Experimental egg activation in lower dipterans (Psychoda, Smittia) by low osmolality. <em>Int. J. Invertebr. Reprod. Dev.</em> <strong>8,</strong> 175–183 (1985).</p>
</div>
<div id="ref-Garcia-Solache2010-lz">
<p>33. García-Solache, M., Jaeger, J. &amp; Akam, M. A systematic analysis of the gap gene system in the moth midge <em>C</em><em>logmia</em> <em>a</em><em>lbipunctata</em>. <em>Dev. Biol.</em> <strong>344,</strong> 306–318 (2010).</p>
</div>
<div id="ref-Choi2018-fg">
<p>34. Choi, H. M. T., Schwarzkopf, M., Fornace, M. E., Acharya, A., Artavanis, G., Stegmaier, J., Cunha, A. &amp; Pierce, N. A. Third-generation <em>in situ</em> hybridization chain reaction: Multiplexed, quantitative, sensitive, versatile, robust. <em>Development</em> <strong>145,</strong> (2018).</p>
</div>
<div id="ref-Schmied2016-oq">
<p>35. Schmied, C. &amp; Tomancak, P. in <em>Drosophila: Methods and protocols</em> (ed. Dahmann, C.) 189–202 (Springer New York, 2016). doi:<a href="https://doi.org/10.1007/978-1-4939-6371-3_10">10.1007/978-1-4939-6371-3_10</a></p>
</div>
<div id="ref-Schindelin2012-di">
<p>36. Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J.-Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. &amp; Cardona, A. Fiji: An open-source platform for biological-image analysis. <em>Nat. Methods</em> <strong>9,</strong> 676–682 (2012).</p>
</div>
<div id="ref-Rueden2017-ky">
<p>37. Rueden, C. T., Schindelin, J., Hiner, M. C., DeZonia, B. E., Walter, A. E., Arena, E. T. &amp; Eliceiri, K. W. ImageJ2: ImageJ for the next generation of scientific image data. <em>BMC Bioinformatics</em> <strong>18,</strong> 529 (2017).</p>
</div>
<div id="ref-Schmid2019-bm">
<p>38. Schmid, B., Tripal, P., Fraaß, T., Kersten, C., Ruder, B., Grüneboom, A., Huisken, J. &amp; Palmisano, R. 3Dscript: Animating 3D/4D microscopy data using a natural-language-based syntax. <em>Nat. Methods</em> <strong>16,</strong> 278–280 (2019).</p>
</div>
<div id="ref-Weigert2018-ti">
<p>39. Weigert, M., Schmidt, U., Boothe, T., Müller, A., Dibrov, A., Jain, A., Wilhelm, B., Schmidt, D., Broaddus, C., Culley, S., Rocha-Martins, M., Segovia-Miranda, F., Norden, C., Henriques, R., Zerial, M., Solimena, M., Rink, J., Tomancak, P., Royer, L., Jug, F. &amp; Myers, E. W. Content-aware image restoration: Pushing the limits of fluorescence microscopy. <em>Nat. Methods</em> <strong>15,</strong> 1090–1097 (2018).</p>
</div>
<div id="ref-Heemskerk2015-kv">
<p>40. Heemskerk, I. &amp; Streichan, S. J. Tissue cartography: Compressing bio-image data by dimensional reduction. <em>Nat. Methods</em> <strong>12,</strong> 1139–1142 (2015).</p>
</div>
<div id="ref-Matlab2015-nd">
<p>41. Matlab. <em>MATLAB</em>. (The MathWorks Inc., 2015). at &lt;<a href="https://www.mathworks.com/products/matlab.html">https://www.mathworks.com/products/matlab.html</a>&gt;</p>
</div>
<div id="ref-Berg2019-ab">
<p>42. Berg, S., Kutra, D., Kroeger, T., Straehle, C. N., Kausler, B. X., Haubold, C., Schiegg, M., Ales, J., Beier, T., Rudy, M., Eren, K., Cervantes, J. I., Xu, B., Beuttenmueller, F., Wolny, A., Zhang, C., Koethe, U., Hamprecht, F. A. &amp; Kreshuk, A. ilastik: Interactive machine learning for (bio)image analysis. <em>Nat. Methods</em> (2019). doi:<a href="https://doi.org/10.1038/s41592-019-0582-9">10.1038/s41592-019-0582-9</a></p>
</div>
<div id="ref-Vellutini2022-ya">
<p>43. Vellutini, B. C. <em>How to make cartographic projections using ImSAnE</em>. (Zenodo, 2022). doi:<a href="https://doi.org/10.5281/zenodo.7628299">10.5281/zenodo.7628299</a></p>
</div>
<div id="ref-Schmied2020-ip">
<p>44. Schmied, C. &amp; Jambor, H. K. Effective image visualization for publications - a workflow using open access tools and concepts. <em>F1000Res.</em> <strong>9,</strong> 1373 (2020).</p>
</div>
<div id="ref-Arganda-Carreras2006-im">
<p>45. Arganda-Carreras, I., Sorzano, C. O. S., Marabini, R., Carazo, J. M., Ortiz-de-Solorzano, C. &amp; Kybic, J. in <em>Computer vision approaches to medical image analysis</em> 85–95 (Springer Berlin Heidelberg, 2006). doi:<a href="https://doi.org/10.1007/11889762_8">10.1007/11889762_8</a></p>
</div>
<div id="ref-Legland2016-cp">
<p>46. Legland, D., Arganda-Carreras, I. &amp; Andrey, P. MorphoLibJ: Integrated library and plugins for mathematical morphology with ImageJ. <em>Bioinformatics</em> <strong>32,</strong> 3532–3534 (2016).</p>
</div>
<div id="ref-Arganda-Carreras2010-la">
<p>47. Arganda-Carreras, I., Fernández-González, R., Muñoz-Barrutia, A. &amp; Ortiz-De-Solorzano, C. 3D reconstruction of histological sections: Application to mammary gland tissue. <em>Microsc. Res. Tech.</em> <strong>73,</strong> 1019–1029 (2010).</p>
</div>
<div id="ref-Tseng2012-ni">
<p>48. Tseng, Q., Duchemin-Pelletier, E., Deshiere, A., Balland, M., Guillou, H., Filhol, O. &amp; Théry, M. Spatial organization of the extracellular matrix regulates cell-cell junction positioning. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>109,</strong> 1506–1511 (2012).</p>
</div>
<div id="ref-Krishna2023-hu">
<p>49. Krishna, A., Szałapak, A., Vellutini, B. C., Cuenca, M. B., Tomančák, P. &amp; Modes, C. D. Model and simulations for: Patterned embryonic invagination evolved in response to mechanical instability. (2023). doi:<a href="https://doi.org/10.5281/zenodo.7784906">10.5281/zenodo.7784906</a></p>
</div>
<div id="ref-The_Inkscape_Project2003-ht">
<p>50. The Inkscape Project. <em>Inkscape: Open source scalable vector graphics editor</em>. (2003). at &lt;<a href="https://inkscape.org">https://inkscape.org</a>&gt;</p>
</div>
<div id="ref-HandBrake_Team2003-fq">
<p>51. HandBrake Team. <em>HandBrake: HandBrake’s main development repository</em>. (Github, 2003). at &lt;<a href="https://github.com/HandBrake/HandBrake">https://github.com/HandBrake/HandBrake</a>&gt;</p>
</div>
<div id="ref-Vellutini2023-ix">
<p>52. Vellutini, B. C., Cuenca, M. B., Krishna, A., Szałapak, A., Modes, C. D. &amp; Tomančák, P. Figures and videos for: Patterned embryonic invagination evolved in response to mechanical instability. (2023). doi:<a href="https://doi.org/10.5281/zenodo.7781916">10.5281/zenodo.7781916</a></p>
</div>
<div id="ref-R-Core-Team1993-jw">
<p>53. R Core Team. <em>R: A language and environment for statistical computing</em>. (R Foundation for Statistical Computing, 1993). at &lt;<a href="http://www.r-project.org">http://www.r-project.org</a>&gt;</p>
</div>
<div id="ref-RStudio-Team2011-vj">
<p>54. RStudio Team. <em>RStudio: Integrated development environment for r</em>. (RStudio, PBC., 2011). at &lt;<a href="http://www.rstudio.com">http://www.rstudio.com</a>&gt;</p>
</div>
<div id="ref-Granger2021-zg">
<p>55. Granger, B. E. &amp; Perez, F. Jupyter: Thinking and storytelling with code and data. <em>Comput. Sci. Eng.</em> <strong>23,</strong> 7–14 (2021).</p>
</div>
<div id="ref-Ashburner2005-ym">
<p>56. Ashburner, M., Golic, K. G. &amp; Hawley, R. S. <em><span style="font-style:normal;">Drosophila</span>: A laboratory handbook</em>. (Cold Spring Harbor Laboratory Press, 2005).</p>
</div>
<div id="ref-Campos-Ortega1985-ma">
<p>57. Campos-Ortega, J. A. &amp; Hartenstein, V. <em>The embryonic development of <span style="font-style:normal;">Drosophila melanogaster</span></em>. (Springer Berlin Heidelberg, 1985). doi:<a href="https://doi.org/10.1007/978-3-662-02454-6">10.1007/978-3-662-02454-6</a></p>
</div>
<div id="ref-Wiegmann2011-ou">
<p>58. Wiegmann, B. M., Trautwein, M. D., Winkler, I. S., Barr, N. B., Kim, J.-W., Lambkin, C., Bertone, M. A., Cassel, B. K., Bayless, K. M., Heimberg, A. M., Wheeler, B. M., Peterson, K. J., Pape, T., Sinclair, B. J., Skevington, J. H., Blagoderov, V., Caravas, J., Kutty, S. N., Schmidt-Ott, U., Kampmeier, G. E., Thompson, F. C., Grimaldi, D. A., Beckenbach, A. T., Courtney, G. W., Friedrich, M., Meier, R. &amp; Yeates, D. K. Episodic radiations in the fly tree of life. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>108,</strong> 5690–5695 (2011).</p>
</div>
<div id="ref-Caroti2015-cy">
<p>59. Caroti, F., Urbansky, S., Wosch, M. &amp; Lemke, S. Germ line transformation and in vivo labeling of nuclei in Diptera: Report on <em>Megaselia abdita</em> (Phoridae) and <em>Chironomus riparius</em> (Chironomidae). <em>Dev. Genes Evol.</em> <strong>225,</strong> 179–186 (2015).</p>
</div>
<div id="ref-Wotton2014-fr">
<p>60. Wotton, K. R., Jiménez-Guri, E., García Matheu, B. &amp; Jaeger, J. A staging scheme for the development of the scuttle fly <em>Megaselia abdita</em>. <em>PLoS One</em> <strong>9,</strong> e84421 (2014).</p>
</div>
<div id="ref-Tomancak2002-yf">
<p>61. Tomancak, P., Beaton, A., Weiszmann, R., Kwan, E., Shu, S., Lewis, S. E., Richards, S., Ashburner, M., Hartenstein, V., Celniker, S. E. &amp; Rubin, G. M. Systematic determination of patterns of gene expression during <em>D</em><em>rosophila</em> embryogenesis. <em>Genome Biol.</em> <strong>3,</strong> research0088.1 (2002).</p>
</div>
<div id="ref-Tomancak2007-db">
<p>62. Tomancak, P., Berman, B. P., Beaton, A., Weiszmann, R., Kwan, E., Hartenstein, V., Celniker, S. E. &amp; Rubin, G. M. Global analysis of patterns of gene expression during <em>D</em><em>rosophila</em> embryogenesis. <em>Genome Biol.</em> <strong>8,</strong> R145 (2007).</p>
</div>
<div id="ref-Lecuyer2007-uy">
<p>63. Lécuyer, E., Yoshida, H., Parthasarathy, N., Alm, C., Babak, T., Cerovina, T., Hughes, T. R., Tomancak, P. &amp; Krause, H. M. Global analysis of mRNA localization reveals a prominent role in organizing cellular architecture and function. <em>Cell</em> <strong>131,</strong> 174–187 (2007).</p>
</div>
</div>
<div style=""></div>
<h1 id="figures">Figures</h1>
<p>High-resolution versions of the figures are available at <a href="https://doi.org/10.5281/zenodo.7781916">https://doi.org/10.5281/zenodo.7781916</a>.<span class="citation" data-cites="Vellutini2023-ix"><sup><a href="#ref-Vellutini2023-ix" role="doc-biblioref">52</a></sup></span></p>
<h2 id="figure-1">Figure 1</h2>
<div id="fig:ectopic-folding" class="fignos">
<figure>
<img src="figures/Fig1.jpg" style="width:100.0%" alt="" /><figcaption><span>Figure 1:</span> Formation of ectopic folds in cephalic furrow mutants. <strong>a</strong>, Overview of key developmental events using a standard developmental staging.<span class="citation" data-cites="Ashburner2005-ym"><sup><a href="#ref-Ashburner2005-ym" role="doc-biblioref">56</a></sup></span> The formation of the cephalic furrow is delayed in <em>prd</em> mutants and absent in <em>btd</em> and <em>eve</em> mutants. In the latter, ectopic folds form near the canonical invagination site of the cephalic furrow. Wildtype and <em>prd</em> embryos form less frequent and smaller ectopic folds more anterior or posterior to the cephalic furrow. Ectopic folds appear later and unfold quicker than the cephalic furrow, and their formation coincides with the appearance of mitotic domains and with the rapid phase of the germ band extension.<span class="citation" data-cites="Campos-Ortega1985-ma"><sup><a href="#ref-Campos-Ortega1985-ma" role="doc-biblioref">57</a></sup></span> <strong>b</strong>, Profile view of wildtype and <em>prd</em> embryos (early stage 8), and <em>btd</em> and <em>eve</em> embryos (late stage 8) showing the divergent morphology of the cephalic furrow and ectopic folds, respectively. Membrane marker = Gap43-mCherry. Scale bars = 20µm. <strong>c</strong>, Lateral view of sibling controls (heterozygotes) and mutant embryos (<em>btd</em> or <em>eve</em> homozygotes). The arrows indicate epithelial folds, the asterisks indicate the position of mitotic domains, and the percentages indicate the extent of germ band extension relative to the egg length. Membrane marker = Gap43-mCherry. Scale bars = 50µm. <strong>d</strong>, Timing of formation of cephalic furrow and ectopic folds in different genetic backgrounds. The cephalic furrow forms about 7 minutes after gastrulation in wildtype (+/+, n=16) and <em>btd</em> heterozygotes (+/−, n=6), and is delayed in <em>eve</em> and <em>prd</em> heterozygotes (+/−, n=7 and n=10, respectively) and in <em>prd</em> homozygotes (−/−, n=5). Ectopic folds form about 20min after gastrulation in <em>btd</em> and <em>eve</em> homozygotes (−/−, n=5 and n=5, respectively). One dot represents one embryo. <strong>e</strong>, Percentage of germ band extension at the time of formation of the cephalic furrow and ectopic folds in different genetic backgrounds. The cephalic furrow appears at 10% germ band extension, except in <em>prd</em> heterozygotes where it varies up to 40%. Ectopic folds form at 30–35% germ band extension. The n values are the same as in <strong>d</strong>. <strong>f</strong>, Variability in the distribution of the cephalic furrow in sibling controls (<em>btd</em> heterozygotes) and of the ectopic folds in <em>btd</em> mutants (n=6). Scale bars = 50µm. <strong>g</strong>, Folded area of the cephalic furrow (CF) and ectopic folds (EF) in <em>btd</em> embryos. The folded region is outlined in yellow on cartographic projections of a representative sibling control <em>btd</em> heterozygote (left) and of a <em>btd</em> homozygote (right). The numbers indicate the average folded area for the sample type in µm<sup>2</sup>×10<sup>3</sup>. <strong>h</strong>, Quantification of the total folded area of epithelial surface in cephalic furrow mutants.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-2">Figure 2</h2>
<div id="fig:mitotic-strain" class="fignos">
<figure>
<img src="figures/Fig2.jpg" style="width:80.0%" alt="" /><figcaption><span>Figure 2:</span> Coincidence of mitotic domains and germ band extension with ectopic folding. <strong>a</strong>, Position of ectopic folds (black lines) in relation to the mitotic domains (colored areas) in <em>btd</em> and <em>eve</em> mutants. The ectopic folds appear between mitotic domains and between the germ band and the mitotic domain 6 (MD6). Scale bars = 50µm. <strong>b</strong>, Apical cell area between mitotic domains MD5 and MD7/6 (top). The infolded area is highlighted in the last frame (4.6 min). Detail of a cell subset in between mitotic domains (bottom) showing non-dividing cells (orange) and adjacent dividing cells (blue). Scale bars = 20µm (this is an approximate value because the pixel resolutions vary across the projection). <strong>c</strong>, Ectopic folding forming between MD6 and the germ band in a <em>btd</em> mutant. The mitotic expansion (double arrouw) precedes the ectopic folding (arrow). Scale bar = 20µm. <strong>d</strong>, Comparison between the timing of cephalic furrow formation (CF), mitotic domain expansion (MD), and ectopic folding (EF) in wildtype and cephalic furrow mutants (<em>btd</em> and <em>eve</em> combined). In wildtype and heterozygote embryos, the cephalic furrow forms before mitotic domains, while in homozygote embryos, the ectopic folds form after mitotic domains. <strong>e</strong>, Compressed epithelial cells between MD6 and the tip of the germ band. Scale bars = 50µm. <strong>f</strong>, Strain rate analysis at the head–trunk (top) and trunk–germ (bottom) regions. The formation of the cephalic furrow (CF), ectopic folds (EF), and mitotic domains (MD) are annotated. The measurements combine isotropic and anisotropic strain rate. <strong>g</strong>, Tissue dynamics at the trunk–germ boundary after laser ablations. We measured the distance between cell vertex pairs over time in control and ablated wildtype embryos. The tracks are color-coded for time. Solid white line represents the distance between vertices at one timepoint. Dashed white line shows the location of the cut. The plots show a smoothed local regression of three samples.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-3">Figure 3</h2>
<!--TODO: Optimize panels for consistency-->
<div id="fig:fold-model" class="fignos">
<figure>
<img src="figures/Fig3.jpg" style="width:100.0%" alt="" /><figcaption><span>Figure 3:</span> Model and simulations of the tissue mechanics at the head–trunk boundary. <strong>a</strong>, Region of interest of the model. One side of a frontal slice across a slightly dorsal plane of the embryonic blastoderm. It follows the embryo shape, its proportions, and the relative positions of mitotic domains, cephalic furrow, and germ band. <strong>b</strong>, Characteristics of the individual components of the model based on particles connected by springs. The drawings exemplify the initial and final states for each component. <strong>c</strong>, Energy equation with a stretching and a bending component, and the dimensionless bending rigidity. Stress-free rod length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>L</mi><mi>o</mi></msub><annotation encoding="application/x-tex">L_o</annotation></semantics></math>), total energy per unit length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>T</mi></msub><annotation encoding="application/x-tex">W_T</annotation></semantics></math>), stretching energy per unit length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>s</mi></msub><annotation encoding="application/x-tex">W_s</annotation></semantics></math>), bending energy per unit length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>b</mi></msub><annotation encoding="application/x-tex">W_b</annotation></semantics></math>), stretching rigidity (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>s</mi></msub><annotation encoding="application/x-tex">K_s</annotation></semantics></math>), bending rigidity (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math>), preferred spring length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">ds_o</annotation></semantics></math>), current spring length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><mi>s</mi></mrow><annotation encoding="application/x-tex">ds</annotation></semantics></math>), preferred curvature (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>κ</mi><mi>o</mi></msub><annotation encoding="application/x-tex">\kappa_o</annotation></semantics></math>), current curvature (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>κ</mi><annotation encoding="application/x-tex">\kappa</annotation></semantics></math>), semi-major embryonic axis (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>L</mi><annotation encoding="application/x-tex">L</annotation></semantics></math>). <strong>d</strong>, Plots showing the energy dynamics across iterations in a typical simulation run. The total energy goes down to equilibrium. The bending energy increases drastically reaching a peak (blue dashed line) that diminishes gradually with the iterations. Energy values are normalized by the initial total energy. The number of folds stabilizes when the bending energy peaks but the fold depth continues to increase until the last iteration (pink dashed line). <strong>e</strong>, Mutant parameter sweep without mitotic domains (no cephalic furrow). The heatmap shows the average number of ectopic folds for different bending rigidities and percentages of germ band extension. Ectopic folding frequency increases with lower bending rigidities (softer tissue) and with greater values of germ band extension. Outlined in white is the baseline condition with neither mitotic domains nor germ band but only ground level noise (i), and the germ band only condition with higher number of folding events (ii). Representative simulations are shown below. <strong>f</strong>, Mutant parameter sweep with mitotic domains (no cephalic furrow). The phase diagram shows an increase in number of folds in relation to <strong>e</strong>. The addition of mitotic domains induces the formation of ectopic folds even without germ band extension (iii). Representative simulations are shown below. <strong>g</strong>, Quantification of ectopic folding to evaluate the effectiveness of the cephalic furrow. The top row shows a control simulating a cephalic furrow mutant condition with mitotic domains and germ band extension. The middle row shows an active cephalic furrow invagination with mitotic domains forming at the same time without delay. The bottom row shows the cephalic furrow with a delay in mitotic domain formation (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>5</mn></mrow><annotation encoding="application/x-tex">t_{MD}=5</annotation></semantics></math>). <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">t_{MD}=1</annotation></semantics></math> corresponds to 10<sup>5</sup> computational timesteps. <strong>h</strong>, Quantification of ectopic folding with variable cephalic furrow positioning.</figcaption>
</figure>
</div>
<!--TODO: Describe properly panel with CF positioning-->
<div style=""></div>
<h2 id="figure-4">Figure 4</h2>
<!--TODO: Rearrange contents from simple to complex-->
<div id="fig:live-experiments" class="fignos">
<figure>
<img src="figures/Fig4.jpg" style="width:100.0%" alt="" /><figcaption><span>Figure 4:</span> Perturbation experiments <em>in vivo</em> using cephalic furrow mutants and wildtype embryos. <strong>a</strong>, Lateral view of <em>btd</em>–<em>stg</em> double mutants showing the absence of ectopic folds at the head–trunk boundary compared to a <em>btd</em> mutant. Scale bar = 50µm. <strong>b</strong>, Profile view of a <em>btd</em>–<em>stg</em> double mutant showing the head–trunk epithelium without mitotic domains or ectopic folds. Scale bar = 20µm. <strong>c</strong>, Cauterized <em>eve</em> mutant in lateral (top) and dorsal view (bottom) at the maximum apical expansion of dividing cells in mitotic domains. The cauterization site is marked by a vermilion circle. The boxed outline (right) shows the progressive expansion of mitotic domains compressing the non-dividing cells between MD1 and MD5. Scale bars = 50µm. <strong>d</strong> Profile view and epithelial trace of non-cauterized and cauterized <em>btd</em> and <em>eve</em> embryos. The trace shows the dynamics of epithelial deformations over time colored from purple to yellow. Cauterized mutants show no ectopic folds and less deformation of the epithelium. Scale bars = 20µm. <strong>e</strong>, Cauterized wildtype embryo in lateral (top) and dorsal view (bottom). The cauterization site is marked by a vermilion circle. The cephalic furrow invaginates normally. Scale bar = 50µm. <strong>f</strong>, Profile view of a non-cauterized and a cauterized wildtype embryo showing a small difference in depth. Scale bar = 20µm. <strong>g</strong>, Tortuosity of the epithelial traces in non-cauterized (<em>btd</em> n=2, <em>eve</em> n=3) and cauterized (<em>btd</em> n=3, <em>eve</em> n=4) embryos from <strong>d</strong>. For each embryo, the left and right side were measured. <strong>h</strong>, Maximum depth of the cephalic furrow in cauterized embryos. Control non-cauterized embryos include wildtype (n=2), and <em>btd</em> (n=1) and <em>eve</em> (n=3) heterozygotes which also form a cephalic furrow. The cauterized group includes wildtype (n=2) and <em>btd</em> (n=1). The cephalic furrow in cauterized embryos is 15% shallower (p=0.0221 in Welch Two Sample t-test). <strong>i</strong>, Recoil velocity of laser ablations around the cephalic furrow at stage 6. The average recoil increases with time for anterior cuts (early n=11, mid n=13, late n=12), and reaches a peak at mid stage 6 for posterior cuts (early n=7, mid n=14, late n=10). The average recoil velocity is smaller at greater distances (−40 and 40) than at shorter distances (−20 and 20) from the initiator cells. In late stage 6 the region with recoil reaches at least 40µm anterior and posterior of the invagination. <strong>j</strong>, Schematic drawings showing the summary of the ectopic folding in double mutants and germ band cauterization experiments. Tissue flows are represented by black arrows, mitotic domains (MD) by light gray patches, and the cephalic furrow (CF) area by a dark gray band. In cephalic furrow mutants the excess of epithelial tissue on the surface, combined with the formation of mitotic domains and the extension of the germ band (GB), leads to mechanical instabilities and the buckling of ectopic folds (EF) at the head–trunk boundary.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-5">Figure 5</h2>
<div id="fig:genetic-patterning" class="fignos">
<figure>
<img src="figures/Fig5.jpg" style="width:100.0%" alt="" /><figcaption><span>Figure 5:</span> Genetic patterning of the head–trunk boundary in <em>Drosophila</em> and <em>Clogmia</em>. <strong>a</strong>, Lateral view of <em>slp</em> mutant heterozygote (control) and mutant embryo at the onset of initiator cell behavior. In <em>slp</em> mutants, the cephalic furrow formation is delayed; it happens when the germ band is extended about 18% of egg length. The position of intiation is shifted forward in mutants at about 73% of egg length. The asterisk indicates mitotic cells. Scale bars = 50µm. <strong>b</strong>, Plots showing the position of the cephalic furrow (CF) and germ band (GB) at the onset of initiator cell behavior in <em>slp</em> mutants. <strong>c</strong>, Lateral view of <em>slp</em> mutants in <em>Drosophila</em> embryos showing the expression of <em>btd</em> and <em>eve</em>. The distance between <em>eve</em> stripe 1 and 2 is larger in <em>slp</em> embryos. The asterisk indicates a region between the acron (ac) and head–trunk (ht) domains of <em>btd</em> where its expression is activated (or de-repressed) in the absence of the <em>slp</em> head domain (sh). Scale bars = 50µm. <strong>d</strong>, Dorsal view showing the ectopic expression of <em>btd</em> (asterisks) and anterior shift of <em>btd</em>–<em>eve</em> overlap in <em>slp</em> mutants. Scale bars = 20µm. <strong>e</strong>, Profile view showing the increased number of cell rows between <em>eve</em> stripe 1 and 2 in <em>slp</em> mutants. Scale bars = 20µm. <strong>f</strong>, Progression of the early expression of <em>btd</em>, <em>eve</em>, and <em>slp</em> around the mitotic cycle 14 in <em>Drosophila</em> and <em>Clogmia</em>. Initially, <em>slp</em> and <em>eve</em> demarcate the head–trunk boundary of both species in broad domains that later become segmented. The head–trunk domain of <em>btd</em> is expressed early in <em>Drosophila</em> but is absent in <em>Clogmia</em>. The anterior domain of <em>btd</em> is present in both species, but is only activated in <em>Drosophila</em> at the onset of gastrulation. <strong>g</strong>, Crop showing the narrowing and sharpening of the expression domains in <em>Drosophila</em>. <em>slp1</em> and <em>eve</em> expression domains are complementary and <em>btd</em> is expressed at this interface. The border between <em>slp1</em> and <em>eve</em> resolve into sharp, non-overlapping stripes. The overlap between <em>btd</em> and <em>eve</em> narrows progressively until resolving to a 1-cell row. Dashed lines mark the future edges of the cephalic furrow. <strong>h</strong>, Close-up of the anterior region showing the absence of a <em>btd</em> head–trunk domain in <em>Clogmia</em> compared to <em>Drosophila</em>. <strong>i</strong>, Comparison of gene expression patterns at the onset of gastrulation. In <em>Drosophila</em>, <em>slp1</em> stripes demarcate the outer edges of the cephalic furrow (dashed lines) and the 1–2-cell wide <em>eve</em>-expressing row is abutted anteriorly and posteriorly by non-overlapping 2-cell wide <em>slp1</em> stripes. The expression pattern of <em>slp</em> and <em>eve</em> in <em>Clogmia</em> is very similar to <em>Drosophila</em>. The main difference is the absence of the head–trunk domain of <em>btd</em> in <em>Clogmia</em>. Dashed lines in <em>Clogmia</em> mark the <em>slp</em> stripes homologous to <em>Drosophila</em>. <strong>j</strong>, Schematic drawings showing the molecular arrangement and putative genetic interactions at the head–trunk boundary of <em>Drosophila</em> and <em>Clogmia</em>. ac: <em>btd</em> acron domain, sh: <em>slp</em> head domain, ht: <em>btd</em> head–trunk domain, fg: <em>btd</em> foregut domain.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-6">Figure 6</h2>
<div id="fig:summary" class="fignos">
<figure>
<img src="figures/Fig6.jpg" style="width:100.0%" alt="" /><figcaption><span>Figure 6:</span> Interplay of genetics and mechanics during cephalic furrow evolution. <strong>a</strong>, Simplified dipteran phylogeny based on<span class="citation" data-cites="Wiegmann2011-ou"><sup><a href="#ref-Wiegmann2011-ou" role="doc-biblioref">58</a></sup></span> with cephalic furrow traits mapped onto the tree. Combined data from this study and Dey <em>et al.</em><span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> (indicated in the figure). The germ band extension and mitotic domains are ancestral, suggesting that compressive stresses at the head–trunk boundary were present since the dawn of Diptera. The cephalic furrow is derived trait, an evolutionary novelty of cyclorraphan flies.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> It is absent in <em>Clogmia</em><span class="citation" data-cites="Jimenez-Guri2014-zs"><sup><a href="#ref-Jimenez-Guri2014-zs" role="doc-biblioref">24</a></sup></span> and <em>Chironomus</em>,<span class="citation" data-cites="Caroti2015-cy"><sup><a href="#ref-Caroti2015-cy" role="doc-biblioref">59</a></sup></span> but present in the common ancestor of <em>Megaselia</em><span class="citation" data-cites="Wotton2014-fr"><sup><a href="#ref-Wotton2014-fr" role="doc-biblioref">60</a></sup></span> and <em>Drosophila</em>. Out-of-plane cell divisions at the head–trunk boundary are present in <em>Clogmia</em> and <em>Chironomus</em>.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> They could be an ancestral trait together with mitotic domains and the long germ. <em>Clogmia</em> lacks a head–trunk domain of <em>btd</em> (this study). In <em>Chironomus</em> <em>btd</em> is expressed in the lateral side, but does not overlap with <em>eve</em> stripe 1.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">13</a></sup></span> Therefore, <em>btd</em>–<em>eve</em> overlap correlates with the presence of the cephalic furrow, and may be associated with its evolution. <strong>b</strong>, Scenario for mechanical instability as a selective pressure for the evolution of morphogenetic innovations. In the ancestral state there was no mechanical instability at the head–trunk boundary (0). The appearance of mitotic domains and germ band extension increased the compressive stresses and ectopic buckling events at the head–trunk boundary (1). This mechanical instability may have had a detrimental effect on individual fitness by affecting developmental robustness or by slowing down embryogenesis. Natural selection favored the establishment of patterned processes that mitigate these compressive stresses at the head–trunk boundary. One solution, present in <em>Clogmia</em> and <em>Chironomus</em>, is the out-of-plane cell divisions which reduce the compression load on the monolayer epithelium (2a). Another solution, present in <em>Drosophila</em> and other cyclorraphan flies, is the formation of an out-of-plane invagination which absorbs the mechanical forces at the head–trunk boundary (2b). These events may have happened through the stabilization of genetic interactions and cooption of existing signaling modules controlling cell and tissue morphogenesis. Tissue mechanics may have been an important factor influencing the evolution of patterned morphogenesis in early embryonic development.</figcaption>
</figure>
</div>
<div style=""></div>
<h1 id="videos">Videos</h1>
<p>High-resolution versions of the videos are available at <a href="https://doi.org/10.5281/zenodo.7781916">https://doi.org/10.5281/zenodo.7781916</a>.<span class="citation" data-cites="Vellutini2023-ix"><sup><a href="#ref-Vellutini2023-ix" role="doc-biblioref">52</a></sup></span></p>
<h2 id="video-1">Video 1</h2>
<div id="fig:vid-lateral-btd" class="fignos">
<figure>
<img src="figures/Vid1.jpg" data-tag="Video 1" style="width:50.0%" alt="" /><figcaption><span>Figure Video 1:</span> Lateral view of ectopic fold formation in <em>btd</em> mutant. The cephalic furrow forms normally in sibling controls (top) but it is absent in <em>btd</em> mutants (bottom). In the mutant, no fold is present at the head–trunk interface until about 20min, when a large ectopic fold appears and quickly unfolds at about 45min. In the sibling control, the cephalic furrow remains partially invaginated for the period shown in the recording (about 110min). Frame rate = 15fps. Scale bars = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-2">Video 2</h2>
<div id="fig:vid-lateral-eve" class="fignos">
<figure>
<img src="figures/Vid2.jpg" data-tag="Video 2" style="width:50.0%" alt="" /><figcaption><span>Figure Video 2:</span> Lateral view of ectopic fold formation in <em>eve</em> mutant. The cephalic furrow forms normally in sibling controls (top) but it is absent in <em>eve</em> mutants (bottom). There is no invagination at the head–trunk boundary at the onset of gastrulation, but an ectopic fold starts forming near the dorsal region as soon as the mitotic domains begin expanding around 24min. The ectopic folds unfolds almost entirely by the end of the recording (about 85min). Additional ectopic folds appear in the trunk region. Frame rate = 10fps. Scale bars = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-3">Video 3</h2>
<div id="fig:vid-profile-btd" class="fignos">
<figure>
<img src="figures/Vid3.jpg" data-tag="Video 3" style="width:50.0%" alt="" /><figcaption><span>Figure Video 3:</span> Profile view of ectopic fold formation in <em>btd</em> mutant. In sibling controls (top), the cephalic furrow initiates at the onset of gastrulation (1.5min) and is fully invaginated when the cell divisions start (about 11min). In <em>btd</em> mutants (bottom), no invagination initiates but some embryos exhibit a bulging of the epithelium due to a residual apical constriction behavior (about 7min). An ectopic fold forms at this position. Its morphology differs greatly from the cephalic furrow (see 10min). Both the cephalic furrow and ectopic folds regress with the extension of the germ band. Frame rate = 10fps. Scale bar = 20µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-4">Video 4</h2>
<div id="fig:vid-profile-eve" class="fignos">
<figure>
<img src="figures/Vid4.jpg" data-tag="Video 4" style="width:50.0%" alt="" /><figcaption><span>Figure Video 4:</span> Profile view of ectopic fold formation in <em>eve</em> mutant. In sibling controls (top), the cephalic furrow intiates at the onset of gastrulation (1.8min). In <em>eve</em> mutants, there are no folds appearing in the epithelium until the formation of mitotic domains (about 10min). Then, a large ectopic fold appears posterior to dividing cells (15min). The epithelium of <em>eve</em> mutants show additional folding events along the head and trunk regions. Frame rate = 10fps. Scale bar = 20µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-5">Video 5</h2>
<div id="fig:vid-mitotic-fold" class="fignos">
<figure>
<img src="figures/Vid5.jpg" data-tag="Video 5" style="width:50.0%" alt="" /><figcaption><span>Figure Video 5:</span> Ectopic folding between mitotic domains in <em>btd</em> mutant. Detailed view of cartographic projections of <em>btd</em> embryos showing the formation of the cephalic furrow (left) and of an ectopic fold (right). In sibling controls, the cephalic furrow initiates from a narrow row of cells and invaginates in a progressive manner before the appearance of mitotic domains. In <em>btd</em> mutants, the ectopic folds only appear after the apical expansion of dividing cells within mitotic domains, quickly buckling and unfolding shortly after. Frame rate = 10fps. Scale bar = 20µm (approximate value for cartographic projection).</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-6">Video 6</h2>
<div id="fig:vid-strain-rate" class="fignos">
<figure>
<img src="figures/Vid6.jpg" data-tag="Video 6" style="width:50.0%" alt="" /><figcaption><span>Figure Video 6:</span> Epithelial strain rate during ectopic folding in <em>btd</em> mutant. Video from Figure <a href="#fig:vid-mitotic-fold">Video 5</a> overlayed with the estimated strain rate across the tissues (color-coded from purple to yellow). Increase in strain rates are associated with tissue infolding and mitotic expansions. The video is looped. Frame rate = 10fps.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-7">Video 7</h2>
<div id="fig:vid-cautlat-eve" class="fignos">
<figure>
<img src="figures/Vid7.jpg" data-tag="Video 7" style="width:50.0%" alt="" /><figcaption><span>Figure Video 7:</span> Lateral views of germ band cauterization in <em>eve</em> embryos. Non-cauterized <em>eve</em> embryo showing the formation of ectopic folds (top, same embryo from Figure <a href="#fig:vid-lateral-eve">Video 2</a>) and a cauterized <em>eve</em> embryo where no ectopic folds appear at the head–trunk interface (bottom). The germ band extension is mechanically blocked by cauterizing the tissue to the vitelline envelope. Mitotic domains form normally but no folding of the surface occurs. Frame rate = 10fps. Scale bar = 50µm.</figcaption>
</figure>
</div>
<!--TODO: Replace eve embryo so that it is not the same used in a previous video?-->
<div style=""></div>
<h2 id="video-8">Video 8</h2>
<div id="fig:vid-cautdor-eve" class="fignos">
<figure>
<img src="figures/Vid8.jpg" data-tag="Video 8" style="width:50.0%" alt="" /><figcaption><span>Figure Video 8:</span> Profile views of germ band cauterization in <em>eve</em> mutant. Same embryo from Figure <a href="#fig:vid-cautlat-eve">Video 7</a>, but showing a surface and a profile view. The cauterization prevents the extension of the germ band. The mitotic domains compress non-dividing cells but these do not buckle. Frame rate = 10fps. Scale bar = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-9">Video 9</h2>
<div id="fig:vid-cautdor-btd" class="fignos">
<figure>
<img src="figures/Vid9.jpg" data-tag="Video 9" style="width:50.0%" alt="" /><figcaption><span>Figure Video 9:</span> Profile views of germ band cauterizations in <em>btd</em> embryos. A non-cauterized <em>btd</em> embryo (top) showing ectopic folds and a cauterized <em>btd</em> embryo showing no ectopic folds (bottom). Frame rate = 10fps. Scale bar = 20µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-10">Video 10</h2>
<div id="fig:vid-double-lateral" class="fignos">
<figure>
<img src="figures/Vid10.jpg" data-tag="Video 10" style="width:50.0%" alt="" /><figcaption><span>Figure Video 10:</span> Lateral view of <em>btd</em>–<em>stg</em> double mutant. A <em>btd</em> homozygote (top) shows the formation of ectopic folds while no ectopic folds form in the <em>btd</em>–<em>stg</em> double mutant (bottom). Frame rate = 10fps. Scale bars = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-11">Video 11</h2>
<div id="fig:vid-double-dorsal" class="fignos">
<figure>
<img src="figures/Vid11.jpg" data-tag="Video 11" style="width:50.0%" alt="" /><figcaption><span>Figure Video 11:</span> Dorsal view of <em>btd</em>–<em>stg</em> double mutant. A <em>btd</em> homozygote shows the formation of an ectopic fold (top). The <em>btd</em>–<em>stg</em> double mutant exhibits no mitotic domains and no ectopic folds (bottom). Frame rate = 10fps. Scale bar = 20µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="video-12">Video 12</h2>
<div id="fig:vid-slp-lateral" class="fignos">
<figure>
<img src="figures/Vid12.jpg" data-tag="Video 12" style="width:50.0%" alt="" /><figcaption><span>Figure Video 12:</span> Lateral view of <em>slp</em> mutant. A <em>slp</em> heterozygote (top) shows a normal cephalic furrow formation. The <em>slp</em> homozygote (bottom) shows a delayed initiator cell behavior and cephalic furrow formation. The embryo also exhibits a more prominent posterior dorsal fold and an ectopic fold forming within the mitotic domain 6 before cell divisions. Frame rate = 10fps. Scale bar = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h1 id="tables">Tables</h1>
<h2 id="table-1">Table 1</h2>
<div id="tbl:gbtime" class="tablenos">
<table>
<caption><span>Table 1:</span> Relative timing differences between the formation of the cephalic furrow (CF), mitotic domains (MD), and ectopic folds (EFs) in different mutant backgrounds. We measured the time after gastrulation (TAG) and the percentage of germ band extension (GBE) at the moment of each developmental event. </caption>
<thead>
<tr class="header">
<th></th>
<th>Zygosity</th>
<th>Event</th>
<th>TAG (min)</th>
<th>GBE (%)</th>
<th>n</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>wildtype</td>
<td>+/+</td>
<td>CF</td>
<td>7.2±2.2</td>
<td>8.7±1.7</td>
<td>16</td>
</tr>
<tr class="even">
<td></td>
<td>+/+</td>
<td>MD</td>
<td>14.6±2.4</td>
<td>21.4±1.9</td>
<td></td>
</tr>
<tr class="odd">
<td><em>btd</em></td>
<td>+/−</td>
<td>CF</td>
<td>8.2±1.7</td>
<td>9.8±1.9</td>
<td>6</td>
</tr>
<tr class="even">
<td></td>
<td>+/−</td>
<td>MD</td>
<td>14.6±3.7</td>
<td>22.1±4.2</td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td>−/−</td>
<td>EF</td>
<td>21.3±4.7</td>
<td>36.3±2.4</td>
<td>5</td>
</tr>
<tr class="even">
<td></td>
<td>−/−</td>
<td>MD</td>
<td>17.0±3.7</td>
<td>28.1±2.3</td>
<td></td>
</tr>
<tr class="odd">
<td><em>eve</em></td>
<td>+/−</td>
<td>CF</td>
<td>12.2±1.7</td>
<td>8.8±2.3</td>
<td>7</td>
</tr>
<tr class="even">
<td></td>
<td>+/−</td>
<td>MD</td>
<td>18.2±1.6</td>
<td>19.9±4.1</td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td>−/−</td>
<td>EF</td>
<td>22.6±2.6</td>
<td>28.5±3.9</td>
<td>5</td>
</tr>
<tr class="even">
<td></td>
<td>−/−</td>
<td>MD</td>
<td>16.9±1.1</td>
<td>19.0±2.7</td>
<td></td>
</tr>
<tr class="odd">
<td><em>prd</em></td>
<td>+/−</td>
<td>CF</td>
<td>11.3±6.2</td>
<td>18.0±12.2</td>
<td>10</td>
</tr>
<tr class="even">
<td></td>
<td>+/−</td>
<td>MD</td>
<td>12.8±1.1</td>
<td>20.4±3.8</td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td>−/−</td>
<td>CF</td>
<td>18.2±5.4</td>
<td>33.0±9.5</td>
<td>5</td>
</tr>
<tr class="even">
<td></td>
<td>−/−</td>
<td>MD</td>
<td>13.0±2.0</td>
<td>23.4±3.0</td>
<td></td>
</tr>
<tr class="odd">
<td><em>btd</em>+<em>eve</em></td>
<td>+/−</td>
<td>CF</td>
<td>10.4±2.6</td>
<td>9.2±2.1</td>
<td>13</td>
</tr>
<tr class="even">
<td></td>
<td>+/−</td>
<td>MD</td>
<td>16.6±3.2</td>
<td>20.9±4.1</td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td>−/−</td>
<td>EF</td>
<td>21.9±3.6</td>
<td>32.4±5.1</td>
<td>10</td>
</tr>
<tr class="even">
<td></td>
<td>−/−</td>
<td>MD</td>
<td>17.0±2.6</td>
<td>23.6±5.3</td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
<h2 id="table-2">Table 2</h2>
<div id="tbl:stats" class="tablenos">
<table>
<caption><span>Table 2:</span> Folding statistics in cephalic furrow mutants. We calculated the percentage of embryos exhibiting a cephalic furrow (CF) or ectopic folds (EF) for each stock and genotype (Frequency), and for each position relative to the head–trunk boundary (Ant=anterior, Mid=middle, Post=posterior). In addition, we calculated the average number of folds per embryo side. For example, 28 out of 36 wildtype embryos show ectopic folds (77.8%); 42.9% of these embryos form ectopic folds anterior to the head–trunk boundary, while 71.4% form them at the posterior position; and each embryo forms 1.1±0.3 ectopic fold on each of its side, on average. The <em>n</em> includes datasets imaged from the lateral and dorsal sides. </caption>
<thead>
<tr class="header">
<th style="text-align: left;">Stock</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Zygosity</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Type</th>
<th style="text-align: right;">n</th>
<th style="text-align: right;">Frequency</th>
<th style="text-align: right;">Ant</th>
<th style="text-align: right;">Mid</th>
<th style="text-align: right;">Post</th>
<th style="text-align: right;">Folds</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">wildtype</td>
<td style="text-align: right;">36</td>
<td style="text-align: center;">+/+</td>
<td style="text-align: right;">36</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">36</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">77.8%</td>
<td style="text-align: right;">42.9%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">71.4%</td>
<td style="text-align: right;">1.1±0.3</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>btd</em></td>
<td style="text-align: right;">46</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">33</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">33</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">18.2%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">13</td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">92.3%</td>
<td style="text-align: right;">50%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">75%</td>
<td style="text-align: right;">2.2±0.4</td>
</tr>
<tr class="even">
<td style="text-align: left;"><em>eve</em></td>
<td style="text-align: right;">36</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">26</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">26.9%</td>
<td style="text-align: right;">14.3%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">85.7%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">10</td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">40%</td>
<td style="text-align: right;">70%</td>
<td style="text-align: right;">90%</td>
<td style="text-align: right;">1.8±0.6</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>prd</em></td>
<td style="text-align: right;">40</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">26</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">26.9%</td>
<td style="text-align: right;">71.4%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">57.1%</td>
<td style="text-align: right;">1.3±0.5</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">14</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">50.0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">71.4%</td>
<td style="text-align: right;">50%</td>
<td style="text-align: right;">80%</td>
<td style="text-align: right;">70%</td>
<td style="text-align: right;">1.9±0.8</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>stg</em></td>
<td style="text-align: right;">46</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">33</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">33</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">36.4%</td>
<td style="text-align: right;">8.3%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">91.7%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">13</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">23.1%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
<h2 id="table-3">Table 3</h2>
<div id="tbl:area" class="tablenos">
<table>
<caption><span>Table 3:</span> Area of ectopic fold formation in cephalic furrow mutants. We calculated surface area (µm<sup>2</sup>×10<sup>3</sup>) of folded cells in the cephalic furrow and ectopic folds of different mutant backgrounds. </caption>
<thead>
<tr class="header">
<th></th>
<th>wildtype</th>
<th><em>btd</em></th>
<th><em>eve</em></th>
<th><em>prd</em></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>CF</td>
<td>11.4±1.2</td>
<td>10.3±1.9</td>
<td>9.7±1.1</td>
<td>6.0±3.4</td>
</tr>
<tr class="even">
<td>EF</td>
<td>0.7±0.5</td>
<td>2.3±1.7</td>
<td>3.2±2.1</td>
<td>1.0±0.6</td>
</tr>
<tr class="odd">
<td>n</td>
<td>CF=16, EF=16</td>
<td>CF=6, EF=13</td>
<td>CF=7, EF=7</td>
<td>CF=14, EF=8</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
<h1 id="extended-data">Extended data</h1>
<!--TODO: Limit to 10 extended data (only figures and tables)-->
<!--TODO: Add single PDF for supplementary methods, tables, discussion, equations, and notes-->
<!--TODO: Convert videos to supplementary standalone files-->
<p>High-resolution versions of the extended data figures and videos are available at <a href="https://doi.org/10.5281/zenodo.7781916">https://doi.org/10.5281/zenodo.7781916</a>.<span class="citation" data-cites="Vellutini2023-ix"><sup><a href="#ref-Vellutini2023-ix" role="doc-biblioref">52</a></sup></span></p>
<div style=""></div>
<h2 id="extended-data-fig.-1">Extended Data Fig. 1</h2>
<div id="fig:initiator" class="fignos">
<figure>
<img src="figures/EDFig1.jpg" data-tag="Extended Data Fig. 1" style="width:100.0%" alt="" /><figcaption><span>Figure Extended Data Fig. 1:</span> Perturbation of the initiator cell behavior in cephalic furrow mutants. <strong>a</strong>, Profile view showing the cephalic furrow formation in wildtype, <em>prd</em>, <em>btd</em>, and <em>eve</em> embryos. The samples are synchronized by the end of cellularization, when the cleavage furrows reach their basal position (frame 0.0min). The arrow indicates the position and timing of the first infolding of the tissue. In wildtype, the shortening of initiator cells begins before cellularization is complete. The apical side of adjacent cells remain in close contact to the initiator row becoming arched in an arrangement typical for the early phase of cephalic furrow formation (wildtype 0.0min). This arrangement is perturbed in mutant embryos to different degrees. In <em>prd</em>, the initiator cells shorten and the tissue invaginates, but the infolding is delayed and the adjacent cells do not arch over the initiator row (<em>prd</em> 6.1min). In <em>btd</em>, there is no cell shortening but some embryos exhibit a certain degree of anisotropic apical constriction which creates a bulge in the epithelium minutes after the end of cellularization (<em>btd</em> 6.0min, see also <strong>b</strong>). This initial bulge often primes the position of ectopic buckling. In <em>eve</em>, the cells show neither shortening nor apical constriction and ectopic folds appear about ten minutes after the end of cellularization (<em>eve</em> 9.7min). Scale bars = 20µm. <strong>b</strong>, Surface view view of cartographic projections showing the head–trunk interface. In wildtype, the anisotropic apical constriction is localized to a narrow stripe adjacent to the initiator row. In <em>prd</em> embryos, the apical constriction occurs but it does not form a clear line of infolding cells preceding the invagination as in wildtype embryos. In <em>btd</em> embryos, there is a similar degree of anisotropic apical constriction occurring but not all embryos form ectopic folds at this region. In <em>eve</em> embryos, the mitotic domain 2 (MD) begins expanding and there is no apical constriction behavior. Time betwee frames is about 4min. Scale bars = 20µm (approximate value).</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-2">Extended Data Fig. 2</h2>
<div id="fig:features" class="fignos">
<figure>
<img src="figures/EDFig2.jpg" data-tag="Extended Data Fig. 2" style="width:100.0%" alt="" /><figcaption><span>Figure Extended Data Fig. 2:</span> Differences between ectopic folding and cephalic furrow formation. <strong>a</strong>, Profile view of a <em>btd</em> mutant embryo showing the presence of two ectopic folds (arrows) forming next to dividing cells (asterisks). Scale bar = 20µm. <strong>b</strong>, Distribution of ectopic folds in <em>eve</em> homozygotes (right) and sibling controls (left). Scale bar = 50µm. <strong>c</strong>, Folded area of the cephalic furrow (CF) and ectopic folds (EF) in <em>eve</em> embryos. The folded region is outlined in yellow on cartographic projections of a representative sibling control <em>eve</em> heterozygote (left) and of a <em>eve</em> homozygote (right). The numbers indicate the average folded area for the sample type in µm<sup>2</sup>×10<sup>3</sup>. <strong>d</strong>, Dynamics of cephalic furrow and ectopic fold formation in <em>eve</em> mutants. <strong>e</strong>, Dynamics, angle directionality, and tortuosity measurements comparing the cephalic furrow and ectopic fold formation. Within the first fifteen minutes after gastrulation, the cephalic furrow exhibits a typical posterior shift on the dorsal side which declines the initial angle of the invagination from 80° to about 60° in relation to the anteroposterior axis. During this period, the invagination begins as a straight line and bends showing a correspondent increase in the measured tortuosity values of the furrow outline. In contrast, ectopic folds show no obvious trend in angular direction tortuosity values over time. For both angle and tortuosity analysis, n=6. <strong>f</strong>, Maximum folding depth of the cephalic furrow and ectopic folds in <em>btd</em> and <em>eve</em> mutants. Ectopic folds are shallower than the cephalic furrow in both genetic backgrounds (<em>btd</em> p=1.278e-09 and <em>eve</em> p=9.728e-05 in a Welch Two Sample t-test). Each dot corresponds to a single fold; each embryo can have multiple folds. The number of embryos analyzed for <em>btd</em> is 22 heterozygotes and 6 homozygotes, and for <em>eve</em> is 14 heterozygotes and 4 homozygotes. <strong>g</strong>, Folded area of ectopic folds in wildtype and mutant embryos (<em>btd</em>, <em>eve</em>, and <em>prd</em>). Ectopic folds in wildtype occupy a smaller area than ectopic folds in cephalic furrow mutants (p=9.107e-06 in a Wilcoxon rank sum exact test). <strong>h</strong>, Lateral views of a wildtype (top) and a <em>prd</em> mutant (bottom) exhibiting ectopic folds. Scale bar = 50µm. <strong>i</strong>, Folded area of the cephalic furrow (CF) and ectopic folds (EF) in the wildtype (top) and <em>prd</em> mutant (bottom) shown in <strong>h</strong>. The folded region is outlined in yellow on a cartographic projection. The numbers indicate the average folded area in µm<sup>2</sup>×10<sup>3</sup> for the CFs and EFs separately. <strong>j</strong>, Comparison of the folded area between the cephalic furrow and the ectopic folds in different genetic backgrounds. <strong>k</strong>, Developmental sequence of the wildtype embryo ectopic fold annotated in <strong>i</strong>. Four cells are temporarily infolded during the mitotic expansion of adjacent cells. Scale bar = 10µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-3">Extended Data Fig. 3</h2>
<div id="fig:apical-strain" class="fignos">
<figure>
<img src="figures/EDFig3.jpg" data-tag="Extended Data Fig. 3" style="width:100.0%" alt="" /><figcaption><span>Figure Extended Data Fig. 3:</span> Complementary analyses of mitotic expansions, strain rate, and tissue compression. <strong>a</strong>, Increase in the apical area of individual cells within mitotic domains. A dividing cell increases its apical area 2.4 times during mitotic rounding. The individual daughter cells retain 66% of the parent apical area. When summed, the apical area of the two daughter cells occupy 1.3 times the original apical area of their parent cell. <strong>b</strong>, Strain rate analysis in <em>btd</em> mutants. Cropped region of cartographic projections of <em>btd</em> sibling controls (top, n=3) and homozygote embryos (bottom, n=3). The membrane marker (Gap43-mCherry) is overlayed with a heatmap indicating the regions of increased strain rate in the tissue. The value is the sum of isotropic and anisotropic strain rates obtained through a particle image velocimetry analysis. We used the strain rates in the regions outlined as head–trunk and trunk–germ to generate the plot in Figure <a href="#fig:mitotic-strain">2</a>f.</figcaption>
</figure>
</div>
<!--TODO: Re-plot apical area expansion using manuscript style-->
<!--TODO: Convert area from px to µm-->
<!--TODO: Add values to plot to be more informative-->
<!--TODO: Add scale bar-->
<div style=""></div>
<h2 id="extended-data-fig.-4">Extended Data Fig. 4</h2>
<div id="fig:model-features" class="fignos">
<figure>
<img src="figures/EDFig4.jpg" data-tag="Extended Data Fig. 4" style="width:100.0%" alt="" /><figcaption><span>Figure Extended Data Fig. 4:</span> Characterization of the model features. <strong>a</strong>, Embryonic proportions in wildtype and cephalic furrow mutants. Approximate relative sizes and positions between embryonic features such as mitotic domains, folds, and the germ band. All values are relative to the embryo length. We used these dimensions as a reference for creating the model. <strong>b</strong>, Representative simulation using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}}=7 \times 10^{-5}</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0.3</mn></mrow><annotation encoding="application/x-tex">g=0.3</annotation></semantics></math> showing the shape of the tissue at <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>t</mi><mo>=</mo><mn>19</mn></mrow><annotation encoding="application/x-tex">t=19</annotation></semantics></math> (blue) and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>t</mi><mo>=</mo><mn>1000</mn></mrow><annotation encoding="application/x-tex">t=1000</annotation></semantics></math> (pink). The respective timepoints are marked in dashed lines in the descriptive plots below. They show the variation in total energy, bending energy, number of folds, and fold depth over the iterations. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>t</mi><mo>=</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">t=1</annotation></semantics></math> corresponds to <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msup><mn>10</mn><mn>5</mn></msup><annotation encoding="application/x-tex">10^{5}</annotation></semantics></math> computational steps and the X axis is in <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>l</mi><mi>o</mi><msub><mi>g</mi><mn>10</mn></msub></mrow><annotation encoding="application/x-tex">log_{10}</annotation></semantics></math> scale to improve the visualization. <strong>c–e</strong>, Parameter sweep without mitotic domains. Plots show (<strong>c</strong>) the number of folds by bending rigidity <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0.3</mn></mrow><annotation encoding="application/x-tex">g=0.3</annotation></semantics></math>, (<strong>d</strong>) the number of folds by germ band extension (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math>) using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}}=1.0 \times 10^{-4}</annotation></semantics></math>, and (<strong>e</strong>) the timing of folding by germ band extension (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math>) using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}}=1.0 \times 10^{-4}</annotation></semantics></math> (right). <strong>f–h</strong>, Parameter sweep with mitotic domains and same plots as above. <strong>i</strong>, Parameter sweep for cephalic furrow simulations. Bending rigidity and cephalic furrow depth at different values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> and germ band extension. Values above 0.2 exhibit a clear peak in bending energy for most conditions and the depth reaches a plateau across iterations. The cephalic furrow depth at the peak of bending energy (max bend) and at the final iteration is similar for simulations with 0% of germ band extension. At higher percentages of germ band extension the folds (both cephalic furrow and ectopic folds) exhibit a greater depth at the last iteration. <strong>j</strong>, Finer-grained parameter sweep of ectopic folding without (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">t_{MD}=0</annotation></semantics></math>) and with (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>5</mn></mrow><annotation encoding="application/x-tex">t_{MD}=5</annotation></semantics></math>) a delay in cephalic furrow formation. With simultaneous formation, only higher values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are effective in buffering the formation of ectopic folds around the cephalic furrow region. This is also limited to low percentages of germ band extension since at higher percentages there is an increase in the frequency of ectopic folding. When a delay in mitotic formation is present, even low values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are effective in preventing the formation of ectopic folds. In this condition, the extension of the germ band increases the formation of ectopic folds, but only at the posterior regions close to the germ band tip. This suggests that the initiation of the cephalic furrow is crucial to its ability to buffer the ectopic folding. Values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are shown in units of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>1</mn><mi>/</mi><mi>L</mi></mrow><annotation encoding="application/x-tex">1/L</annotation></semantics></math>. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">t_{MD}=1</annotation></semantics></math> corresponds to 10<sup>5</sup> computational timesteps. <strong>k</strong>, Representative simulations in <strong>g</strong> at 0 and 20% germ band extension.</figcaption>
</figure>
</div>
<!--TODO: Update model-features references in the text-->
<!--TODO: Move full cf-sweep to supplementary or just leave in repository?-->
<!--TODO: Replace with newer simulations-->
<div style=""></div>
<h2 id="extended-data-fig.-5">Extended Data Fig. 5</h2>
<!--TODO: Add more contents to this figure-->
<div id="fig:live-details" class="fignos">
<figure>
<img src="figures/EDFig5.jpg" data-tag="Extended Data Fig. 5" style="width:100.0%" alt="" /><figcaption><span>Figure Extended Data Fig. 5:</span> Additional <em>in vivo</em> experiments in cephalic furrow mutants. <strong>a</strong>, Lateral 3D renderings of <em>stg</em> mutant embryos. <em>stg</em> homozygotes show no cell divisions after gastrulation but the early morphogenetic movements of gastrulation occur normally. Asterisks indicate mitotic domains. The cephalic furrow forms without delays and exhibits a similar dynamics of invagination compared to wildtype embryos. The only noticeable difference is that the dorsal portion does not shift as posteriorly as in sibling controls, which could be due to the absence of mitotic domains in the head. Scale bar = 50µm. <strong>b</strong>, Optical slice of profile views. The initiator cell behaviors are not perturbed in <em>stg</em> mutants and the morphology of the invagination is intact. In fact, because of the lack of cell divisions, the epithelium remains more uniform during gastrulation when compared to sibling controls or wildtype embryos. Scale bar = 20µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-6">Extended Data Fig. 6</h2>
<div id="fig:genetic-features" class="fignos">
<figure>
<img src="figures/EDFig6.jpg" data-tag="Extended Data Fig. 6" style="width:100.0%" alt="" /><figcaption><span>Figure Extended Data Fig. 6:</span> Additional genetic patterning of the head–trunk boundary in <em>Drosophila</em>. <strong>a</strong>, Expression of <em>btd</em> and <em>eve</em> in wildtype and <em>slp</em> mutants at mid and late mitotic cycle 14.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-7">Extended Data Fig. 7</h2>
<div id="fig:genetic-comparative" class="fignos">
<figure>
<img src="figures/EDFig7.jpg" data-tag="Extended Data Fig. 7" style="width:100.0%" alt="" /><figcaption><span>Figure Extended Data Fig. 7:</span> Additional genetic patterning of the head–trunk boundary in other species. <strong>a</strong>, Expression of…</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-video-s1">Extended Data Video S1</h2>
<div id="fig:vid-initiator" class="fignos">
<figure>
<img src="figures/VidS1.jpg" data-tag="Video S1" style="width:50.0%" alt="" /><figcaption><span>Figure Video S1:</span> Residual apical constriction in <em>btd</em> mutants. Lateral view (left) and cartographic projection (right) of the head–trunk interface in four individual <em>btd</em> mutants. Cells exhibit different degrees of apical constriction. The video is looped to highlight the changes in apical cell area. Frame rate = 10fps.</figcaption>
</figure>
</div>
<!--TODO: Add scale bars-->
<div style=""></div>
<h2 id="extended-data-video-s2">Extended Data Video S2</h2>
<div id="fig:vid-dynamics-btd" class="fignos">
<figure>
<img src="figures/VidS2.jpg" data-tag="Video S2" style="width:50.0%" alt="" /><figcaption><span>Figure Video S2:</span> Dynamics of ectopic folding in <em>btd</em> mutant. The cephalic furrow in sibling controls (top) and the ectopic folds in <em>btd</em> mutants (bottom) are annotated in white to visualize the dynamics in position, extension, and shape during their formation. Frame rate = 10fps. Scale bars = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-video-s3">Extended Data Video S3</h2>
<div id="fig:vid-variability-btd" class="fignos">
<figure>
<img src="figures/VidS3.jpg" data-tag="Video S3" style="width:50.0%" alt="" /><figcaption><span>Figure Video S3:</span> Variability of ectopic folding in <em>btd</em> mutants. The video shows four individual <em>btd</em> mutants where each display a different pattern and number of ectopic folds at the head–trunk interface. The video is looped to highlight the dynamics of ectopic folding. Frame rate = 15fps. Scale bars = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-video-s4">Extended Data Video S4</h2>
<div id="fig:vid-stg-lateral" class="fignos">
<figure>
<img src="figures/VidS4.jpg" data-tag="Video S4" style="width:50.0%" alt="" /><figcaption><span>Figure Video S4:</span> Lateral view of cephalic furrow formation in <em>stg</em> mutants. Sibling control (top) and <em>stg</em> mutant (bottom) during gastrulation. The formation of the cephalic furrow almost identical to the control embryo. The other morphogenetic movements also occur normally until about 35min. At this point the cells in the <em>stg</em> mutant are notably larger than the control. Frame rate = 15fps. Scale bars = 50µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-video-s5">Extended Data Video S5</h2>
<div id="fig:vid-stg-dorsal" class="fignos">
<figure>
<img src="figures/VidS5.jpg" data-tag="Video S5" style="width:50.0%" alt="" /><figcaption><span>Figure Video S5:</span> Dorsal view of cephalic furrow formation in <em>stg</em> mutants. Sibling control (top) and <em>stg</em> mutant (bottom) during gastrulation. The cephalic furrow in <em>stg</em> mutants initiates without delay and shows identical morphology to the control until cell divisions begin in the latter. The cells dividing within the cephalic furrow of control embryos alter its morphology, it becomes curved and lengthier. In contrast, the cephalic furrow in the <em>stg</em> mutant retains its initial morphology until it unfolds. Frame rate = 10fps. Scale bar = 20µm.</figcaption>
</figure>
</div>
<!--TODO: Fix the time to show countdown to gastrulation-->
<!--TODO: Add slp-mutant video from 1-analyses/slp-mutants/3-snaps/-->
<div style=""></div>
<h1 id="supplementary-information">Supplementary information</h1>
<h2 id="supplementary-note-1">Supplementary Note 1</h2>
<h3 id="comparing-our-reference-bending-rigidity-to-direct-measurements">Comparing our reference bending rigidity to direct measurements</h3>
<!--TODO: Revise this text-->
<p>The goal is to compare the reference bending rigidity that we estimate for the <em>Drosophila</em> blastoderm with a direct measurement from the literature. Trushko et al. <span class="citation" data-cites="Trushko2020-gf"><sup><a href="#ref-Trushko2020-gf" role="doc-biblioref">16</a></sup></span> reports that in a MDCK monolayer <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>b</mi></msub><mo>=</mo><mn>5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>13</mn></mrow></msup><mi>N</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">K_b = 5\times10^{-13} Nm</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>s</mi></msub><mo>=</mo><mn>0.15</mn><mi>N</mi><msup><mi>m</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_s = 0.15 Nm^{-1}</annotation></semantics></math>. With these values we can compute the dimensionless bending rigidity <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mo>*</mo></msubsup><annotation encoding="application/x-tex">K_b^*</annotation></semantics></math>. However, the thickness of the MDCK monolayer and the <em>Drosophila</em> blastoderm are very different, and hence we need to correct for this before computing <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mo>*</mo></msubsup><annotation encoding="application/x-tex">K_b^*</annotation></semantics></math> for MDCK monolayer. The bending rigidity scales with square of the thickness of the tissue. Hence the corrected <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math> can be computed as <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mo>=</mo><msubsup><mi>K</mi><mi>b</mi><mo>*</mo></msubsup><mo stretchy="false" form="prefix">(</mo><msub><mi>h</mi><mrow><mi>b</mi><mi>l</mi><mi>a</mi><mi>s</mi><mi>t</mi><mi>o</mi><mi>d</mi><mi>e</mi><mi>r</mi><mi>m</mi></mrow></msub><mi>/</mi><msub><mi>h</mi><mrow><mi>M</mi><mi>D</mi><mi>C</mi><mi>K</mi></mrow></msub><msup><mo stretchy="false" form="postfix">)</mo><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_b^{&#39;} = K_b^*(h_{blastoderm}/h_{MDCK})^2</annotation></semantics></math>. This gives us <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mo>=</mo><mn>2</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>12</mn></mrow></msup><mi>N</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">K_b^{&#39;} =2\times10^{-12} Nm</annotation></semantics></math>. Now we can compute <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mo>*</mo></msubsup><mo>=</mo><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mi>/</mi><msub><mi>K</mi><mi>s</mi></msub><msup><mi>L</mi><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_b^* = K_b^{&#39;}/K_s L^2</annotation></semantics></math> where we put <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>L</mi><mo>=</mo><mn>255</mn><mi>μ</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">L = 255\mu m</annotation></semantics></math>. Here we scale <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mi>/</mi><msub><mi>K</mi><mi>s</mi></msub></mrow><annotation encoding="application/x-tex">K_b^{&#39;}/K_s</annotation></semantics></math> by the relevant lengthscale which is the semi-major axis of the Drosophila embryo. This gives us <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mo>*</mo></msubsup><mo>=</mo><mn>2.05</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_b^* = 2.05\times 10^{-4}</annotation></semantics></math>. This is the estimated <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mo>*</mo></msubsup><annotation encoding="application/x-tex">K_b^*</annotation></semantics></math> for the MDCK monolayer that has the same thickness as the <em>Drosophila</em> blastoderm (referring to the height correction) and has been put in the same geometrical conditions as the <em>Drosophila</em> blastoderm (referring to the <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>L</mi><annotation encoding="application/x-tex">L</annotation></semantics></math> used in the calculation).</p>
<h2 id="supplementary-note-2">Supplementary Note 2</h2>
<h3 id="live-imaging-screen-for-cephalic-furrow-genes">Live-imaging screen for cephalic furrow genes</h3>
<!--TODO: Add figure with screen outline-->
<!--TODO: Add table with candidate genes-->
<!--TODO: Add table with screen results-->
<!--TODO: Add figure with screen results-->
<p>To uncover other genes directly involved in cephalic furrow formation in addition to <em>btd</em>, <em>eve</em>, and <em>prd</em>, we performed a live-imaging screen in strains containing loss-of-function alleles for a selection of candidate genes expressed at the head–trunk region.<span class="citation" data-cites="Tomancak2002-yf Tomancak2007-db Lecuyer2007-uy"><sup><a href="#ref-Tomancak2002-yf" role="doc-biblioref">61</a>–<a href="#ref-Lecuyer2007-uy" role="doc-biblioref">63</a></sup></span> Because the cephalic furrow is transient and leaves no trace, the live-imaging approach is critical to recognize altered phenotypes. From about 50 genes, we only detected three showing abnormal cephalic furrow formation to different degrees besides the previously described genes (see Table <a href="#tbl:screen">Supplementary Table 1</a>). The strongest cephalic furrow phenotype was present in flies mutant for the <em>sloppy paired</em> (<em>slp</em>) genes.</p>
<h2 id="supplementary-note-3">Supplementary Note 3</h2>
<h3 id="genetic-interactions-at-the-headtrunk-boundary">Genetic interactions at the head–trunk boundary</h3>
<p>early slp activity represses eve at the anterior end</p>
<p><em>btd</em> is de-repressed between the acron and head–trunk boundary (Figure <a href="#fig:genetic-patterning">5</a>c–d)</p>
<h2 id="supplementary-method-1">Supplementary Method 1</h2>
<div id="fig:tortuosity" class="fignos">
<figure>
<img src="figures/SIFig1.jpg" data-tag="Supplementary Method 1" style="width:100.0%" alt="" /><figcaption><span>Figure Supplementary Method 1:</span> Image processing pipeline for the tortuosity analysis in cauterized mutants. <strong>a</strong>, We acquired a single slice in profile view of the head–trunk epithelium. First, we straightened the epithelial monolayer along the curvature of the vitelline envelope using the Straighten tool in ImageJ. We then cropped a window to standardize the size of the analyzed area for all embryos. <strong>b</strong>, Then, we applied a gaussian blur which allows to capture the deformations in the epithelium caused by the ectopic folds after thresholding. We create a mask and detect edges and invert to retain a single pixel line corresponding to the outline of the epithelium. The image is inverted for downstream processing. <strong>c</strong>, We applied a temporal color-coded projections to visualize the epithelial dynamics over time, and created a montage with all timepoints to extract the length of the outline using the skeletonize plugin in ImageJ.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-methods-2">Supplementary Methods 2</h2>
<div id="fig:ablation" class="fignos">
<figure>
<img src="figures/SIFig2.jpg" data-tag="Supplementary Note 2" style="width:80.0%" alt="" /><figcaption><span>Figure Supplementary Note 2:</span> Image processing pipeline for the ablation analysis. <strong>a</strong>, Example of laser ablation near the cephalic furrow with the membrane signal (top left), the generated kymograph (right), and the linear fit over the distance between edges extracted from the kymograph (bottom left). The position of the laser cut is annotated in a vermilion line, the cell edges are marked in black circles, and the edge distances in a blue line. The distance between edges increase over time. <strong>b</strong>, Example of a laser ablation far from the cephalic furrow where the distance between edges does not increase over time.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-table-1">Supplementary Table 1</h2>
<div id="tbl:screen" class="tablenos">
<table>
<caption><span>Table Supplementary Table 1:</span> Summary of live-imaging screening results for cephalic furrow genes. </caption>
<colgroup>
<col style="width: 20%" />
<col style="width: 25%" />
<col style="width: 34%" />
<col style="width: 20%" />
</colgroup>
<thead>
<tr class="header">
<th>gene name</th>
<th>gene symbol</th>
<th>allele/deficiency</th>
<th>phenotype</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><em>buttonhead</em></td>
<td><em>btd</em></td>
<td><em>btd</em><sup>XA</sup></td>
<td>absent<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">7</a></sup></span></td>
</tr>
<tr class="even">
<td><em>even skipped</em></td>
<td><em>eve</em></td>
<td><em>eve</em><sup>3</sup></td>
<td>absent<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">7</a></sup></span></td>
</tr>
<tr class="odd">
<td><em>paired</em></td>
<td><em>prd</em></td>
<td><em>prd</em><sup>4</sup></td>
<td>delayed<span class="citation" data-cites="Blankenship2001-tm"><sup><a href="#ref-Blankenship2001-tm" role="doc-biblioref">14</a></sup></span> and abnormal</td>
</tr>
<tr class="even">
<td><em>sloppy paired 1</em> and <em>2</em></td>
<td><em>slp1</em>, <em>slp2</em></td>
<td><em>slp</em><sup>Δ34B</sup></td>
<td>delayed and displaced anteriorly</td>
</tr>
<tr class="odd">
<td><em>sloppy paired 1</em></td>
<td><em>slp1</em></td>
<td><em>slp1</em><sup>1</sup></td>
<td>delayed and displaced anteriorly</td>
</tr>
<tr class="even">
<td><em>giant</em></td>
<td><em>gt</em></td>
<td><em>gt</em><sup>X11</sup></td>
<td>perturbed formation</td>
</tr>
<tr class="odd">
<td><em>knirps</em>, <em>knirps-like</em></td>
<td><em>kni</em>, <em>knrl</em></td>
<td>Df(3L)BSC448</td>
<td>ventral portion displaced anteriorly</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
</body>
</html>
