<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Bruno C. Vellutini1,✉, Marina B. Cuenca1, Abhijeet Krishna1,2,3, Alicja Szałapak1,2,3, Carl D. Modes1,2,3, and Pavel Tomancak1,2,3,✉" />
  <title>Patterned invagination prevents mechanical instability during gastrulation</title>
  <style>
    html {
      line-height: 1.5;
      font-family: Georgia, serif;
      font-size: 20px;
      color: #1a1a1a;
      background-color: #fdfdfd;
    }
    body {
      margin: 0 auto;
      max-width: 36em;
      padding-left: 50px;
      padding-right: 50px;
      padding-top: 50px;
      padding-bottom: 50px;
      hyphens: auto;
      word-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    @media (max-width: 600px) {
      body {
        font-size: 0.9em;
        padding: 1em;
      }
    }
    @media print {
      body {
        background-color: transparent;
        color: black;
        font-size: 12pt;
      }
      p, h2, h3 {
        orphans: 3;
        widows: 3;
      }
      h2, h3, h4 {
        page-break-after: avoid;
      }
    }
    p {
      margin: 1em 0;
    }
    a {
      color: #1a1a1a;
    }
    a:visited {
      color: #1a1a1a;
    }
    img {
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.4em;
    }
    h5, h6 {
      font-size: 1em;
      font-style: italic;
    }
    h6 {
      font-weight: normal;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    blockquote {
      margin: 1em 0 1em 1.7em;
      padding-left: 1em;
      border-left: 2px solid #e6e6e6;
      color: #606060;
    }
    code {
      font-family: Menlo, Monaco, 'Lucida Console', Consolas, monospace;
      font-size: 85%;
      margin: 0;
    }
    pre {
      margin: 1em 0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
    }
    .sourceCode {
     background-color: transparent;
     overflow: visible;
    }
    hr {
      background-color: #1a1a1a;
      border: none;
      height: 1px;
      margin: 1em 0;
    }
    table {
      margin: 1em 0;
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
      font-variant-numeric: lining-nums tabular-nums;
    }
    table caption {
      margin-bottom: 0.75em;
    }
    tbody {
      margin-top: 0.5em;
      border-top: 1px solid #1a1a1a;
      border-bottom: 1px solid #1a1a1a;
    }
    th {
      border-top: 1px solid #1a1a1a;
      padding: 0.25em 0.5em 0.25em 0.5em;
    }
    td {
      padding: 0.125em 0.5em 0.25em 0.5em;
    }
    header {
      margin-bottom: 4em;
      text-align: center;
    }
    #TOC li {
      list-style: none;
    }
    #TOC a:not(:hover) {
      text-decoration: none;
    }
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  
</head>
<body>
<header id="title-block-header">
<h1 class="title">Patterned invagination prevents mechanical instability during gastrulation</h1>
<p class="author">Bruno C. Vellutini<sup>1,✉</sup>, Marina B. Cuenca<sup>1</sup>, Abhijeet Krishna<sup>1,2,3</sup>, Alicja Szałapak<sup>1,2,3</sup>, Carl D. Modes<sup>1,2,3</sup>, and Pavel Tomancak<sup>1,2,3,✉</sup></p>
</header>
<p><sup>1</sup> Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, Germany<br />
<sup>2</sup> Center for Systems Biology Dresden, Dresden, Germany<br />
<sup>3</sup> Cluster of Excellence Physics of Life, Technische Universität Dresden, Dresden, Germany</p>
<p><sup>✉</sup> Correspondence: <a href="mailto:vellutini@mpi-cbg.de">Bruno C. Vellutini &lt;vellutini@mpi-cbg.de&gt;</a>, <a href="mailto:tomancak@mpi-cbg.de">Pavel Tomancak &lt;tomancak@mpi-cbg.de&gt;</a></p>
<h1 id="abstract">Abstract</h1>
<!--TODO: Revise abstract adding references-->
<!--SUMMARY PARAGRAPH: Papers start with a fully referenced, bold paragraph, ideally of about 200 words, aimed at readers in other disciplines. Numbers, abbreviations, acronyms or measurements should be avoided unless essential. The summary paragraph consists of 2 to 3 sentences of basic-level introduction to the field; a brief account of the background and rationale of the work; a statement of the main conclusions (introduced by the phrase 'Here we show' or its equivalent); and a conclusion of 2 to 3 sentences putting the main findings into general context so it is clear how the results described in the paper have moved the field forward. A downloadable, annotated example is available here.-->
<p>Mechanical forces are crucial for driving and shaping tissue morphogenesis during embryonic development, but their relevance for the evolution of development remains poorly understood. Here, we show that a morphogenetic innovation present in fly embryos—a deep epithelial fold known as the cephalic furrow—plays a mechanical role during <em>Drosophila</em> gastrulation. By integrating <em>in vivo</em> experiments and <em>in silico</em> simulations, we find that the formation of the cephalic furrow prevents mechanical instabilities at the head–trunk epithelium by absorbing the compressive stresses generated by concurrent morphogenetic movements of gastrulation, the expansion of mitotic domains and the germ band extension. Furthermore, by comparing the expression of known and novel genes involved in cephalic furrow formation between fly species, we find that the presence of the cephalic furrow is linked to changes in the expression of <em>buttonhead</em> transcription factor at the head–trunk boundary. These data suggest that the genetic control of cephalic furrow formation was established through the integration of a new player into the ancestral head–trunk patterning system, and that mechanical instability may have been the selective pressure associated with the evolution of the cephalic furrow. Our findings uncover empirical evidence for how mechanical forces can influence the evolution of morphogenetic innovations in early development.</p>
<h1 id="main">Main</h1>
<!--TODO: Reduce and focus intro to 500 words-->
<!--MAIN TEXT: If further introductory material is necessary, the main text can begin with up to 500 words of introduction expanding on the background to the work (some overlap with the summary is acceptable), before proceeding to a concise, focused account of the findings, and ending with 1 or 2 short paragraphs of discussion. Sections are separated with subheadings (up to 40 characters including spaces) to aid navigation.-->
<p>Morphogenesis is a physical process that shapes embryonic tissues through cell-generated mechanical forces.<span class="citation" data-cites="Keller2012-ca Heisenberg2013-pl"><sup><a href="#ref-Keller2012-ca" role="doc-biblioref">1</a>,<a href="#ref-Heisenberg2013-pl" role="doc-biblioref">2</a></sup></span> These forces drive tissues to bend, extend, and invaginate—movements that also push and pull on their neighboring regions. Such physical interactions provide essential information to embryonic cells throughout development and ultimately shape the final morphology of tissues and organs.<span class="citation" data-cites="Collinet2021-qa"><sup><a href="#ref-Collinet2021-qa" role="doc-biblioref">3</a></sup></span> However, how mechanical forces influence the evolution of morphogenesis in early embryonic development remains unclear. To investigate the interplay between genetics and mechanics during the evolution of morphogenesis, we studied a prominent but enigmatic epithelial fold that forms at the head–trunk boundary of flies during gastrulation—the cephalic furrow.<span class="citation" data-cites="Hartenstein1985-zs Spencer2015-td"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a>,<a href="#ref-Spencer2015-td" role="doc-biblioref">5</a></sup></span></p>
<!--Morphogenesis is a physical process [@Keller2012-ca; @Heisenberg2013-pl].-->
<!--Embryonic cells drive this processes by generating force that makes tissues bend.-->
<!--When tissues extend or invaginate, they also push and pull on their neighboring tissues.-->
<!--These mechanical interactions provide essential information to embryonic cells throughout development and ultimately shape the final morphology of tissues and organs [@Collinet2021-qa].-->
<!--TODO: Revise drafts below, incorporate?-->
<!--Morphogenesis is a complex sequence of patterning and physical events.-->
<!--Under genetic control, tissues can fold via intrinsic cell-generated forces.-->
<!--Moving tissues generate extrinsic tissue-scale forces.-->
<!--Self organized morphogenesis is important for regulating development.-->
<!--Mechanical forces are crucial for the morphogenesis of embryonic tissues.-->
<!--But it is unclear whether tissue mechanics can influence the origin and evolution of patterned morphogenetic processes.-->
<!--What are the selective pressures and evolutionary events leading to an evolutionary innovation in early embryos where the tissues are confined in their shells?-->
<!--How does the mechanics of embryonic tissues impact the evolution of morphogenetic processes?-->
<!--How do novel morphogenetic processes in the embryo originate?-->
<!--Embryos develop through orchestrated cell and tissue movements known as morphogenesis.-->
<!--Morphogenetic movements are usually under tight genetic control The control of morphogenesis in early embryonic development can be patterned or self-organized, but it is usually an intricate mixture of both [@Collinet2021-qa].-->
<!--Genes determine the position and timing of cell shape changes which will drive coordinated tissue movements like in gastrulation.-->
<!--Morphogenetic processes are usually under tight genetic control.-->
<!--The expression of maternal or zygotic genes in early development determines cell fates and activate behaviors.-->
<!--The position and timing of morphogenetic processes is usually under tight genetic control and regulated by genes which instruct active cell shape changes driving morphogenesis [@Collinet2021-qa].-->
<!--Epithelial folding, tissue extension, and cell delamination are examples of morphogenetic processes occurring during embryonic development.-->
<!--These events are usually under tight genetic control.-->
<!--The position and timing is determined by the expression of maternal or zygotic genes which control cell shape changes that drive morphogenesis [@Collinet2021-qa].-->
<!--When tissues fold, they push and pull on their neighboring tissues, and these mechanical cues have been shown to be crucial as a source of information defining the final morphology.-->
<!--Epithelial folding is a fundamental process of animal embryogenesis [@Wang2021-iy].-->
<!--Tissues fold through intrinsic cellular mechanisms or extrinsic tissue-scale forces [@Denk-Lobnig2020-pr].-->
<!--Some theories suggest that mechanical forces may be crucial to generate novelties in animal development [@Newman2000-oz].-->
<!--However, these ideas have not been tested empirically.-->
<!--However, the selective pressures and evolutionary events leading to the evolution of novel morphogenetic processes in early embryos are poorly understood.-->
<p>Cephalic furrow formation is under strict genetic control. In <em>Drosophila</em>, it is one of the earliest morphogenetic events of gastrulation, beginning as paired lateral indentations which quickly invaginate to form a deep epithelial fold at the boundary between the head (procephalon) and the trunk (germ band).<span class="citation" data-cites="Hartenstein1985-zs Foe1989-cw Spencer2015-td"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a>–<a href="#ref-Foe1989-cw" role="doc-biblioref">6</a></sup></span> The position of the cephalic furrow along the anteroposterior axis is determined with remarkable accuracy<span class="citation" data-cites="Liu2013-lc"><sup><a href="#ref-Liu2013-lc" role="doc-biblioref">7</a></sup></span> by the zygotic expression of two transcription factors, <em>buttonhead</em> (<em>btd</em>) and <em>even skipped</em> (<em>eve</em>), whose domains overlap at the head–trunk boundary by a narrow row of blastoderm cells.<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">8</a></sup></span> These so-called initiator cells shorten along the apical–basal axis through lateral myosin contractility.<span class="citation" data-cites="Eritano2020-mi"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a></sup></span> The shortening of initiators drives the initial infolding of the tissue while the mechanical coupling between cells ensures the propagation of a precise, bidirectional morphogenetic wave of tissue folding.<span class="citation" data-cites="Eritano2020-mi Popkova2024-eg"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a>,<a href="#ref-Popkova2024-eg" role="doc-biblioref">10</a></sup></span> The resulting fold spans the entire lateral surface, from dorsal to ventral, making the cephalic furrow a landmark of <em>Drosophila</em> gastrulation.<span class="citation" data-cites="Hartenstein1985-zs Foe1989-cw"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a>,<a href="#ref-Foe1989-cw" role="doc-biblioref">6</a></sup></span></p>
<p>Despite its prominence and underlying genetic control, the cephalic furrow has no obvious function during development. Unlike other embryonic invaginations such as the ventral furrow, which gives rise to mesodermal precursors, and the midgut invaginations, which give rise to endodermal tissues, the cephalic furrow does not give rise to any specific structure; it simply unfolds after a couple of hours, leaving no trace.<span class="citation" data-cites="Hartenstein1985-zs"><sup><a href="#ref-Hartenstein1985-zs" role="doc-biblioref">4</a></sup></span> While the cephalic furrow has been thought to serve as a temporary tissue storage,<span class="citation" data-cites="Turner1977-ol"><sup><a href="#ref-Turner1977-ol" role="doc-biblioref">11</a></sup></span> or as a tissue anchor during gastrulation,<span class="citation" data-cites="Costa1993-qj Dicko2017-wz"><sup><a href="#ref-Costa1993-qj" role="doc-biblioref">12</a>,<a href="#ref-Dicko2017-wz" role="doc-biblioref">13</a></sup></span> these hypotheses have not been investigated <em>in vivo</em> or considered in a phylogenetic context. Remarkably, recent evidence indicates that the cephalic furrow is an evolutionary novelty of some dipteran flies,<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> making it an ideal model for investigating how patterned morphogenetic processes in early embryonic development evolve.</p>
<!--Redowan2023-ew cephalic furrow formation model-->
<!--The invagination is only present in a derived group of cyclorraphan flies, but is absent in the remaining basally-branching dipteran flies [@Dey2023-mo].-->
<!--The clear origin in the phylogenetic tree makes the cephalic furrow an ideal model for investigating how patterned morphogenetic processes evolve.-->
<p>Our work integrates genetics and mechanics to uncover the developmental role and evolutionary origins of the cephalic furrow. First, we analyze how perturbing cephalic furrow formation impacts gastrulation in <em>Drosophila</em> by live imaging mutant embryos, and find that the absence of the cephalic furrow increases the mechanical instability of the blastoderm epithelium. Using a combination of <em>in vivo</em> experiments and <em>in silico</em> simulations, we identify the expansion of mitotic domains and germ band extension as the sources of mechanical stress and show that the cephalic furrow prevents this epithelial instability by absorbing compressive stresses. This suggests the cephalic furrow plays a mechanical role during gastrulation. Next, to uncover the changes in genetic patterning associated with the evolution of the cephalic furrow, we compared the expression of head–trunk genes between species with and without the cephalic furrow. While the expression of the patterning genes is mostly conserved, we find the <em>btd–eve</em> overlap, which is required for the cephalic furrow specification in <em>Drosophila</em>, is absent in species without the cephalic furrow. This suggests that changes in the expression of <em>btd</em> at the head–trunk boundary may be associated with the evolution of the cephalic furrow. Taken together, these data suggest that the evolution of the cephalic furrow patterning system occurred through the cooption of a novel genetic player, and that the underlying selective pressure may have been the mechanical instability during gastrulation. Our findings reveal an interplay between genetic patterning and mechanical forces during the evolution of morphogenesis in early development.</p>
<!--The cephalic furrow retains a significant number of blastoderm cells beneath the surface of the embryo during gastrulation (about 10% according to recent estimates [@Stern2021-zv]),-->
<!--The mechanisms controlling the formation of the cephalic furrow remain poorly understood [@Gilmour2017-um]-->
<!--But the interactions of the cephalic furrow with other morphogenetic movements have not been investigated *in vivo* [@Kong2017-ac].-->
<!--Current models for the cephalic furrow account for the mechanics of the invagination [@Eritano2020-mi; @Jeffery2023; @OlderModel], but not the global context of how it interacts with neighbouring tissues during gastrulation.-->
<h1 id="analyses-of-cephalic-furrow-mutants">Analyses of cephalic furrow mutants</h1>
<p>To understand the physical consequences of perturbing the formation of the cephalic furrow in <em>Drosophila</em>, we investigated the tissue dynamics at the head–trunk boundary in known cephalic furrow mutants—<em>btd</em>, <em>eve</em>, and <em>prd</em>.<span class="citation" data-cites="Vincent1997-fa Blankenship2001-tm"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">8</a>,<a href="#ref-Blankenship2001-tm" role="doc-biblioref">15</a></sup></span> We generated fluorescent lines carrying a loss of function allele of these genes, and imaged the embryos with high temporal resolution using <em>in toto</em> lightsheet microscopy, characterizing differences in timing and dynamics of developmental events during gastrulation (Fig. <a href="#fig:ectopic-folding">1</a>a).</p>
<!--*btd* and *eve*, where the invagination is absent [@Vincent1997-fa], and *prd* where the invagination is delayed [@Blankenship2001-tm].-->
<h2 id="formation-of-ectopic-folds">Formation of ectopic folds</h2>
<!--TODO: Rearrange figure panels to match text-->
<p>We first analyzed the behavior of initiator cells in the three mutant backgrounds. The typical shortening of initiators is perturbed in <em>prd</em> mutants and severely disrupted in <em>btd</em> and <em>eve</em> mutants (<a href="#fig:initiator">Extended Data Fig. 1</a>, <a href="#fig:vid-initiator">Supplementary Video 1</a>, Supplementary Note 1). The absence of initiator shortening in <em>btd</em> and <em>eve</em> is associated with a strong phenotype: the formation of late epithelial folds near the canonical site of cephalic furrow invagination (Fig. <a href="#fig:ectopic-folding">1</a>b,c, <a href="#fig:initiator">Extended Data Fig. 1</a>a). While the presence of late folds was already observed in <em>eve</em> mutants<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">8</a></sup></span> and, more recently, in <em>btd</em> mutants,<span class="citation" data-cites="Eritano2020-mi"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a></sup></span> their origin and relation to the cephalic furrow have not been investigated. Therefore, to understand the mechanisms causing these epithelial folds, we characterized their formation in different genetic backgrounds.</p>
<p>These ectopic folds, as they will be referred to from hereon, appear around the head–trunk boundary of <em>btd</em> and <em>eve</em> embryos, about 20 min after gastrulation when the germ band is already extended 35% of the egg length (Fig. <a href="#fig:ectopic-folding">1</a>a–e, <a href="#fig:vid-lateral-btd">Supplementary Video 2</a>, <a href="#fig:vid-lateral-eve">Supplementary Video 3</a>, Table <a href="#tbl:gbtime">1</a>). Ectopic folds can superficially resemble a cephalic furrow, but they lack the typical symmetric morphology of the wildtype invagination.<span class="citation" data-cites="Spencer2015-td"><sup><a href="#ref-Spencer2015-td" role="doc-biblioref">5</a></sup></span> Instead, ectopic folds in <em>btd</em> and <em>eve</em> mutants have a loose, often asymmetric cleft without wedge- or arched-shaped cells (Fig. <a href="#fig:ectopic-folding">1</a>b, <a href="#fig:vid-profile-btd">Supplementary Video 4</a>, <a href="#fig:vid-profile-eve">Supplementary Video 5</a>), and occupy <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mfrac><mn>1</mn><mn>4</mn></mfrac><annotation encoding="application/x-tex">\frac{1}{4}</annotation></semantics></math> of the area and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mfrac><mn>1</mn><mn>5</mn></mfrac><annotation encoding="application/x-tex">\frac{1}{5}</annotation></semantics></math> of the depth of the cephalic furrow (Fig. <a href="#fig:ectopic-folding">1</a>g,h, <a href="#fig:ectopic-features">Extended Data Fig. 2</a>b,f, Table <a href="#tbl:area">2</a>, Table <a href="#tbl:depth">3</a>). Ectopic folds also fold and unfold faster, lacking the typical kinetics of cephalic furrow formation (Fig. <a href="#fig:ectopic-folding">1</a>a,b, <a href="#fig:initiator">Extended Data Fig. 1</a>a, <a href="#fig:ectopic-features">Extended Data Fig. 2</a>c–e, <a href="#fig:vid-mitotic-fold">Supplementary Video 6</a>). Finally, the formation of ectopic folds is more variable than that of the cephalic furrow, as their position along the head–trunk boundary differs between individual mutant embryos (Fig. <a href="#fig:ectopic-folding">1</a>f, <a href="#fig:ectopic-features">Extended Data Fig. 2</a>a, <a href="#fig:ectopic-variability">Supplementary Fig. 1</a>, <a href="#fig:vid-dynamics-btd">Supplementary Video 7</a>, <a href="#fig:vid-variability-btd">Supplementary Video 8</a>). Altogether, these differences in morphology, kinetics, timing, position, and variability, suggest that ectopic folds and cephalic furrow form via distinct mechanisms.</p>
<!--TODO: Revise this paragraph-->
<!--TODO: Move paragraph to supplement?-->
<p>When applying our ectopic folding scoring pipeline, we noted that heterozygote and wildtype embryos can also form ectopic folds located anterior or posterior to the cephalic furrow. The frequency of ectopic folding in these embryos is lower than in homozygotes, but not negligible. While nearly every <em>btd</em> and <em>eve</em> homozygote embryos (&gt;92%) show one or more ectopic fold per side (2.2±0.4 and 1.8±0.6, respectively) (<a href="#fig:ectopic-features">Extended Data Fig. 2</a>h, <a href="#fig:vid-variability-btd">Supplementary Video 8</a>), between 18–27% of heterozygotes and about 78% of wildtype embryos have one ectopic fold in addition to the cephalic furrow (<a href="#fig:ectopic-features">Extended Data Fig. 2</a>i,j, Table <a href="#tbl:freq">4</a>). However, we find a fundamental difference. The area of the ectopic folds in heterozygote and wildtype embryos is significantly smaller (about 4x) compared to the area of ectopic folds in <em>btd</em> and <em>eve</em> embryos (<a href="#fig:ectopic-features">Extended Data Fig. 2</a>g,k–n, Table <a href="#tbl:area">2</a>). This data shows the head–trunk interface of <em>Drosophila</em> during gastrulation is a region prone for the formation of ectopic folds and provides evidence that the absence of the cephalic furrow leads to an increase in the magnitude of these folding events.</p>
<!--while *prd* homozygotes have a relatively lower frequency (71.4%) with 1.9±0.8 ectopic folds per embryo side-->
<!--TODO:AB: Quantify the variability of ectopic folds in some way (plot it)-->
<h2 id="evidence-of-tissue-compression">Evidence of tissue compression</h2>
<!--TODO: Rearrange figure panels to match text-->
<p>The variability in ectopic folding suggests that, unlike the cephalic furrow, the ectopic folds are not under genetic control and form as the result of physical interactions in the tissue. Our analysis shows that the formation of ectopic folds coincides spatially and temporally with two other processes of gastrulation: the expansion of mitotic domains and the extension of the germ band (Fig. <a href="#fig:ectopic-folding">1</a>a,c).</p>
<p>Mitotic domains are groups of blastoderm cells that divide in synchrony during nuclear cycle 14, first appearing 20 min after gastrulation on the embryo’s head.<span class="citation" data-cites="Foe1989-cw"><sup><a href="#ref-Foe1989-cw" role="doc-biblioref">6</a></sup></span> Our analysis of the ectopic folds in <em>btd</em> and <em>eve</em> mutants shows that they form between mitotic domains or between the posterior Mitotic Domain 6 (MD6) and the extending germ band (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>a,b). When mitotic cells begin to divide, they lose their basal attachment, round up at the apical side, and more than double their apical area during anaphase (<a href="#fig:apical-strain">Supplementary Fig. 2</a>a). This apical expansion compresses the adjacent, non-dividing cells, which are the first to fold inward (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>b). Mitotic expansions always precede ectopic folding (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>c,d, <a href="#fig:initiator">Extended Data Fig. 1</a>a, Table <a href="#tbl:gbtime">1</a>). This suggests the formation of mitotic domains may generate buckling instability in the monolayer epithelium, contributing to the appearance of ectopic folds.</p>
<!--TODO: Add pvalues for MD/EF timing somewhere?-->
<p>To estimate the forces acting on the buckling tissue, we measured the rate of tissue deformation (strain rate) at the head–trunk and trunk–germ boundaries using particle image velocimetry (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>e). Control embryos exhibit a peak of strain rate around the head–trunk interface that correlates with late phase of the cephalic furrow invagination, when the initiator cells move into the yolk (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>f, <a href="#fig:apical-strain">Supplementary Fig. 2</a>b, <a href="#fig:vid-strain-rate">Supplementary Video 9</a>). This is absent in <em>btd</em> mutants. Instead, the mutants show a higher peak of strain rate that coincides with the maximum expansion of the mitotic domains during telophase and ectopic folding (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>f, <a href="#fig:apical-strain">Supplementary Fig. 2</a>b, <a href="#fig:vid-strain-rate">Supplementary Video 9</a>). This suggests the tissue deformation is associated with the mitotic expansions.</p>
<!--TODO: Revise this paragraph-->
<p>At the trunk–germ interface, the cells between MD6 and the extending germ band become increasingly anisotropic (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>e). We find that the strain rate in this region steadily increases over time (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>f), suggesting that the tissue is under compression. To test this hypothesis, we performed laser cuts at the trunk–germ interface in wildtype embryos. We ablated the apical membrane of multiple cells (3–4) with cuts oriented orthogonal to the direction of the germ band extension, and then tracked the distance between non-ablated cells on each side of the cut (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>g). This distance remains constant in uncut control embryos, but decreases over the first five minutes after the cut (<a href="#fig:mitotic-strain">Extended Data Fig. 3</a>g). This is evidence that the tissue may be “collapsing on itself” after the ablation, which supports the hypothesis that the trunk–germ interface is under compression from the extending germ band.</p>
<p>Taken together, these analyses suggest the mitotic expansions and germ band extension are potential sources of compressive stress capable of inducing tissue buckling, and thus could contribute to the formation of ectopic folds at the head–trunk boundary during gastrulation.</p>
<!--TODO:AB: Are there force measurements?-->
<!--TODO: Include cell shape analyses with density et al. around here-->
<!--TODO: Quantify compression of trunk-germ cells in btd mutants-->
<h2 id="mitotic-domains-and-germ-band">Mitotic domains and germ band</h2>
<p>To test the individual role of mitotic domains and germ band to the mechanical stability of the blastoderm, we performed a series of perturbation experiments <em>in vivo</em>.</p>
<p>We first asked if mitotic expansions are required for the formation of ectopic folds. To that end, we generated a double-mutant line lacking both cephalic furrow and mitotic domains, using the loss-of-function alleles of <em>btd</em> and <em>string</em> (<em>stg</em>)—the <em>cdc25</em> phosphatase ortholog that regulates the formation of mitotic domains in <em>Drosophila</em>.<span class="citation" data-cites="Edgar1989-hv"><sup><a href="#ref-Edgar1989-hv" role="doc-biblioref">16</a></sup></span> The absence of mitotic domains in <em>stg</em> mutants does not affect the formation of the cephalic furrow or other morphogenetic movements of gastrulation<span class="citation" data-cites="Edgar1989-hv"><sup><a href="#ref-Edgar1989-hv" role="doc-biblioref">16</a></sup></span> (Fig. <a href="#fig:live-experiments">2</a>a, <a href="#fig:live-details">Extended Data Fig. 4</a>a,b, <a href="#fig:vid-stg-lateral">Supplementary Video 10</a>, <a href="#fig:vid-stg-dorsal">Supplementary Video 11</a>). However, the absence of mitotic domains suppresses the formation of ectopic folds in <em>btd</em>–<em>stg</em> double-mutants; these embryos show no ectopic folds at their head–trunk interface (Fig. <a href="#fig:live-experiments">2</a>b,c, <a href="#fig:vid-double-lateral">Supplementary Video 12</a>, <a href="#fig:vid-double-dorsal">Supplementary Video 13</a>). This indicates that mitotic expansions are necessary for the appearance of ectopic folds in cephalic furrow mutants.</p>
<p>Next, we asked if the extension of the germ band is required for the formation of ectopic folds. To prevent the germ band from extending, we cauterized a patch of posterodorsal tissue at the onset of gastrulation to mechanically attach it to the vitelline envelope (Fig. <a href="#fig:live-experiments">2</a>d,f). Blocking the germ band in wildtype embryos does not prevent the formation of the cephalic furrow (Fig. <a href="#fig:live-experiments">2</a>d,e), as the shortening of the initiator cells occurs normally and correlates with an increase in tension around the head–trunk boundary (Fig. <a href="#fig:live-experiments">2</a>i). We only detect a mild influence of the germ band on the final depth of the invagination, since the cephalic furrow in cauterized embryos is about 15% shallower than in control embryos (Fig. <a href="#fig:live-experiments">2</a>e,j). This data corroborates the understanding that the initial formation of the cephalic furrow is an active process independent of other morphogenetic movements</p>
<p>We then performed the cauterization experiment in <em>btd</em> and <em>eve</em> embryos. When the germ band extension is blocked in these mutants, no ectopic folds appear at the head–trunk interface (Fig. <a href="#fig:live-experiments">2</a>f, <a href="#fig:vid-cautlat-eve">Supplementary Video 14</a>, <a href="#fig:vid-cautdor-eve">Supplementary Video 15</a>, <a href="#fig:vid-cautdor-btd">Supplementary Video 16</a>), and their epithelium undergoes less deformation and buckling events compared to non-cauterized mutant embryos (Fig. <a href="#fig:live-experiments">2</a>h,k). The expansion of cell apices in mitotic domains still compress the neighboring, non-dividing cells, but no buckling occurs (Fig. <a href="#fig:live-experiments">2</a>g). These experiments reveal that the germ band extension is necessary for the appearance of ectopic folds in cephalic furrow mutants.</p>
<p>Overall, we conclude that neither mitotic expansions nor the germ band extension can induce ectopic folds individually, but when both events happen concomitantly, the epithelial monolayer becomes unstable and buckles.</p>
<!--Conversely, the experiment reveals the compression exerted by the germ band extension alone is insufficient to induce ectopic folding.-->
<!--and, conversely, that mitotic expansions alone are insufficient to cause epithelial buckling at the head--trunk interface.-->
<!--Mitotic activity and germ band elongation contributes to the depth of the cephalic furrow [@Campos-Ortega1997-rd].-->
<h1 id="physical-model-of-folding-dynamics">Physical model of folding dynamics</h1>
<p>To determine the relative contribution of mitotic domains and germ band as sources of mechanical stress on the head–trunk boundary, we created a physical model of the blastoderm and ran <em>in silico</em> simulations of the tissue mechanics in mutant and wildtype conditions.</p>
<h2 id="design-and-general-properties">Design and general properties</h2>
<p>Our model represents an epithelial monolayer confined inside a rigid shell (Fig. <a href="#fig:fold-model">3</a>a, Fig. <a href="#fig:model-features">Extended Data Fig. 5</a>a, Supplementary Note 2). The monolayer is composed of equidistant particles connected by springs. Mitotic domains are regions with a higher density of particles, which tend expand; the cephalic furrow is a region with an intrinsic negative curvature, which tends to invaginate; and the germ band is a static limit to the posterior end of the monolayer (Fig. <a href="#fig:fold-model">3</a>b). In this tissue, the total energy per unit length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>T</mi></msub><annotation encoding="application/x-tex">W_T</annotation></semantics></math>) is the sum of stretching (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>s</mi></msub><annotation encoding="application/x-tex">W_s</annotation></semantics></math>) and bending (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>b</mi></msub><annotation encoding="application/x-tex">W_b</annotation></semantics></math>) energy components, each associated with a stretching (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>s</mi></msub><annotation encoding="application/x-tex">K_s</annotation></semantics></math>) and a bending (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math>) rigidity. We combined these two parameters into a single, dimensionless bending rigidity (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math>), and used it as the main parameter in our model and simulations (Fig. <a href="#fig:fold-model">3</a>c).</p>
<p>To characterize the energy dynamics of the system, we ran simulations using a single bending rigidity value and percentage of germ band extension. When a fold begins to form, the bending energy increases, releasing a large amount of stretching energy, which, decreases the total energy of the system (Fig. <a href="#fig:fold-model">3</a>d). The increase in bending energy coincides with a rapid deepening of the fold. Once the bending energy reaches a peak, the fold continues to deepen more gradually, but the number of folds rarely changes afterward (Fig. <a href="#fig:fold-model">3</a>d, <a href="#fig:model-features">Extended Data Fig. 5</a>b). As folding events are stochastic, happening at distinct iterations in each simulation, we used this peak of bending energy as a reference point to standardize the comparison between simulations.</p>
<!--($K_{b}^{\text{*}} = 0.8 \times 10^{−4}$)-->
<!--To run the simulations, we defined a ground level of random noise and iterated towards the total energy equilibrium (see Methods).-->
<!--We then set several simulations with different bending rigidity values and combinations of presence/absence of mitotic domains, cephalic furrow, and percentages of germ band extension to quantify the position, frequency, and depth of ectopic folding in the epithelium.-->
<!--Without the germ band, the tissue is almost stress-free and no ectopic folding occurs ({@fig:fold-model}e).-->
<!--We begin to observe folds in the simulations with higher progression of germ band extension.-->
<h2 id="ectopic-folding-in-mutant-conditions">Ectopic folding in mutant conditions</h2>
<!--TODO:AB: Describe overfolding events. Mark them on phase diagram. Discuss as limitation-->
<p>To obtain realistic values of the dimensionless bending rigidity <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math>, we performed a sweep across the parameter space for the cephalic furrow mutant conditions. In general, we find that the number of ectopic folds is higher in softer conditions, where the bending rigidity is lower (Fig. <a href="#fig:fold-model">3</a>e,f, <a href="#fig:model-features">Extended Data Fig. 5</a>c,f). In simulations without mitotic domains, we observe no folding events without the germ band (Fig. <a href="#fig:fold-model">3</a>e, column i). With the extension of the germ band, the probability of buckling increases (<a href="#fig:model-features">Extended Data Fig. 5</a>d) and the time to folding decreases (<a href="#fig:model-features">Extended Data Fig. 5</a>e). The parameter sweep shows a clear transition in the phase space with the buckling probability reaching a plateau around <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>≈</mo><mn>1.2</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} \approx 1.2 \times 10^{−4}</annotation></semantics></math> (Fig. <a href="#fig:fold-model">3</a>e). In these stiffer conditions, the germ band, even at its maximum extension, cannot drive the formation of ectopic folds.</p>
<p>Adding mitotic domains to the simulations changed the phase diagram, increasing the probability of folding. We find that, in softer conditions (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>≤</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} \leq 1.0 \times 10^{−4}</annotation></semantics></math>), mitotic domains alone can induce ectopic folding (Fig. <a href="#fig:fold-model">3</a>f, column iii). For lower values of germ band extension, the number of folds is higher (<a href="#fig:model-features">Extended Data Fig. 5</a>g) and the time to folding is lower (<a href="#fig:model-features">Extended Data Fig. 5</a>h) compared to the simulations without mitotic domains. These simulations reveal that, depending on the bending rigidity, the germ band or mitotic domains alone can drive ectopic folding, and that their combined action increases the mechanical instability of the tissues.</p>
<p>To determine where our embryo lies in this parameter space, we considered the insight from our experimental data that neither mitotic domain nor germ band can promote ectopic folding by themselves. With this information, we identified the bending rigidity value that recapitulates these biological behaviors in the simulations. For that, we determined at which bending rigidity the average number of folds falls below 1 in mitotic-domains-only and germ-band-only conditions (<a href="#fig:model-features">Extended Data Fig. 5</a>c,f). The criterion is fulfilled when the bending rigidity is <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>≈</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} \approx 1.0 \times 10^{−4}</annotation></semantics></math>.</p>
<p>To compare this value with existing direct measurements, we calculated <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> for the bending rigidity estimates in cultured cells (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>b</mi></msub><mo>≈</mo><mn>5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>13</mn></mrow></msup><mi>N</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">K_{b} \approx 5 \times 10^{-13} Nm</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>s</mi></msub><mo>≈</mo><mn>0.2</mn><mi>N</mi><msup><mi>m</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{s} \approx 0.2 Nm^{-1}</annotation></semantics></math>)<span class="citation" data-cites="Trushko2020-gf Fouchard2020-km"><sup><a href="#ref-Trushko2020-gf" role="doc-biblioref">17</a>,<a href="#ref-Fouchard2020-km" role="doc-biblioref">18</a></sup></span> (Supplementary Note 3). Given that in elastic sheets the bending and stretching rigidity ratio is proportional to the square of the height (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>b</mi></msub><mi>/</mi><msub><mi>K</mi><mi>s</mi></msub><mo>∝</mo><msup><mi>h</mi><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_{b}/K_{s} \propto h^2</annotation></semantics></math>),<span class="citation" data-cites="Efrati2009-ve"><sup><a href="#ref-Efrati2009-ve" role="doc-biblioref">19</a></sup></span> we adjusted the calculation to the thickness of the epithelial monolayer in culture (~18 µm).<span class="citation" data-cites="Fouchard2020-km"><sup><a href="#ref-Fouchard2020-km" role="doc-biblioref">18</a></sup></span> The value we obtained in this way for the effective <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> of a height-adjusted epithelial monolayer, <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mo>≈</mo><mn>1.8</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">\approx 1.8 \times 10^{-4}</annotation></semantics></math>, is within a factor of 2 of our own reference bending rigidity, indicating that the relevant region of parameter space we identified in our model is consistent with existing direct measurements of bending rigidity in other tissues. This value establishes an important bridge between our <em>in vivo</em> and <em>in silico</em> data by revealing where the embryo resides within the parameter space of the model, and by providing a biologically relevant reference for subsequent simulations.</p>
<!--The simulations show that the average depth of the cephalic furrow correlates with the percentage of germ band extension ({@fig:fold-model}i).-->
<!--TODO: Move definition of a fold to methods-->
<!--TODO: Add panel with the threshold for counting folds-->
<h2 id="role-of-the-cephalic-furrow">Role of the cephalic furrow</h2>
<p>The formation of ectopic folds in cephalic furrow mutants led us to hypothesize that the wildtype invagination may contribute to absorbing the compressive stresses generated by mitotic domains and germ band in normal conditions. To explore the role of the cephalic furrow as a mechanical buffer, we analyzed how the presence of the invagination affects the dynamics of ectopic folding in the simulations.</p>
<p>We programmed the cephalic furrow in our model by establishing an intrinsic negative curvature (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math>) to a narrow region of the particle–spring blastoderm that matches the span of the initiator cells <em>in vivo</em> (Fig. <a href="#fig:fold-model">3</a>a,b). Using our reference bending rigidity value (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} = 1.0 \times 10^{−4}</annotation></semantics></math>), we ran a parameter sweep for different <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> and established a baseline (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><mo>&gt;</mo><mn>0.3</mn></mrow><annotation encoding="application/x-tex">\kappa_o^{CF}&gt;0.3</annotation></semantics></math>) where the invagination forms in a robust manner with minimal variability, phenocopying the cephalic furrow <em>in vivo</em> (<a href="#fig:model-features">Extended Data Fig. 5</a>i).</p>
<p>We first evaluated how the strength of the cephalic furrow impacts the frequency and placement of ectopic folding. In conditions without the germ band (<em>g</em>=0), we find that an active invagination reduces the spread and frequency of ectopic folding at the head–trunk boundary (Fig. <a href="#fig:fold-model">3</a>g, <a href="#fig:model-features">Extended Data Fig. 5</a>l) and that this reduction in variability correlates with the strength of the pull (<a href="#fig:model-features">Extended Data Fig. 5</a>j). In conditions with an extended germ band, however, this buffering effect was diminished due to an increase in ectopic folding at the posterior region, which dominates the landscape and even inhibits the formation of the cephalic furrow in embryos with <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><mo>≤</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">\kappa_o^{CF} \leq 1</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>≥</mo><mn>0.2</mn></mrow><annotation encoding="application/x-tex">g \geq 0.2</annotation></semantics></math> (<a href="#fig:model-features">Extended Data Fig. 5</a>j). Nevertheless, in general, these simulations show that higher values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are more effective at preventing ectopic folds.</p>
<p>Since the cephalic furrow forms around 15 min before mitotic domains in wildtype embryos, we asked whether their relative timing of formation may influence the effectiveness of the cephalic furrow. In contrast to the simulations above where the cephalic furrow and mitotic domains form at the same time, we added a delay to the formation of mitotic domains to match the <em>in vivo</em> characteristics. In this condition, the cephalic furrow is more effective in preventing ectopic folding, even for lower <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> values (Fig. <a href="#fig:fold-model">3</a>g, <a href="#fig:model-features">Extended Data Fig. 5</a>j,k,l). Similar to our <em>in vivo</em> observations of ectopic folding in wildtype embryos (<a href="#fig:ectopic-features">Extended Data Fig. 2</a>i–m, Table <a href="#tbl:freq">4</a>), we also observe some ectopic folding in these simulations, especially at greater percentages of germ band extension (<a href="#fig:model-features">Extended Data Fig. 5</a>k). These experiments reveal that relative timing, rather than force, is more important at preventing ectopic folding.</p>
<p>Finally, we tested how the position of the cephalic furrow impacts its ability to prevent ectopic folding. When the germ band is not extended (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0.0</mn></mrow><annotation encoding="application/x-tex">g=0.0</annotation></semantics></math>), a cephalic furrow positioned anteriorly (&gt;70% embryo length) effectively prevents ectopic folding in the head (Fig. <a href="#fig:fold-model">3</a>h), similar to the simulations at the wildtype position. However, when the cephalic furrow is positioned more posteriorly (&lt;50% embryo length), ectopic folding around mitotic domains becomes more frequent. Conversely, when the germ band is extended (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0.4</mn></mrow><annotation encoding="application/x-tex">g=0.4</annotation></semantics></math>), a posteriorly located cephalic furrow (&lt;30% embryo length) prevents ectopic folding in the posterior region, while an anteriorly located invagination (&gt;50% embryo length) fails to do so (Fig. <a href="#fig:fold-model">3</a>h). These simulations show that when the cephalic furrow is positioned between 40-60% embryo length, it is the most effective at preventing ectopic folding throughout the different phases of germ band extension.</p>
<p>Taken together, our physical model provides a theoretical basis that a fold such as the cephalic furrow—by forming before other morphogenetic events around the central region of the anteroposterior axis—can absorb compressive stresses and prevent, to a substantial degree, mechanical instabilities in embryonic tissues during gastrulation.</p>
<!--TODO: Add full parameter sweeps as supplementary notes-->
<!--TODO: Timing effectiveness is not exemplified in the last panel-->
<!--TODO: Add evolutionary question here or after genes?-->
<h1 id="evolution-of-gene-expression">Evolution of gene expression</h1>
<p>As described above, our analyses suggest that the effectiveness of the cephalic furrow in preventing epithelial instabilities depends on the position and time of the invagination. In <em>Drosophila</em>, this spatiotemporal control is determined genetically by the combinatorial expression of <em>btd</em>, <em>eve</em>, and <em>prd</em> at the head–trunk boundary.<span class="citation" data-cites="Vincent1997-fa Blankenship2001-tm"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">8</a>,<a href="#ref-Blankenship2001-tm" role="doc-biblioref">15</a></sup></span> However, it remains unclear which elements of this genetic patterning cascade are associated with the evolution of the cephalic furrow. To address this question, we first sought to identify additional genes involved in cephalic furrow formation, and then asked how the expression of these cephalic furrow genes compare between dipteran species with and without the cephalic furrow.</p>
<h2 id="patterning-of-the-cephalic-furrow">Patterning of the cephalic furrow</h2>
<!--TODO: Rearrange figure panels to match text-->
<!--TODO: Move slp from genetic-patterning and genetic-features-->
<!--TODO: Create additional genetic-mutants EDfig-->
<p>We performed a live-imaging screen (Supplementary Note 4) and found that the <em>sloppy paired</em> (<em>slp</em>) transcription factors plays a role in the positioning of initiator cells (<a href="#fig:slp-analyses">Extended Data Fig. 6</a>, <a href="#fig:vid-slp-lateral">Supplementary Video 17</a>, Supplementary Note 5).</p>
<p>By analyzing the expression of <em>slp1</em> relative to <em>btd</em>, <em>eve</em>, and <em>prd</em> in wildtype <em>Drosophila</em> embryos, we find that the early <em>slp1</em> and <em>eve</em> expression domains demarcate the head–trunk boundary from the onset of zygotic activation until gastrulation (Fig. <a href="#fig:genetic-patterning">4</a>a, <a href="#fig:genetic-features">Extended Data Fig. 7</a>a). Early <em>slp1</em> transcripts are limited to the anterior end while <em>eve</em> transcripts, which are initially ubiquitous,<span class="citation" data-cites="Andrioli2012-fv"><sup><a href="#ref-Andrioli2012-fv" role="doc-biblioref">20</a></sup></span> begin to clear from the anterior end at cycle 11, and become limited to the posterior region of the body from cycle 12 (<a href="#fig:genetic-features">Extended Data Fig. 7</a>a). At this stage, the two genes form broad, complementary territories that correspond to the head and trunk regions of the embryo, with the domains juxtaposed at ~70% of the embryo length (<a href="#fig:genetic-features">Extended Data Fig. 7</a>a). We first detect <em>btd</em> and <em>prd</em> transcripts at this interface (<a href="#fig:genetic-features">Extended Data Fig. 7</a>a,b). During subsequent stages, the <em>slp1</em>–<em>eve</em> boundary progressively resolves into narrow abutting stripes giving rise to the row of initiator cells (Fig. <a href="#fig:genetic-patterning">4</a>a, <a href="#fig:genetic-features">Extended Data Fig. 7</a>a). Altogether, the data suggests that <em>slp1</em> activity contributes to restricting the anterior boundary of <em>eve</em> expression during early stages of zygotic activation, an interaction that determines the site of invagination of the cephalic furrow along the anteroposterior axis.</p>
<p>At the onset of gastrulation, the expression of <em>btd</em>, <em>eve</em>, <em>slp</em>, and <em>prd</em> at the head–trunk boundary of <em>Drosophila</em>, forms a unique combinatorial code that coincides with the different portions of the cephalic furrow (Fig. <a href="#fig:genetic-patterning">4</a>b, <a href="#fig:genetic-features">Extended Data Fig. 7</a>c). The central row of <em>eve</em>-expressing initiator cells are surrounded by <em>slp1</em>-expressing adjacent cells, with <em>prd</em> expression offset by a single row of cells relative to the inner <em>slp1</em> stripes (Fig. <a href="#fig:genetic-patterning">4</a>b–d, <a href="#fig:genetic-features">Extended Data Fig. 7</a>b,c). Moreover, <em>slp1</em>-expressing cells also demarcate the outer edges of the invagination (Fig. <a href="#fig:genetic-patterning">4</a>d). This molecular arrangement is disrupted in mutants that exhibit cephalic furrow defects (<em>btd</em>, <em>eve</em>, and <em>prd</em>) (<a href="#fig:mutant-expression">Extended Data Fig. 8</a>). The combinatorial expression suggests that each row has a unique transcriptional identity and that this specific molecular profile is important for the patterning and morphogenesis of the invagination in <em>Drosophila</em> (Fig. <a href="#fig:genetic-patterning">4</a>e).</p>
<!--TODO: Add supplementary note commenting on mutant expression?-->
<!--Since *slp* is crucial to maintain parasegment polarity by activating *wingless* and repressing *engrailed* [@Cadigan1994-hz], the expression of *slp* may also be important for establishing the individual cell identities around the cephalic furrow.-->
<h2 id="innovation-at-the-headtrunk-boundary">Innovation at the head–trunk boundary</h2>
<p>To uncover the differences in the head–trunk genetic patterning associated with the evolution of the cephalic furrow, we compared the expression patterns in <em>Drosophila</em> to three other dipteran species. One species, the Mediterranean fruit fly <em>Ceratitis capitata</em> (Tephritidae),<span class="citation" data-cites="Strobl2022-og Strobl2024-kh"><sup><a href="#ref-Strobl2022-og" role="doc-biblioref">21</a>,<a href="#ref-Strobl2024-kh" role="doc-biblioref">22</a></sup></span> belongs to a family known to form a cephalic furrow. The other two species, the malaria mosquito <em>Anopheles stephensi</em> (Culicidae)<span class="citation" data-cites="Cheatle-Jarvela2020-go Cheatle-Jarvela2023-gd"><sup><a href="#ref-Cheatle-Jarvela2020-go" role="doc-biblioref">23</a>,<a href="#ref-Cheatle-Jarvela2023-gd" role="doc-biblioref">24</a></sup></span> and the drain fly <em>Clogmia albipunctata</em> (Psychodidae),<span class="citation" data-cites="Jimenez-Guri2014-zs Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a>,<a href="#ref-Jimenez-Guri2014-zs" role="doc-biblioref">25</a></sup></span> belong to families where the cephalic furrow has not been observed.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span></p>
<p>The three species show early, juxtaposing domains of <em>slp</em> and <em>eve</em> demarcating the head and trunk regions in a pattern highly similar to that of <em>Drosophila</em> (Fig. <a href="#fig:genetic-patterning">4</a>a,f, <a href="#fig:genetic-comparative">Extended Data Fig. 9</a>a,b,c). Moreover, the late pattern of abutting <em>slp</em> and <em>eve</em> stripes is nearly identical between the four species (Fig. <a href="#fig:genetic-patterning">4</a>a,f,g). Only the expression of <em>prd</em> in <em>Clogmia</em> differs from <em>Drosophila</em>, since the <em>prd</em>-expressing cells are not offset from <em>slp</em> and <em>eve</em> (<a href="#fig:genetic-comparative">Extended Data Fig. 9</a>d,e) These observations suggest that the genetic interactions that establish the head–trunk boundary position in <em>Drosophila</em>, and much of the late patterning, might be conserved in other dipteran species.</p>
<p>However, we identified a key difference in the expression of <em>btd</em> between species with and without the cephalic furrow. While the head–trunk domain of <em>btd</em> overlaps with <em>eve</em> stripe 1 in <em>Drosophila</em> and <em>Ceratitis</em>, it does not overlap with <em>eve</em> in <em>Anopheles</em> and is entirely absent in <em>Clogmia</em> (Fig. <a href="#fig:genetic-patterning">4</a>g,h). Moreover, the <em>btd</em>–<em>eve</em> overlap is absent in <em>Chironomus</em>, another dipteran that lacks a cephalic furrow.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> This comparative expression data suggests the evolution of the cephalic furrow may have been associated with changes in the localization of <em>btd</em> along the head–trunk boundary and emergence of a <em>btd</em>–<em>eve</em> overlap in this region. The establishment of this regulatory framework may have been essential to stabilize the cephalic furrow as a patterned morphogenetic process, and thus enable its mechanical role in absorbing compressive stresses and preventing epithelial instabilities during dipteran gastrulation.</p>
<!--from early stages and a more anterior domain localized in the acron is activated at the onset of gastrulation ({@fig:genetic-patterning}f).-->
<!--In *Clogmia*, the *btd* acron domain is expressed much earlier and, strikingly, the head--trunk domain is not present; *btd* is only expressed at a ventral patch in the foregut area ({@fig:genetic-patterning}f,h).-->
<!--Additionally, the combinatorial expression of *slp*, *eve*, and *prd* at the onset of gastrulation is conserved between *Clogmia* and *Drosophila* ({@fig:genetic-patterning}j).-->
<!--These data shows that both flies share a conserved head--trunk patterning system, and that the only significant difference is the presence of a head--trunk domain of *btd* in *Drosophila*.-->
<!--Since *btd* is a crucial gene for cephalic furrow formation, this suggests that the origin of a novel expression domain of *btd* at the head--trunk boundary was a key event associated with the evolution of the cephalic furrow.-->
<!--Clogmia lacks the head--trunk domain of btd-->
<!--- As an aside, curiously, the eye domain of btd is expressed very early in Clogmia while the same domain is delayed tremendously in Drosophila. Clogmia has larval eyes while Drosophila doesn’t. It could be that some portions of head development in Drosophila were just turned off.-->
<!--It is probable that the expansion of btd along the head--trunk was or still is or originated with some other role in the patterning of the head, and that only later on it got coopted into the cephalic furrow program.-->
<!--Could it be that this is a re-use of the anterior midgut program? There btd and eve overlap and there is an invagination. I wonder if it is also driven by lateral myosin.-->
<!--Clogmia is the sister group to Megaselia+Episyrphus+Drosophila [@Jimenez-Guri2013]-->
<!--Compare the dorsoventral patterning between Clogmia and Drosophila [@Wotton2017]. Apparently, Clogmia is ventralized while Drosophila dorsalized. The origin of this head--trunk btd domain could be that it expanded dorsally with the dorsalization.-->
<h1 id="discussion">Discussion</h1>
<!--TODO: Reduce discussion a lot-->
<p>Our work investigates the developmental function and patterning evolution of the cephalic furrow—an epithelial invagination that forms at the head–trunk boundary of dipteran flies. We find that without the cephalic furrow, the head–trunk tissues become unstable during gastrulation and buckle due to compressive stresses exerted by the expansion of mitotic domains and the germ band extension.</p>
<p>Compression is a key mechanism driving the formation of epithelial folds in development,<span class="citation" data-cites="Nelson2016-gz Collinet2021-qa"><sup><a href="#ref-Collinet2021-qa" role="doc-biblioref">3</a>,<a href="#ref-Nelson2016-gz" role="doc-biblioref">26</a></sup></span> particularly when tissues are under confinement.<span class="citation" data-cites="Trushko2020-gf"><sup><a href="#ref-Trushko2020-gf" role="doc-biblioref">17</a></sup></span> Cell divisions can increase epithelial instability due to the in-plane, outward forces generated during the elongation phase<span class="citation" data-cites="Gupta2021-oo"><sup><a href="#ref-Gupta2021-oo" role="doc-biblioref">27</a></sup></span> and to the imbalance caused by the basal detachment of dividing cells.<span class="citation" data-cites="Ko2020-wk"><sup><a href="#ref-Ko2020-wk" role="doc-biblioref">28</a></sup></span> This is the case for the tracheal placode of flies<span class="citation" data-cites="Kondo2013-gy"><sup><a href="#ref-Kondo2013-gy" role="doc-biblioref">29</a></sup></span> and intestinal villi of mice,<span class="citation" data-cites="Freddo2016-et"><sup><a href="#ref-Freddo2016-et" role="doc-biblioref">30</a></sup></span> where mitotic rounding induces the formation of epithelial folds. Interestingly, the folding in these tissues only occur when the epithelium is under compression, similar to our findings that only the combined action of mitotic expansions and germ band extension can induce ectopic folds (Fig. <a href="#fig:live-experiments">2</a>). Moreover, complementary experiments performed in an independent study corroborate the role of mitotic domains and the germ band as sources of mechanical stress.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> The consolidated data, therefore, indicates that the head–trunk tissues of flies are under increased compressive stress during gastrulation. We provide evidence that the cephalic furrow counteracts these stresses. An early invagination effectively reduces the appearance of ectopic folds <em>in vivo</em> and in simulations, supporting the hypothesis that the cephalic furrow prevents the build up of compressive stresses at the head–trunk boundary, and thus, accomplishes a mechanical role during <em>Drosophila</em> gastrulation.</p>
<!--The fact that *eve* mutants have no cell intercalation [@Collinet2015-af] suggests that the formation of ectopic folds depends on the rapid phase of germ band extension [@Campos-Ortega1985-ma], driven mainly by the posterior midgut invagination.-->
<!--This raises an intriguing evolutionary question---has the cephalic furrow evolved in response to mechanical instability?-->
<p>This physical role raises the intriguing idea that the cephalic furrow evolved as a response to mechanical instability. In this case, we expect that increased epithelial instabilities might have a detrimental effect on embryonic development and potentially decrease the fitness of individuals. Mechanical compression can trigger ATP release,<span class="citation" data-cites="Bao2004-dx"><sup><a href="#ref-Bao2004-dx" role="doc-biblioref">31</a></sup></span> calcium signaling,<span class="citation" data-cites="Locovei2006-qe"><sup><a href="#ref-Locovei2006-qe" role="doc-biblioref">32</a></sup></span> and even DNA damage<span class="citation" data-cites="Shah2021-gh"><sup><a href="#ref-Shah2021-gh" role="doc-biblioref">33</a></sup></span> in mechanosensitive cells. Moreover, variable tissue buckling can disrupt short-range signaling, cell-to-cell interactions, or potentially slow down embryogenesis at the head–trunk boundary. Investigating these effects <em>in vivo</em> is challenging, and our efforts to prevent cephalic furrow formation without affecting other developmental processes have failed so far. However, there is evidence that inhibiting the invagination via optogenetics increases the frequency of a distorted ventral midline,<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> suggesting that mechanical instability can impact the robustness of morphogenetic processes. In this sense, the appearance of a patterned head invagination may have improved developmental robustness and speed and been positively selected during evolution.</p>
<p>The distribution of cephalic-furrow-related traits onto the dipteran phylogeny is consistent with the hypothesis of mechanical instability as a selective pressure for the evolution of morphogenesis (Fig. <a href="#fig:summary">5</a>a). Mitotic domains and the germ band extension are ancestral traits, common to Diptera, while the cephalic furrow is a derived trait—an evolutionary novelty of cyclorraphan flies<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> (Fig. <a href="#fig:summary">5</a>a). Since these sources of stress were probably present at the dawn of dipterans, their head–trunk interface may have endured mechanical instabilities long before the evolution of the cephalic furrow. Remarkably, flies without a cephalic furrow (e.g., <em>Clogmia</em> and <em>Chironomus</em>) exhibit out-of-plane cell divisions at the head–trunk boundary, suggesting that they evolved an alternative solution to mitigate the effect of tissue compression during gastrulation<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> (Fig. <a href="#fig:summary">5</a>a). Moreover, neither <em>Clogmia</em>, <em>Anopheles</em>, nor <em>Chironomus</em> exhibit the <em>btd</em>–<em>eve</em> expression overlap at the head–trunk boundary<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> (Fig. <a href="#fig:genetic-patterning">4</a>), a trait essential to specify the initiator cells in species with cephalic furrow like <em>Drosophila</em>. Therefore, the establishment of a <em>btd</em>–<em>eve</em> overlap was probably a key event associated with the origin of the cephalic furrow and that might explain the divergent morphogenetic solutions to a similar selective pressure of mechanical instability (Fig. <a href="#fig:summary">5</a>b).</p>
<!--Facilitated by this novel transcriptional background-->
<!--the signaling pathway regulating lateral myosin activation-->
<!--coopted to control the behavior of initiator cells-->
<!--and genetically stabilized by natural selection.-->
<!--Altogether, the genetic patterning and tissue mechanics data obtained in our study and in the study by Dey et al. raises the following evolutionary scenario for the cephalic furrow ({@fig:summary}b).-->
<!--The evolution of the long germ in Diptera had as a consequence the increase in global compressive stresses happening during fly gastrulation.-->
<!--These stresses led to more frequent and potentially detrimental mechanical instability in embryonic tissues.-->
<!--The establishment of out-of-plane divisions was one solution to mitigate and it probably occurred by the fixation of genetic mechanisms determining the orientation.-->
<!--Cyclorraphan took another route.-->
<!--The genetic stabilization occurred through the btd--eve overlap via cooption of yet unknown genetic signaling controlling lateral myosin activation.-->
<!--The invention of the cephalic furrow may have allowed for shorter developmental times or may be the responsible factor leading to increased mechanical instability.-->
<!--One idea is that shorter developmental times concatenated the formation of mitotic domains and germ band extension, increasing the mechanical instabilities at the head--trunk boundary, or alternatively, that flies that evolved the cephalic furrow allowed for shorter developmental times.-->
<!--Broader implications for the interplay between mechanics and patterned morphogenesis-->
<!--Genetic stabilization of self-organized morphogenesis by natural selection-->
<!--Empirical support for this mechanism of generating morphogenetic innovations-->
<p>Our data reveals how the interplay between genetic patterning and tissue mechanics may have shaped the evolution of morphogenetic processes in the early embryonic development of flies. These findings, however, uncover a potentially more general mechanism about how tissue mechanics can influence developmental evolution. Classical theoretical work by Newman and Müller raised the hypothesis that self-organized morphogenesis was critical to generate morphogenetic innovations at the dawn of animal evolution.<span class="citation" data-cites="Newman2000-oz"><sup><a href="#ref-Newman2000-oz" role="doc-biblioref">34</a></sup></span> Our work provides initial empirical evidence supporting this hypothesis, showing that mechanical forces might have had a critical role in generating morphogenetic innovations not only at the transition to multicellularity, but also after the establishment of developmental patterning systems across evolutionary time. We propose that the genetic resolution of mechanical conflicts between distinct embryonic processes may be a broadly occurring mechanism that contributes to generating the remarkable morphogenetic diversity of early animal embryogenesis.</p>
<!--While the interplay between tissue mechanics and developmental evolution is poorly understood, classical theoretical work by Newman and Müller have raised the hypothesis that self-organized morphogenesis were critical to generate morphogenetic innovations at the dawn of animal evolution [@Newman2000-oz].-->
<!--Or, in other words, that before genome guided embryonic development evolved, tissues would fold due to mechanical interactions between them and the environment, and the patterned morphogenetic programs we know today were fixed secondarily by natural selection [@Newman2000-oz].-->
<!--Our work support this hypothesis by bringing empirical evidence from developmental and comparative data that mechanical forces might have a critical role in generating morphogenetic innovations in early development.-->
<!--In the case of the cephalic furrow, compressive stresses originating from concurrent gastrulation processes and resulting in ectopic epithelial buckling may have had a detrimental effect to the fitness of individuals.-->
<!--This mechanical instability could have acted as a selective pressure underlying the cooption of novel head--trunk gene expression domains and the establishment of a patterned cephalic furrow invagination.-->
<!--These speculations, however, require further testing with experiments that prevent cephalic furrow formation without other genetic side effects, as well as a systematic comparative analysis of mitotic domains, germ band extension, and tissue mechanics across the fly phylogeny.-->
<!--Overall, our work proposes that the cephalic furrow plays a mechanical role in absorbing compressive stresses in fly gastrulation, and introduces the cephalic furrow formation as powerful context to test the hypothesis that mechanical interactions within embryonic tissues may play an important role for the evolution of morphogenesis in early development.-->
<!--Alternatively, the evolution of the cephalic furrow was in fact the facilitator to allowing for faster development.-->
<!--Physical interaction of embryonic tissues with the egg shell are important for morphogenesis [@Bailles2019-qs; @Munster2019-ww]-->
<!--Our simulations also revealed the buckling events are influenced by the roundness of the vitelline envelope. Buckling occurs more easily on straighter shells. This might explain why buckling at the tips is less frequent. Friction plays a role in the shape of the fold [Trushko]. Recent works show how interactions of the embryonic tissues with the shell [@Munster;@Bailles] and between gastrulation movements [@Guo2021] are important to embryonic development. Insect eggs have an outstanding diversity [@Church]. This observation might have implications to how egg shape might have influenced the evolution of development. And reveal a glimpse of the complex mechanical underpinnings of gastrulation dynamics around the head--trunk boundary of bilaterians.-->
<!--Mechanical instability perturbs patterning and causes the loss of ATP, which may lead to  developmental delays-->
<!--Our findings also explain why so few cephalic furrow mutants have been identified. Because even though the cephalic furrow is disrupted, the tissue compression from other gastrulation movements will still make the epithelium fold, making the cephalic furrow phenotypes less obvious-->
<!--Dividing cells can’t do morphogenesis. That’s why mitotic domains occur in patches. Otherwise, gastrulation movements would fail.-->
<h1 id="methods">Methods</h1>
<h2 id="drosophila-stocks-and-genetic-crossings"><em>Drosophila</em> stocks and genetic crossings</h2>
<p>To generate fluorescent cephalic furrow mutants, we performed genetic crosses using the loss-of-function alleles <em>btd<sup>XA</sup></em> (FBal0030657), <em>eve<sup>3</sup></em> (FBal0003885), <em>prd<sup>4</sup></em> (FBal0013967), <em>slp<sup>Δ34B</sup></em> (FBal0035631), and <em>stg<sup>2</sup></em> (FBal0247234); the membrane fluorescent marker <em>Gap43-mCherry</em> (FBal0258719, gift from Kassiani Skouloudaki); and the green fluorescent balancers <em>FM7c, Kr-GFP</em> (FBst0005193), <em>CyO, twi-GFP</em> (gift from Akanksha Jain), and <em>TM3, Kr-GFP</em> (FBst0005195). We established stable lines balancing the loss-of-function alleles with fluorescent balancers, and used the lack of GFP signal to identify homozygous embryos in our live-imaging recordings. For genes on chromosomes 1 and 2 (<em>btd</em>, <em>eve</em>, and <em>prd</em>), we added the membrane marker on chromosome 3 ([<em>btd<sup>XA</sup>/FM7c, Kr-GFP;; Gap43-mCherry/MKRS</em>], [<em>eve<sup>3</sup>/CyO, twi-GFP; Gap43-mCherry/MKRS</em>], and [<em>slp<sup>Δ34B</sup>/CyO, twi-GFP; Gap43-mCherry/TM6B</em>]). For <em>stg</em>, which is located on chromosome 3, we recombined the allele with Gap (<em>Gap43-mCherry, stg<sup>2</sup>/TM3, Kr-GFP</em>). Since the <em>btd</em>–<em>stg</em> double mutant stable line is weak, we imaged the progeny of <em>btd<sup>XA</sup>/FM7c, Kr-GFP;; Gap43-mCherry, stg<sup>2</sup>/Gap43-mCherry</em> flies, identifying <em>btd</em> homozygozity by the GFP, and <em>stg</em> homozygozity by the lack of cell divisions after gastrulation. For laser ablations, we used a <em>moe-GFP</em> line (gift from Elizabeth Knust). The wildtype stocks contain the Gap43-mCherry marker in the Oregon-R genetic background. We obtained the founder fly stocks from the Bloomington Drosophila Stock Center and the Kyoto Stock Center and deposited the lines in the MPI-CBG stock collection. The complete list of FlyBase<span class="citation" data-cites="Gramates2022-xe"><sup><a href="#ref-Gramates2022-xe" role="doc-biblioref">35</a></sup></span> accession numbers and genotypes is available in the project’s data repository.<span class="citation" data-cites="Vellutini2023-ng"><sup><a href="#ref-Vellutini2023-ng" role="doc-biblioref">36</a></sup></span></p>
<h2 id="animal-husbandry-and-embryo-collection">Animal husbandry and embryo collection</h2>
<p>We maintained the <em>Drosophila</em> stocks in 50 mL hard plastic vials containing standard fly food and enclosed with a foam lid to allow air exchange. They were kept in an incubator with a constant 25° C temperature and 65% humidity and a 12:12 h light cycle. For imaging, we first amplified the stocks in larger 200 mL vials for a few weeks. We then narcotized the flies with CO<sub>2</sub>, and transferred them to a cage with a plate attached to one end containing a layer of apple juice agar and a slab of yeast paste on top. The flies were left to acclimatize in the cage for two days before the experiments. To guarantee the embryos are at a similar developmental stage, we exchanged the agar plate once per hour at least twice (pre-lays), and let the flies lay the eggs on the agar for one hour before collecting the plate. After filling the plate with water, we used a brush to release the eggs from the agar and transferred them to a cell strainer with 100 µm nylon mesh (VWR). To remove the chorion, we immersed the embryos in 20% bleach (sodium hypochlorite solution, Merck 1.05614.2500) for 90 s, washed abundantly with water, and proceeded to mounting for live imaging.</p>
<p>We maintained <em>Clogmia</em> flies in 9cm-wide plastic Petri dishes with a layer of wet cotton at room temperature and fed weekly with powder parsley. To obtain embryos for fixation, we collected the adult flies in a 200 mL hard plastic vial with wet cotton, and let them mate for 2–3 days. Then, we anesthetized the flies with CO<sub>2</sub>, dissected the ovaries from ripe females, and released the eggs using tweezers in deionized water, which activates embryonic development.<span class="citation" data-cites="Sander1985-sg Garcia-Solache2010-lz"><sup><a href="#ref-Sander1985-sg" role="doc-biblioref">37</a>,<a href="#ref-Garcia-Solache2010-lz" role="doc-biblioref">38</a></sup></span> We let embryos develop in deionized water at room temperature until the desired stage. To remove the chorion, we transferred the embryos to a glass vial with 0.5x PBS using a fine brush, exchanged the medium for 5% bleach in 0.5x PBS for 2 min, and washed abundantly with 0.5x PBS. Using the diluted PBS solution instead of water prevents the embryos from bursting after dechorionation.</p>
<p>We obtained pupae of the EgyptII wild type strain of <em>Ceratitis</em> from the Insect Pest Control Laboratory of the International Atomic Energy Agency (IAEA). Adult flies were kept at 25° C, 65% humidity, and 12:12 h light cycle, inside 49x30x30 cm plexiglass cages with the front and back ends covered by a nylon mesh. We provided water through a soaked towel and food as a 3:1 sugar:yeast mixture. As ripe females lay the eggs through the nylon mesh, we placed a plastic container with water at the back end of the cage for several hours to collect eggs. We dechorionated <em>Ceratitis</em> embryos using the <em>Drosophila</em> protocol.</p>
<p>We performed the collection of <em>Anopheles</em> embryos at the Center for Integrative Infectious Diseases Research (CIID) at Heidelberg University. To collect embryos, we placed a glass container with water and filter paper inside a cage with 300 mated females, which were fed a blood meal 72 h before, and put the cage in the dark for 2 h at 29° C. We then removed the container from the cage and let the embryos develop until the desired stage. To dechorionate, we collected the embryos on a cell strainer, incubated in 5% bleach for 75 s, and washed them thoroughly with deionized water.</p>
<h2 id="embryo-fixation-and-in-situ-hybridization">Embryo fixation and <em>in situ</em> hybridization</h2>
<p>For <em>Drosophila</em> and <em>Ceratitis</em>, we transferred dechorionated embryos to a glass vial containing equal volumes of 4% formaldehyde in PBS and n-Heptane, and let the vial shaking at 215 rpm for 45 min. For <em>Clogmia</em>, we diluted the fixative in 0.5x PBS. After removing the fixative (lower phase) using a glass pipette, we added an equal volume of 100% methanol, and shook the vial vigorously by hand for 1 min. We then removed the n-Heptane (upper phase) and collected the embryos on the bottom of an Eppendorf tube and washed several times with 100% methanol. For <em>Anopheles</em>, we followed a similar protocol that includes a longer 30 min wash in water after fixation, a 30 s boiling water step followed by 15 min in ice-cold water, until the final methanol washes.<span class="citation" data-cites="Cheatle-Jarvela2020-go"><sup><a href="#ref-Cheatle-Jarvela2020-go" role="doc-biblioref">23</a></sup></span> All the samples were stored in 100% methanol at -20° C.</p>
<p>We performed the <em>in situ</em> hybridization of <em>btd</em>, <em>eve</em>, <em>prd</em>, and <em>slp</em> genes using the Hybridization Chain Reaction v3.0 (HCR<sup>TM</sup>)<span class="citation" data-cites="Choi2018-fg"><sup><a href="#ref-Choi2018-fg" role="doc-biblioref">39</a></sup></span> reagents, except for the probe sets, which we designed using a custom script. The oligos were obtained from Sigma-Aldrich. We selected the HCR<sup>TM</sup> amplifiers to allow for triple (multiplexed) <em>in situ</em> combinations of <em>btd</em>+<em>eve</em>+<em>slp</em> or <em>prd</em>+<em>eve</em>+<em>slp</em>. Before starting, we manually devitenillized <em>Anopheles</em> embryos using fine tweezers. Then, we rehydrated the embryos in 100% methanol with a series of washes to 100% PBT. We permeabilized the embryos with 1:5000 dilution of ProteinaseK (20 mg/mL) for 5min, except for <em>Drosophila</em>. All samples were re-fixed in 4% formaldehyde for 40 min and washed thoroughly with PBT. We then followed the <em>“In situ HCR v3.0 protocol for whole-mount fruit fly embryos Revision 4 (2019-02-21)”</em> from Molecular Instruments <a href="https://www.molecularinstruments.com/hcr-rnafish-protocols">molecularinstruments.com/hcr-rnafish-protocols</a>. After the protocol, we stained the embryos with 1:1000 DAPI in 5x SSCT solution for 2 h and mounted the embryos in 80% glycerol in 5x SSCT for imaging.</p>
<h2 id="sample-mounting-for-microscopy">Sample mounting for microscopy</h2>
<p>For most of our live imaging, we used a Zeiss Lightsheet Z.1 microscope. To increase the throughput of samples imaged in one session, we optimized a mounting strategy developed previously in our laboratory.<span class="citation" data-cites="Schmied2016-oq"><sup><a href="#ref-Schmied2016-oq" role="doc-biblioref">40</a></sup></span> First, we cut a 22x22 mm glass coverslip (0.17 mm thickness) into 6x15 mm strips using a diamond knife, and attached a single strip to a custom sample holder using silicon glue, letting it harden for 15 min. We then coated the coverslip strip with a thin layer of heptane glue and let it dry while preparing the embryos. Using a fine brush, we transferred the embryos collected in the cell strainer onto an agar pad, and oriented them manually with a blunt cactus spine under a stereomicroscope. We aligned about 20 embryos in a single line (head to tail) along the main axis of the strip with the left or ventral sides up, depending on the experiment. To attach the embryos to the coverslip, we carefully lowered the sample holder over the agar pad until the glass coated with heptane glue touched the embryos. We placed the sample holder into the microscope chamber filled with water, and rotated it so that the samples are facing the detection objective directly, and the coverslip is orthogonal to the detection objective; this is important to prevent the lightsheet from hitting the glass edges. With the embryos oriented vertically along the coverslip, the lightsheet generated from the illumination objectives coming from the sides only needs to pass through the width of the embryo (about 200 µm). This approach gives the best results for recording lateral and dorsal views and is ideal for live imaging homozygote embryos since they are only about one fourth of the total number of imaged embryos. For fixed imaging of <em>in situ</em> samples, we used an inverted Zeiss LSM 700 Confocal microscope. We mounted the samples immersed in 80% glycerol between a slide and a glass coverslip supported by tape.</p>
<h2 id="microscopy-acquisition-parameters">Microscopy acquisition parameters</h2>
<p>For the lightsheet lateral datasets, we used a Zeiss 20x/1NA Plan-Apochromat water immersion objective to acquire stacks with 0.28 µm XY-resolution and 3 µm Z-resolution covering half of the embryo’s volume in a single view. This Z-resolution was restored to 1 µm during image processing (see below). For the dorsal datasets, we used a Zeiss 40x/1NA Plan-Apochromat water immersion objective to acquire stacks with 0.14 µm XY-resolution and 3 µm Z-resolution covering a volume around in the middle section of the anterior end of the embryo. We adjusted the time resolution between 45–60 s per frame to maximize the number of embryos acquired in one session. To visualize both the membrane signal (mCherry) and the green balancer signal (GFP), we acquired two channels simultaneously using the 488 and 561 nm lasers at 3% power with an image splitter cube containing a LP560 dichromatic mirror with SP550 and LP585 emission filters. All live imaging recordings were performed at 25° C. For the confocal datasets, we used a 20x/0.8 Plan-Apochromat Zeiss air objective to acquire 4-channels using 3 tracks (405, 488 and 639, and 555 nm, respectively) with a BP575-640 emission filter and about 0.4 µm XY-resolution and 2 µm Z-resolution covering about half the embryo’s volume.</p>
<!--using a pixel dwell time of x with 2 line averaging and 1 airy units.-->
<h2 id="image-processing-and-visualization">Image processing and visualization</h2>
<p>We converted the raw imaging datasets into individual TIFF stacks for downstream processing using a custom ImageJ macro in Fiji.<span class="citation" data-cites="Schindelin2012-di Rueden2017-ky"><sup><a href="#ref-Schindelin2012-di" role="doc-biblioref">41</a>,<a href="#ref-Rueden2017-ky" role="doc-biblioref">42</a></sup></span> To visualize the presence and dynamics of ectopic folds, we generated 3D renderings of the surface of embryos in lateral recordings using the plugin 3Dscript in Fiji.<span class="citation" data-cites="Schmid2019-bm"><sup><a href="#ref-Schmid2019-bm" role="doc-biblioref">43</a></sup></span> For analyzing the entire epithelial surface, we first improved the signal-to-noise ratio and Z-resolution of lateral datasets from 3 µm to 1 µm by training a deep learning upsampling model using CARE.<span class="citation" data-cites="Weigert2018-ti"><sup><a href="#ref-Weigert2018-ti" role="doc-biblioref">44</a></sup></span> Then, we created cartographic projections of the lateral recordings using the ImSAnE toolbox<span class="citation" data-cites="Heemskerk2015-kv"><sup><a href="#ref-Heemskerk2015-kv" role="doc-biblioref">45</a></sup></span> by loading the restored data in MATLAB,<span class="citation" data-cites="Matlab2015-nd"><sup><a href="#ref-Matlab2015-nd" role="doc-biblioref">46</a></sup></span> segmenting the epithelial surface using ilastik,<span class="citation" data-cites="Berg2019-ab"><sup><a href="#ref-Berg2019-ab" role="doc-biblioref">47</a></sup></span> and generating 3D cartographic projections of lateral views following a workflow established for fly embryos.<span class="citation" data-cites="Vellutini2022-ya"><sup><a href="#ref-Vellutini2022-ya" role="doc-biblioref">48</a></sup></span> To visualize <em>in situ</em> hybridization data, we performed maximum intensity projections or extracted single slices from the raw volumes. For all microscopy images, we only performed minimal linear intensity adjustments to improve their contrast and brightness.<span class="citation" data-cites="Schmied2020-ip"><sup><a href="#ref-Schmied2020-ip" role="doc-biblioref">49</a></sup></span></p>
<h2 id="ectopic-fold-analyses">Ectopic fold analyses</h2>
<p>To characterize the relative timing of ectopic folding, we annotated the germ band position and the number of frames after the onset of gastrulation at the initial buckling, when the first cells disappear from the surface in the lateral 3D renderings. We defined the onset of gastrulation (T=0) as the moment immediately after the end of cellularization, and immediately before the beginning of the ventral furrow invagination. To visualize the variability of ectopic folding, we manually traced the fold outlines in lateral recordings using Fiji. Because embryos have different sizes, we first used the plugin <em>bUnwarpJ</em><span class="citation" data-cites="Arganda-Carreras2006-im"><sup><a href="#ref-Arganda-Carreras2006-im" role="doc-biblioref">50</a></sup></span> (<a href="https://imagej.net/plugins/bunwarpj/">imagej.net/plugins/bunwarpj</a>) to register individual frames and then applied the same transformation to the fold traces for a standardized comparison. We analyzed the dynamics of ectopic folds by measuring the relative angle and tortuosity of the segmented line traces over time, and to visualize the kinetics we generated color-coded temporal projections using the script <em>Temporal Color Code</em> (<a href="https://imagej.net/plugins/temporal-color-code">imagej.net/plugins/temporal-color-code</a>) with the perceptually uniform <em>mpl-viridis</em> color map (<a href="https://bids.github.io/colormap/">bids.github.io/colormap</a>) bundled in Fiji.</p>
<p>To estimate the folded area in the cephalic furrow and ectopic folds, we annotated the region of the blastoderm before gastrulation that infolded in the cartographic projections using Fiji, and calculated the area correcting the pixel dimensions according to the coordinates in the projection. For the fold depth, we measured the distance between the vitelline envelope to the tip of the fold at the moment of maximum depth in the dorsal recordings. For the analysis of the epithelial surface, we used the plugin <em>MorphoLibJ</em><span class="citation" data-cites="Legland2016-cp"><sup><a href="#ref-Legland2016-cp" role="doc-biblioref">51</a></sup></span> (<a href="https://imagej.net/plugins/morpholibj">imagej.net/plugins/morpholibj</a>) to segment, measure, and color-code the cell apical areas, and the plugin <em>Linear Stack Alignment with SIFT</em> (<a href="https://imagej.net/plugins/linear-stack-alignment-with-sift">imagej.net/plugins/linear-stack-alignment-with-sift</a>) to register cells between timepoints.</p>
<h2 id="laser-cauterization-experiments">Laser cauterization experiments</h2>
<p>We performed laser cauterization experiments in two microscope setups, a Luxendo MuVi SPIM with a photomanipulation module and a Zeiss LSM 780 NLO with multiphoton excitation. For the MuVi SPIM, we embedded dechorionated embryos in 2% low-melting agarose and mounted the samples in glass capillaries to obtain <em>in toto</em> recordings. We used a pulsed infrared laser 1030–1040 nm with 200 fs pulse duration and 1.5W power to cauterize the posterior region of the dorsal embryonic surface, attaching the blastoderm to the vitelline envelope. Using an Olympus 20x/1.0NA water immersion objective, we acquired stacks with 0.29 µm XY-resolution and 1 µm Z-resolution of four different angles every one minute. For the Zeiss microscope, we attached the embryos with the dorsal side down onto coverslips using heptane glue and immersing in halocarbon oil. We cauterized the embryos sequentially using a near infrared 800 nm laser (Chameleon Vision II) through a single pixel line (210 nm/px and 100 µs/px) around the same dorsal region to block the germ band extension. We used a Zeiss 25x/0.8NA LD LCI Plan-Apochromat glycerol immersion objective to acquire every 2:38 min two different planes of the blastoderm, (i) the surface to monitor the germ band extension, and (ii) 40 µm deep in the equatorial region to monitor the occurrence of ectopic folding. The stacks had 0.21 µm XY-resolution and one minute time resolution. To obtain a quantitative measure of ectopic folding, we analyzed the degree by which the tissues deform between non-cauterized and cauterized mutants, using as a proxy the tortuosity of the epithelium outline. For that, we took the profile slices from dorsal recordings and transformed the curved vitelline envelope into a straight line using the <em>Straighten</em> tool of ImageJ (<a href="#fig:tortuosity">Supplementary Fig. 3</a>a). We then cropped a 200×25 µm region along the head–trunk interface and applied Gaussian blur, thresholding, and edge detection to obtain the epithelium outline for individual timepoints covering about 50 min after gastrulation (<a href="#fig:tortuosity">Supplementary Fig. 3</a>a,b). We extracted measurements from the epithelium outlines using the ImageJ plugin <em>Analyze Skeleton</em><span class="citation" data-cites="Arganda-Carreras2010-la"><sup><a href="#ref-Arganda-Carreras2010-la" role="doc-biblioref">52</a></sup></span> (<a href="https://imagej.net/plugins/analyze-skeleton/">imagej.net/plugins/analyze-skeleton</a>), and generated the color-coded temporal projections as described above.</p>
<!--TODO: Temperature in MuVi and Zeiss?-->
<h2 id="laser-ablation-experiments">Laser ablation experiments</h2>
<p>We performed laser ablations in a Yokogawa CSU-X1 confocal spinning disk, an EMCCD camera (Andor iXon DU-888), and the software AndorIQ for image acquisition. We attached dechorionated embryos laterally to a MatTek glass-bottom petri dish and covered the samples with water, and performed the ablations using a Titanium Sapphire Chameleon Ultra II (Coherent) laser at 800 nm tuned down from 80 MHz to 20 kHz with a pulse-picker. The laser power measured before the microscope port was 6 mW and the pixel dwell time for scanning was 2 µs. To ensure the cut, we repeated the scan ten consecutive times along a single cell, acquiring a single slice with a 60x/1.2NA water immersion objective with 0.18 µm XY-resolution and 200 ms time-steps. We ablated each embryo just once. The temperature was maintained at 28° C. To analyze the ablation data, we created a line crossing the edges of the ablated cell perpendicular to the cut and generated a kymograph using the <em>Multi Kymograph</em> Fiji plugin (<a href="#fig:ablation">Supplementary Fig. 4</a>). We then binarized the kymographs, measured the distance between cell edges over the first 30 s after the cut, and performed a linear fit of the data to obtain the recoil velocity (<a href="#fig:ablation">Supplementary Fig. 4</a>).</p>
<!--TODO: What is the brand of the objective?-->
<!--TODO: Does a higher temperature affects the quantifications?-->
<h2 id="strain-rate-analysis">Strain rate analysis</h2>
<p>To estimate the strain rates, we first performed particle image velocimetry on cartographic projections using the ImageJ plugin <em>iterativePIV</em><span class="citation" data-cites="Tseng2012-ni"><sup><a href="#ref-Tseng2012-ni" role="doc-biblioref">53</a></sup></span> (<a href="https://sites.google.com/site/qingzongtseng/piv">sites.google.com/site/qingzongtseng/piv</a>). Then, we used the equation <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>E</mi><mo>=</mo><mrow><mo stretchy="true" form="prefix">|</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><mover><mi>∇</mi><mo accent="true">⃗</mo></mover><mi>.</mi><mover><mi>v</mi><mo accent="true">⃗</mo></mover><mo stretchy="true" form="postfix">)</mo></mrow><mo>+</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><msub><mi>∂</mi><mi>x</mi></msub><msub><mi>v</mi><mi>y</mi></msub><mo>+</mo><msub><mi>∂</mi><mi>y</mi></msub><msub><mi>v</mi><mi>x</mi></msub><mo stretchy="true" form="postfix">)</mo></mrow><mo stretchy="true" form="postfix">|</mo></mrow></mrow><annotation encoding="application/x-tex">E = \left| \frac{1}{2} \left( \vec \nabla . \vec v \right) + \frac{1}{2} \left( \partial_x v_y + \partial_y v_x \right) \right|</annotation></semantics></math> to define and calculate the magnitude of the strain rate, where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>v</mi><annotation encoding="application/x-tex">v</annotation></semantics></math> is the displacement obtained in the PIV divided by the time in minutes. The measurements combine isotropic and anisotropic strain rate. We used these values to create a color-coded overlay for the strain rate (<a href="#fig:apical-strain">Supplementary Fig. 2</a>b). To generate the line plots, we averaged the strain rate along the dorsoventral axis in two predefined regions, the head–trunk (canonical cephalic furrow position) and the trunk–germ (posterior to the Mitotic Domain 6) (<a href="#fig:apical-strain">Supplementary Fig. 2</a>b).</p>
<h2 id="model-and-simulations">Model and simulations</h2>
<p>Our model follows an approach similar to a previously published model of epithelial buckling under confinement.<span class="citation" data-cites="Trushko2020-gf"><sup><a href="#ref-Trushko2020-gf" role="doc-biblioref">17</a></sup></span> It represents the monolayer epithelium of the early <em>Drosophila</em> embryo in a cross-section as a single line through the apical–basal midline of the epithelial cells. The tissue is modeled as an elastic rod with a stretching energy per unit length <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>s</mi></msub><annotation encoding="application/x-tex">W_s</annotation></semantics></math> and bending energy per unit length <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>b</mi></msub><annotation encoding="application/x-tex">W_b</annotation></semantics></math> so that the total energy per unit length is <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>W</mi><mi>T</mi></msub><mo>=</mo><msub><mi>W</mi><mi>s</mi></msub><mo>+</mo><msub><mi>W</mi><mi>b</mi></msub></mrow><annotation encoding="application/x-tex">W_T = W_s + W_b</annotation></semantics></math>. In full, <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>W</mi><mi>T</mi></msub><mo>=</mo><msub><mo>∫</mo><msub><mi>L</mi><mi>o</mi></msub></msub><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>s</mi></msub><msup><mrow><mo stretchy="true" form="prefix">(</mo><mfrac><mrow><mi>d</mi><mi>s</mi></mrow><mrow><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow></mfrac><mo>−</mo><mn>1</mn><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub><mo>+</mo><msub><mo>∫</mo><msub><mi>L</mi><mi>o</mi></msub></msub><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>b</mi></msub><msup><mrow><mo stretchy="true" form="prefix">(</mo><mi>κ</mi><mo>−</mo><msub><mi>κ</mi><mi>o</mi></msub><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">W_T = \int_{L_o} \frac{1}{2} K_s \left( \frac{ds}{ds_o} - 1 \right)^2 ds_o + \int_{L_o} \frac{1}{2} K_b \left( \kappa - \kappa_o \right)^2 ds_o</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>s</mi></msub><annotation encoding="application/x-tex">K_s</annotation></semantics></math> is the stretching rigidity and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math> is the bending rigidity of the tissue; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">ds_o</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><mi>s</mi></mrow><annotation encoding="application/x-tex">ds</annotation></semantics></math> are the preferred and current lengths of the curve, respectively; and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>κ</mi><annotation encoding="application/x-tex">\kappa</annotation></semantics></math> is the curvature of the rod. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>L</mi><mi>o</mi></msub><annotation encoding="application/x-tex">L_o</annotation></semantics></math> is the total length of the tissue in a stress-free condition. To perform numerics, we discretize the curve into <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>N</mi><annotation encoding="application/x-tex">N</annotation></semantics></math> particles indexed by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math>. The total energy per unit length for this discretized model is given by <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>W</mi><mi>T</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>s</mi></msub><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>2</mn></mrow><mrow><mi>N</mi><mo>−</mo><mn>3</mn></mrow></munderover><msup><mrow><mo stretchy="true" form="prefix">(</mo><mfrac><mrow><mi>Δ</mi><msub><mi>r</mi><mi>i</mi></msub></mrow><mrow><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow></mfrac><mo>−</mo><mn>1</mn><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub><mo>+</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><msub><mi>K</mi><mi>b</mi></msub><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>2</mn></mrow><mrow><mi>N</mi><mo>−</mo><mn>3</mn></mrow></munderover><msup><mrow><mo stretchy="true" form="prefix">(</mo><msub><mi>κ</mi><mi>i</mi></msub><mo>−</mo><msub><mi>κ</mi><mrow><mi>o</mi><mo>,</mo><mi>i</mi></mrow></msub><mo stretchy="true" form="postfix">)</mo></mrow><mn>2</mn></msup><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">W_T^{\text{*}} = \frac{1}{2} K_s \sum_{i = 2}^{N-3} \left( \frac{\Delta r_i}{\Delta r_o} - 1 \right)^2 \Delta r_o + \frac{1}{2} K_b \sum_{i = 2}^{N-3} \left( \kappa_i - \kappa_{o,i} \right)^2 \Delta r_o</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">\Delta r_o</annotation></semantics></math> is the preferred length of springs connecting consecutive points (equal for all springs); <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mi>i</mi></msub></mrow><annotation encoding="application/x-tex">\Delta r_i</annotation></semantics></math> is the current length between <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">i+1</annotation></semantics></math>; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>κ</mi><mi>i</mi></msub><annotation encoding="application/x-tex">\kappa_i</annotation></semantics></math> is the discretized curvature at point <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math>; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>κ</mi><mrow><mi>o</mi><mo>,</mo><mi>i</mi></mrow></msub><annotation encoding="application/x-tex">\kappa_{o,i}</annotation></semantics></math> is the preferred curvature at point <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math> (equal to 0, except when specified). The first and last two points of the curve are fixed in space. To obtain a physically meaningful dimensionless bending rigidity, we divide the bending rigidity by the factor <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>s</mi></msub><msup><mi>L</mi><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_sL^2</annotation></semantics></math> as <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mfrac><msub><mi>K</mi><mi>b</mi></msub><mrow><msub><mi>K</mi><mi>s</mi></msub><msup><mi>L</mi><mn>2</mn></msup></mrow></mfrac></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} = \frac{K_b}{K_sL^2}</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>L</mi><annotation encoding="application/x-tex">L</annotation></semantics></math> is the semi-major axis of the embryo. To minimize the total energy, we add a ground level of noise to the particles and let the particles move in the direction of the forces. The motion of the particles is governed by <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mfrac><mrow><mi>Δ</mi><msub><munder><mi>r</mi><mo accent="true">_</mo></munder><mi>i</mi></msub></mrow><mrow><mi>Δ</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mo>−</mo><mfrac><mi>L</mi><mrow><msub><mi>K</mi><mi>s</mi></msub><mi>τ</mi></mrow></mfrac><mfrac><mrow><mi>∂</mi><msup><mi>W</mi><mtext mathvariant="normal">*</mtext></msup></mrow><mrow><mi>∂</mi><msub><munder><mi>r</mi><mo accent="true">_</mo></munder><mi>i</mi></msub></mrow></mfrac><mo>+</mo><msub><munder><mi>ζ</mi><mo accent="true">_</mo></munder><mi>i</mi></msub></mrow><annotation encoding="application/x-tex">\frac{\Delta\underline{r}_i}{\Delta t} = - \frac{L}{K_s\tau}\frac{\partial W^{\text{*}}}{\partial \underline{r}_i} + \underline{\zeta}_i</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><munder><mi>r</mi><mo accent="true">_</mo></munder><mi>i</mi></msub><annotation encoding="application/x-tex">\underline{r}_i</annotation></semantics></math> is the current position of the <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>i</mi><annotation encoding="application/x-tex">i</annotation></semantics></math>th particle; <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>τ</mi><annotation encoding="application/x-tex">\tau</annotation></semantics></math> represents an arbitrary timescale introduced here to balance dimensions (set to 1); <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><mi>t</mi></mrow><annotation encoding="application/x-tex">\Delta t</annotation></semantics></math> are the timesteps (set to <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>5</mn></mrow></msup><mo>×</mo><mi>τ</mi><msub><mi>K</mi><mi>s</mi></msub><mi>/</mi><mi>L</mi></mrow><annotation encoding="application/x-tex">10^{-5} \times \tau K_s/L</annotation></semantics></math>); and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><munder><mi>ζ</mi><mo accent="true">_</mo></munder><mi>i</mi></msub><annotation encoding="application/x-tex">\underline{\zeta}_i</annotation></semantics></math> is the noise, chosen from a Gaussian distribution with mean 0 and standard distribution <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>5</mn></mrow></msup><mo>×</mo><mi>L</mi></mrow><annotation encoding="application/x-tex">10^{-5}\times L</annotation></semantics></math>. In our model, the position of the germ band corresponds to the position of the last particle in the curve on the semi-ellipse that represents the embryonic blastoderm. The extent of the germ band is given by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math>, which is the projection of the germ band arclength onto the mid-axis of the embryo normalized by the embryo length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>2</mn><mi>L</mi></mrow><annotation encoding="application/x-tex">2L</annotation></semantics></math>). When <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">g=0</annotation></semantics></math> the tissue is free of stretching stress, but at any other <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>0</mn><mo>&lt;</mo><mi>g</mi><mo>&lt;</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">0&lt;g&lt;1</annotation></semantics></math>, the blastoderm will be compressed. The preferred lengths of the individual springs is obtained by dividing the elliptical arclength into <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>N</mi><annotation encoding="application/x-tex">N</annotation></semantics></math> equal segments. The length of each segment is given by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mtext mathvariant="normal">o</mtext></msub><mo>=</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><mi>L</mi><msubsup><mo>∫</mo><mn>0</mn><mi>π</mi></msubsup><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>e</mi><mn>2</mn></msup><msup><mo>cos</mo><mn>2</mn></msup><mo stretchy="false" form="prefix">(</mo><mi>u</mi><mo stretchy="false" form="postfix">)</mo></mrow></msqrt><mi>d</mi><mi>u</mi><mo stretchy="true" form="postfix">)</mo></mrow></mrow><annotation encoding="application/x-tex">\Delta r_{\text{o}} = \frac{1}{N} \left( L\int_{0}^{\pi} \sqrt{1 - e^2 \cos^2(u)} du \right)</annotation></semantics></math>. To find the initial lengths of the springs, we use <math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><mi>r</mi><mo stretchy="false" form="prefix">(</mo><mi>t</mi><mo>=</mo><mn>0</mn><mo stretchy="false" form="postfix">)</mo><mo>=</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><mrow><mo stretchy="true" form="prefix">(</mo><mi>L</mi><msubsup><mo>∫</mo><mrow><mi>u</mi><mi>′</mi></mrow><mi>π</mi></msubsup><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mi>e</mi><mn>2</mn></msup><msup><mo>cos</mo><mn>2</mn></msup><mo stretchy="false" form="prefix">(</mo><mi>u</mi><mo stretchy="false" form="postfix">)</mo></mrow></msqrt><mi>d</mi><mi>u</mi><mo stretchy="true" form="postfix">)</mo></mrow></mrow><annotation encoding="application/x-tex">\Delta r(t = 0) = \frac{1}{N} \left( L\int_{u&#39;}^{\pi} \sqrt{1 - e^2 \cos^2(u)} du \right)</annotation></semantics></math> where <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>e</mi><mo>=</mo><msqrt><mrow><mn>1</mn><mo>−</mo><msup><mrow><mo stretchy="false" form="prefix">(</mo><mn>0.4</mn><mo stretchy="false" form="postfix">)</mo></mrow><mn>2</mn></msup></mrow></msqrt></mrow><annotation encoding="application/x-tex">e = \sqrt{1 - {(0.4)}^{2}}</annotation></semantics></math> and the angle <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>u</mi><mi>′</mi></mrow><annotation encoding="application/x-tex">u&#39;</annotation></semantics></math> corresponds to the position of the blastoderm end. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>u</mi><mi>′</mi></mrow><annotation encoding="application/x-tex">u&#39;</annotation></semantics></math> is obtained for a given value of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math> by <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>u</mi><mi>′</mi><mo>=</mo><msup><mo>cos</mo><mrow><mo>−</mo><mn>1</mn></mrow></msup><mo stretchy="false" form="prefix">(</mo><mn>1</mn><mo>−</mo><mn>2</mn><mi>g</mi><mo stretchy="false" form="postfix">)</mo></mrow><annotation encoding="application/x-tex">u&#39; = \cos^{-1}(1 - 2g)</annotation></semantics></math>. Here, we obtained the initial lengths by dividing the compressed blastoderm into <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>N</mi><annotation encoding="application/x-tex">N</annotation></semantics></math> equal segments. For any simulation, the value of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math> is constant (blastoderm end is static in position). To model mitotic domains, we introduced new particles and springs on the mid-points between two particles in specific regions of length <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>0.5</mn><mi>L</mi></mrow><annotation encoding="application/x-tex">0.5L</annotation></semantics></math>. The new springs are given the same <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Δ</mi><msub><mi>r</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">\Delta r_{o}</annotation></semantics></math> as the rest of the springs in the tissue. The blastoderm is confined by a rigid boundary in the shape of a semi-ellipse. Any particle that lands outside this boundary at any timestep is re-positioned onto the rigid boundary. This new position is prescribed by taking the intersection point of the rigid boundary curve and the line segment that connects the position before this iteration (which was inside or on the vitelline envelope) and the position outside the vitelline envelope. Finally, we define and count a <em>fold</em> when we find that a particle’s distance from the rigid boundary is greater than a threshold value. To calculate this threshold, we measure the maximum distance that particles can achieve when the tissue is in a stress-free state. This threshold was calculated to be <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>0.035</mn><mi>L</mi></mrow><annotation encoding="application/x-tex">0.035L</annotation></semantics></math>. The code for the model and simulations is available in a Zenodo repository.<span class="citation" data-cites="Krishna2023-hu"><sup><a href="#ref-Krishna2023-hu" role="doc-biblioref">54</a></sup></span></p>
<h2 id="data-visualization-and-figure-assembly">Data visualization and figure assembly</h2>
<p>We created illustrations and assembled the final figure plates using Inkscape v1.2.2.<span class="citation" data-cites="The_Inkscape_Project2003-ht"><sup><a href="#ref-The_Inkscape_Project2003-ht" role="doc-biblioref">55</a></sup></span> For microscopy videos, we exported the original stacks as AVI without compression with 10–15 fps using Fiji and post-processed them to MPEG-4 format 1080p resolution using the H.264 encoding at a constant bitrate quality factor of 18 for visualization using HandBrake v1.6.1.<span class="citation" data-cites="HandBrake_Team2003-fq"><sup><a href="#ref-HandBrake_Team2003-fq" role="doc-biblioref">56</a></sup></span> The high-resolution figures and videos are available in a Zenodo repository.<span class="citation" data-cites="Vellutini2023-ix"><sup><a href="#ref-Vellutini2023-ix" role="doc-biblioref">57</a></sup></span> We performed the data wrangling, statistical analyses, and plotting in R v4.2.1<span class="citation" data-cites="R-Core-Team1993-jw"><sup><a href="#ref-R-Core-Team1993-jw" role="doc-biblioref">58</a></sup></span> using RStudio v2022.7.2.576,<span class="citation" data-cites="RStudio-Team2011-vj"><sup><a href="#ref-RStudio-Team2011-vj" role="doc-biblioref">59</a></sup></span> and Python 3.10.7 using Jupyter notebooks.<span class="citation" data-cites="Granger2021-zg"><sup><a href="#ref-Granger2021-zg" role="doc-biblioref">60</a></sup></span> All the data and analyses pipelines were deposited in a Zenodo repository.<span class="citation" data-cites="Vellutini2023-ng"><sup><a href="#ref-Vellutini2023-ng" role="doc-biblioref">36</a></sup></span></p>
<h2 id="data-availability">Data availability</h2>
<ul>
<li>Main repository with data, analyses, figures, and text source files: <a href="https://zenodo.org/record/7781947">zenodo.org/record/7781947</a>.<span class="citation" data-cites="Vellutini2023-ng"><sup><a href="#ref-Vellutini2023-ng" role="doc-biblioref">36</a></sup></span></li>
<li>Theory repository with model code and simulation outputs: <a href="https://zenodo.org/record/7784906">zenodo.org/record/7784906</a>.<span class="citation" data-cites="Krishna2023-hu"><sup><a href="#ref-Krishna2023-hu" role="doc-biblioref">54</a></sup></span></li>
<li>Media repository with high-resolution figures and videos: <a href="https://zenodo.org/record/7781916">zenodo.org/record/7781916</a>.<span class="citation" data-cites="Vellutini2023-ix"><sup><a href="#ref-Vellutini2023-ix" role="doc-biblioref">57</a></sup></span></li>
</ul>
<h1 id="acknowledgments">Acknowledgments</h1>
<p>We thank all current and former LoPaTs for discussions and support during this project. Akanksha Jain and Vladimir Ulman for the initial assistance with cartographic projections. Giulia Serafini for the help with fly crosses. Anaïs Bailles for the constructive feedback on the physics. Michaela Burkon and Pavel Mejstrik for technical support. Jan Brugués and Keisuke Ishihara for the laser ablation setup. MPI-CBG’s Light Microscopy Facility and Computer Services Facility for the assistance with data acquisition and processing. Sven Ssykor and Cornelia Maas for the support with fly stocks. Radim Žídek for the essential help in construct design. Steffen Lemke and Yu-Chiun Wang for the cephalic furrow discussions and sharing of unpublished data. Lemke Lab for the generous help in setting up <em>Clogmia</em> cultures. Freddy Frischknecht and Miriam Reinig for enabling and supporting the collection of <em>Anopheles</em> embryos. Konstantinos Bourtzis for kindly providing <em>Ceratitis</em> pupae. Carles Blanch-Mercader for feedback on simulations. Juliana Roscito for the text revisions. Michele Marass for crucial editorial input. Michael Akam for drawing BCV’s attention to the cephalic furrow. And two anonymous reviewers and Alfonso Martinez Arias for the comments and constructive feedback.</p>
<h1 id="funding">Funding</h1>
<p>This work was supported by the MPI-CBG core funding from CDM and PT, and by an European Research Council Advanced Grant (ERC-AdG 885504 GHOSTINTHESHELL) awarded to PT. AS was supported by funding from the European Union’s Horizon 2020 Research and Innovation Programme under grant agreement no. 829010 (PRIME). BCV was supported by an EMBO Long Term Fellowship (ALTF 74-2018).</p>
<h1 id="author-contributions">Author contributions</h1>
<p>BCV and PT conceived the study. BCV designed experiments, generated fly stocks, acquired microscopy data, performed <em>in situ</em> hybridization, and processed and analyzed the <em>in vivo</em> data. MBC conceived and conducted the laser ablation and cauterization experiments, and analyzed the laser ablation and strain rate data. CDM, AK, and AS designed the model. AK and AS programmed the model, performed the simulations, and analyzed the <em>in silico</em> data. BCV wrote the manuscript. All authors revised and contributed to the text.</p>
<h1 id="references">References</h1>
<!--TODO: Ensure limit and split reference lists-->
<!--REFERENCES: As a guideline, most papers should include no more than 50 main text references; all additional references can be cited in (and listed after) the Methods section, as detailed below.-->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-Keller2012-ca" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Keller, R. Developmental biology. Physical biology returns to morphogenesis. <em>Science</em> <strong>338,</strong> 201–203 (2012).</div>
</div>
<div id="ref-Heisenberg2013-pl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Heisenberg, C.-P. &amp; Bellaïche, Y. Forces in tissue morphogenesis and patterning. <em>Cell</em> <strong>153,</strong> 948–962 (2013).</div>
</div>
<div id="ref-Collinet2021-qa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Collinet, C. &amp; Lecuit, T. Programmed and self-organized flow of information during morphogenesis. <em>Nat. Rev. Mol. Cell Biol.</em> <strong>22,</strong> 245–265 (2021).</div>
</div>
<div id="ref-Hartenstein1985-zs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Hartenstein, V. &amp; Campos-Ortega, J. A. Fate-mapping in wild-type<em><span>D</span>rosophila melanogaste</em>r: I. The spatio-temporal pattern of embryonic cell divisions. <em>Wilhelm Roux’ Archiv</em> <strong>194,</strong> 181–195 (1985).</div>
</div>
<div id="ref-Spencer2015-td" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Spencer, A. K., Siddiqui, B. A. &amp; Thomas, J. H. Cell shape change and invagination of the cephalic furrow involves reorganization of <span>F</span>-actin. <em>Dev. Biol.</em> <strong>402,</strong> 192–207 (2015).</div>
</div>
<div id="ref-Foe1989-cw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Foe, V. E. Mitotic domains reveal early commitment of cells in <em><span>D</span>rosophila</em> embryos. <em>Development</em> <strong>107,</strong> 1–22 (1989).</div>
</div>
<div id="ref-Liu2013-lc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Liu, F., Morrison, A. H. &amp; Gregor, T. Dynamic interpretation of maternal inputs by the <em><span>D</span>rosophila</em> segmentation gene network. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>110,</strong> 6724–6729 (2013).</div>
</div>
<div id="ref-Vincent1997-fa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Vincent, A., Blankenship, J. T. &amp; Wieschaus, E. Integration of the head and trunk segmentation systems controls cephalic furrow formation in <em><span>D</span>rosophila</em>. <em>Development</em> <strong>124,</strong> 3747–3754 (1997).</div>
</div>
<div id="ref-Eritano2020-mi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Eritano, A. S., Bromley, C. L., Bolea Albero, A., Schütz, L., Wen, F.-L., Takeda, M., Fukaya, T., Sami, M. M., Shibata, T., Lemke, S. &amp; Wang, Y.-C. Tissue-scale mechanical coupling reduces morphogenetic noise to ensure precision during epithelial folding. <em>Dev. Cell</em> <strong>53,</strong> 212–228.e12 (2020).</div>
</div>
<div id="ref-Popkova2024-eg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Popkova, A., Andrenšek, U., Pagnotta, S., Ziherl, P., Krajnc, M. &amp; Rauzi, M. A mechanical wave travels along a genetic guide to drive the formation of an epithelial furrow during <em>drosophila</em> gastrulation. <em>Dev. Cell</em> <strong>59,</strong> 400–414.e5 (2024).</div>
</div>
<div id="ref-Turner1977-ol" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Turner, F. R. &amp; Mahowald, A. P. Scanning electron microscopy of <em><span>D</span>rosophila melanogaster</em> embryogenesis. II. Gastrulation and segmentation. <em>Dev. Biol.</em> <strong>57,</strong> 403–416 (1977).</div>
</div>
<div id="ref-Costa1993-qj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Costa, M., Sweeton, D. &amp; Wieschaus, E. in <em>The development of <em><span>D</span>rosophila melanogaster</em></em> (eds. Bate, M. &amp; Arias, A. M.) <strong>1,</strong> 425–465 (Cold Spring Harbor Laboratory Press, 1993).</div>
</div>
<div id="ref-Dicko2017-wz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Dicko, M., Saramito, P., Blanchard, G. B., Lye, C. M., Sanson, B. &amp; Étienne, J. Geometry can provide long-range mechanical guidance for embryogenesis. <em>PLoS Comput. Biol.</em> <strong>13,</strong> e1005443 (2017).</div>
</div>
<div id="ref-Dey2023-mo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Dey, B., Kaul, V., Kale, G., Scorcelletti, M., Takeda, M., Wang, Y.-C. &amp; Lemke, S. Divergent evolutionary strategies preempt tissue collision in fly gastrulation. <em>bioRxiv</em> (2023). doi:<a href="https://doi.org/10.1101/2023.10.09.561568">10.1101/2023.10.09.561568</a></div>
</div>
<div id="ref-Blankenship2001-tm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Blankenship, J. T. &amp; Wieschaus, E. Two new roles for the <em><span>D</span>rosophila</em> <span>AP</span> patterning system in early morphogenesis. <em>Development</em> <strong>128,</strong> 5129–5138 (2001).</div>
</div>
<div id="ref-Edgar1989-hv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Edgar, B. A. &amp; O’Farrell, P. H. Genetic control of cell division patterns in the <em><span>D</span>rosophila</em> embryo. <em>Cell</em> <strong>57,</strong> 177–187 (1989).</div>
</div>
<div id="ref-Trushko2020-gf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Trushko, A., Di Meglio, I., Merzouki, A., Blanch-Mercader, C., Abuhattum, S., Guck, J., Alessandri, K., Nassoy, P., Kruse, K., Chopard, B. &amp; Roux, A. Buckling of an epithelium growing under spherical confinement. <em>Dev. Cell</em> <strong>54,</strong> 655–668.e6 (2020).</div>
</div>
<div id="ref-Fouchard2020-km" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Fouchard, J., Wyatt, T. P. J., Proag, A., Lisica, A., Khalilgharibi, N., Recho, P., Suzanne, M., Kabla, A. &amp; Charras, G. Curling of epithelial monolayers reveals coupling between active bending and tissue tension. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>117,</strong> 9377–9383 (2020).</div>
</div>
<div id="ref-Efrati2009-ve" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Efrati, E., Sharon, E. &amp; Kupferman, R. Elastic theory of unconstrained non-euclidean plates. <em>J. Mech. Phys. Solids</em> <strong>57,</strong> 762–775 (2009).</div>
</div>
<div id="ref-Andrioli2012-fv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Andrioli, L. P., Digiampietri, L. A., Barros, L. P. de &amp; Machado-Lima, A. Huckebein is part of a combinatorial repression code in the anterior blastoderm. <em>Dev. Biol.</em> <strong>361,</strong> 177–185 (2012).</div>
</div>
<div id="ref-Strobl2022-og" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Strobl, F., Schetelig, M. F. &amp; Stelzer, E. H. K. In toto light sheet fluorescence microscopy live imaging datasets of <em>ceratitis capitata</em> embryonic development. <em>Sci. Data</em> <strong>9,</strong> 340 (2022).</div>
</div>
<div id="ref-Strobl2024-kh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Strobl, F., Schmitz, A., Schetelig, M. F. &amp; Stelzer, E. H. K. A two-level staging system for the embryonic morphogenesis of the mediterranean fruit fly (medfly) ceratitis capitata. <em>PLoS One</em> <strong>19,</strong> e0316391 (2024).</div>
</div>
<div id="ref-Cheatle-Jarvela2020-go" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Cheatle Jarvela, A. M., Trelstad, C. S. &amp; Pick, L. Regulatory gene function handoff allows essential gene loss in mosquitoes. <em>Commun. Biol.</em> <strong>3,</strong> 540 (2020).</div>
</div>
<div id="ref-Cheatle-Jarvela2023-gd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Cheatle Jarvela, A. M., Trelstad, C. S. &amp; Pick, L. Anterior-posterior patterning of segments in <span><em>Anopheles stephensi</em></span> offers insights into the transition from sequential to simultaneous segmentation in holometabolous insects. <em>J. Exp. Zool. B Mol. Dev. Evol.</em> <strong>340,</strong> 116–130 (2023).</div>
</div>
<div id="ref-Jimenez-Guri2014-zs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Jiménez-Guri, E., Wotton, K. R., Gavilán, B. &amp; Jaeger, J. A staging scheme for the development of the moth midge <em><span>C</span>logmia albipunctata</em>. <em>PLoS One</em> <strong>9,</strong> e84422 (2014).</div>
</div>
<div id="ref-Nelson2016-gz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Nelson, C. M. On buckling morphogenesis. <em>J. Biomech. Eng.</em> <strong>138,</strong> 021005 (2016).</div>
</div>
<div id="ref-Gupta2021-oo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Gupta, V. K., Nam, S., Yim, D., Camuglia, J., Martin, J. L., Sanders, E. N., O’Brien, L. E., Martin, A. C., Kim, T. &amp; Chaudhuri, O. The nature of cell division forces in epithelial monolayers. <em>J. Cell Biol.</em> <strong>220,</strong> (2021).</div>
</div>
<div id="ref-Ko2020-wk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Ko, C. S., Kalakuntla, P. &amp; Martin, A. C. Apical constriction reversal upon mitotic entry underlies different morphogenetic outcomes of cell division. <em>Mol. Biol. Cell</em> <strong>31,</strong> 1663–1674 (2020).</div>
</div>
<div id="ref-Kondo2013-gy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Kondo, T. &amp; Hayashi, S. Mitotic cell rounding accelerates epithelial invagination. <em>Nature</em> <strong>494,</strong> 125–129 (2013).</div>
</div>
<div id="ref-Freddo2016-et" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Freddo, A. M., Shoffner, S. K., Shao, Y., Taniguchi, K., Grosse, A. S., Guysinger, M. N., Wang, S., Rudraraju, S., Margolis, B., Garikipati, K., Schnell, S. &amp; Gumucio, D. L. Coordination of signaling and tissue mechanics during morphogenesis of murine intestinal villi: A role for mitotic cell rounding. <em>Integr. Biol.</em> <strong>8,</strong> 918–928 (2016).</div>
</div>
<div id="ref-Bao2004-dx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Bao, L., Locovei, S. &amp; Dahl, G. Pannexin membrane channels are mechanosensitive conduits for ATP. <em>FEBS Lett.</em> <strong>572,</strong> 65–68 (2004).</div>
</div>
<div id="ref-Locovei2006-qe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Locovei, S., Wang, J. &amp; Dahl, G. Activation of pannexin 1 channels by ATP through P2Y receptors and by cytoplasmic calcium. <em>FEBS Lett.</em> <strong>580,</strong> 239–244 (2006).</div>
</div>
<div id="ref-Shah2021-gh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Shah, P., Hobson, C. M., Cheng, S., Colville, M. J., Paszek, M. J., Superfine, R. &amp; Lammerding, J. Nuclear deformation causes DNA damage by increasing replication stress. <em>Curr. Biol.</em> <strong>31,</strong> 753–765.e6 (2021).</div>
</div>
<div id="ref-Newman2000-oz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Newman, S. A. &amp; Müller, G. B. Epigenetic mechanisms of character origination. <em>J. Exp. Zool.</em> <strong>288,</strong> 304–317 (2000).</div>
</div>
<div id="ref-Gramates2022-xe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Gramates, L. S., Agapite, J., Attrill, H., Calvi, B. R., Crosby, M. A., Dos Santos, G., Goodman, J. L., Goutte-Gattat, D., Jenkins, V. K., Kaufman, T., Larkin, A., Matthews, B. B., Millburn, G., Strelets, V. B. &amp; the FlyBase Consortium. FlyBase: A guided tour of highlighted features. <em>Genetics</em> <strong>220,</strong> iyac035 (2022).</div>
</div>
<div id="ref-Vellutini2023-ng" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Vellutini, B. C., Cuenca, M. B., Krishna, A., Szałapak, A., Modes, C. D. &amp; Tomančák, P. Repository for: Patterned embryonic invagination evolved in response to mechanical instability. (2023). doi:<a href="https://doi.org/10.5281/zenodo.7781947">10.5281/zenodo.7781947</a></div>
</div>
<div id="ref-Sander1985-sg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Sander, K. Experimental egg activation in lower dipterans (<span>P</span>sychoda, <span>S</span>mittia) by low osmolality. <em>Int. J. Invertebr. Reprod. Dev.</em> <strong>8,</strong> 175–183 (1985).</div>
</div>
<div id="ref-Garcia-Solache2010-lz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">García-Solache, M., Jaeger, J. &amp; Akam, M. A systematic analysis of the gap gene system in the moth midge <span><em>C</em></span><em>logmia</em> <span><em>a</em></span><em>lbipunctata</em>. <em>Dev. Biol.</em> <strong>344,</strong> 306–318 (2010).</div>
</div>
<div id="ref-Choi2018-fg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Choi, H. M. T., Schwarzkopf, M., Fornace, M. E., Acharya, A., Artavanis, G., Stegmaier, J., Cunha, A. &amp; Pierce, N. A. Third-generation <em>in situ</em> hybridization chain reaction: Multiplexed, quantitative, sensitive, versatile, robust. <em>Development</em> <strong>145,</strong> (2018).</div>
</div>
<div id="ref-Schmied2016-oq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Schmied, C. &amp; Tomancak, P. in <em>Drosophila: Methods and protocols</em> (ed. Dahmann, C.) 189–202 (Springer New York, 2016). doi:<a href="https://doi.org/10.1007/978-1-4939-6371-3_10">10.1007/978-1-4939-6371-3_10</a></div>
</div>
<div id="ref-Schindelin2012-di" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J.-Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. &amp; Cardona, A. Fiji: An open-source platform for biological-image analysis. <em>Nat. Methods</em> <strong>9,</strong> 676–682 (2012).</div>
</div>
<div id="ref-Rueden2017-ky" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Rueden, C. T., Schindelin, J., Hiner, M. C., DeZonia, B. E., Walter, A. E., Arena, E. T. &amp; Eliceiri, K. W. ImageJ2: ImageJ for the next generation of scientific image data. <em>BMC Bioinformatics</em> <strong>18,</strong> 529 (2017).</div>
</div>
<div id="ref-Schmid2019-bm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Schmid, B., Tripal, P., Fraaß, T., Kersten, C., Ruder, B., Grüneboom, A., Huisken, J. &amp; Palmisano, R. 3Dscript: Animating 3D/4D microscopy data using a natural-language-based syntax. <em>Nat. Methods</em> <strong>16,</strong> 278–280 (2019).</div>
</div>
<div id="ref-Weigert2018-ti" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline">Weigert, M., Schmidt, U., Boothe, T., Müller, A., Dibrov, A., Jain, A., Wilhelm, B., Schmidt, D., Broaddus, C., Culley, S., Rocha-Martins, M., Segovia-Miranda, F., Norden, C., Henriques, R., Zerial, M., Solimena, M., Rink, J., Tomancak, P., Royer, L., Jug, F. &amp; Myers, E. W. Content-aware image restoration: Pushing the limits of fluorescence microscopy. <em>Nat. Methods</em> <strong>15,</strong> 1090–1097 (2018).</div>
</div>
<div id="ref-Heemskerk2015-kv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Heemskerk, I. &amp; Streichan, S. J. Tissue cartography: Compressing bio-image data by dimensional reduction. <em>Nat. Methods</em> <strong>12,</strong> 1139–1142 (2015).</div>
</div>
<div id="ref-Matlab2015-nd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">Matlab. <em>MATLAB</em>. (The MathWorks Inc., 2015). at &lt;<a href="https://www.mathworks.com/products/matlab.html">https://www.mathworks.com/products/matlab.html</a>&gt;</div>
</div>
<div id="ref-Berg2019-ab" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Berg, S., Kutra, D., Kroeger, T., Straehle, C. N., Kausler, B. X., Haubold, C., Schiegg, M., Ales, J., Beier, T., Rudy, M., Eren, K., Cervantes, J. I., Xu, B., Beuttenmueller, F., Wolny, A., Zhang, C., Koethe, U., Hamprecht, F. A. &amp; Kreshuk, A. <span class="nocase">i</span>lastik: Interactive machine learning for (bio)image analysis. <em>Nat. Methods</em> (2019). doi:<a href="https://doi.org/10.1038/s41592-019-0582-9">10.1038/s41592-019-0582-9</a></div>
</div>
<div id="ref-Vellutini2022-ya" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Vellutini, B. C. <em>How to make cartographic projections using <span>ImSAnE</span></em>. (Zenodo, 2022). doi:<a href="https://doi.org/10.5281/zenodo.7628299">10.5281/zenodo.7628299</a></div>
</div>
<div id="ref-Schmied2020-ip" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Schmied, C. &amp; Jambor, H. K. Effective image visualization for publications - a workflow using open access tools and concepts. <em>F1000Res.</em> <strong>9,</strong> 1373 (2020).</div>
</div>
<div id="ref-Arganda-Carreras2006-im" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">Arganda-Carreras, I., Sorzano, C. O. S., Marabini, R., Carazo, J. M., Ortiz-de-Solorzano, C. &amp; Kybic, J. in <em>Computer vision approaches to medical image analysis</em> 85–95 (Springer Berlin Heidelberg, 2006). doi:<a href="https://doi.org/10.1007/11889762_8">10.1007/11889762_8</a></div>
</div>
<div id="ref-Legland2016-cp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Legland, D., Arganda-Carreras, I. &amp; Andrey, P. MorphoLibJ: Integrated library and plugins for mathematical morphology with <span>I</span>mage<span>J</span>. <em>Bioinformatics</em> <strong>32,</strong> 3532–3534 (2016).</div>
</div>
<div id="ref-Arganda-Carreras2010-la" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline">Arganda-Carreras, I., Fernández-González, R., Muñoz-Barrutia, A. &amp; Ortiz-De-Solorzano, C. 3D reconstruction of histological sections: Application to mammary gland tissue. <em>Microsc. Res. Tech.</em> <strong>73,</strong> 1019–1029 (2010).</div>
</div>
<div id="ref-Tseng2012-ni" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline">Tseng, Q., Duchemin-Pelletier, E., Deshiere, A., Balland, M., Guillou, H., Filhol, O. &amp; Théry, M. Spatial organization of the extracellular matrix regulates cell-cell junction positioning. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>109,</strong> 1506–1511 (2012).</div>
</div>
<div id="ref-Krishna2023-hu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline">Krishna, A., Szałapak, A., Vellutini, B. C., Cuenca, M. B., Tomančák, P. &amp; Modes, C. D. Model and simulations for: Patterned embryonic invagination evolved in response to mechanical instability. (2023). doi:<a href="https://doi.org/10.5281/zenodo.7784906">10.5281/zenodo.7784906</a></div>
</div>
<div id="ref-The_Inkscape_Project2003-ht" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline">The Inkscape Project. <em>Inkscape: Open source scalable vector graphics editor</em>. (2003). at &lt;<a href="https://inkscape.org">https://inkscape.org</a>&gt;</div>
</div>
<div id="ref-HandBrake_Team2003-fq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline">HandBrake Team. <em>HandBrake: HandBrake’s main development repository</em>. (Github, 2003). at &lt;<a href="https://github.com/HandBrake/HandBrake">https://github.com/HandBrake/HandBrake</a>&gt;</div>
</div>
<div id="ref-Vellutini2023-ix" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline">Vellutini, B. C., Cuenca, M. B., Krishna, A., Szałapak, A., Modes, C. D. &amp; Tomančák, P. Figures and videos for: Patterned embryonic invagination evolved in response to mechanical instability. (2023). doi:<a href="https://doi.org/10.5281/zenodo.7781916">10.5281/zenodo.7781916</a></div>
</div>
<div id="ref-R-Core-Team1993-jw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline">R Core Team. <em>R: A language and environment for statistical computing</em>. (R Foundation for Statistical Computing, 1993). at &lt;<a href="http://www.r-project.org">http://www.r-project.org</a>&gt;</div>
</div>
<div id="ref-RStudio-Team2011-vj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline">RStudio Team. <em>RStudio: Integrated development environment for r</em>. (RStudio, PBC., 2011). at &lt;<a href="http://www.rstudio.com">http://www.rstudio.com</a>&gt;</div>
</div>
<div id="ref-Granger2021-zg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline">Granger, B. E. &amp; Perez, F. Jupyter: Thinking and storytelling with code and data. <em>Comput. Sci. Eng.</em> <strong>23,</strong> 7–14 (2021).</div>
</div>
<div id="ref-Campos-Ortega1985-ma" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline">Campos-Ortega, J. A. &amp; Hartenstein, V. <em>The embryonic development of <em><span>D</span>rosophila <span class="nocase">m</span>elanogaster</em></em>. (Springer Berlin Heidelberg, 1985). doi:<a href="https://doi.org/10.1007/978-3-662-02454-6">10.1007/978-3-662-02454-6</a></div>
</div>
<div id="ref-Ashburner2005-ym" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline">Ashburner, M., Golic, K. G. &amp; Hawley, R. S. <em><em><span>D</span>rosophila</em>: A laboratory handbook</em>. (Cold Spring Harbor Laboratory Press, 2005).</div>
</div>
<div id="ref-Wiegmann2011-ou" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline">Wiegmann, B. M., Trautwein, M. D., Winkler, I. S., Barr, N. B., Kim, J.-W., Lambkin, C., Bertone, M. A., Cassel, B. K., Bayless, K. M., Heimberg, A. M., Wheeler, B. M., Peterson, K. J., Pape, T., Sinclair, B. J., Skevington, J. H., Blagoderov, V., Caravas, J., Kutty, S. N., Schmidt-Ott, U., Kampmeier, G. E., Thompson, F. C., Grimaldi, D. A., Beckenbach, A. T., Courtney, G. W., Friedrich, M., Meier, R. &amp; Yeates, D. K. Episodic radiations in the fly tree of life. <em>Proc. Natl. Acad. Sci. U. S. A.</em> <strong>108,</strong> 5690–5695 (2011).</div>
</div>
<div id="ref-Tomancak2002-yf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline">Tomancak, P., Beaton, A., Weiszmann, R., Kwan, E., Shu, S., Lewis, S. E., Richards, S., Ashburner, M., Hartenstein, V., Celniker, S. E. &amp; Rubin, G. M. Systematic determination of patterns of gene expression during <span><em>D</em></span><em>rosophila</em> embryogenesis. <em>Genome Biol.</em> <strong>3,</strong> research0088.1 (2002).</div>
</div>
<div id="ref-Tomancak2007-db" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline">Tomancak, P., Berman, B. P., Beaton, A., Weiszmann, R., Kwan, E., Hartenstein, V., Celniker, S. E. &amp; Rubin, G. M. Global analysis of patterns of gene expression during <span><em>D</em></span><em>rosophila</em> embryogenesis. <em>Genome Biol.</em> <strong>8,</strong> R145 (2007).</div>
</div>
<div id="ref-Lecuyer2007-uy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline">Lécuyer, E., Yoshida, H., Parthasarathy, N., Alm, C., Babak, T., Cerovina, T., Hughes, T. R., Tomancak, P. &amp; Krause, H. M. Global analysis of <span class="nocase">mRNA</span> localization reveals a prominent role in organizing cellular architecture and function. <em>Cell</em> <strong>131,</strong> 174–187 (2007).</div>
</div>
<div id="ref-Andrioli2004-qi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline">Andrioli, L. P., Oberstein, A. L., Corado, M. S. G., Yu, D. &amp; Small, S. Groucho-dependent repression by <span>S</span>loppy-paired 1 differentially positions anterior pair-rule stripes in the <span><em>D</em></span><em>rosophila</em> embryo. <em>Dev. Biol.</em> <strong>276,</strong> 541–551 (2004).</div>
</div>
</div>
<div style=""></div>
<h1 id="figures">Figures</h1>
<!--TODO: Reduce legends adding relevant information-->
<!--FIGURE LEGENDS: These should be listed sequentially after the main text references and not in the figure files. Each legend should begin with a brief title for the whole figure and continue with a short description of each panel and the symbols used. Legends should not exceed 300 words each. Each figure legend should contain, for each panel where relevant, the following information:-->
<h2 id="figure-1">Figure 1</h2>
<!--TODO: Delete or move the original long legend-->
<!--Formation of ectopic folds in cephalic furrow mutants.-->
<!--**a**, Overview of key developmental events in the different genetic backgrounds. Cephalic furrow formation is delayed in *prd* mutants and absent in *btd* and *eve* mutants. Ectopic folds form around the head--trunk boundary, appearing later and unfolding quicker than the cephalic furrow. Their formation coincides with the appearance of mitotic domains and rapid phase of germ band extension.-->
<!--**b**, Profile views of wildtype and *prd* embryos at early stage 8 and *btd* and *eve* embryos at late stage 8, showing the divergent morphologies of the cephalic furrow and ectopic folds. Scale bars = 20\ µm.-->
<!--**c**, Lateral view of sibling controls (heterozygotes) and mutant embryos (*btd* or *eve* homozygotes). Arrows indicate epithelial folds. Asterisks indicate the position of mitotic domains. Percentages indicate the extent of germ band extension relative to egg length. Scale bars = 50\ µm.-->
<!--**d**, Timing of cephalic furrow formation and ectopic folding in different genetic backgrounds. The cephalic furrow forms about 7 minutes after gastrulation in wildtype (n = 16). *btd* heterozygotes show no difference (n = 6, p = 0.283), while *eve* heterozygotes show a slight delay (n = 7, p < 0.001); *prd* heterozygotes are variable (n = 10). In homozygotes, the ectopic folds form delayed relative to wildtype cephalic furrow formation, about 20\ min after gastrulation, in *btd* (n = 5, p = 0.001), *eve* (n = 5, p = 0.001), and *prd* (n = 5, p = 0.003).-->
<!--**e**, Percentage of germ band extension at the time of cephalic furrow and ectopic fold formation. While the cephalic furrow appears at 10% extension, the ectopic folds form at 30--35% extension in *btd*, *eve*, and *prd* homozygotes (p < 0.001). n numbers are the same as in **d**.-->
<!--**f**, Variability in the positioning of the cephalic furrow in *btd* heterozygotes (n = 6) and ectopic folds in *btd* mutants (n = 5). Scale bars = 50\ µm.-->
<!--**g**, Folded area (yellow outline) of the cephalic furrow (CF) in a *btd* heterozygote (left, n = 6) and of ectopic folds (EF) in a *btd* homozygote (right, n = 5). The numbers indicate the average folded area in µm^2^ for each condition. Cartographic projections of lateral view datasets. Scale bars = 50\ µm (approximated value since pixel sizes vary across the projection).-->
<!--**h**, Total area of tissues folded in the cephalic furrow and ectopic folds. While the folded area of *btd* heterozygotes (n = 6) is similar to wildtype embryos (n = 16, p = 0.133), it is reduced in *eve* (n = 7, p < 0.001) and *prd* (n = 9, p < 0.001) heterozygotes. Ectopic folds in *btd* (n = 5), *eve* (n = 4), and *prd* (n = 5) homozygotes have a smaller folded area compared to the wildtype cephalic furrow (p < 0.001).-->
<!--In all box plots, the center depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range; each data point represents one embryo; asterisks indicate p < 0.05 in a two-sided Mann--Whitney *U* test between condition and wildtype.-->
<!--The presence of ectopic folds in the absence of the cephalic furrow was highly repeatable across 7 independent experiments in *btd* mutants (total of 46 embryos) and 5 independent experiments in *eve* mutants (total of 36 embryos).-->
<!--The membrane marker in panels **b**, **c**, **f**, and **g** is Gap43-mCherry.-->
<!--Embryonic staging based on standard developmental tables [@Campos-Ortega1985-ma; @Ashburner2005-ym].-->
<div id="fig:ectopic-folding" class="fignos">
<figure>
<img src="figures/Fig1.jpg" style="width:100.0%" alt="Figure 1: Formation of ectopic folds in cephalic furrow mutants. a, Timing of developmental events in wildtype compared to prd, btd, and eve mutants. b, Profile view of the cephalic furrow in wildtype and prd mutants at early stage 8 and of ectopic folds in btd and eve mutants at late stage 8. Arrows indicate epithelial folds and asterisks indicate mitotic domains. Scale bars = 20 µm. c, Lateral view of heterozygote (sibling controls) homozygote embryos (mutants) of btd and eve during gastrulation. Percentages indicate the extent of germ band extension relative to egg length. Scale bars = 50 µm. d, Differences in the timing of cephalic furrow formation between wildtype (n = 16) and heterozygote embryos of btd (n = 6, p = 0.283), eve (n = 7, p &lt; 0.001), and prd (n = 10); and between the wildtype cephalic furrow and the ectopic folds in homozygote embryos of btd (n = 5, p = 0.001), eve (n = 5, p = 0.001), and prd (n = 5, p = 0.003). e, Differences in the percentage of germ band extension at the time of cephalic furrow in wildtype and ectopic fold formation in btd, eve, and prd homozygotes (p &lt; 0.001). n numbers are the same as in d. f, Variability in the position of the cephalic furrow and ectopic folds in btd heterozygotes (n = 6) and homozygotes (n = 5), respectively. Scale bars = 50 µm. g, Cartographic projections of lateral views showing the folded area (yellow outline) of the cephalic furrow (CF) in a btd heterozygote (n = 6) and of ectopic folds (EF) in a btd homozygote (n = 5). Numbers indicate the average and standard deviation for each condition. Scale bars ≈ 50 µm (pixel size varies across the projection). h, Differences in the total folded area of the cephalic furrow between wildtype (n = 16) and heterozygote embryos of btd (n = 6, p = 0.133) eve (n = 7, p &lt; 0.001), and prd (n = 9, p &lt; 0.001); and between the wildtype cephalic furrow and the ectopic folds in homozygote embryos of btd (n = 5), eve (n = 4), and prd (n = 5) (p &lt; 0.001 for each). In box plots, the center depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. Each data point represents one embryo. Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney U test contrasting the condition against wildtype. The presence of ectopic folds was highly repeatable across 7 independent experiments in btd mutants (46 embryos) and 5 independent experiments in eve mutants (36 embryos). The membrane marker in panels b, c, f, and g is Gap43-mCherry. Embryonic staging based on standard developmental tables.61,62" /><figcaption aria-hidden="true"><span>Figure 1:</span> Formation of ectopic folds in cephalic furrow mutants. <strong>a</strong>, Timing of developmental events in wildtype compared to <em>prd</em>, <em>btd</em>, and <em>eve</em> mutants. <strong>b</strong>, Profile view of the cephalic furrow in wildtype and <em>prd</em> mutants at early stage 8 and of ectopic folds in <em>btd</em> and <em>eve</em> mutants at late stage 8. Arrows indicate epithelial folds and asterisks indicate mitotic domains. Scale bars = 20 µm. <strong>c</strong>, Lateral view of heterozygote (sibling controls) homozygote embryos (mutants) of <em>btd</em> and <em>eve</em> during gastrulation. Percentages indicate the extent of germ band extension relative to egg length. Scale bars = 50 µm. <strong>d</strong>, Differences in the timing of cephalic furrow formation between wildtype (n = 16) and heterozygote embryos of <em>btd</em> (n = 6, p = 0.283), <em>eve</em> (n = 7, p &lt; 0.001), and <em>prd</em> (n = 10); and between the wildtype cephalic furrow and the ectopic folds in homozygote embryos of <em>btd</em> (n = 5, p = 0.001), <em>eve</em> (n = 5, p = 0.001), and <em>prd</em> (n = 5, p = 0.003). <strong>e</strong>, Differences in the percentage of germ band extension at the time of cephalic furrow in wildtype and ectopic fold formation in <em>btd</em>, <em>eve</em>, and <em>prd</em> homozygotes (p &lt; 0.001). n numbers are the same as in <strong>d</strong>. <strong>f</strong>, Variability in the position of the cephalic furrow and ectopic folds in <em>btd</em> heterozygotes (n = 6) and homozygotes (n = 5), respectively. Scale bars = 50 µm. <strong>g</strong>, Cartographic projections of lateral views showing the folded area (yellow outline) of the cephalic furrow (CF) in a <em>btd</em> heterozygote (n = 6) and of ectopic folds (EF) in a <em>btd</em> homozygote (n = 5). Numbers indicate the average and standard deviation for each condition. Scale bars ≈ 50 µm (pixel size varies across the projection). <strong>h</strong>, Differences in the total folded area of the cephalic furrow between wildtype (n = 16) and heterozygote embryos of <em>btd</em> (n = 6, p = 0.133) <em>eve</em> (n = 7, p &lt; 0.001), and <em>prd</em> (n = 9, p &lt; 0.001); and between the wildtype cephalic furrow and the ectopic folds in homozygote embryos of <em>btd</em> (n = 5), <em>eve</em> (n = 4), and <em>prd</em> (n = 5) (p &lt; 0.001 for each). In box plots, the center depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. Each data point represents one embryo. Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney <em>U</em> test contrasting the condition against wildtype. The presence of ectopic folds was highly repeatable across 7 independent experiments in <em>btd</em> mutants (46 embryos) and 5 independent experiments in <em>eve</em> mutants (36 embryos). The membrane marker in panels <strong>b</strong>, <strong>c</strong>, <strong>f</strong>, and <strong>g</strong> is Gap43-mCherry. Embryonic staging based on standard developmental tables.<span class="citation" data-cites="Campos-Ortega1985-ma Ashburner2005-ym"><sup><a href="#ref-Campos-Ortega1985-ma" role="doc-biblioref">61</a>,<a href="#ref-Ashburner2005-ym" role="doc-biblioref">62</a></sup></span></figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-2">Figure 2</h2>
<!--TODO: Remove or move old long figure legend-->
<!--Perturbation experiments *in vivo* using cephalic furrow mutants and wildtype embryos.-->
<!--**a**, Lateral and profile views of *stg* mutants showing the formation of the cephalic furrow and germ band extension are unaffected. This phenotype is highly repeatable based on 3 independent experiments in *stg* mutants (46 embryos). Scale bars = 50\ µm and 20\ µm.-->
<!--**b**, Lateral and profile views of *btd* mutants showing the formation of ectopic folds. Samples already described in {@fig:ectopic-folding}c,b. Scale bars = 50\ µm and 20\ µm.-->
<!--**c**, Lateral and profile views of *btd*--*stg* double mutants, which lack both the cephalic furrow and mitotic domains, showing the absence of ectopic folds at the head--trunk interface. This phenotype was consistent in 5 double homozygote embryos obtained from 6 independent experiments with *btd*--*stg* mutants (35 embryos). Scale bar = 50\ µm and 20\ µm.-->
<!--**d**, Cauterized wildtype embryo in lateral (left) and dorsal view (right). Cauterization site is outlined in orange. The germ band is blocked from extending, but the cephalic furrow formation occurs normally. We only obtained one successful cauterization in a wildtype embryo under multiview lightsheet microscopy. We corroborated and quantified the phenotype in different experiments under confocal microscopy (**e**). Scale bar = 50\ µm.-->
<!--**e**, Profile view of a non-cauterized and cauterized wildtype embryos showing a small difference in depth. See **j** for the quantification. Scale bar = 20\ µm.-->
<!--**f**, Cauterized *eve* mutant in lateral (left) and dorsal view (right) at the maximum expansion of dividing cells in mitotic domains. Cauterization site is outlined in orange. The boxed outline (right) shows panel **g**. We only obtained one successful cauterization in an *eve* embryo under multiview lightsheet microscopy. We corroborated and quantified the phenotype in different experiments under confocal microscopy (**h**). Scale bar = 50\ µm.-->
<!--**g**, Inset from panel **f** showing the progressive expansion of mitotic domains compressing the non-dividing cells between MD1 and MD5. Scale bar = 50\ µm.-->
<!--**h** Profile view and epithelial trace of non-cauterized and cauterized *btd* and *eve* embryos. The trace shows the dynamics of epithelial deformations over time, colored from purple to yellow. Cauterized mutants show no ectopic folds and less deformation of the epithelium. The difference in tortuosity was consistent across 6 and 4 independent experiments in *btd* (6 embryos) and *eve* (7 embryos), respectively, where the cauterization successfully blocked the germ band extension. Scale bars = 20\ µm.-->
<!--**i**, Recoil velocity of laser ablations around the cephalic furrow at stage 6. The average recoil of anterior cuts increases with time for early (n = 11), mid (n = 13), and late (n = 12) stages. In posterior cuts, it increases between early (n = 7) and mid (n = 14) stages and decreases in the late stage (n = 10). The average recoil velocity is higher closer to the initiator cells (20\ µm). The center of the box plot depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range.-->
<!--**j**, Maximum depth of the cephalic furrow in cauterized embryos. We pooled wildtype and heterozygote embryos for non-cauterized (n = 7 total from wt = 3, *btd* = 1, and *eve* = 3) and cauterized (n = 3 total from wt = 2 and *btd* = 1) samples. The cephalic furrow in cauterized embryos is 15% shallower (p = 0.058 in a one-sided Mann--Whitney *U* test). The box plot representation is the same as in **i**.-->
<!--**k**, Tortuosity of the epithelial traces in non-cauterized (n = 2) and cauterized (n = 3) *btd* embryos, and non-cauterized (n = 3) and cauterized (n = 4) *eve* embryos from **h**. We measured both the left and right sides of each embryo. The plots show the mean predicted values from the regression of each condition with a shaded band indicating the 95% confidence interval for the mean trend.-->
<div id="fig:live-experiments" class="fignos">
<figure>
<img src="figures/Fig2.jpg" style="width:100.0%" alt="Figure 2: Experiments in cephalic furrow mutants in vivo. a, stg mutants. The phenotype was highly repeatable in 3 independent experiments (46 embryos). Scale bars = 50 µm. b, btd mutants. Samples described in Fig. 1c,b. Scale bars = 50 µm. c, btd–stg double mutants. The phenotype was consistent in 5 double homozygote embryos from 6 independent experiments (35 embryos). Scale bar = 50 µm. d, Germ band cauterization (orange) in wildtype embryo under multiview lightsheet microscopy. We quantified and corroborated the phenotype under confocal microscopy (see h). Scale bar = 50 µm. e, Cephalic furrow in non-cauterized and cauterized wildtype embryos. See j for quantification. Scale bar = 20 µm. f, Germ band cauterization (orange) in eve mutant under multiview lightsheet microscopy. We quantified and corroborated the phenotype under confocal microscopy (h). Scale bar = 50 µm. g, Inset from f showing compressed non-dividing cells between mitotic domains. Scale bar = 50 µm. h Germ band cauterizations in btd and eve mutants under confocal microscopy showing a trace with the epithelial deformations over time. The phenotype was was consistent across 6 and 4 independent experiments in btd (6 embryos) and eve (7 embryos), respectively. Scale bars = 20 µm. i, Cell membrane recoil velocity from laser ablations around the cephalic furrow in gastrulating wildtype embryos. Anterior cuts (green) for early (n = 11), mid (n = 13), and late (n = 12) stage 6, and posterior cuts (magenta) for early (n = 7), mid (n = 14), and late (n = 10) stage 6. The box plot center shows the median, the lower/upper hinges correspond to the first/third quartiles, and the whiskers extend 1.5 times the inter-quartile range. j, Maximum depth of the cephalic furrow with germ band cauterization (pooled wildtype and heterozygote embryos). Non-cauterized (n = 7, from wt = 3, btd = 1, eve = 3) and cauterized (n = 3, from wt = 2 and btd = 1). Fold in cauterized embryos is 15% shallower, but the difference is not statistically significant (p = 0.058, one-sided Mann–Whitney U test). Box plot as in i. k, Tortuosity of epithelial traces in cauterized mutants from h. btd non-cauterized (n = 2) and cauterized (n = 3), and eve non-cauterized (n = 3) and cauterized (n = 4). Plots show mean predicted values from regression with a 95% confidence interval shaded band." /><figcaption aria-hidden="true"><span>Figure 2:</span> Experiments in cephalic furrow mutants <em>in vivo</em>. <strong>a</strong>, <em>stg</em> mutants. The phenotype was highly repeatable in 3 independent experiments (46 embryos). Scale bars = 50 µm. <strong>b</strong>, <em>btd</em> mutants. Samples described in Fig. <a href="#fig:ectopic-folding">1</a>c,b. Scale bars = 50 µm. <strong>c</strong>, <em>btd</em>–<em>stg</em> double mutants. The phenotype was consistent in 5 double homozygote embryos from 6 independent experiments (35 embryos). Scale bar = 50 µm. <strong>d</strong>, Germ band cauterization (orange) in wildtype embryo under multiview lightsheet microscopy. We quantified and corroborated the phenotype under confocal microscopy (see <strong>h</strong>). Scale bar = 50 µm. <strong>e</strong>, Cephalic furrow in non-cauterized and cauterized wildtype embryos. See <strong>j</strong> for quantification. Scale bar = 20 µm. <strong>f</strong>, Germ band cauterization (orange) in <em>eve</em> mutant under multiview lightsheet microscopy. We quantified and corroborated the phenotype under confocal microscopy (<strong>h</strong>). Scale bar = 50 µm. <strong>g</strong>, Inset from <strong>f</strong> showing compressed non-dividing cells between mitotic domains. Scale bar = 50 µm. <strong>h</strong> Germ band cauterizations in <em>btd</em> and <em>eve</em> mutants under confocal microscopy showing a trace with the epithelial deformations over time. The phenotype was was consistent across 6 and 4 independent experiments in <em>btd</em> (6 embryos) and <em>eve</em> (7 embryos), respectively. Scale bars = 20 µm. <strong>i</strong>, Cell membrane recoil velocity from laser ablations around the cephalic furrow in gastrulating wildtype embryos. Anterior cuts (green) for early (n = 11), mid (n = 13), and late (n = 12) stage 6, and posterior cuts (magenta) for early (n = 7), mid (n = 14), and late (n = 10) stage 6. The box plot center shows the median, the lower/upper hinges correspond to the first/third quartiles, and the whiskers extend 1.5 times the inter-quartile range. <strong>j</strong>, Maximum depth of the cephalic furrow with germ band cauterization (pooled wildtype and heterozygote embryos). Non-cauterized (n = 7, from wt = 3, <em>btd</em> = 1, <em>eve</em> = 3) and cauterized (n = 3, from wt = 2 and <em>btd</em> = 1). Fold in cauterized embryos is 15% shallower, but the difference is not statistically significant (p = 0.058, one-sided Mann–Whitney <em>U</em> test). Box plot as in <strong>i</strong>. <strong>k</strong>, Tortuosity of epithelial traces in cauterized mutants from <strong>h</strong>. <em>btd</em> non-cauterized (n = 2) and cauterized (n = 3), and <em>eve</em> non-cauterized (n = 3) and cauterized (n = 4). Plots show mean predicted values from regression with a 95% confidence interval shaded band.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-3">Figure 3</h2>
<!--TODO: Add n = 20 for simulations-->
<!--TODO: Copy dfs folders to local github repo-->
<!--TODO: Structure directories for github repo-->
<!--TODO: Copy modeslab/data_version_5 to cephalic furrow fileserver-->
<div id="fig:fold-model" class="fignos">
<figure>
<img src="figures/Fig3.jpg" style="width:100.0%" alt="Figure 3: Model and simulations of the tissue mechanics at the head–trunk boundary. a, Region of interest of the model. From a frontal slice across a dorsal plane, we select one side of the blastoderm, following the embryo’s shape, proportions, and relative position of mitotic domains, cephalic furrow, and germ band. b, Characteristics of the model components which is based on particles connected by springs. Drawings exemplify the initial and final states for each component. c, Energy equation with a stretching and a bending component, and the dimensionless bending rigidity. Stress-free rod length (L_o), total energy per unit length (W_T), stretching energy per unit length (W_s), bending energy per unit length (W_b), stretching rigidity (K_s), bending rigidity (K_b), preferred spring length (ds_o), current spring length (ds), preferred curvature (\kappa_o), current curvature (\kappa), semi-major embryonic axis (L). d, Plots showing the energy dynamics across iterations in a typical simulation. Total energy goes down to equilibrium. Bending energy increases drastically reaching a peak (blue dashed line) that diminishes gradually with each iteration. Energy values are normalized by the initial total energy. The number of folds stabilizes when the bending energy peaks but the fold depth continues to increase until the last iteration (pink dashed line). The line represents the mean value and the shaded area the standard deviation across simulations (n = 20). e, Parameter sweep for a mutant condition (no cephalic furrow) without mitotic domains. The heatmap shows the average number of ectopic folds for different bending rigidities and percentages of germ band extension. Ectopic folding frequency increases with lower bending rigidities (softer tissues) and with greater values of germ band extension. Outlined in white are the baseline conditions with neither mitotic domains nor germ band but only ground level noise (i), and the germ-band-only condition with more folding events (ii). A visual rendering of representative simulations are shown below. f, Parameter sweep for a mutant condition (no cephalic furrow) with mitotic domains. The phase diagram shows an increase in number of folds in relation to e. The addition of mitotic domains induces the formation of ectopic folds even without germ band extension (iii). Representative simulations are shown below. g, Quantification of ectopic folding to evaluate the effectiveness of the cephalic furrow. A mutant condition with mitotic domains and germ band extension show extensive ectopic folding (top row). With the presence of the cephalic furrow, ectopic folding decreases around the head–trunk region, especially in lower percentage of germ band extension (middle row). Adding a delay of t_{MD}=5 to mitotic domain formation to mimic the in vivo condition results in less frequent ectopic folding throughout the blastoderm and germ band extension (bottom row). t_{MD}=1 corresponds to 105 computational timesteps. The black line represents the mean values and the gray shaded area the standard deviation of the data across simulations (n = 20). h, Quantification of ectopic folding with the cephalic furrow at different positions along the anteroposterior axis. Displacing the cephalic furrow more anteriorly results in ectopic folding at the posterior end (top row) while displacing it more posteriorly results in more ectopic folding at the anterior region (bottom row). Placing the cephalic furrow around the central region prevents ectopic folding more broadly. The black line represents the mean values and the gray shaded area the standard deviation of the data across simulations (n = 20)." /><figcaption aria-hidden="true"><span>Figure 3:</span> Model and simulations of the tissue mechanics at the head–trunk boundary. <strong>a</strong>, Region of interest of the model. From a frontal slice across a dorsal plane, we select one side of the blastoderm, following the embryo’s shape, proportions, and relative position of mitotic domains, cephalic furrow, and germ band. <strong>b</strong>, Characteristics of the model components which is based on particles connected by springs. Drawings exemplify the initial and final states for each component. <strong>c</strong>, Energy equation with a stretching and a bending component, and the dimensionless bending rigidity. Stress-free rod length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>L</mi><mi>o</mi></msub><annotation encoding="application/x-tex">L_o</annotation></semantics></math>), total energy per unit length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>T</mi></msub><annotation encoding="application/x-tex">W_T</annotation></semantics></math>), stretching energy per unit length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>s</mi></msub><annotation encoding="application/x-tex">W_s</annotation></semantics></math>), bending energy per unit length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>b</mi></msub><annotation encoding="application/x-tex">W_b</annotation></semantics></math>), stretching rigidity (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>s</mi></msub><annotation encoding="application/x-tex">K_s</annotation></semantics></math>), bending rigidity (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math>), preferred spring length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><msub><mi>s</mi><mi>o</mi></msub></mrow><annotation encoding="application/x-tex">ds_o</annotation></semantics></math>), current spring length (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>d</mi><mi>s</mi></mrow><annotation encoding="application/x-tex">ds</annotation></semantics></math>), preferred curvature (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>κ</mi><mi>o</mi></msub><annotation encoding="application/x-tex">\kappa_o</annotation></semantics></math>), current curvature (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>κ</mi><annotation encoding="application/x-tex">\kappa</annotation></semantics></math>), semi-major embryonic axis (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>L</mi><annotation encoding="application/x-tex">L</annotation></semantics></math>). <strong>d</strong>, Plots showing the energy dynamics across iterations in a typical simulation. Total energy goes down to equilibrium. Bending energy increases drastically reaching a peak (blue dashed line) that diminishes gradually with each iteration. Energy values are normalized by the initial total energy. The number of folds stabilizes when the bending energy peaks but the fold depth continues to increase until the last iteration (pink dashed line). The line represents the mean value and the shaded area the standard deviation across simulations (n = 20). <strong>e</strong>, Parameter sweep for a mutant condition (no cephalic furrow) without mitotic domains. The heatmap shows the average number of ectopic folds for different bending rigidities and percentages of germ band extension. Ectopic folding frequency increases with lower bending rigidities (softer tissues) and with greater values of germ band extension. Outlined in white are the baseline conditions with neither mitotic domains nor germ band but only ground level noise (i), and the germ-band-only condition with more folding events (ii). A visual rendering of representative simulations are shown below. <strong>f</strong>, Parameter sweep for a mutant condition (no cephalic furrow) with mitotic domains. The phase diagram shows an increase in number of folds in relation to <strong>e</strong>. The addition of mitotic domains induces the formation of ectopic folds even without germ band extension (iii). Representative simulations are shown below. <strong>g</strong>, Quantification of ectopic folding to evaluate the effectiveness of the cephalic furrow. A mutant condition with mitotic domains and germ band extension show extensive ectopic folding (top row). With the presence of the cephalic furrow, ectopic folding decreases around the head–trunk region, especially in lower percentage of germ band extension (middle row). Adding a delay of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>5</mn></mrow><annotation encoding="application/x-tex">t_{MD}=5</annotation></semantics></math> to mitotic domain formation to mimic the <em>in vivo</em> condition results in less frequent ectopic folding throughout the blastoderm and germ band extension (bottom row). <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">t_{MD}=1</annotation></semantics></math> corresponds to 10<sup>5</sup> computational timesteps. The black line represents the mean values and the gray shaded area the standard deviation of the data across simulations (n = 20). <strong>h</strong>, Quantification of ectopic folding with the cephalic furrow at different positions along the anteroposterior axis. Displacing the cephalic furrow more anteriorly results in ectopic folding at the posterior end (top row) while displacing it more posteriorly results in more ectopic folding at the anterior region (bottom row). Placing the cephalic furrow around the central region prevents ectopic folding more broadly. The black line represents the mean values and the gray shaded area the standard deviation of the data across simulations (n = 20).</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-4">Figure 4</h2>
<!--**a**, Lateral view of *slp* heterozygote (control) and mutant embryo at the onset of initiator cell behavior. In *slp* mutants, the cephalic furrow formation is delayed; it happens when the germ band is extended about 18% of egg length. The position of initiator cells is shifted forward in mutants to about 73% of egg length. The asterisk indicates mitotic cells. The forward-shifted phenotype was consistent on 6 independent experiments in *slp* mutants (39 embryos). Scale bars = 50\ µm.-->
<!--**b**, Plots showing the position of the cephalic furrow (CF) and germ band (GB) at the onset of initiator cell behavior in *slp* mutants. The center of the box plot depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range.-->
<div id="fig:genetic-patterning" class="fignos">
<figure>
<img src="figures/Fig4.jpg" style="width:100.0%" alt="Figure 4: Genetic patterning of the head–trunk boundary in Drosophila, Ceratitis, Anopheles, and Clogmia. a, Progression of the early expression of btd, eve, and slp1 around the nuclear cycle 14 in Drosophila. slp1 and eve initially demarcate the head–trunk boundary in broad domains which become segmented, narrow and resolve into sharp, non-overlapping stripes. btd transcripts are localized at the head–trunk interface. The expression pattern was repeatable across 3 experiments in wildtype samples (26 embryos). Scale bars = 50 µm. b, Lateral view of the expression at the head–trunk boundary in Drosophila. slp1 stripes demarcate the outer edges of the cephalic furrow (dashed lines). Scale bar = 50 µm. c, Profile view of the gene expression in initiator cells of Drosophila. The eve-expressing row is abutted anteriorly and posteriorly by non-overlapping slp1 stripes. btd and eve overlap by 1-cell row. prd expression is offset with slp1. Dashed lines mark the future edges of the cephalic furrow. Scale bar = 20 µm. d, Profile view of the gene expression after cephalic furrow invagination in Drosophila. The eve-expressing initiator cells are located deep into the yolk, while the slp head domain (sh) and stripe 3 demarcate the outer edges of the invagination. Scale bar = 20 µm. e, Schematic drawings showing the combinatorial molecular arrangement at the head–trunk boundary of Drosophila. f, Gene expression domains of btd, eve and slp during nuclear cycles 13 and 14 in Ceratitis, Anopheles, and Clogmia. The patterns were consistent across 3 experiments in Ceratitis (38 embryos), 4 experiments in Anopheles (43 embryos), and 4 experiments in Clogmia (44 embryos). Scale bars = 100 µm. g, Comparison of expression patterns at the head–trunk boundary before gastrulation between species. The expression of slp and eve are similar across all species. Both Drosophila and Ceratitis, species with a cephalic furrow, show a head–trunk domain of btd that overlaps with eve stripe 1. In Anopheles, the head–trunk btd domain does not overlap with eve stripe 1, and in Clogmia this btd domain is absent. Scale bars = 50 µm. h, Profile view of the gene expression at the head–trunk boundary cells in Ceratitis, Anopheles, and Clogmia. Scale bars = 20 µm. ac: btd acron domain, sh: slp head domain, ht: btd head–trunk domain, fg: btd foregut domain." /><figcaption aria-hidden="true"><span>Figure 4:</span> Genetic patterning of the head–trunk boundary in <em>Drosophila</em>, <em>Ceratitis</em>, <em>Anopheles</em>, and <em>Clogmia</em>. <strong>a</strong>, Progression of the early expression of <em>btd</em>, <em>eve</em>, and <em>slp1</em> around the nuclear cycle 14 in <em>Drosophila</em>. <em>slp1</em> and <em>eve</em> initially demarcate the head–trunk boundary in broad domains which become segmented, narrow and resolve into sharp, non-overlapping stripes. <em>btd</em> transcripts are localized at the head–trunk interface. The expression pattern was repeatable across 3 experiments in wildtype samples (26 embryos). Scale bars = 50 µm. <strong>b</strong>, Lateral view of the expression at the head–trunk boundary in <em>Drosophila</em>. <em>slp1</em> stripes demarcate the outer edges of the cephalic furrow (dashed lines). Scale bar = 50 µm. <strong>c</strong>, Profile view of the gene expression in initiator cells of <em>Drosophila</em>. The <em>eve</em>-expressing row is abutted anteriorly and posteriorly by non-overlapping <em>slp1</em> stripes. <em>btd</em> and <em>eve</em> overlap by 1-cell row. <em>prd</em> expression is offset with <em>slp1</em>. Dashed lines mark the future edges of the cephalic furrow. Scale bar = 20 µm. <strong>d</strong>, Profile view of the gene expression after cephalic furrow invagination in <em>Drosophila</em>. The <em>eve</em>-expressing initiator cells are located deep into the yolk, while the <em>slp</em> head domain (sh) and stripe 3 demarcate the outer edges of the invagination. Scale bar = 20 µm. <strong>e</strong>, Schematic drawings showing the combinatorial molecular arrangement at the head–trunk boundary of <em>Drosophila</em>. <strong>f</strong>, Gene expression domains of <em>btd</em>, <em>eve</em> and <em>slp</em> during nuclear cycles 13 and 14 in <em>Ceratitis</em>, <em>Anopheles</em>, and <em>Clogmia</em>. The patterns were consistent across 3 experiments in <em>Ceratitis</em> (38 embryos), 4 experiments in <em>Anopheles</em> (43 embryos), and 4 experiments in <em>Clogmia</em> (44 embryos). Scale bars = 100 µm. <strong>g</strong>, Comparison of expression patterns at the head–trunk boundary before gastrulation between species. The expression of <em>slp</em> and <em>eve</em> are similar across all species. Both <em>Drosophila</em> and <em>Ceratitis</em>, species with a cephalic furrow, show a head–trunk domain of <em>btd</em> that overlaps with <em>eve</em> stripe 1. In <em>Anopheles</em>, the head–trunk <em>btd</em> domain does not overlap with <em>eve</em> stripe 1, and in <em>Clogmia</em> this <em>btd</em> domain is absent. Scale bars = 50 µm. <strong>h</strong>, Profile view of the gene expression at the head–trunk boundary cells in <em>Ceratitis</em>, <em>Anopheles</em>, and <em>Clogmia</em>. Scale bars = 20 µm. ac: <em>btd</em> acron domain, sh: <em>slp</em> head domain, ht: <em>btd</em> head–trunk domain, fg: <em>btd</em> foregut domain.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="figure-5">Figure 5</h2>
<div id="fig:summary" class="fignos">
<figure>
<img src="figures/Fig5.jpg" style="width:100.0%" alt="Figure 5: Interplay of genetics and mechanics during cephalic furrow evolution. a, Simplified dipteran phylogeny based on Ref63 with cephalic furrow traits mapped onto the tree. The germ band extension and mitotic domains are ancestral, suggesting that compressive stresses at the head–trunk boundary were present since the dawn of Diptera. The cephalic furrow is a derived trait, an evolutionary novelty of cyclorraphan flies present in the common ancestor of Megaselia and Drosophila, and the out-of-plane cell divisions at the head–trunk boundary are present in non-cyclorraphan flies such as Clogmia and Chironomus.14 The presence of a btd–eve overlap correlates with the presence of the cephalic furrow, and may be associated with its evolution. Data sources for the figure are annotated with geometrical symbols; this study (black circle), Dey et al.14 (black triangle), and Eritano et al.9 and Vincent et al.8 (black square). b, Scenario for mechanical instability as a selective pressure for the evolution of morphogenetic innovations. In the ancestral state, there was no mechanical instability at the head–trunk boundary [0]. The appearance of mitotic domains and germ band extension increased the compressive stresses and ectopic buckling events at the head–trunk boundary [1]. This mechanical instability may have had a detrimental effect on individual fitness by affecting developmental robustness or by slowing down embryogenesis. Natural selection favored the establishment of patterned processes that mitigate these compressive stresses at the head–trunk boundary. One solution, present in Clogmia and Chironomus, is the out-of-plane cell divisions, which reduce the compression load on the monolayer epithelium [2a]. Another solution, present in Drosophila and other cyclorraphan flies, is the formation of an out-of-plane invagination which absorbs the mechanical forces at the head–trunk boundary [2b]. These events may have happened through the stabilization of genetic interactions and cooption of existing signaling modules controlling cell and tissue morphogenesis. Tissue mechanics may have been an important factor influencing the evolution of patterned morphogenesis in early embryonic development." /><figcaption aria-hidden="true"><span>Figure 5:</span> Interplay of genetics and mechanics during cephalic furrow evolution. <strong>a</strong>, Simplified dipteran phylogeny based on Ref<span class="citation" data-cites="Wiegmann2011-ou"><sup><a href="#ref-Wiegmann2011-ou" role="doc-biblioref">63</a></sup></span> with cephalic furrow traits mapped onto the tree. The germ band extension and mitotic domains are ancestral, suggesting that compressive stresses at the head–trunk boundary were present since the dawn of Diptera. The cephalic furrow is a derived trait, an evolutionary novelty of cyclorraphan flies present in the common ancestor of <em>Megaselia</em> and <em>Drosophila</em>, and the out-of-plane cell divisions at the head–trunk boundary are present in non-cyclorraphan flies such as <em>Clogmia</em> and <em>Chironomus</em>.<span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> The presence of a <em>btd</em>–<em>eve</em> overlap correlates with the presence of the cephalic furrow, and may be associated with its evolution. Data sources for the figure are annotated with geometrical symbols; this study (black circle), Dey <em>et al.</em><span class="citation" data-cites="Dey2023-mo"><sup><a href="#ref-Dey2023-mo" role="doc-biblioref">14</a></sup></span> (black triangle), and Eritano et al.<span class="citation" data-cites="Eritano2020-mi"><sup><a href="#ref-Eritano2020-mi" role="doc-biblioref">9</a></sup></span> and Vincent et al.<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">8</a></sup></span> (black square). <strong>b</strong>, Scenario for mechanical instability as a selective pressure for the evolution of morphogenetic innovations. In the ancestral state, there was no mechanical instability at the head–trunk boundary [0]. The appearance of mitotic domains and germ band extension increased the compressive stresses and ectopic buckling events at the head–trunk boundary [1]. This mechanical instability may have had a detrimental effect on individual fitness by affecting developmental robustness or by slowing down embryogenesis. Natural selection favored the establishment of patterned processes that mitigate these compressive stresses at the head–trunk boundary. One solution, present in <em>Clogmia</em> and <em>Chironomus</em>, is the out-of-plane cell divisions, which reduce the compression load on the monolayer epithelium [2a]. Another solution, present in <em>Drosophila</em> and other cyclorraphan flies, is the formation of an out-of-plane invagination which absorbs the mechanical forces at the head–trunk boundary [2b]. These events may have happened through the stabilization of genetic interactions and cooption of existing signaling modules controlling cell and tissue morphogenesis. Tissue mechanics may have been an important factor influencing the evolution of patterned morphogenesis in early embryonic development.</figcaption>
</figure>
</div>
<div style=""></div>
<h1 id="tables">Tables</h1>
<h2 id="table-1">Table 1</h2>
<div id="tbl:gbtime" class="tablenos">
<table id="tbl:gbtime">
<caption><span>Table 1:</span> Relative timing differences between developmental events in cephalic furrow mutants. We measured the time after gastrulation (TAG) and the percentage of germ band extension (GBE) at the moment of formation of the cephalic furrow (CF), mitotic domains (MD), and ectopic folds (EFs). For a more general comparison, we also pooled the data for mutants where the cephalic furrow is absent (<em>btd</em>+<em>eve</em>). </caption>
<thead>
<tr class="header">
<th style="text-align: left;">Stock</th>
<th style="text-align: right;">n</th>
<th>Zygosity</th>
<th style="text-align: right;">n</th>
<th>Event</th>
<th>TAG (min)</th>
<th>GBE (%)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">wildtype</td>
<td style="text-align: right;">16</td>
<td>+/+</td>
<td style="text-align: right;">16</td>
<td>CF</td>
<td>7.2±2.2</td>
<td>8.7±1.7</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>14.6±2.4</td>
<td>21.4±1.9</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>btd</em></td>
<td style="text-align: right;">11</td>
<td>+/−</td>
<td style="text-align: right;">6</td>
<td>CF</td>
<td>8.2±1.7</td>
<td>9.8±1.9</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>14.6±3.7</td>
<td>22.1±4.2</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td>−/−</td>
<td style="text-align: right;">5</td>
<td>EF</td>
<td>21.3±4.7</td>
<td>36.3±2.4</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>17.0±3.7</td>
<td>28.1±2.3</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>eve</em></td>
<td style="text-align: right;">12</td>
<td>+/−</td>
<td style="text-align: right;">7</td>
<td>CF</td>
<td>12.2±1.7</td>
<td>8.8±2.3</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>18.2±1.6</td>
<td>19.9±4.1</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td>−/−</td>
<td style="text-align: right;">5</td>
<td>EF</td>
<td>22.6±2.6</td>
<td>28.5±3.9</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>16.9±1.1</td>
<td>19.0±2.7</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>prd</em></td>
<td style="text-align: right;">15</td>
<td>+/−</td>
<td style="text-align: right;">10</td>
<td>CF</td>
<td>11.3±6.2</td>
<td>18.0±12.2</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>12.8±1.1</td>
<td>20.4±3.8</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td>−/−</td>
<td style="text-align: right;">5</td>
<td>CF</td>
<td>18.2±5.4</td>
<td>33.0±9.5</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>13.0±2.0</td>
<td>23.4±3.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>btd</em>+<em>eve</em></td>
<td style="text-align: right;">23</td>
<td>+/−</td>
<td style="text-align: right;">13</td>
<td>CF</td>
<td>10.4±2.6</td>
<td>9.2±2.1</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>16.6±3.2</td>
<td>20.9±4.1</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td>−/−</td>
<td style="text-align: right;">10</td>
<td>EF</td>
<td>21.9±3.6</td>
<td>32.4±5.1</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td></td>
<td style="text-align: right;"></td>
<td>MD</td>
<td>17.0±2.6</td>
<td>23.6±5.3</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
<h2 id="table-2">Table 2</h2>
<div id="tbl:area" class="tablenos">
<table id="tbl:area">
<caption><span>Table 2:</span> Surface area of ectopic folds in cephalic furrow mutants. Values in µm<sup>2</sup>×10<sup>3</sup>. </caption>
<thead>
<tr class="header">
<th style="text-align: left;">Stock</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Zygosity</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Type</th>
<th style="text-align: right;">n</th>
<th style="text-align: right;">Area</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">wildtype</td>
<td style="text-align: right;">16</td>
<td style="text-align: center;">+/+</td>
<td style="text-align: right;">16</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">16</td>
<td style="text-align: right;">11.4±1.2</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">0.6±0.4</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>btd</em></td>
<td style="text-align: right;">11</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">6</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">10.3±1.9</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">5</td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">2.3±1.8</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>eve</em></td>
<td style="text-align: right;">11</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">7</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">9.7±1.1</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">4</td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">3.2±2.1</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>prd</em></td>
<td style="text-align: right;">14</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">9</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">7.6±2.6</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">0.8±0.5</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">5</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">7.2</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">1.3±0.9</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
<h2 id="table-3">Table 3</h2>
<div id="tbl:depth" class="tablenos">
<table id="tbl:depth">
<caption><span>Table 3:</span> Maximum depth of ectopic folds in cephalic furrow mutants. Values in µm. </caption>
<thead>
<tr class="header">
<th style="text-align: left;">Stock</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Zygosity</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Type</th>
<th style="text-align: right;">n</th>
<th style="text-align: right;">Depth</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><em>btd</em></td>
<td style="text-align: right;">39</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">32</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">52</td>
<td style="text-align: right;">71.6±8.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">32.5±3.6</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">7</td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">52.5±12.1</td>
</tr>
<tr class="even">
<td style="text-align: left;"><em>eve</em></td>
<td style="text-align: right;">24</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">20</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">34</td>
<td style="text-align: right;">59.0±6.8</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">36.1±4.4</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">4</td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">42.1±11.7</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
<h2 id="table-4">Table 4</h2>
<div id="tbl:freq" class="tablenos">
<table id="tbl:freq">
<caption><span>Table 4:</span> Folding statistics in cephalic furrow mutants. We calculated the percentage of embryos having a cephalic furrow (CF) or ectopic folds (EF) for each stock and genotype (Frequency), including the position of folding along the head–trunk boundary (Ant=anterior, Mid=middle, Post=posterior). In addition, we calculated the average number of folds per embryo side (Folds). For example, 28 out of 36 wildtype embryos show ectopic folds (77.8%); 42.9% of these embryos have folds at the anterior region and 71.4% form them at posterior to the head–trunk boundary; each embryo forms 1.1±0.3 folds on each side. The <em>n</em> includes datasets imaged from the lateral and dorsal sides. </caption>
<thead>
<tr class="header">
<th style="text-align: left;">Stock</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Zygosity</th>
<th style="text-align: right;">n</th>
<th style="text-align: center;">Type</th>
<th style="text-align: right;">n</th>
<th style="text-align: right;">Frequency</th>
<th style="text-align: right;">Ant</th>
<th style="text-align: right;">Mid</th>
<th style="text-align: right;">Post</th>
<th style="text-align: right;">Folds</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">wildtype</td>
<td style="text-align: right;">36</td>
<td style="text-align: center;">+/+</td>
<td style="text-align: right;">36</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">36</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">77.8%</td>
<td style="text-align: right;">42.9%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">71.4%</td>
<td style="text-align: right;">1.1±0.3</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>btd</em></td>
<td style="text-align: right;">46</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">33</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">33</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">18.2%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">13</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">92.3%</td>
<td style="text-align: right;">50%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">75%</td>
<td style="text-align: right;">2.2±0.4</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>eve</em></td>
<td style="text-align: right;">36</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">26</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">26.9%</td>
<td style="text-align: right;">14.3%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">85.7%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">10</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">40%</td>
<td style="text-align: right;">70%</td>
<td style="text-align: right;">90%</td>
<td style="text-align: right;">1.8±0.6</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>prd</em></td>
<td style="text-align: right;">40</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">26</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">26.9%</td>
<td style="text-align: right;">71.4%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">57.1%</td>
<td style="text-align: right;">1.3±0.5</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">14</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">50.0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">71.4%</td>
<td style="text-align: right;">50%</td>
<td style="text-align: right;">80%</td>
<td style="text-align: right;">70%</td>
<td style="text-align: right;">1.9±0.8</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>stg</em></td>
<td style="text-align: right;">46</td>
<td style="text-align: center;">+/−</td>
<td style="text-align: right;">33</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">33</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">36.4%</td>
<td style="text-align: right;">8.3%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">91.7%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">−/−</td>
<td style="text-align: right;">13</td>
<td style="text-align: center;">CF</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;"></td>
<td style="text-align: right;"></td>
<td style="text-align: center;">EF</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">23.1%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">0%</td>
<td style="text-align: right;">100%</td>
<td style="text-align: right;">1.0±0.0</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
<h1 id="extended-data">Extended data</h1>
<div style=""></div>
<h2 id="extended-data-fig.-1">Extended Data Fig. 1</h2>
<div id="fig:initiator" class="fignos">
<figure>
<img src="figures/FigED1.jpg" data-tag="Extended Data Fig. 1" style="width:100.0%" alt="Figure Extended Data Fig. 1: Perturbation of the initiator cell behavior in cephalic furrow mutants. a, Profile view showing the cephalic furrow formation in wildtype, prd, btd, and eve embryos. The samples are synchronized by the end of cellularization, when the cleavage furrows reach their basal position (frame 0.0 min). The arrow indicates the position and timing of the first infolding of the tissue. In wildtype, the shortening of initiator cells begins before cellularization is complete (wildtype 0.0 min). Cells adjacent to the initiators remain in close contact to the initiator row. Their apical membranes are tightly connected and become arched in an arrangement typical to the early phase of the invagination of the cephalic furrow (CF). This arrangement is perturbed in mutant embryos to different degrees. In prd, the initiator cells shorten and the tissue invaginates, but the infolding is delayed, and the adjacent cells do not arch over the initiator row (prd 6.1 min). In btd, there is no cell shortening, but some embryos exhibit a certain degree of anisotropic apical constriction which creates a bulge in the epithelium minutes after the end of cellularization (btd 6.0 min, see also b). This initial bulge often primes the position for the formation of ectopic folds (EF) of ectopic buckling. In eve, the cells show neither shortening nor apical constriction and ectopic folds appear about ten minutes after the end of cellularization (eve 9.7 min). The initiator behavior phenotypes were highly consistent for wildtype (1 experiment, 20 embryos), btd (3 experiments, 29 embryos) and eve (2 experiments, 23 embryos) and more variable for prd mutants (2 experiments, 26 embryos). Scale bars = 20 µm. b, Surface view of cartographic projections showing the head–trunk interface. In wildtype, the anisotropic apical constriction is localized to a narrow stripe adjacent to the initiator row. In prd embryos, the apical constriction occurs, but it does not form a clear line of infolding cells preceding the invagination as in wildtype embryos. In btd embryos, there is a similar degree of anisotropic apical constriction occurring but not all embryos form ectopic folds at this region. In eve embryos, the mitotic domain 2 (MD) begins expanding and there is no apical constriction behavior. Time between frames is about 4 min. The initiator behavior phenotypes were highly consistent for wildtype (1 experiment, 16 embryos), btd (3 experiments, 14 embryos) and eve (3 experiments, 12 embryos) and more variable for prd mutants (1 experiment, 14 embryos). Scale bars = 20 µm (approximated value since pixel sizes vary across the projection)." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 1:</span> Perturbation of the initiator cell behavior in cephalic furrow mutants. <strong>a</strong>, Profile view showing the cephalic furrow formation in wildtype, <em>prd</em>, <em>btd</em>, and <em>eve</em> embryos. The samples are synchronized by the end of cellularization, when the cleavage furrows reach their basal position (frame 0.0 min). The arrow indicates the position and timing of the first infolding of the tissue. In wildtype, the shortening of initiator cells begins before cellularization is complete (wildtype 0.0 min). Cells adjacent to the initiators remain in close contact to the initiator row. Their apical membranes are tightly connected and become arched in an arrangement typical to the early phase of the invagination of the cephalic furrow (CF). This arrangement is perturbed in mutant embryos to different degrees. In <em>prd</em>, the initiator cells shorten and the tissue invaginates, but the infolding is delayed, and the adjacent cells do not arch over the initiator row (<em>prd</em> 6.1 min). In <em>btd</em>, there is no cell shortening, but some embryos exhibit a certain degree of anisotropic apical constriction which creates a bulge in the epithelium minutes after the end of cellularization (<em>btd</em> 6.0 min, see also <strong>b</strong>). This initial bulge often primes the position for the formation of ectopic folds (EF) of ectopic buckling. In <em>eve</em>, the cells show neither shortening nor apical constriction and ectopic folds appear about ten minutes after the end of cellularization (<em>eve</em> 9.7 min). The initiator behavior phenotypes were highly consistent for wildtype (1 experiment, 20 embryos), <em>btd</em> (3 experiments, 29 embryos) and <em>eve</em> (2 experiments, 23 embryos) and more variable for <em>prd</em> mutants (2 experiments, 26 embryos). Scale bars = 20 µm. <strong>b</strong>, Surface view of cartographic projections showing the head–trunk interface. In wildtype, the anisotropic apical constriction is localized to a narrow stripe adjacent to the initiator row. In <em>prd</em> embryos, the apical constriction occurs, but it does not form a clear line of infolding cells preceding the invagination as in wildtype embryos. In <em>btd</em> embryos, there is a similar degree of anisotropic apical constriction occurring but not all embryos form ectopic folds at this region. In <em>eve</em> embryos, the mitotic domain 2 (MD) begins expanding and there is no apical constriction behavior. Time between frames is about 4 min. The initiator behavior phenotypes were highly consistent for wildtype (1 experiment, 16 embryos), <em>btd</em> (3 experiments, 14 embryos) and <em>eve</em> (3 experiments, 12 embryos) and more variable for <em>prd</em> mutants (1 experiment, 14 embryos). Scale bars = 20 µm (approximated value since pixel sizes vary across the projection).</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-2">Extended Data Fig. 2</h2>
<!--TODO: Reduce figure legend-->
<!--**e**, Angle directionality and tortuosity during cephalic furrow and ectopic fold formation. Within the first fifteen minutes after gastrulation, the cephalic furrow exhibits a typical posterior shift on the dorsal side which declines the initial angle of the invagination from 80° to about 60° in relation to the anteroposterior axis. During this period, the invagination begins as a straight line and bends showing a correspondent increase in the measured tortuosity values of the furrow outline. In contrast, ectopic folds show no obvious trend in angular direction tortuosity values over time. For both angle and tortuosity analysis, n = 6.-->
<div id="fig:ectopic-features" class="fignos">
<figure>
<img src="figures/FigED2.jpg" data-tag="Extended Data Fig. 2" style="width:100.0%" alt="Figure Extended Data Fig. 2: Differences between ectopic folding and cephalic furrow formation. a, Distribution of ectopic folds in eve homozygotes (right) and sibling controls (left). Scale bars = 50 µm. b, Folded area of the cephalic furrow (CF) and ectopic folds (EF) in eve embryos. The folded region is outlined in yellow on cartographic projections of a representative sibling control eve heterozygote (left) and of a eve homozygote (right). The numbers indicate the average folded area for the sample type in µm2×103. Scale bars = 50 µm (approximated value since pixel sizes vary across the projection). c, Dynamics of cephalic furrow and ectopic fold formation in eve mutants. d, Dynamics of cephalic furrow and ectopic fold formation in btd mutants. e, Angle directionality and tortuosity during cephalic furrow and ectopic fold formation. The cephalic furrow angle declines from 80° to about 60° and bends, increasing its tortuosity within the first 15 min, while the ectopic folds show no obvious trend (n = 6). f, Maximum depth of the cephalic furrow and ectopic folds in btd and eve mutants. Ectopic folds are shallower than the cephalic furrow in both genetic backgrounds (p &lt; 0.001). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney U test. Each point corresponds to a single fold; one embryo can have multiple folds. The number of embryos analyzed for btd is 32 heterozygotes (52 CFs and 6 EFs measured in total) and 7 homozygotes (28 EFs), and for eve is 20 heterozygotes (34 CFs and 4 EFs) and 4 homozygotes (15 EFs). g, Folded area of ectopic folds in wildtype and mutant embryos (btd, eve, and prd). Ectopic folds in wildtype occupy a smaller area than ectopic folds in cephalic furrow mutants (p &lt; 0.001). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney U test. The number of embryos analyzed for wildtype is 16 (14 EFs in total), for heterozygotes is 9 (7 EFs), and for homozygotes is 14 (28 EFs). h, Profile view of a btd mutant embryo, showing the presence of two ectopic folds (arrows) forming next to dividing cells (asterisks). Scale bar = 20 µm. i, Proportion of embryos showing different fold types (CF and EF) in wildtype (n = 36), btd heterozygotes (n = 33), btd homozygotes (n = 13), eve heterozygotes (n = 26), eve homozygotes (n = 10), prd heterozygotes (n = 26), prd homozygotes (n = 14), stg heterozygotes (n = 33), and stg homozygotes (n = 13). j, Proportion of ectopic folding (EF) at different positions of the head–trunk boundary (anterior, mid, posterior) in wildtype (n = 28), btd heterozygotes (n = 6), btd homozygotes (n = 12), eve heterozygotes (n = 7), eve homozygotes (n = 10), prd heterozygotes (n = 7), prd homozygotes (n = 10), stg heterozygotes (n = 12), and stg homozygotes (n = 3). k, Lateral views of a wildtype (left) and a prd mutant (right) exhibiting ectopic folds. Scale bar = 50 µm. l, Folded area of the cephalic furrow (CF) and ectopic folds (EF) in the wildtype (left) and prd mutant (right) shown in k. The folded region is outlined in yellow on a cartographic projection. The numbers indicate the average folded area in µm2×103 for the CFs and EFs separately. Scale bars = 50 µm (approximated value since pixel sizes vary across the projection). m, Developmental sequence of the wildtype embryo ectopic fold annotated in l. Four cells are temporarily infolded during the mitotic expansion of adjacent cells. Scale bar = 10 µm. n, Folded area distinguishing between cephalic furrow and ectopic folds in different genetic backgrounds. For wildtype embryos n = 16 (16 CFs and 14 EFs), btd heterozygotes n = 6 (6 CFs), btd homozygotes n = 5 (12 EFs), eve heterozygotes n = 7 (7 CFs), eve homozygotes n = 4 (7 EFs), prd heterozygotes n = 9 (9 CFs and 7 EFs), and prd homozygotes n = 5 (1 CF and 9 EFs). In all box plots, the center depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 2:</span> Differences between ectopic folding and cephalic furrow formation. <strong>a</strong>, Distribution of ectopic folds in <em>eve</em> homozygotes (right) and sibling controls (left). Scale bars = 50 µm. <strong>b</strong>, Folded area of the cephalic furrow (CF) and ectopic folds (EF) in <em>eve</em> embryos. The folded region is outlined in yellow on cartographic projections of a representative sibling control <em>eve</em> heterozygote (left) and of a <em>eve</em> homozygote (right). The numbers indicate the average folded area for the sample type in µm<sup>2</sup>×10<sup>3</sup>. Scale bars = 50 µm (approximated value since pixel sizes vary across the projection). <strong>c</strong>, Dynamics of cephalic furrow and ectopic fold formation in <em>eve</em> mutants. <strong>d</strong>, Dynamics of cephalic furrow and ectopic fold formation in <em>btd</em> mutants. <strong>e</strong>, Angle directionality and tortuosity during cephalic furrow and ectopic fold formation. The cephalic furrow angle declines from 80° to about 60° and bends, increasing its tortuosity within the first 15 min, while the ectopic folds show no obvious trend (n = 6). <strong>f</strong>, Maximum depth of the cephalic furrow and ectopic folds in <em>btd</em> and <em>eve</em> mutants. Ectopic folds are shallower than the cephalic furrow in both genetic backgrounds (p &lt; 0.001). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney <em>U</em> test. Each point corresponds to a single fold; one embryo can have multiple folds. The number of embryos analyzed for <em>btd</em> is 32 heterozygotes (52 CFs and 6 EFs measured in total) and 7 homozygotes (28 EFs), and for <em>eve</em> is 20 heterozygotes (34 CFs and 4 EFs) and 4 homozygotes (15 EFs). <strong>g</strong>, Folded area of ectopic folds in wildtype and mutant embryos (<em>btd</em>, <em>eve</em>, and <em>prd</em>). Ectopic folds in wildtype occupy a smaller area than ectopic folds in cephalic furrow mutants (p &lt; 0.001). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney <em>U</em> test. The number of embryos analyzed for wildtype is 16 (14 EFs in total), for heterozygotes is 9 (7 EFs), and for homozygotes is 14 (28 EFs). <strong>h</strong>, Profile view of a <em>btd</em> mutant embryo, showing the presence of two ectopic folds (arrows) forming next to dividing cells (asterisks). Scale bar = 20 µm. <strong>i</strong>, Proportion of embryos showing different fold types (CF and EF) in wildtype (n = 36), <em>btd</em> heterozygotes (n = 33), <em>btd</em> homozygotes (n = 13), <em>eve</em> heterozygotes (n = 26), <em>eve</em> homozygotes (n = 10), <em>prd</em> heterozygotes (n = 26), <em>prd</em> homozygotes (n = 14), <em>stg</em> heterozygotes (n = 33), and <em>stg</em> homozygotes (n = 13). <strong>j</strong>, Proportion of ectopic folding (EF) at different positions of the head–trunk boundary (anterior, mid, posterior) in wildtype (n = 28), <em>btd</em> heterozygotes (n = 6), <em>btd</em> homozygotes (n = 12), <em>eve</em> heterozygotes (n = 7), <em>eve</em> homozygotes (n = 10), <em>prd</em> heterozygotes (n = 7), <em>prd</em> homozygotes (n = 10), <em>stg</em> heterozygotes (n = 12), and <em>stg</em> homozygotes (n = 3). <strong>k</strong>, Lateral views of a wildtype (left) and a <em>prd</em> mutant (right) exhibiting ectopic folds. Scale bar = 50 µm. <strong>l</strong>, Folded area of the cephalic furrow (CF) and ectopic folds (EF) in the wildtype (left) and <em>prd</em> mutant (right) shown in <strong>k</strong>. The folded region is outlined in yellow on a cartographic projection. The numbers indicate the average folded area in µm<sup>2</sup>×10<sup>3</sup> for the CFs and EFs separately. Scale bars = 50 µm (approximated value since pixel sizes vary across the projection). <strong>m</strong>, Developmental sequence of the wildtype embryo ectopic fold annotated in <strong>l</strong>. Four cells are temporarily infolded during the mitotic expansion of adjacent cells. Scale bar = 10 µm. <strong>n</strong>, Folded area distinguishing between cephalic furrow and ectopic folds in different genetic backgrounds. For wildtype embryos n = 16 (16 CFs and 14 EFs), <em>btd</em> heterozygotes n = 6 (6 CFs), <em>btd</em> homozygotes n = 5 (12 EFs), <em>eve</em> heterozygotes n = 7 (7 CFs), <em>eve</em> homozygotes n = 4 (7 EFs), <em>prd</em> heterozygotes n = 9 (9 CFs and 7 EFs), and <em>prd</em> homozygotes n = 5 (1 CF and 9 EFs). In all box plots, the center depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-3">Extended Data Fig. 3</h2>
<div id="fig:mitotic-strain" class="fignos">
<figure>
<img src="figures/FigED3.jpg" data-tag="Extended Data Fig. 3" style="width:100.0%" alt="Figure Extended Data Fig. 3: Coincidence of mitotic domains and germ band extension with ectopic folding. a, Representative positions of folding (black lines) in relation to mitotic domains (colored areas) in wildtype embryos and btd and eve mutants. Ectopic folds appear between mitotic domains and between the germ band and the mitotic domain 6 (MD6). While the position of ectopic folds vary between individual embryos, the MD2 phenotype was consistent in 4 and 3 independent experiments for btd (16 embryos) and eve (12 embryos), respectively. Scale bars = 50 µm. b, Apical area of cells between the mitotic domains MD5 and MD7 (top) until the formation of a ectopic fold; folded area is highlighted at the last frame (4.6 min). The bottom panels show an annotated subset of non-dividing cells (orange) and dividing cells (blue). The apical area expansion was consistent in all embryos with mitotic domains. Scale bars = 20 µm (approximated value since pixel sizes vary across the projection). c, Ectopic folding between MD6 and germ band (not visible) in a btd mutant. Mitotic expansions (double arrow) always precede ectopic folding (arrow), as quantified in d. Scale bar = 20 µm. d, Timing of cephalic furrow (CF) formation relative to mitotic domain (MD) expansion. In wildtype (n = 16) and heterozygote embryos (n = 13, btd and eve combined), the cephalic furrow forms before mitotic domains (p &lt; 0.001 and p = 0.002, respectively). In homozygote embryos (n = 10), the ectopic folds form after mitotic domains (p = 0.002). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney U test between CF and MD timing. The box plot center depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. e, Lateral view of sibling control and btd mutant showing the head–trunk (HT) and trunk–germ (TG) regions. The apical surface of cells at the TG, between MD6 and the tip of the germ band, exhibits a compressed profile. Only a subset of cephalic furrow mutants exhibit this phenotype as it depends on the timing and position of ectopic folding. Scale bars = 50 µm. f, Strain rate analysis at the head–trunk (top) and trunk–germ (bottom) regions of btd heterozygotes (n = 3) and homozygotes (n = 3). Filled triangles indicate the beginning of cephalic furrow (CF), mitotic domains (MD), and ectopic folds (EF) formation. MDi shows the initial metaphase expansion and MDii the moment of cell division during telophase. Empty triangles indicate the peaks of strain rate. Letters A–D show the exact frames from Supplementary Fig. 2b. The measurements combine isotropic and anisotropic strain rate. The line data shows the mean predicted values from the regression with the shaded area representing the 95% confidence interval of the mean. g, Dynamics of trunk–germ tissues after laser cuts in wildtype embryos. We measured the distance between pairs of cell vertices over time in control (n = 3) and ablated embryos (n = 3). Tracks are color-coded for time. Solid white line represents the distance between vertices at one timepoint. Dashed white line shows the location of the cut. Performing the ablations consistently in different embryos was challenging; we only analyzed embryos where the vitelline envelope remained intact after the laser pulse. The plots show a smoothed local regression of the data with a shaded band indicating the 95% confidence interval for the mean trend. Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 3:</span> Coincidence of mitotic domains and germ band extension with ectopic folding. <strong>a</strong>, Representative positions of folding (black lines) in relation to mitotic domains (colored areas) in wildtype embryos and <em>btd</em> and <em>eve</em> mutants. Ectopic folds appear between mitotic domains and between the germ band and the mitotic domain 6 (MD6). While the position of ectopic folds vary between individual embryos, the MD2 phenotype was consistent in 4 and 3 independent experiments for <em>btd</em> (16 embryos) and <em>eve</em> (12 embryos), respectively. Scale bars = 50 µm. <strong>b</strong>, Apical area of cells between the mitotic domains MD5 and MD7 (top) until the formation of a ectopic fold; folded area is highlighted at the last frame (4.6 min). The bottom panels show an annotated subset of non-dividing cells (orange) and dividing cells (blue). The apical area expansion was consistent in all embryos with mitotic domains. Scale bars = 20 µm (approximated value since pixel sizes vary across the projection). <strong>c</strong>, Ectopic folding between MD6 and germ band (not visible) in a <em>btd</em> mutant. Mitotic expansions (double arrow) always precede ectopic folding (arrow), as quantified in <strong>d</strong>. Scale bar = 20 µm. <strong>d</strong>, Timing of cephalic furrow (CF) formation relative to mitotic domain (MD) expansion. In wildtype (n = 16) and heterozygote embryos (n = 13, <em>btd</em> and <em>eve</em> combined), the cephalic furrow forms before mitotic domains (p &lt; 0.001 and p = 0.002, respectively). In homozygote embryos (n = 10), the ectopic folds form after mitotic domains (p = 0.002). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney <em>U</em> test between CF and MD timing. The box plot center depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. <strong>e</strong>, Lateral view of sibling control and <em>btd</em> mutant showing the head–trunk (HT) and trunk–germ (TG) regions. The apical surface of cells at the TG, between MD6 and the tip of the germ band, exhibits a compressed profile. Only a subset of cephalic furrow mutants exhibit this phenotype as it depends on the timing and position of ectopic folding. Scale bars = 50 µm. <strong>f</strong>, Strain rate analysis at the head–trunk (top) and trunk–germ (bottom) regions of <em>btd</em> heterozygotes (n = 3) and homozygotes (n = 3). Filled triangles indicate the beginning of cephalic furrow (CF), mitotic domains (MD), and ectopic folds (EF) formation. MD<sub>i</sub> shows the initial metaphase expansion and MD<sub>ii</sub> the moment of cell division during telophase. Empty triangles indicate the peaks of strain rate. Letters A–D show the exact frames from <a href="#fig:apical-strain">Supplementary Fig. 2</a>b. The measurements combine isotropic and anisotropic strain rate. The line data shows the mean predicted values from the regression with the shaded area representing the 95% confidence interval of the mean. <strong>g</strong>, Dynamics of trunk–germ tissues after laser cuts in wildtype embryos. We measured the distance between pairs of cell vertices over time in control (n = 3) and ablated embryos (n = 3). Tracks are color-coded for time. Solid white line represents the distance between vertices at one timepoint. Dashed white line shows the location of the cut. Performing the ablations consistently in different embryos was challenging; we only analyzed embryos where the vitelline envelope remained intact after the laser pulse. The plots show a smoothed local regression of the data with a shaded band indicating the 95% confidence interval for the mean trend. Scale bar = 20 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-4">Extended Data Fig. 4</h2>
<!--TODO: Add more contents to this figure-->
<div id="fig:live-details" class="fignos">
<figure>
<img src="figures/FigED4.jpg" data-tag="Extended Data Fig. 4" style="width:100.0%" alt="Figure Extended Data Fig. 4: Developmental sequence of stg mutants. a, Lateral 3D renderings of stg homozygotes. They show no cell divisions after gastrulation, but the early morphogenetic movements of gastrulation occur normally. Asterisks indicate mitotic domains in controls. The cephalic furrow forms without delays and exhibits a similar dynamics of invagination compared to wildtype embryos. The only noticeable difference is that the dorsal portion does not shift as posteriorly as in sibling controls, which could be due to the absence of mitotic domains in the head. The phenotype was highly consistent across 2 independent experiments in stg mutants (29 embryos). Scale bar = 50 µm. b, Optical slice of profile views in stg homozygotes. The initiator cell behaviors are not perturbed and the morphology of the invagination is intact. In fact, because of the lack of cell divisions, the epithelium remains more uniform during gastrulation when compared to sibling controls or wildtype embryos. The phenotype was highly consistent in stg samples (1 experiment, 29 embryos). Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 4:</span> Developmental sequence of <em>stg</em> mutants. <strong>a</strong>, Lateral 3D renderings of <em>stg</em> homozygotes. They show no cell divisions after gastrulation, but the early morphogenetic movements of gastrulation occur normally. Asterisks indicate mitotic domains in controls. The cephalic furrow forms without delays and exhibits a similar dynamics of invagination compared to wildtype embryos. The only noticeable difference is that the dorsal portion does not shift as posteriorly as in sibling controls, which could be due to the absence of mitotic domains in the head. The phenotype was highly consistent across 2 independent experiments in <em>stg</em> mutants (29 embryos). Scale bar = 50 µm. <strong>b</strong>, Optical slice of profile views in <em>stg</em> homozygotes. The initiator cell behaviors are not perturbed and the morphology of the invagination is intact. In fact, because of the lack of cell divisions, the epithelium remains more uniform during gastrulation when compared to sibling controls or wildtype embryos. The phenotype was highly consistent in <em>stg</em> samples (1 experiment, 29 embryos). Scale bar = 20 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-5">Extended Data Fig. 5</h2>
<div id="fig:model-features" class="fignos">
<figure>
<img src="figures/FigED5.jpg" data-tag="Extended Data Fig. 5" style="width:100.0%" alt="Figure Extended Data Fig. 5: Model general properties and parameter sweeps. a, Embryonic proportions in wildtype and cephalic furrow mutants. Approximate relative sizes and positions between embryonic features such as mitotic domains, folds, and the germ band. All values are relative to the embryo length. We used these dimensions as a reference for creating the model. b, Representative simulation using K_{b}^{\text{*}}=7 \times 10^{-5} and g=0.3 showing the shape of the tissue at t=19 (blue) and t=1000 (pink). The respective timepoints are marked in dashed lines in the descriptive plots below. They show the variation in total energy, bending energy, number of folds, and fold depth over the iterations. t=1 corresponds to 10^{5} computational steps and the X axis is in log_{10} scale to improve the visualization. c–e, Parameter sweep without mitotic domains. Plots show (c) the number of folds by bending rigidity K_{b}^{\text{*}} using g=0.3, (d) the number of folds by germ band extension (g) using K_{b}^{\text{*}}=1.0 \times 10^{-4}, and (e) the timing of folding by germ band extension (g) using K_{b}^{\text{*}}=1.0 \times 10^{-4} (right). The line represents the mean value and the error bar the standard deviation of the data across simulations (n = 20). f–h, Parameter sweep with mitotic domains and same plots as above. i, Parameter sweep for cephalic furrow simulations. Bending rigidity and cephalic furrow depth at different values of \kappa_o^{CF} and germ band extension. Values above 0.2 exhibit a clear peak in bending energy for most conditions and the depth reaches a plateau across iterations. The cephalic furrow depth at the peak of bending energy (max bend) and at the final iteration is similar for simulations with 0% of germ band extension. At higher percentages of germ band extension the folds (both cephalic furrow and ectopic folds) exhibit a greater depth at the last iteration. j, Fine-grained parameter sweep of ectopic folding without a delay in the formation of mitotic domains (t_{MD}=0). With simultaneous formation, only higher values of \kappa_o^{CF} are effective in buffering the formation of ectopic folds around the cephalic furrow region. This is also limited to low percentages of germ band extension since at higher percentages there is an increase in the frequency of ectopic folding. k, Fine-grained parameter sweep of ectopic folding with a delay in the formation of mitotic domains (t_{MD}=5). When a delay in mitotic formation is present, even low values of \kappa_o^{CF} are effective in preventing the formation of ectopic folds. In this condition, the extension of the germ band increases the formation of ectopic folds, but only at the posterior regions close to the germ band tip. This suggests that the initiation of the cephalic furrow is crucial to its ability to buffer the ectopic folding. Values of \kappa_o^{CF} are shown in units of 1/L. t_{MD}=1 corresponds to 105 computational timesteps. l, Representative simulations from Fig. 3g at 0 and 20% of germ band extension." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 5:</span> Model general properties and parameter sweeps. <strong>a</strong>, Embryonic proportions in wildtype and cephalic furrow mutants. Approximate relative sizes and positions between embryonic features such as mitotic domains, folds, and the germ band. All values are relative to the embryo length. We used these dimensions as a reference for creating the model. <strong>b</strong>, Representative simulation using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>7</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}}=7 \times 10^{-5}</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0.3</mn></mrow><annotation encoding="application/x-tex">g=0.3</annotation></semantics></math> showing the shape of the tissue at <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>t</mi><mo>=</mo><mn>19</mn></mrow><annotation encoding="application/x-tex">t=19</annotation></semantics></math> (blue) and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>t</mi><mo>=</mo><mn>1000</mn></mrow><annotation encoding="application/x-tex">t=1000</annotation></semantics></math> (pink). The respective timepoints are marked in dashed lines in the descriptive plots below. They show the variation in total energy, bending energy, number of folds, and fold depth over the iterations. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>t</mi><mo>=</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">t=1</annotation></semantics></math> corresponds to <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msup><mn>10</mn><mn>5</mn></msup><annotation encoding="application/x-tex">10^{5}</annotation></semantics></math> computational steps and the X axis is in <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>l</mi><mi>o</mi><msub><mi>g</mi><mn>10</mn></msub></mrow><annotation encoding="application/x-tex">log_{10}</annotation></semantics></math> scale to improve the visualization. <strong>c–e</strong>, Parameter sweep without mitotic domains. Plots show (<strong>c</strong>) the number of folds by bending rigidity <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi><mo>=</mo><mn>0.3</mn></mrow><annotation encoding="application/x-tex">g=0.3</annotation></semantics></math>, (<strong>d</strong>) the number of folds by germ band extension (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math>) using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}}=1.0 \times 10^{-4}</annotation></semantics></math>, and (<strong>e</strong>) the timing of folding by germ band extension (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math>) using <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>1.0</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}}=1.0 \times 10^{-4}</annotation></semantics></math> (right). The line represents the mean value and the error bar the standard deviation of the data across simulations (n = 20). <strong>f–h</strong>, Parameter sweep with mitotic domains and same plots as above. <strong>i</strong>, Parameter sweep for cephalic furrow simulations. Bending rigidity and cephalic furrow depth at different values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> and germ band extension. Values above 0.2 exhibit a clear peak in bending energy for most conditions and the depth reaches a plateau across iterations. The cephalic furrow depth at the peak of bending energy (max bend) and at the final iteration is similar for simulations with 0% of germ band extension. At higher percentages of germ band extension the folds (both cephalic furrow and ectopic folds) exhibit a greater depth at the last iteration. <strong>j</strong>, Fine-grained parameter sweep of ectopic folding without a delay in the formation of mitotic domains (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">t_{MD}=0</annotation></semantics></math>). With simultaneous formation, only higher values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are effective in buffering the formation of ectopic folds around the cephalic furrow region. This is also limited to low percentages of germ band extension since at higher percentages there is an increase in the frequency of ectopic folding. <strong>k</strong>, Fine-grained parameter sweep of ectopic folding with a delay in the formation of mitotic domains (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>5</mn></mrow><annotation encoding="application/x-tex">t_{MD}=5</annotation></semantics></math>). When a delay in mitotic formation is present, even low values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are effective in preventing the formation of ectopic folds. In this condition, the extension of the germ band increases the formation of ectopic folds, but only at the posterior regions close to the germ band tip. This suggests that the initiation of the cephalic furrow is crucial to its ability to buffer the ectopic folding. Values of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>κ</mi><mi>o</mi><mrow><mi>C</mi><mi>F</mi></mrow></msubsup><annotation encoding="application/x-tex">\kappa_o^{CF}</annotation></semantics></math> are shown in units of <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mn>1</mn><mi>/</mi><mi>L</mi></mrow><annotation encoding="application/x-tex">1/L</annotation></semantics></math>. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>t</mi><mrow><mi>M</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>1</mn></mrow><annotation encoding="application/x-tex">t_{MD}=1</annotation></semantics></math> corresponds to 10<sup>5</sup> computational timesteps. <strong>l</strong>, Representative simulations from Fig. <a href="#fig:fold-model">3</a>g at 0 and 20% of germ band extension.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="extended-data-fig.-6">Extended Data Fig. 6</h2>
<div id="fig:slp-analyses" class="fignos">
<figure>
<img src="figures/FigED6.jpg" data-tag="Extended Data Fig. 6" style="width:100.0%" alt="Figure Extended Data Fig. 6: Analyses of slp mutants in Drosophila. a, Lateral view of slp heterozygote (control) and mutant embryo at the onset of initiator cell behavior. In slp mutants, the formation of the cephalic furrow is delayed and shifted forward. The phenotype was consistent in 6 independent experiments with slp mutants (39 embryos). Scale bars = 50 µm. b, Position of the cephalic furrow (CF) and germ band (GB) in slp mutants. The CF is displaced anteriorly and the GB is further extended (p &lt; 0.001) when comparing heterozygotes (n = 26) and homozygotes (n = 7). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney U test. The center of the box plot depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. c, Expression of btd and eve in slp mutants. Scale bars = 50 µm. d, Lateral view of slp mutants in Drosophila embryos showing the expression of btd and eve. The distance between eve stripe 1 and 2 is larger in slp embryos. The asterisk indicates a region between the acron (ac) and head–trunk (ht) domains of btd where its expression is activated (or de-repressed) in the absence of the slp head domain (sh). The pattern was consistent across 4 independent experiments in slp mutants (30 embryos). Scale bars = 50 µm. e, Dorsal view showing the ectopic expression of btd (asterisks) and anterior shift of btd–eve overlap in slp mutants. Scale bars = 20 µm. f, Profile view showing the increased number of cell rows between eve stripe 1 and 2 in slp mutants. Scale bars = 20 µm. g, Initiator cell behavior in slp mutants. Scale bar = 20 µm. h, Asymmetric cephalic furrow in slp mutants. Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 6:</span> Analyses of <em>slp</em> mutants in <em>Drosophila</em>. <strong>a</strong>, Lateral view of <em>slp</em> heterozygote (control) and mutant embryo at the onset of initiator cell behavior. In <em>slp</em> mutants, the formation of the cephalic furrow is delayed and shifted forward. The phenotype was consistent in 6 independent experiments with <em>slp</em> mutants (39 embryos). Scale bars = 50 µm. <strong>b</strong>, Position of the cephalic furrow (CF) and germ band (GB) in <em>slp</em> mutants. The CF is displaced anteriorly and the GB is further extended (p &lt; 0.001) when comparing heterozygotes (n = 26) and homozygotes (n = 7). Asterisks indicate p &lt; 0.05 in a two-sided Mann–Whitney <em>U</em> test. The center of the box plot depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. <strong>c</strong>, Expression of <em>btd</em> and <em>eve</em> in <em>slp</em> mutants. Scale bars = 50 µm. <strong>d</strong>, Lateral view of <em>slp</em> mutants in <em>Drosophila</em> embryos showing the expression of <em>btd</em> and <em>eve</em>. The distance between <em>eve</em> stripe 1 and 2 is larger in <em>slp</em> embryos. The asterisk indicates a region between the acron (ac) and head–trunk (ht) domains of <em>btd</em> where its expression is activated (or de-repressed) in the absence of the <em>slp</em> head domain (sh). The pattern was consistent across 4 independent experiments in <em>slp</em> mutants (30 embryos). Scale bars = 50 µm. <strong>e</strong>, Dorsal view showing the ectopic expression of <em>btd</em> (asterisks) and anterior shift of <em>btd</em>–<em>eve</em> overlap in <em>slp</em> mutants. Scale bars = 20 µm. <strong>f</strong>, Profile view showing the increased number of cell rows between <em>eve</em> stripe 1 and 2 in <em>slp</em> mutants. Scale bars = 20 µm. <strong>g</strong>, Initiator cell behavior in <em>slp</em> mutants. Scale bar = 20 µm. <strong>h</strong>, Asymmetric cephalic furrow in <em>slp</em> mutants. Scale bar = 20 µm.</figcaption>
</figure>
</div>
<!--TODO: Finish proper figure legends.-->
<div style=""></div>
<h2 id="extended-data-fig.-7">Extended Data Fig. 7</h2>
<div id="fig:genetic-features" class="fignos">
<figure>
<img src="figures/FigED7.jpg" data-tag="Extended Data Fig. 7" style="width:100.0%" alt="Figure Extended Data Fig. 7: Genetic patterning of the head–trunk boundary in Drosophila. a, Expression of slp1, eve, and btd in wildtype from nuclear cycle (nc) 11 to gastrulation. The patterns were highly consistent across experiment as indicated in {fig:genetic-patterning}f. Scale bars = 50 µm. b, Expression of prd, slp1, and eve in wildtype embryos in lateral view. The patterns were consistent across 4 experiments in wildtype samples (10 embryos). Scale bars = 50 µm. c, Expression of prd, slp1, and eve in wildtype embryos with head and profile views. Scale bars = 20 µm." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 7:</span> Genetic patterning of the head–trunk boundary in <em>Drosophila</em>. <strong>a</strong>, Expression of <em>slp1</em>, <em>eve</em>, and <em>btd</em> in wildtype from nuclear cycle (nc) 11 to gastrulation. The patterns were highly consistent across experiment as indicated in {fig:genetic-patterning}f. Scale bars = 50 µm. <strong>b</strong>, Expression of <em>prd</em>, <em>slp1</em>, and <em>eve</em> in wildtype embryos in lateral view. The patterns were consistent across 4 experiments in wildtype samples (10 embryos). Scale bars = 50 µm. <strong>c</strong>, Expression of <em>prd</em>, <em>slp1</em>, and <em>eve</em> in wildtype embryos with head and profile views. Scale bars = 20 µm.</figcaption>
</figure>
</div>
<!--TODO: Finish proper figure legends.-->
<div style=""></div>
<h2 id="extended-data-fig.-8">Extended Data Fig. 8</h2>
<div id="fig:mutant-expression" class="fignos">
<figure>
<img src="figures/FigED8.jpg" data-tag="Extended Data Fig. 8" style="width:100.0%" alt="Figure Extended Data Fig. 8: Gene expression at the head–trunk boundary of cephalic furrow mutants in Drosophila. a, Disruption of btd, eve, and slp1 expression in the head of btd, eve, and prd mutant embryos. The patterns were consistent across 5 experiment in btd mutants (20 embryos), 2 experiments in eve mutants (12 embryos), and 2 experiments in prd mutants (12 embryos). Scale bars = 50 µm. b, Profile views of g showing the gene expression at the head–trunk boundary cells. Scale bars = 20 µm. c, Lateral views of btd, eve, and slp1 expression patterns in btd, eve, and prd mutants after gastrulation. Scale bars = 50 µm." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 8:</span> Gene expression at the head–trunk boundary of cephalic furrow mutants in <em>Drosophila</em>. <strong>a</strong>, Disruption of <em>btd</em>, <em>eve</em>, and <em>slp1</em> expression in the head of <em>btd</em>, <em>eve</em>, and <em>prd</em> mutant embryos. The patterns were consistent across 5 experiment in <em>btd</em> mutants (20 embryos), 2 experiments in <em>eve</em> mutants (12 embryos), and 2 experiments in <em>prd</em> mutants (12 embryos). Scale bars = 50 µm. <strong>b</strong>, Profile views of <strong>g</strong> showing the gene expression at the head–trunk boundary cells. Scale bars = 20 µm. <strong>c</strong>, Lateral views of <em>btd</em>, <em>eve</em>, and <em>slp1</em> expression patterns in <em>btd</em>, <em>eve</em>, and <em>prd</em> mutants after gastrulation. Scale bars = 50 µm.</figcaption>
</figure>
</div>
<!--TODO: Finish proper figure legends.-->
<div style=""></div>
<h2 id="extended-data-fig.-9">Extended Data Fig. 9</h2>
<div id="fig:genetic-comparative" class="fignos">
<figure>
<img src="figures/FigED9.jpg" data-tag="Extended Data Fig. 9" style="width:100.0%" alt="Figure Extended Data Fig. 9: Genetic patterning of the head–trunk boundary in other dipteran species. The patterns were highly consistent across experiments as indicated in {fig:genetic-patterning}j. a, Expression of slp1, eve, and btd in Ceratitis developmental stages. Scale bars = 100 µm. b, Expression of slp2, eve, and btd in Anopheles developmental stages. Scale bars = 50 µm. c, Expression of slp1, eve, and btd in Clogmia developmental stages. Scale bars = 50 µm. d, Expression of slp1, eve, and prd in Clogmia lateral views. The patterns were consistent among the 20 embryos analyzed (1 experiment). Scale bars = 50 µm. e, Expression of slp1, eve, and prd in Clogmia at the head–trunk boundary in profile views. Scale bars = 20 µm." /><figcaption aria-hidden="true"><span>Figure Extended Data Fig. 9:</span> Genetic patterning of the head–trunk boundary in other dipteran species. The patterns were highly consistent across experiments as indicated in {fig:genetic-patterning}j. <strong>a</strong>, Expression of <em>slp1</em>, <em>eve</em>, and <em>btd</em> in <em>Ceratitis</em> developmental stages. Scale bars = 100 µm. <strong>b</strong>, Expression of <em>slp2</em>, <em>eve</em>, and <em>btd</em> in <em>Anopheles</em> developmental stages. Scale bars = 50 µm. <strong>c</strong>, Expression of <em>slp1</em>, <em>eve</em>, and <em>btd</em> in <em>Clogmia</em> developmental stages. Scale bars = 50 µm. <strong>d</strong>, Expression of <em>slp1</em>, <em>eve</em>, and <em>prd</em> in <em>Clogmia</em> lateral views. The patterns were consistent among the 20 embryos analyzed (1 experiment). Scale bars = 50 µm. <strong>e</strong>, Expression of <em>slp1</em>, <em>eve</em>, and <em>prd</em> in <em>Clogmia</em> at the head–trunk boundary in profile views. Scale bars = 20 µm.</figcaption>
</figure>
</div>
<!--TODO: Finish proper figure legends.-->
<div style=""></div>
<h1 id="supplementary-information">Supplementary information</h1>
<!--TODO: Add supplementary guide here (index)-->
<div style=""></div>
<h2 id="supplementary-figure-1">Supplementary Figure 1</h2>
<div id="fig:ectopic-variability" class="fignos">
<figure>
<img src="figures/FigS1.jpg" data-tag="Supplementary Fig. 1" style="width:100.0%" alt="Figure Supplementary Fig. 1: Individual variability of ectopic folding in btd and eve mutants. Cephalic furrow and ectopic folds are traced in white lines. The combined views illustrate the position of folding overlaid over a single embryo after embryo registration." /><figcaption aria-hidden="true"><span>Figure Supplementary Fig. 1:</span> Individual variability of ectopic folding in <em>btd</em> and <em>eve</em> mutants. Cephalic furrow and ectopic folds are traced in white lines. The combined views illustrate the position of folding overlaid over a single embryo after embryo registration.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-figure-2">Supplementary Figure 2</h2>
<div id="fig:apical-strain" class="fignos">
<figure>
<img src="figures/FigS2.jpg" data-tag="Supplementary Fig. 2" style="width:100.0%" alt="Figure Supplementary Fig. 2: Complementary analyses of mitotic expansions and strain rate. a, Increase in the apical area of individual cells within mitotic domains. A dividing cell increases its apical area 2.4 times during mitotic rounding. The individual daughter cells retain 66% of the parent apical area. When summed, the apical area of the two daughter cells occupy 1.3 times the original apical area of their parent cell. Scale bar = 10 µm. Data from 9 tracked cells on a single btd heterozygote embryo. The center of the box plot depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. b, Strain rate analysis in btd mutants. Cropped region of cartographic projections of btd sibling controls (top, n=3) and homozygote embryos (bottom, n=3). The membrane marker (Gap43-mCherry) is overlayed with a heatmap indicating the regions of increased strain rate in the tissue. The value is the sum of isotropic and anisotropic strain rates obtained through a particle image velocimetry analysis. We used the strain rates in the regions outlined as head–trunk and trunk–germ to generate the plot in Extended Data Fig. 3f. Scale bar = 20 µm (approximated value since pixel sizes vary across the projection)." /><figcaption aria-hidden="true"><span>Figure Supplementary Fig. 2:</span> Complementary analyses of mitotic expansions and strain rate. <strong>a</strong>, Increase in the apical area of individual cells within mitotic domains. A dividing cell increases its apical area 2.4 times during mitotic rounding. The individual daughter cells retain 66% of the parent apical area. When summed, the apical area of the two daughter cells occupy 1.3 times the original apical area of their parent cell. Scale bar = 10 µm. Data from 9 tracked cells on a single <em>btd</em> heterozygote embryo. The center of the box plot depicts the median, the lower and upper hinges correspond to the first and third quartiles (25 and 75th percentiles), and the whiskers extend from the hinges until 1.5 times the inter-quartile range. <strong>b</strong>, Strain rate analysis in <em>btd</em> mutants. Cropped region of cartographic projections of <em>btd</em> sibling controls (top, n=3) and homozygote embryos (bottom, n=3). The membrane marker (Gap43-mCherry) is overlayed with a heatmap indicating the regions of increased strain rate in the tissue. The value is the sum of isotropic and anisotropic strain rates obtained through a particle image velocimetry analysis. We used the strain rates in the regions outlined as head–trunk and trunk–germ to generate the plot in <a href="#fig:mitotic-strain">Extended Data Fig. 3</a>f. Scale bar = 20 µm (approximated value since pixel sizes vary across the projection).</figcaption>
</figure>
</div>
<!--TODO: Re-plot apical area expansion using manuscript style-->
<!--TODO: Convert area from px to µm-->
<!--TODO: Add values to plot to be more informative-->
<!--TODO: Add scale bar-->
<div style=""></div>
<h2 id="supplementary-figure-3">Supplementary Figure 3</h2>
<div id="fig:tortuosity" class="fignos">
<figure>
<img src="figures/FigS3.jpg" data-tag="Supplementary Fig. 3" style="width:100.0%" alt="Figure Supplementary Fig. 3: Image processing pipeline for the tortuosity analysis in cauterized mutants. a, We acquired a single slice in profile view of the head–trunk epithelium. First, we straightened the epithelial monolayer along the curvature of the vitelline envelope using the Straighten tool in ImageJ. We then cropped a window to standardize the size of the analyzed area for all embryos. b, Then, we applied a Gaussian blur which allows capturing the deformations in the epithelium caused by the ectopic folds after thresholding. We create a mask and detect edges and invert to retain a single pixel line corresponding to the outline of the epithelium. The image is inverted for downstream processing. c, We applied a temporal color-coded projections to visualize the epithelial dynamics over time, and created a montage with all timepoints to extract the length of the outline using the skeletonize plugin in ImageJ." /><figcaption aria-hidden="true"><span>Figure Supplementary Fig. 3:</span> Image processing pipeline for the tortuosity analysis in cauterized mutants. <strong>a</strong>, We acquired a single slice in profile view of the head–trunk epithelium. First, we straightened the epithelial monolayer along the curvature of the vitelline envelope using the Straighten tool in ImageJ. We then cropped a window to standardize the size of the analyzed area for all embryos. <strong>b</strong>, Then, we applied a Gaussian blur which allows capturing the deformations in the epithelium caused by the ectopic folds after thresholding. We create a mask and detect edges and invert to retain a single pixel line corresponding to the outline of the epithelium. The image is inverted for downstream processing. <strong>c</strong>, We applied a temporal color-coded projections to visualize the epithelial dynamics over time, and created a montage with all timepoints to extract the length of the outline using the skeletonize plugin in ImageJ.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-figure-4">Supplementary Figure 4</h2>
<div id="fig:ablation" class="fignos">
<figure>
<img src="figures/FigS4.jpg" data-tag="Supplementary Fig. 4" style="width:100.0%" alt="Figure Supplementary Fig. 4: Image processing pipeline for the ablation analysis. a, Example of laser ablation near the cephalic furrow with the membrane signal (top left), the generated kymograph (right), and the linear fit over the distance between edges extracted from the kymograph (bottom left). The position of the laser cut is annotated in a vermilion line, the cell edges are marked in black circles, and the edge distances in a blue line. The distance between edges increases over time. b, Example of a laser ablation far from the cephalic furrow where the distance between edges does not increase over time." /><figcaption aria-hidden="true"><span>Figure Supplementary Fig. 4:</span> Image processing pipeline for the ablation analysis. <strong>a</strong>, Example of laser ablation near the cephalic furrow with the membrane signal (top left), the generated kymograph (right), and the linear fit over the distance between edges extracted from the kymograph (bottom left). The position of the laser cut is annotated in a vermilion line, the cell edges are marked in black circles, and the edge distances in a blue line. The distance between edges increases over time. <strong>b</strong>, Example of a laser ablation far from the cephalic furrow where the distance between edges does not increase over time.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-1">Supplementary Video 1</h2>
<!--TODO: Merge Vid1+2-->
<!--TODO: Merge Vid3+4-->
<!--TODO: Merge Vid5+6-->
<!--TODO: Merge Vid7+8-->
<!--TODO: Lower speed Vid9-->
<div id="fig:vid-initiator" class="fignos">
<figure>
<img src="figures/Vid1.jpg" data-tag="Supplementary Video 1" style="width:50.0%" alt="Figure Supplementary Video 1: Reduced apical constriction in btd mutants. Lateral view (left) and cartographic projection (right) of the head–trunk interface in four individual btd mutants. Putative initiator cells (center) exhibit a reduced degree of apical constriction. The video is looped to highlight the changes in apical cell area. Frame rate = 10 fps. Scale bar = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 1:</span> Reduced apical constriction in <em>btd</em> mutants. Lateral view (left) and cartographic projection (right) of the head–trunk interface in four individual <em>btd</em> mutants. Putative initiator cells (center) exhibit a reduced degree of apical constriction. The video is looped to highlight the changes in apical cell area. Frame rate = 10 fps. Scale bar = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-2">Supplementary Video 2</h2>
<div id="fig:vid-lateral-btd" class="fignos">
<figure>
<img src="figures/Vid2.jpg" data-tag="Supplementary Video 2" style="width:50.0%" alt="Figure Supplementary Video 2: Lateral view of ectopic fold formation in btd mutant. The cephalic furrow forms normally in sibling controls (top) but is absent in btd mutants (bottom). In the mutant, no fold is present at the head–trunk interface until about 20 min, when a large ectopic fold appears and quickly unfolds at about 45 min. In the sibling control, the cephalic furrow remains partially invaginated for the period shown in the recording (about 110 min). Frame rate = 15 fps. Scale bars = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 2:</span> Lateral view of ectopic fold formation in <em>btd</em> mutant. The cephalic furrow forms normally in sibling controls (top) but is absent in <em>btd</em> mutants (bottom). In the mutant, no fold is present at the head–trunk interface until about 20 min, when a large ectopic fold appears and quickly unfolds at about 45 min. In the sibling control, the cephalic furrow remains partially invaginated for the period shown in the recording (about 110 min). Frame rate = 15 fps. Scale bars = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-3">Supplementary Video 3</h2>
<div id="fig:vid-lateral-eve" class="fignos">
<figure>
<img src="figures/Vid3.jpg" data-tag="Supplementary Video 3" style="width:50.0%" alt="Figure Supplementary Video 3: Lateral view of ectopic fold formation in eve mutant. The cephalic furrow forms normally in sibling controls (top) but is absent in eve mutants (bottom). There is no invagination at the head–trunk boundary at the onset of gastrulation, but an ectopic fold starts forming near the dorsal region as soon as the mitotic domains begin expanding around 24 min. The ectopic folds unfold almost entirely by the end of the recording (about 85 min). Additional ectopic folds appear in the trunk region. Frame rate = 10 fps. Scale bars = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 3:</span> Lateral view of ectopic fold formation in <em>eve</em> mutant. The cephalic furrow forms normally in sibling controls (top) but is absent in <em>eve</em> mutants (bottom). There is no invagination at the head–trunk boundary at the onset of gastrulation, but an ectopic fold starts forming near the dorsal region as soon as the mitotic domains begin expanding around 24 min. The ectopic folds unfold almost entirely by the end of the recording (about 85 min). Additional ectopic folds appear in the trunk region. Frame rate = 10 fps. Scale bars = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-4">Supplementary Video 4</h2>
<div id="fig:vid-profile-btd" class="fignos">
<figure>
<img src="figures/Vid4.jpg" data-tag="Supplementary Video 4" style="width:50.0%" alt="Figure Supplementary Video 4: Profile view of ectopic fold formation in btd mutant. In sibling controls (top), the cephalic furrow initiates at the onset of gastrulation (1.5 min) and is fully invaginated when the cell divisions start (about 11 min). In btd mutants (bottom), no invagination initiates, but some embryos exhibit a bulging of the epithelium due to a residual apical constriction behavior (about 7 min). An ectopic fold forms at this position. Its morphology differs greatly from the cephalic furrow (see 10 min). Both the cephalic furrow and ectopic folds regress with the extension of the germ band. Frame rate = 10 fps. Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 4:</span> Profile view of ectopic fold formation in <em>btd</em> mutant. In sibling controls (top), the cephalic furrow initiates at the onset of gastrulation (1.5 min) and is fully invaginated when the cell divisions start (about 11 min). In <em>btd</em> mutants (bottom), no invagination initiates, but some embryos exhibit a bulging of the epithelium due to a residual apical constriction behavior (about 7 min). An ectopic fold forms at this position. Its morphology differs greatly from the cephalic furrow (see 10 min). Both the cephalic furrow and ectopic folds regress with the extension of the germ band. Frame rate = 10 fps. Scale bar = 20 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-5">Supplementary Video 5</h2>
<div id="fig:vid-profile-eve" class="fignos">
<figure>
<img src="figures/Vid5.jpg" data-tag="Supplementary Video 5" style="width:50.0%" alt="Figure Supplementary Video 5: Profile view of ectopic fold formation in eve mutant. In sibling controls (top), the cephalic furrow initiates at the onset of gastrulation (1.8 min). In eve mutants, there are no folds appearing in the epithelium until the formation of mitotic domains (about 10 min). Then, a large ectopic fold appears posterior to dividing cells (15 min). The epithelium of eve mutants show additional folding events along the head and trunk regions. Frame rate = 10 fps. Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 5:</span> Profile view of ectopic fold formation in <em>eve</em> mutant. In sibling controls (top), the cephalic furrow initiates at the onset of gastrulation (1.8 min). In <em>eve</em> mutants, there are no folds appearing in the epithelium until the formation of mitotic domains (about 10 min). Then, a large ectopic fold appears posterior to dividing cells (15 min). The epithelium of <em>eve</em> mutants show additional folding events along the head and trunk regions. Frame rate = 10 fps. Scale bar = 20 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-6">Supplementary Video 6</h2>
<div id="fig:vid-mitotic-fold" class="fignos">
<figure>
<img src="figures/Vid6.jpg" data-tag="Supplementary Video 6" style="width:50.0%" alt="Figure Supplementary Video 6: Ectopic folding between mitotic domains in btd mutant. Detailed view of cartographic projections of btd embryos, showing the formation of the cephalic furrow (left) and of an ectopic fold (right). In sibling controls, the cephalic furrow initiates from a narrow row of cells and invaginates in a progressive manner before the appearance of mitotic domains. In btd mutants, the ectopic folds only appear after the apical expansion of dividing cells within mitotic domains, quickly buckling and unfolding shortly after. Frame rate = 10 fps. Scale bar = 20 µm (approximated value since pixel sizes vary across the projection)." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 6:</span> Ectopic folding between mitotic domains in <em>btd</em> mutant. Detailed view of cartographic projections of <em>btd</em> embryos, showing the formation of the cephalic furrow (left) and of an ectopic fold (right). In sibling controls, the cephalic furrow initiates from a narrow row of cells and invaginates in a progressive manner before the appearance of mitotic domains. In <em>btd</em> mutants, the ectopic folds only appear after the apical expansion of dividing cells within mitotic domains, quickly buckling and unfolding shortly after. Frame rate = 10 fps. Scale bar = 20 µm (approximated value since pixel sizes vary across the projection).</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-7">Supplementary Video 7</h2>
<div id="fig:vid-dynamics-btd" class="fignos">
<figure>
<img src="figures/Vid7.jpg" data-tag="Supplementary Video 7" style="width:50.0%" alt="Figure Supplementary Video 7: Dynamics of ectopic folding in btd mutant. The cephalic furrow in sibling controls (top) and the ectopic folds in btd mutants (bottom) are annotated in white to visualize the dynamics in position, extension, and shape during their formation. Frame rate = 10 fps. Scale bars = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 7:</span> Dynamics of ectopic folding in <em>btd</em> mutant. The cephalic furrow in sibling controls (top) and the ectopic folds in <em>btd</em> mutants (bottom) are annotated in white to visualize the dynamics in position, extension, and shape during their formation. Frame rate = 10 fps. Scale bars = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-8">Supplementary Video 8</h2>
<div id="fig:vid-variability-btd" class="fignos">
<figure>
<img src="figures/Vid8.jpg" data-tag="Supplementary Video 8" style="width:50.0%" alt="Figure Supplementary Video 8: Variability of ectopic folding in btd mutants. The video shows four individual btd mutants, where each display a different pattern and number of ectopic folds at the head–trunk interface. The video is looped to highlight the dynamics of ectopic folding. Frame rate = 15 fps. Scale bars = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 8:</span> Variability of ectopic folding in <em>btd</em> mutants. The video shows four individual <em>btd</em> mutants, where each display a different pattern and number of ectopic folds at the head–trunk interface. The video is looped to highlight the dynamics of ectopic folding. Frame rate = 15 fps. Scale bars = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-9">Supplementary Video 9</h2>
<div id="fig:vid-strain-rate" class="fignos">
<figure>
<img src="figures/Vid9.jpg" data-tag="Supplementary Video 9" style="width:50.0%" alt="Figure Supplementary Video 9: Epithelial strain rate during ectopic folding in btd mutant. Video from Supplementary Video 6 overlaid with the estimated strain rate across the tissues (color-coded from purple to yellow). Increase in strain rates are associated with tissue infolding and mitotic expansions. The video is looped. Frame rate = 10 fps. Scale bar = 20 µm (approximated value since pixel sizes vary across the projection)." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 9:</span> Epithelial strain rate during ectopic folding in <em>btd</em> mutant. Video from <a href="#fig:vid-mitotic-fold">Supplementary Video 6</a> overlaid with the estimated strain rate across the tissues (color-coded from purple to yellow). Increase in strain rates are associated with tissue infolding and mitotic expansions. The video is looped. Frame rate = 10 fps. Scale bar = 20 µm (approximated value since pixel sizes vary across the projection).</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-10">Supplementary Video 10</h2>
<div id="fig:vid-stg-lateral" class="fignos">
<figure>
<img src="figures/Vid10.jpg" data-tag="Supplementary Video 10" style="width:50.0%" alt="Figure Supplementary Video 10: Lateral view of cephalic furrow formation in stg mutants. Sibling control (top) and stg mutant (bottom) during gastrulation. The formation of the cephalic furrow is almost identical to the control embryo. The other morphogenetic movements also occur normally until about 35 min. At this point, the cells in the stg mutant are notably larger than the control. Frame rate = 15 fps. Scale bars = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 10:</span> Lateral view of cephalic furrow formation in <em>stg</em> mutants. Sibling control (top) and <em>stg</em> mutant (bottom) during gastrulation. The formation of the cephalic furrow is almost identical to the control embryo. The other morphogenetic movements also occur normally until about 35 min. At this point, the cells in the <em>stg</em> mutant are notably larger than the control. Frame rate = 15 fps. Scale bars = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-11">Supplementary Video 11</h2>
<div id="fig:vid-stg-dorsal" class="fignos">
<figure>
<img src="figures/Vid11.jpg" data-tag="Supplementary Video 11" style="width:50.0%" alt="Figure Supplementary Video 11: Dorsal view of cephalic furrow formation in stg mutants. Sibling control (top) and stg mutant (bottom) during gastrulation. The cephalic furrow in stg mutants initiates without delay and shows identical morphology to the control until cell divisions begin in the latter. The cells dividing within the cephalic furrow of control embryos alter its morphology, it becomes curved and lengthier. In contrast, the cephalic furrow in the stg mutant retains its initial morphology until it unfolds. Frame rate = 10 fps. Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 11:</span> Dorsal view of cephalic furrow formation in <em>stg</em> mutants. Sibling control (top) and <em>stg</em> mutant (bottom) during gastrulation. The cephalic furrow in <em>stg</em> mutants initiates without delay and shows identical morphology to the control until cell divisions begin in the latter. The cells dividing within the cephalic furrow of control embryos alter its morphology, it becomes curved and lengthier. In contrast, the cephalic furrow in the <em>stg</em> mutant retains its initial morphology until it unfolds. Frame rate = 10 fps. Scale bar = 20 µm.</figcaption>
</figure>
</div>
<!--TODO: Fix the time to show countdown to gastrulation-->
<div style=""></div>
<h2 id="supplementary-video-12">Supplementary Video 12</h2>
<div id="fig:vid-double-lateral" class="fignos">
<figure>
<img src="figures/Vid12.jpg" data-tag="Supplementary Video 12" style="width:50.0%" alt="Figure Supplementary Video 12: Lateral view of btd–stg double mutant. A btd homozygote (top) shows the formation of ectopic folds, while no ectopic folds form in the btd–stg double mutant (bottom). Frame rate = 10 fps. Scale bars = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 12:</span> Lateral view of <em>btd</em>–<em>stg</em> double mutant. A <em>btd</em> homozygote (top) shows the formation of ectopic folds, while no ectopic folds form in the <em>btd</em>–<em>stg</em> double mutant (bottom). Frame rate = 10 fps. Scale bars = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-13">Supplementary Video 13</h2>
<div id="fig:vid-double-dorsal" class="fignos">
<figure>
<img src="figures/Vid13.jpg" data-tag="Supplementary Video 13" style="width:50.0%" alt="Figure Supplementary Video 13: Dorsal view of btd–stg double mutant. A btd homozygote shows the formation of an ectopic fold (top). The btd–stg double mutant exhibits no mitotic domains and no ectopic folds (bottom). Frame rate = 10 fps. Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 13:</span> Dorsal view of <em>btd</em>–<em>stg</em> double mutant. A <em>btd</em> homozygote shows the formation of an ectopic fold (top). The <em>btd</em>–<em>stg</em> double mutant exhibits no mitotic domains and no ectopic folds (bottom). Frame rate = 10 fps. Scale bar = 20 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-14">Supplementary Video 14</h2>
<div id="fig:vid-cautlat-eve" class="fignos">
<figure>
<img src="figures/Vid14.jpg" data-tag="Supplementary Video 14" style="width:50.0%" alt="Figure Supplementary Video 14: Lateral views of germ band cauterization in eve embryos. Non-cauterized eve embryo showing the formation of ectopic folds (top, same embryo from Supplementary Video 3) and a cauterized eve embryo where no ectopic folds appear at the head–trunk interface (bottom). The germ band extension is mechanically blocked by cauterizing the tissue to the vitelline envelope. Mitotic domains form normally, but no folding of the surface occurs. Frame rate = 10 fps. Scale bar = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 14:</span> Lateral views of germ band cauterization in <em>eve</em> embryos. Non-cauterized <em>eve</em> embryo showing the formation of ectopic folds (top, same embryo from <a href="#fig:vid-lateral-eve">Supplementary Video 3</a>) and a cauterized <em>eve</em> embryo where no ectopic folds appear at the head–trunk interface (bottom). The germ band extension is mechanically blocked by cauterizing the tissue to the vitelline envelope. Mitotic domains form normally, but no folding of the surface occurs. Frame rate = 10 fps. Scale bar = 50 µm.</figcaption>
</figure>
</div>
<!--TODO: Replace eve embryo so that it is not the same used in a previous video?-->
<div style=""></div>
<h2 id="supplementary-video-15">Supplementary Video 15</h2>
<div id="fig:vid-cautdor-eve" class="fignos">
<figure>
<img src="figures/Vid15.jpg" data-tag="Supplementary Video 15" style="width:50.0%" alt="Figure Supplementary Video 15: Profile views of germ band cauterization in eve mutant. Same embryo from Supplementary Video 14, but showing a surface and a profile view. The cauterization prevents the extension of the germ band. The mitotic domains compress non-dividing cells, but these do not buckle. Frame rate = 10 fps. Scale bar = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 15:</span> Profile views of germ band cauterization in <em>eve</em> mutant. Same embryo from <a href="#fig:vid-cautlat-eve">Supplementary Video 14</a>, but showing a surface and a profile view. The cauterization prevents the extension of the germ band. The mitotic domains compress non-dividing cells, but these do not buckle. Frame rate = 10 fps. Scale bar = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-16">Supplementary Video 16</h2>
<div id="fig:vid-cautdor-btd" class="fignos">
<figure>
<img src="figures/Vid16.jpg" data-tag="Supplementary Video 16" style="width:50.0%" alt="Figure Supplementary Video 16: Profile views of germ band cauterizations in btd embryos. A non-cauterized btd embryo (top) showing ectopic folds and a cauterized btd embryo showing no ectopic folds (bottom). Frame rate = 10 fps. Scale bar = 20 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 16:</span> Profile views of germ band cauterizations in <em>btd</em> embryos. A non-cauterized <em>btd</em> embryo (top) showing ectopic folds and a cauterized <em>btd</em> embryo showing no ectopic folds (bottom). Frame rate = 10 fps. Scale bar = 20 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-video-17">Supplementary Video 17</h2>
<div id="fig:vid-slp-lateral" class="fignos">
<figure>
<img src="figures/Vid17.jpg" data-tag="Supplementary Video 17" style="width:50.0%" alt="Figure Supplementary Video 17: Lateral view of slp mutant. A slp heterozygote (top) shows a normal cephalic furrow formation. The slp homozygote (bottom) shows a delayed initiator cell behavior and cephalic furrow formation. The embryo also exhibits a more prominent posterior dorsal fold and an ectopic fold forming within the mitotic domain 6 before cell divisions. Frame rate = 10 fps. Scale bar = 50 µm." /><figcaption aria-hidden="true"><span>Figure Supplementary Video 17:</span> Lateral view of <em>slp</em> mutant. A <em>slp</em> heterozygote (top) shows a normal cephalic furrow formation. The <em>slp</em> homozygote (bottom) shows a delayed initiator cell behavior and cephalic furrow formation. The embryo also exhibits a more prominent posterior dorsal fold and an ectopic fold forming within the mitotic domain 6 before cell divisions. Frame rate = 10 fps. Scale bar = 50 µm.</figcaption>
</figure>
</div>
<div style=""></div>
<h2 id="supplementary-note-1">Supplementary Note 1</h2>
<h3 id="disruption-of-initiator-cell-behavior-in-cephalic-furrow-mutants">Disruption of initiator cell behavior in cephalic furrow mutants</h3>
<p>We analyzed how the behavior of initiator cells is perturbed in the <em>btd</em>, <em>eve</em>, and <em>prd</em> mutant backgrounds using lightsheet microscopy. In wildtype embryos, the initiator cells shorten and undergo anisotropic apical constriction minutes before the end of cellularization<span class="citation" data-cites="Spencer2015-td"><sup><a href="#ref-Spencer2015-td" role="doc-biblioref">5</a></sup></span> (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b). In <em>prd</em> mutants, both behaviors are delayed in initiator cells, and the adjacent cells lack the typical arched profile present in the wildtype invagination<span class="citation" data-cites="Spencer2015-td"><sup><a href="#ref-Spencer2015-td" role="doc-biblioref">5</a></sup></span> (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b). Moreover, the tissue only invaginates after gastrulation and the resulting fold is abnormal compared to a wildtype invagination. In <em>btd</em> and <em>eve</em> mutants, the cephalic furrow phenotypes are stronger. Initiator cells in <em>btd</em> embryos exhibit a reduced degree of apical constriction but do not shorten (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b, <a href="#fig:vid-initiator">Supplementary Video 1</a>). In contrast, <em>eve</em> mutants show neither apical constriction nor cell shortening; the epithelium remains flat for several minutes after gastrulation (<a href="#fig:initiator">Extended Data Fig. 1</a>a,b). These observations reveal that initiator cell behavior in <em>prd</em> mutants is not only delayed but also perturbed, and that the cellular mechanism that drives cephalic furrow formation is severely disrupted in <em>btd</em> and <em>eve</em> mutants.</p>
<!--TODO: Create video for btd initiator cells {!@fig:vid-initiator-btd}-->
<!--TODO: Create video for prd initiator cells {!@fig:vid-initiator-prd}-->
<h2 id="supplementary-note-2">Supplementary Note 2</h2>
<h3 id="design-of-physical-model-and-simulations">Design of physical model and simulations</h3>
<p>Our model represents an epithelial monolayer confined inside a rigid shell. It embodies one side of a frontal slice between the midline and the dorsal apex of a <em>Drosophila</em> embryo with its typical morphological proportions (Fig. <a href="#fig:fold-model">3</a>a, Fig. <a href="#fig:model-features">Extended Data Fig. 5</a>a). The blastoderm is modeled by an elliptical arc of equidistant particles connected by springs and enclosed on one side by a rigid barrier representing the vitelline envelope (Fig. <a href="#fig:fold-model">3</a>b). The total energy per unit length of this tissue (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>T</mi></msub><annotation encoding="application/x-tex">W_T</annotation></semantics></math>) is a sum of a stretching energy component (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>s</mi></msub><annotation encoding="application/x-tex">W_s</annotation></semantics></math>) and a bending energy component (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>W</mi><mi>b</mi></msub><annotation encoding="application/x-tex">W_b</annotation></semantics></math>) (Fig. <a href="#fig:fold-model">3</a>c). Each of these components has a rigidity associated with them. <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>s</mi></msub><annotation encoding="application/x-tex">K_s</annotation></semantics></math> is the stretching rigidity and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math> is the bending rigidity. These two parameters can be combined into a single dimensionless bending rigidity, <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> (Fig. <a href="#fig:fold-model">3</a>c).</p>
<p>To simulate the physical interactions between mitotic domains, germ band, and cephalic furrow, we defined the mitotic domains as compressed regions which tend to expand (they contain more particles compared to the surrounding regions), and the cephalic furrow as a narrow region having an intrinsic negative curvature predisposing the tissue to invaginate (Fig. <a href="#fig:fold-model">3</a>b). The germ band is defined by the position of the posterior end of the tissue (<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>g</mi><annotation encoding="application/x-tex">g</annotation></semantics></math>), which is fixed at different fractions of egg length for each simulation (Fig. <a href="#fig:fold-model">3</a>a,b). Thus, the effect of germ band extension appears as a global compression in the blastoderm. To run the simulations, we defined a ground level of random noise and iterated towards equilibrium of the total energy in the system (see Methods). We then set several simulations with different bending rigidity values and combinations of presence/absence of mitotic domains, cephalic furrow, and percentages of germ band extension to quantify the position, frequency, and depth of ectopic folding in the epithelium.</p>
<p>To characterize the model properties and energy dynamics, we ran simulations using initially a single bending rigidity value, without mitotic domains, and at different percentages of germ band extension. Without the germ band, the tissue is almost stress-free and no ectopic folding occurs (Fig. <a href="#fig:fold-model">3</a>e). We begin to observe folds in the simulations with higher progression of germ band extension. Folding events are stochastic and happen at distinct iterations for each simulation. When a fold begins to form, the bending energy increases releasing a larger amount of stretching energy, which, in turn, decreases the total energy of the system over each iteration (Fig. <a href="#fig:fold-model">3</a>d). The increase in bending energy coincides with a rapid deepening of the fold. Once the bending energy reaches a peak, we find that the fold continues to deepen more gradually, but the number of folds rarely changes afterward (Fig. <a href="#fig:fold-model">3</a>d, <a href="#fig:model-features">Extended Data Fig. 5</a>b). Therefore, this peak of bending energy provides an informative reference point, which we used to standardize the comparison across simulations.</p>
<h2 id="supplementary-note-3">Supplementary Note 3</h2>
<h3 id="comparing-our-reference-bending-rigidity-to-direct-measurements">Comparing our reference bending rigidity to direct measurements</h3>
<p>Here we compare the reference bending rigidity established for the <em>Drosophila</em> blastoderm with our experiments to direct measurements from the literature. Trushko et al.<span class="citation" data-cites="Trushko2020-gf"><sup><a href="#ref-Trushko2020-gf" role="doc-biblioref">17</a></sup></span> reports that in a MDCK monolayer <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>b</mi></msub><mo>=</mo><mn>5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>13</mn></mrow></msup><mi>N</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">K_b = 5\times10^{-13} Nm</annotation></semantics></math> and <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>K</mi><mi>s</mi></msub><mo>=</mo><mn>0.15</mn><mi>N</mi><msup><mi>m</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_s = 0.15 Nm^{-1}</annotation></semantics></math>. With these values, we can compute the dimensionless bending rigidity <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math>. However, the thickness of the MDCK monolayer and the <em>Drosophila</em> blastoderm are very different, and hence we need to correct for this before computing <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> for MDCK monolayer. The bending rigidity scales with the square of the thickness of the tissue. Hence, the corrected <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>K</mi><mi>b</mi></msub><annotation encoding="application/x-tex">K_b</annotation></semantics></math> can be computed as <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mo>=</mo><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo stretchy="false" form="prefix">(</mo><msub><mi>h</mi><mrow><mi>b</mi><mi>l</mi><mi>a</mi><mi>s</mi><mi>t</mi><mi>o</mi><mi>d</mi><mi>e</mi><mi>r</mi><mi>m</mi></mrow></msub><mi>/</mi><msub><mi>h</mi><mrow><mi>M</mi><mi>D</mi><mi>C</mi><mi>K</mi></mrow></msub><msup><mo stretchy="false" form="postfix">)</mo><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_b^{&#39;} = K_{b}^{\text{*}}(h_{blastoderm}/h_{MDCK})^2</annotation></semantics></math>. This gives us <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mo>=</mo><mn>2</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>12</mn></mrow></msup><mi>N</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">K_b^{&#39;} =2\times10^{-12} Nm</annotation></semantics></math>. Now we can compute <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mi>/</mi><msub><mi>K</mi><mi>s</mi></msub><msup><mi>L</mi><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} = K_b^{&#39;}/K_s L^2</annotation></semantics></math> where we put <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>L</mi><mo>=</mo><mn>255</mn><mi>μ</mi><mi>m</mi></mrow><annotation encoding="application/x-tex">L = 255\mu m</annotation></semantics></math>. Here we scale <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mi>′</mi></msubsup><mi>/</mi><msub><mi>K</mi><mi>s</mi></msub></mrow><annotation encoding="application/x-tex">K_b^{&#39;}/K_s</annotation></semantics></math> by the relevant lengthscale which is the semi-major axis of the <em>Drosophila</em> embryo. This gives us <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><mo>=</mo><mn>2.05</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow><annotation encoding="application/x-tex">K_{b}^{\text{*}} = 2.05\times 10^{-4}</annotation></semantics></math>. This is the estimated <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msubsup><mi>K</mi><mi>b</mi><mtext mathvariant="normal">*</mtext></msubsup><annotation encoding="application/x-tex">K_{b}^{\text{*}}</annotation></semantics></math> for the MDCK monolayer that has the same thickness as the <em>Drosophila</em> blastoderm (referring to the height correction) and has been put in the same geometrical conditions as the <em>Drosophila</em> blastoderm (referring to the <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>L</mi><annotation encoding="application/x-tex">L</annotation></semantics></math> used in the calculation).</p>
<h2 id="supplementary-note-4">Supplementary Note 4</h2>
<h3 id="live-imaging-screen-for-cephalic-furrow-genes">Live-imaging screen for cephalic furrow genes</h3>
<!--TODO: Add figure with screen outline-->
<!--TODO: Add table with candidate genes-->
<!--TODO: Add table with screen results-->
<!--TODO: Add figure with screen results-->
<p>To uncover other genes directly involved in cephalic furrow formation in addition to <em>btd</em>, <em>eve</em>, and <em>prd</em>, we performed a live-imaging screen in strains containing loss-of-function alleles for a selection of candidate genes expressed at the head–trunk region.<span class="citation" data-cites="Tomancak2002-yf Tomancak2007-db Lecuyer2007-uy"><sup><a href="#ref-Tomancak2002-yf" role="doc-biblioref">64</a>–<a href="#ref-Lecuyer2007-uy" role="doc-biblioref">66</a></sup></span> Because the cephalic furrow is transient and leaves no trace, the live-imaging approach is critical to recognize altered phenotypes. From about 50 genes, we only detected three showing abnormal cephalic furrow formation to different degrees besides the previously described genes (see Table <a href="#tbl:screen">Supplementary Table 1</a>). The strongest cephalic furrow phenotype was present in flies mutant for the <em>sloppy paired</em> (<em>slp</em>) genes.</p>
<h2 id="supplementary-note-5">Supplementary Note 5</h2>
<h3 id="role-of-sloppy-paired-in-cephalic-furrow-patterning">Role of <em>sloppy paired</em> in cephalic furrow patterning</h3>
<!--TODO: Create a slp-analyses figure only with slp-->
<!--TODO: Replace genetic-patterning and genetic-features-->
<p>To identify other cephalic furrow genes, we performed a live-imaging screen using loss-of-function alleles of several candidates expressed at the head–trunk interface (Supplementary Note 4). We found that null mutants for the <em>sloppy paired</em> (<em>slp</em>) transcription factors, <em>slp1</em> and <em>slp2</em>, show a strong phenotype where the cephalic furrow is delayed and shifted towards the anterior end by ~6% of the egg length (control = 67.6±1.4%, n=26; <em>slp</em> = 73.2±0.7%, n=7) (Fig. <a href="#fig:slp-analyses">Extended Data Fig. 6</a>a,b, <a href="#fig:vid-slp-lateral">Supplementary Video 17</a>). With this anterior shift, <em>slp</em> mutants exhibit a more prominent posterior dorsal fold and an early ectopic fold within MD6 appearing before cell divisions (Fig. <a href="#fig:slp-analyses">Extended Data Fig. 6</a>a, <a href="#fig:vid-slp-lateral">Supplementary Video 17</a>). These observations are congruent with the increase in posterior mechanical instability present in our simulations where the cephalic furrow is shifted forward (Fig. <a href="#fig:fold-model">3</a>h).</p>
<p>Since <em>slp1</em> is a known anterior repressor that positions anterior pair rule stripes,<span class="citation" data-cites="Andrioli2004-qi Andrioli2012-fv"><sup><a href="#ref-Andrioli2012-fv" role="doc-biblioref">20</a>,<a href="#ref-Andrioli2004-qi" role="doc-biblioref">67</a></sup></span> we wondered if the shift in the position of the cephalic furrow in <em>slp</em> mutants coincided with a shift in typical overlap between <em>btd</em> and <em>eve</em> stripe 1 at the head–trunk boundary. We find that the expression patterns and typical overlap between the two genes remains almost unaltered, except for the small ectopic expression <em>btd</em> in the head and for the wider gap between <em>eve</em> stripe 1 and 2 (Fig. <a href="#fig:slp-analyses">Extended Data Fig. 6</a>c–e). The anterior shift in cells expressing <em>btd</em> and <em>eve</em> corresponds to a few rows of blastoderm cells (control = 4.7±0.5 rows, n=4; <em>slp</em> = 7.3±0.7 rows, n=5) (Fig. <a href="#fig:slp-analyses">Extended Data Fig. 6</a>f). Despite the displacement, most <em>slp</em> embryos exhibit initiator cell behaviors, suggesting that the patterning is not entirely perturbed (<a href="#fig:slp-analyses">Extended Data Fig. 6</a>g). However, the symmetry of the resulting fold is altered (<a href="#fig:slp-analyses">Extended Data Fig. 6</a>h), indicating that <em>slp1</em> may contribute not only to the positioning but also to the patterning of individual cells that give rise to the cephalic furrow.</p>
<!--## Supplementary Note-->
<!--### Genetic interactions at the head--trunk boundary-->
<!--- btd also regulates slp stripe 2-->
<!--- slp positions btd/eve, then btd positions slp stripe 2-->
<!--- late *slp* expression demarcates cephalic furrow edges-->
<!--- early slp activity represses eve at the anterior end-->
<!--- *btd* is de-repressed between the acron and head--trunk boundary ({@fig:genetic-patterning}c--d)-->
<!--TODO: Write proper findings regarding genetic interactions.-->
<div style=""></div>
<h2 id="supplementary-table-1">Supplementary Table 1</h2>
<div id="tbl:screen" class="tablenos">
<table id="tbl:screen" data-tag="Supplementary Table 1">
<caption><span>Table Supplementary Table 1:</span> Summary of live-imaging screening results for cephalic furrow genes. </caption>
<colgroup>
<col style="width: 20%" />
<col style="width: 25%" />
<col style="width: 34%" />
<col style="width: 20%" />
</colgroup>
<thead>
<tr class="header">
<th>gene name</th>
<th>gene symbol</th>
<th>allele/deficiency</th>
<th>phenotype</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><em>buttonhead</em></td>
<td><em>btd</em></td>
<td><em>btd</em><sup>XA</sup></td>
<td>absent<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">8</a></sup></span></td>
</tr>
<tr class="even">
<td><em>even skipped</em></td>
<td><em>eve</em></td>
<td><em>eve</em><sup>3</sup></td>
<td>absent<span class="citation" data-cites="Vincent1997-fa"><sup><a href="#ref-Vincent1997-fa" role="doc-biblioref">8</a></sup></span></td>
</tr>
<tr class="odd">
<td><em>paired</em></td>
<td><em>prd</em></td>
<td><em>prd</em><sup>4</sup></td>
<td>delayed<span class="citation" data-cites="Blankenship2001-tm"><sup><a href="#ref-Blankenship2001-tm" role="doc-biblioref">15</a></sup></span> and abnormal</td>
</tr>
<tr class="even">
<td><em>sloppy paired 1</em> and <em>2</em></td>
<td><em>slp1</em>, <em>slp2</em></td>
<td><em>slp</em><sup>Δ34B</sup></td>
<td>delayed and displaced anteriorly</td>
</tr>
<tr class="odd">
<td><em>sloppy paired 1</em></td>
<td><em>slp1</em></td>
<td><em>slp1</em><sup>1</sup></td>
<td>delayed and displaced anteriorly</td>
</tr>
<tr class="even">
<td><em>giant</em></td>
<td><em>gt</em></td>
<td><em>gt</em><sup>X11</sup></td>
<td>perturbed formation</td>
</tr>
<tr class="odd">
<td><em>knirps</em>, <em>knirps-like</em></td>
<td><em>kni</em>, <em>knrl</em></td>
<td>Df(3L)BSC448</td>
<td>ventral portion displaced anteriorly</td>
</tr>
</tbody>
</table>
</div>
<div style=""></div>
</body>
</html>
